# Class: activity

a provence-generating activity

URI:
<https://microbiomedata/schema/workflow_execution_activity/Activity>

[![img](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BAgent%5D,%5BAgent%5D%3Cwas%20associated%20with%200..1-++%5BActivity%7Cid:string;name:string%20%3F;started_at_time:string%20%3F;ended_at_time:string%20%3F;used:string%20%3F%5D,%5BActivity%5D%3Cwas%20informed%20by%200..1-%20%5BActivity%5D,%5BAttributeValue%5D-%20was%20generated%20by%200..1%3E%5BActivity%5D,%5BWorkflowExecutionActivity%5D-%20was%20informed%20by%201..1%3E%5BActivity%5D,%5BActivity%5D%5E-%5BWorkflowExecutionActivity%5D,%5BAttributeValue%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BAgent%5D,%5BAgent%5D%3Cwas%20associated%20with%200..1-++%5BActivity%7Cid:string;name:string%20%3F;started_at_time:string%20%3F;ended_at_time:string%20%3F;used:string%20%3F%5D,%5BActivity%5D%3Cwas%20informed%20by%200..1-%20%5BActivity%5D,%5BAttributeValue%5D-%20was%20generated%20by%200..1%3E%5BActivity%5D,%5BWorkflowExecutionActivity%5D-%20was%20informed%20by%201..1%3E%5BActivity%5D,%5BActivity%5D%5E-%5BWorkflowExecutionActivity%5D,%5BAttributeValue%5D)

## Children

-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
    -Represents an instance of an execution of a particular workflow

## Referenced by Class

-   **None** [was generated by](was_generated_by.md) 0..1
    [Activity](Activity.md)
-   **None** [was informed by](was_informed_by.md) 0..1
    [Activity](Activity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md) [workflow
    execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md) 1..1
    [Activity](Activity.md)

## Attributes

### Own

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [started at time](started_at_time.md) 0..1
    -   Range: [String](types/String.md)
-   [ended at time](ended_at_time.md) 0..1
    -   Range: [String](types/String.md)
-   [was informed by](was_informed_by.md) 0..1
    -   Range: [Activity](Activity.md)
-   [was associated with](was_associated_with.md) 0..1
    -   Range: [Agent](Agent.md)
-   [used](used.md) 0..1
    -   Range: [String](types/String.md)

## Other properties

+----------------------------------+----------------------------------+
| **Mappings:**                    | prov:Activity                    |
+==================================+==================================+
| Class: agent ============        |                                  |
+----------------------------------+----------------------------------+
| a provence-gen                   | erating agent                    |
+----------------------------------+----------------------------------+
| URI: <https://m>                 | icrobiomedata/schema/w           |
|                                  | orkflow_execution_activity/Agent |
| [![image1](https://yu            |                                  |
| ml.me/diagram/nofunky;dir:TB/cla |                                  |
| ss/%5BActivity%5D%3Cwas%20inform |                                  |
| ed%20by%200..1-%20%5BAgent%5D,%5 |                                  |
| BAgent%5D%3Cacted%20on%20behalf% |                                  |
| 20of%200..1-++%5BAgent%5D,%5BAct |                                  |
| ivity%5D++-%20was%20associated%2 |                                  |
| 0with%200..1%3E%5BAgent%5D,%5BAc |                                  |
| tivity%5D)](https://yuml.me/diag |                                  |
| ram/nofunky;dir:TB/class/%5BActi |                                  |
| vity%5D%3Cwas%20informed%20by%20 |                                  |
| 0..1-%20%5BAgent%5D,%5BAgent%5D% |                                  |
| 3Cacted%20on%20behalf%20of%200.. |                                  |
| 1-++%5BAgent%5D,%5BActivity%5D++ |                                  |
| -%20was%20associated%20with%200. |                                  |
| .1%3E%5BAgent%5D,%5BActivity%5D) |                                  |
+----------------------------------+----------------------------------+
|                                  | -by-class-1:                     |
+----------------------------------+----------------------------------+
| Referenced by                    | Class                            |
+----------------------------------+----------------------------------+
| ------------------------------   | ------------------------------   |
+----------------------------------+----------------------------------+
| \- **None** \`a                  | cted on behalf of                |
|                                  | \<acted_on_behalf_of.md\>\`\_\_  |
|                                  | 0..1                             |
+----------------------------------+----------------------------------+
| > \`Agent \<Age                  | nt.md\>\`\_\_                    |
+----------------------------------+----------------------------------+
| \- **None** \`w                  | as associated with               |
|                                  | \<was_associated_with.md\>\`\_\_ |
|                                  | 0..1                             |
+----------------------------------+----------------------------------+
| > \`Agent \<Age                  | nt.md\>\`\_\_                    |
+----------------------------------+----------------------------------+
| ## Attributes {#attributes-1}    | -1:                              |
|                                  |                                  |
| ### Own {#own-1}                 |                                  |
+----------------------------------+----------------------------------+
| -   \`acted on b                 | ehalf of                         |
|                                  | \<acted_on_behalf_of.md\>\`\_\_  |
|                                  | 0..1                             |
+----------------------------------+----------------------------------+
| > -   Range: \`                  | Agent \<Agent.md\>\`\_\_         |
+----------------------------------+----------------------------------+
| -   \`was inform                 | ed by                            |
|                                  | \<was_informed_by.md\>\`\_\_     |
|                                  | 0..1                             |
+----------------------------------+----------------------------------+
| > -   Range: \`                  | Activity \<Activity.md\>\`\_\_   |
+----------------------------------+----------------------------------+
|                                  | erties-1:                        |
+----------------------------------+----------------------------------+
| Other properti                   | es                               |
+----------------------------------+----------------------------------+
| ------------------------------   | \--                              |
+----------------------------------+----------------------------------+
| =============                    | ==========                       |
+----------------------------------+----------------------------------+
| **Mappings:**                    | > prov:Agent                     |
+----------------------------------+----------------------------------+

# Class: attribute value

The value for any value of a attribute for a sample. This object can
hold both the un-normalized atomic value and the structured value

URI:
<https://microbiomedata/schema/workflow_execution_activity/AttributeValue>

[![image2](https://yuml.me/diagram/nofunky;dir:TB/class/%5BUrlValue%5D,%5BTimestampValue%5D,%5BTextValue%5D,%5BQuantityValue%5D,%5BPersonValue%5D,%5BIntegerValue%5D,%5BImageValue%5D,%5BGeolocationValue%5D,%5BControlledTermValue%5D,%5BBooleanValue%5D,%5BActivity%5D%3Cwas%20generated%20by%200..1-%20%5BAttributeValue%7Chas_raw_value:string%20%3F;type:string%20%3F%5D,%5BAttributeValue%5D%5E-%5BUrlValue%5D,%5BAttributeValue%5D%5E-%5BTimestampValue%5D,%5BAttributeValue%5D%5E-%5BTextValue%5D,%5BAttributeValue%5D%5E-%5BQuantityValue%5D,%5BAttributeValue%5D%5E-%5BPersonValue%5D,%5BAttributeValue%5D%5E-%5BIntegerValue%5D,%5BAttributeValue%5D%5E-%5BImageValue%5D,%5BAttributeValue%5D%5E-%5BGeolocationValue%5D,%5BAttributeValue%5D%5E-%5BControlledTermValue%5D,%5BAttributeValue%5D%5E-%5BBooleanValue%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BUrlValue%5D,%5BTimestampValue%5D,%5BTextValue%5D,%5BQuantityValue%5D,%5BPersonValue%5D,%5BIntegerValue%5D,%5BImageValue%5D,%5BGeolocationValue%5D,%5BControlledTermValue%5D,%5BBooleanValue%5D,%5BActivity%5D%3Cwas%20generated%20by%200..1-%20%5BAttributeValue%7Chas_raw_value:string%20%3F;type:string%20%3F%5D,%5BAttributeValue%5D%5E-%5BUrlValue%5D,%5BAttributeValue%5D%5E-%5BTimestampValue%5D,%5BAttributeValue%5D%5E-%5BTextValue%5D,%5BAttributeValue%5D%5E-%5BQuantityValue%5D,%5BAttributeValue%5D%5E-%5BPersonValue%5D,%5BAttributeValue%5D%5E-%5BIntegerValue%5D,%5BAttributeValue%5D%5E-%5BImageValue%5D,%5BAttributeValue%5D%5E-%5BGeolocationValue%5D,%5BAttributeValue%5D%5E-%5BControlledTermValue%5D,%5BAttributeValue%5D%5E-%5BBooleanValue%5D,%5BActivity%5D)

## Children

-   [BooleanValue](BooleanValue.md) - A value that is a boolean
-   [ControlledTermValue](ControlledTermValue.md) - A controlled term or
    class from an ontology
-   [GeolocationValue](GeolocationValue.md) - A normalized value for a
    location on the earth's surface
-   [ImageValue](ImageValue.md) - An attribute value representing an
    image.
-   [IntegerValue](IntegerValue.md) - A value that is an integer
-   [PersonValue](PersonValue.md) - An attribute value representing a
    person
-   [QuantityValue](QuantityValue.md) - A simple quantity, e.g. 2cm
-   [TextValue](TextValue.md) - A basic string value
-   [TimestampValue](TimestampValue.md) - A value that is a timestamp.
    The range should be ISO-8601
-   [UrlValue](UrlValue.md) - A value that is a string that conforms to
    URL syntax

## Referenced by Class {#referenced-by-class-2}

## Attributes {#attributes-2}

### Own {#own-2}

-   [has raw value](has_raw_value.md) 0..1
    -   Description: The value that was specified for an annotation in
        raw form, i.e. a string. E.g. "2 cm" or "2-4 cm"
    -   Range: [String](types/String.md)
-   [was generated by](was_generated_by.md) 0..1
    -   Range: [Activity](Activity.md)
-   [attribute value➞type](attribute_value_type.md) 0..1
    -   Description: An optional string that specified the type of
        object.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None

# Class: biosample

A material sample. It may be environmental (encompassing many organisms)
or isolate or tissue. An environmental sample containing genetic
material from multiple individuals is commonly referred to as a
biosample.

URI: [nmdc:Biosample](https://microbiomedata/meta/Biosample)

[![image3](https://yuml.me/diagram/nofunky;dir:TB/class/%5BTimestampValue%5D,%5BTextValue%5D,%5BQuantityValue%5D,%5BOmicsProcessing%5D,%5BNamedThing%5D,%5BGeolocationValue%5D,%5BDatabase%5D,%5BControlledTermValue%5D,%5BBiosampleProcessing%5D,%5BQuantityValue%5D%3Csubsurface_depth2%200..1-++%5BBiosample%7Ctype:string%20%3F;ecosystem:string%20%3F;ecosystem_category:string%20%3F;ecosystem_type:string%20%3F;ecosystem_subtype:string%20%3F;specific_ecosystem:string%20%3F;add_date:string%20%3F;community:string%20%3F;habitat:string%20%3F;host_name:string%20%3F;identifier:string%20%3F;location:string%20%3F;mod_date:string%20%3F;ncbi_taxonomy_name:string%20%3F;proport_woa_temperature:string%20%3F;salinity_category:string%20%3F;sample_collection_site:string%20%3F;soluble_iron_micromol:string%20%3F;GOLD_sample_identifiers:external_identifier%20*;INSDC_biosample_identifiers:external_identifier%20*;INSDC_secondary_sample_identifiers:external_identifier%20*;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BQuantityValue%5D%3Csubsurface_depth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdepth2%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cwater_content_soil_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cwater_content%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_phosp%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctot_nitro_content_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_nitro_content%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctot_org_c_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_org_carb%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_diss_nitro%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_depth_water_col%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_carb%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctidal_stage%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctillage%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctexture_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctexture%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctemp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csulfide%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csulfate%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cstore_cond%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csoil_type_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csoil_type%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csodium%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cslope_aspect%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cslope_gradient%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csize_frac_up%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csize_frac_low%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csieving%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cseason_precpt%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cseason_temp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csamp_vol_we_dna_ext%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csamp_store_temp%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csamp_store_loc%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csamp_store_dur%200..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Csamp_mat_process%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csamp_collect_device%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csalinity_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csalinity%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Credox_potential%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cprofile_position%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cprevious_land_use_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cprevious_land_use%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpressure%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpotassium%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cpool_dna_extracts%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cphosplipid_fatt_acid%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cphaeopigments%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cph_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cph%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpetroleum_hydrocarb%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cperturbation%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpart_org_carb%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Coxy_stat_samp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Corganism_count%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Corg_nitro%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Corg_matter%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cnitrite%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cnitrate%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cn_alkanes%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmisc_param%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cmicrobial_biomass_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmicrobial_biomass%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmean_peak_frict_vel%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmean_frict_vel%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmagnesium%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clocal_class_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clocal_class%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clink_climate_info%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clink_class_info%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clink_addit_analys%200..1-++%5BBiosample%5D,%5BGeolocationValue%5D%3Clat_lon%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Chorizon_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Chorizon%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cheavy_metals_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cheavy_metals%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cglucosidase_act%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cgeo_loc_name%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Cflooding%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Cfire%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cfao_class%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Cextreme_event%200..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cenv_medium%201..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cenv_local_scale%201..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cenv_broad_scale%201..1-++%5BBiosample%5D,%5BTextValue%5D%3Cenv_package%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Celev%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cdrainage_class%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_oxygen%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_org_nitro%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_org_carb%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_inorg_phosp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_inorg_carb%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_hydrogen%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_carb_dioxide%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdepth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdensity%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccrop_rotation%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccur_vegetation_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccur_vegetation%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccur_land_use%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Ccollection_date%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cchlorophyll%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cchloride%200..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cchem_administration%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ccarb_nitro_ratio%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ccalcium%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cbromide%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cbishomohopanol%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cbacteria_carb_prod%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cannual_temp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cannual_precpt%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cammonium%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Caminopept_act%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cal_sat_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cal_sat%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Calt%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Calkyl_diethers%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Calkalinity_method%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Calkalinity%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cagrochem_addition%200..1-++%5BBiosample%5D,%5BNamedThing%5D%3Cpart%20of%200..*-%20%5BBiosample%5D,%5BBiosampleProcessing%5D-%20has%20input%200..*%3E%5BBiosample%5D,%5BDatabase%5D++-%20biosample%20set%200..*%3E%5BBiosample%5D,%5BOmicsProcessing%5D-%20has%20input(i)%201..*%3E%5BBiosample%5D,%5BNamedThing%5D%5E-%5BBiosample%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BTimestampValue%5D,%5BTextValue%5D,%5BQuantityValue%5D,%5BOmicsProcessing%5D,%5BNamedThing%5D,%5BGeolocationValue%5D,%5BDatabase%5D,%5BControlledTermValue%5D,%5BBiosampleProcessing%5D,%5BQuantityValue%5D%3Csubsurface_depth2%200..1-++%5BBiosample%7Ctype:string%20%3F;ecosystem:string%20%3F;ecosystem_category:string%20%3F;ecosystem_type:string%20%3F;ecosystem_subtype:string%20%3F;specific_ecosystem:string%20%3F;add_date:string%20%3F;community:string%20%3F;habitat:string%20%3F;host_name:string%20%3F;identifier:string%20%3F;location:string%20%3F;mod_date:string%20%3F;ncbi_taxonomy_name:string%20%3F;proport_woa_temperature:string%20%3F;salinity_category:string%20%3F;sample_collection_site:string%20%3F;soluble_iron_micromol:string%20%3F;GOLD_sample_identifiers:external_identifier%20*;INSDC_biosample_identifiers:external_identifier%20*;INSDC_secondary_sample_identifiers:external_identifier%20*;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BQuantityValue%5D%3Csubsurface_depth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdepth2%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cwater_content_soil_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cwater_content%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_phosp%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctot_nitro_content_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_nitro_content%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctot_org_c_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_org_carb%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_diss_nitro%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_depth_water_col%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctot_carb%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctidal_stage%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctillage%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ctexture_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctexture%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ctemp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csulfide%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csulfate%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cstore_cond%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csoil_type_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csoil_type%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csodium%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cslope_aspect%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cslope_gradient%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csize_frac_up%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csize_frac_low%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csieving%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cseason_precpt%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cseason_temp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csamp_vol_we_dna_ext%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csamp_store_temp%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csamp_store_loc%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csamp_store_dur%200..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Csamp_mat_process%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csamp_collect_device%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Csalinity_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Csalinity%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Credox_potential%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cprofile_position%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cprevious_land_use_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cprevious_land_use%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpressure%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpotassium%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cpool_dna_extracts%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cphosplipid_fatt_acid%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cphaeopigments%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cph_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cph%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpetroleum_hydrocarb%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cperturbation%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cpart_org_carb%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Coxy_stat_samp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Corganism_count%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Corg_nitro%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Corg_matter%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cnitrite%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cnitrate%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cn_alkanes%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmisc_param%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cmicrobial_biomass_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmicrobial_biomass%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmean_peak_frict_vel%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmean_frict_vel%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cmagnesium%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clocal_class_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clocal_class%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clink_climate_info%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clink_class_info%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Clink_addit_analys%200..1-++%5BBiosample%5D,%5BGeolocationValue%5D%3Clat_lon%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Chorizon_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Chorizon%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cheavy_metals_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cheavy_metals%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cglucosidase_act%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cgeo_loc_name%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Cflooding%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Cfire%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cfao_class%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Cextreme_event%200..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cenv_medium%201..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cenv_local_scale%201..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cenv_broad_scale%201..1-++%5BBiosample%5D,%5BTextValue%5D%3Cenv_package%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Celev%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cdrainage_class%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_oxygen%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_org_nitro%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_org_carb%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_inorg_phosp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_inorg_carb%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_hydrogen%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdiss_carb_dioxide%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdepth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cdensity%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccrop_rotation%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccur_vegetation_meth%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccur_vegetation%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Ccur_land_use%200..1-++%5BBiosample%5D,%5BTimestampValue%5D%3Ccollection_date%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cchlorophyll%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cchloride%200..1-++%5BBiosample%5D,%5BControlledTermValue%5D%3Cchem_administration%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ccarb_nitro_ratio%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Ccalcium%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cbromide%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cbishomohopanol%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cbacteria_carb_prod%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cannual_temp%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cannual_precpt%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cammonium%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Caminopept_act%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Cal_sat_meth%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cal_sat%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Calt%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Calkyl_diethers%200..1-++%5BBiosample%5D,%5BTextValue%5D%3Calkalinity_method%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Calkalinity%200..1-++%5BBiosample%5D,%5BQuantityValue%5D%3Cagrochem_addition%200..1-++%5BBiosample%5D,%5BNamedThing%5D%3Cpart%20of%200..*-%20%5BBiosample%5D,%5BBiosampleProcessing%5D-%20has%20input%200..*%3E%5BBiosample%5D,%5BDatabase%5D++-%20biosample%20set%200..*%3E%5BBiosample%5D,%5BOmicsProcessing%5D-%20has%20input(i)%201..*%3E%5BBiosample%5D,%5BNamedThing%5D%5E-%5BBiosample%5D)

## Identifier prefixes

-   GOLD

## Parents

-   is_a: [NamedThing](NamedThing.md) - a databased entity or
    concept/class

## Referenced by Class {#referenced-by-class-3}

-   [BiosampleProcessing](BiosampleProcessing.md) [biosample
    processing➞has input](biosample_processing_has_input.md) 0..\*
    [Biosample](Biosample.md)
-   [Database](Database.md) [biosample set](biosample_set.md) 0..\*
    [Biosample](Biosample.md)
-   [OmicsProcessing](OmicsProcessing.md) [omics processing➞has
    input](omics_processing_has_input.md) 1..\*
    [Biosample](Biosample.md)

## Attributes {#attributes-3}

### Own {#own-3}

-   [type](type.md) 0..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [agrochem_addition](agrochem_addition.md) 0..1
    -   Description: "Addition of fertilizers, pesticides, etc. - amount
        and time of applications"
    -   Range: [QuantityValue](QuantityValue.md)
-   [alkalinity](alkalinity.md) 0..1
    -   Description: "Alkalinity, the ability of a solution to
        neutralize acids to the equivalence point of carbonate or
        bicarbonate"
    -   Range: [QuantityValue](QuantityValue.md)
-   [alkalinity_method](alkalinity_method.md) 0..1
    -   Description: Method used for alkalinity measurement
    -   Range: [TextValue](TextValue.md)
-   [alkyl_diethers](alkyl_diethers.md) 0..1
    -   Description: Concentration of alkyl diethers
    -   Range: [QuantityValue](QuantityValue.md)
-   [alt](alt.md) 0..1
    -   Description: "Altitude is a term used to identify heights of
        objects such as airplanes, space shuttles, rockets, atmospheric
        balloons and heights of places such as atmospheric layers and
        clouds. It is used to measure the height of an object which is
        above the earthbs surface. In this context, the altitude
        measurement is the vertical distance between the earth's surface
        above sea level and the sampled position in the air"
    -   Range: [QuantityValue](QuantityValue.md)
    -   in subsets: (environment)
-   [al_sat](al_sat.md) 0..1
    -   Description: Aluminum saturation (esp. For tropical soils)
    -   Range: [QuantityValue](QuantityValue.md)
-   [al_sat_meth](al_sat_meth.md) 0..1
    -   Description: Reference or method used in determining Al
        saturation
    -   Range: [TextValue](TextValue.md)
-   [aminopept_act](aminopept_act.md) 0..1
    -   Description: Measurement of aminopeptidase activity
    -   Range: [QuantityValue](QuantityValue.md)
-   [ammonium](ammonium.md) 0..1
    -   Description: Concentration of ammonium in the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [annual_precpt](annual_precpt.md) 0..1
    -   Description: "The average of all annual precipitation values
        known, or an estimated equivalent value derived by such methods
        as regional indexes or Isohyetal maps."
    -   Range: [QuantityValue](QuantityValue.md)
-   [annual_temp](annual_temp.md) 0..1
    -   Description: Mean annual temperature
    -   Range: [QuantityValue](QuantityValue.md)
-   [bacteria_carb_prod](bacteria_carb_prod.md) 0..1
    -   Description: Measurement of bacterial carbon production
    -   Range: [QuantityValue](QuantityValue.md)
-   [bishomohopanol](bishomohopanol.md) 0..1
    -   Description: Concentration of bishomohopanol
    -   Range: [QuantityValue](QuantityValue.md)
-   [bromide](bromide.md) 0..1
    -   Description: Concentration of bromide
    -   Range: [QuantityValue](QuantityValue.md)
-   [calcium](calcium.md) 0..1
    -   Description: Concentration of calcium in the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [carb_nitro_ratio](carb_nitro_ratio.md) 0..1
    -   Description: Ratio of amount or concentrations of carbon to
        nitrogen
    -   Range: [QuantityValue](QuantityValue.md)
-   [chem_administration](chem_administration.md) 0..1
    -   Description: "List of chemical compounds administered to the
        host or site where sampling occurred, and when
        (e.g. Antibiotics, n fertilizer, air filter); can include
        multiple compounds. For chemical entities of biological interest
        ontology (chebi) (v 163),
        <http://purl.bioontology.org/ontology/chebi>"
    -   Range: [ControlledTermValue](ControlledTermValue.md)
-   [chloride](chloride.md) 0..1
    -   Description: Concentration of chloride in the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [chlorophyll](chlorophyll.md) 0..1
    -   Description: Concentration of chlorophyll
    -   Range: [QuantityValue](QuantityValue.md)
-   [collection_date](collection_date.md) 0..1
    -   Description: "The time of sampling, either as an instance
        (single point in time) or interval. In case no exact time is
        available, the date/time can be right truncated i.e. all of
        these are valid times: 2008-01-23T19:23:10+00:00;
        2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01;
        2008 all are ISO8601 compliant"
    -   Range: [TimestampValue](TimestampValue.md)
    -   in subsets: (environment)
-   [cur_land_use](cur_land_use.md) 0..1
    -   Description: Present state of sample site
    -   Range: [TextValue](TextValue.md)
-   [cur_vegetation](cur_vegetation.md) 0..1
    -   Description: "Vegetation classification from one or more
        standard classification systems, or agricultural crop"
    -   Range: [TextValue](TextValue.md)
-   [cur_vegetation_meth](cur_vegetation_meth.md) 0..1
    -   Description: Reference or method used in vegetation
        classification
    -   Range: [TextValue](TextValue.md)
-   [crop_rotation](crop_rotation.md) 0..1
    -   Description: "Whether or not crop is rotated, and if yes,
        rotation schedule"
    -   Range: [TextValue](TextValue.md)
-   [density](density.md) 0..1
    -   Description: "Density of the sample, which is its mass per unit
        volume (aka volumetric mass density)"
    -   Range: [QuantityValue](QuantityValue.md)
-   [depth](depth.md) 0..1
    -   Description: Please refer to the definitions of depth in the
        environmental packages
    -   Range: [QuantityValue](QuantityValue.md)
    -   in subsets: (environment)
-   [diss_carb_dioxide](diss_carb_dioxide.md) 0..1
    -   Description: Concentration of dissolved carbon dioxide in the
        sample or liquid portion of the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [diss_hydrogen](diss_hydrogen.md) 0..1
    -   Description: Concentration of dissolved hydrogen
    -   Range: [QuantityValue](QuantityValue.md)
-   [diss_inorg_carb](diss_inorg_carb.md) 0..1
    -   Description: "Dissolved inorganic carbon concentration in the
        sample, typically measured after filtering the sample using a
        0.45 micrometer filter"
    -   Range: [QuantityValue](QuantityValue.md)
-   [diss_inorg_phosp](diss_inorg_phosp.md) 0..1
    -   Description: Concentration of dissolved inorganic phosphorus in
        the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [diss_org_carb](diss_org_carb.md) 0..1
    -   Description: "Concentration of dissolved organic carbon in the
        sample, liquid portion of the sample, or aqueous phase of the
        fluid"
    -   Range: [QuantityValue](QuantityValue.md)
-   [diss_org_nitro](diss_org_nitro.md) 0..1
    -   Description: Dissolved organic nitrogen concentration measured
        as; total dissolved nitrogen - NH4 - NO3 - NO2
    -   Range: [QuantityValue](QuantityValue.md)
-   [diss_oxygen](diss_oxygen.md) 0..1
    -   Description: Concentration of dissolved oxygen
    -   Range: [QuantityValue](QuantityValue.md)
-   [drainage_class](drainage_class.md) 0..1
    -   Description: Drainage classification from a standard system such
        as the USDA system
    -   Range: [TextValue](TextValue.md)
-   [elev](elev.md) 0..1
    -   Description: "Elevation of the sampling site is its height above
        a fixed reference point, most commonly the mean sea level.
        Elevation is mainly used when referring to points on the earth's
        surface, while altitude is used for points above the surface,
        such as an aircraft in flight or a spacecraft in orbit"
    -   Range: [QuantityValue](QuantityValue.md)
    -   in subsets: (environment)
-   [env_package](env_package.md) 0..1
    -   Description: "MIxS extension for reporting of measurements and
        observations obtained from one or more of the environments where
        the sample was obtained. All environmental packages listed here
        are further defined in separate subtables. By giving the name of
        the environmental package, a selection of fields can be made
        from the subtables and can be reported"
    -   Range: [TextValue](TextValue.md)
    -   in subsets: (mixs extension)
-   [biosample➞env_broad_scale](biosample_env_broad_scale.md) 1..1
    -   Description: "In this field, report which major environmental
        system your sample or specimen came from. The systems identified
        should have a coarse spatial grain, to provide the general
        environmental context of where the sampling was done (e.g. were
        you in the desert or a rainforest?). We recommend using
        subclasses of ENVOUs biome class:
        <http://purl.obolibrary.org/obo/ENVO_00000428>. Format (one
        term): termLabel \[termID\], Format (multiple terms): termLabel
        \[termID\]termLabel \[termID\]. Example: Annotating a water
        sample from the photic zone in middle of the Atlantic Ocean,
        consider: oceanic epipelagic zone biome \[ENVO:01000033\].
        Example: Annotating a sample from the Amazon rainforest
        consider: tropical moist broadleaf forest biome
        \[ENVO:01000228\]. If needed, request new terms on the ENVO
        tracker, identified here:
        <http://www.obofoundry.org/ontology/envo.html>"
    -   Range: [ControlledTermValue](ControlledTermValue.md)
    -   in subsets: (environment)
-   [biosample➞env_local_scale](biosample_env_local_scale.md) 1..1
    -   Description: "In this field, report the entity or entities which
        are in your sample or specimenUs local vicinity and which you
        believe have significant causal influences on your sample or
        specimen. Please use terms that are present in ENVO and which
        are of smaller spatial grain than your entry for
        env_broad_scale. Format (one term): termLabel \[termID\]; Format
        (multiple terms): termLabel \[termID\]termLabel \[termID\].
        Example: Annotating a pooled sample taken from various
        vegetation layers in a forest consider: canopy
        \[ENVO:00000047\]litter layer \[ENVO:01000338\]shrub layer
        \[ENVO:01000336\]. If needed, request new terms on the ENVO
        tracker, identified here:
        <http://www.obofoundry.org/ontology/envo.html>"
    -   Range: [ControlledTermValue](ControlledTermValue.md)
    -   in subsets: (environment)
-   [biosample➞env_medium](biosample_env_medium.md) 1..1
    -   Description: "In this field, report which environmental material
        or materials (pipe separated) immediately surrounded your sample
        or specimen prior to sampling, using one or more subclasses of
        ENVOUs environmental material class:
        <http://purl.obolibrary.org/obo/ENVO_00010483>. Format (one
        term): termLabel \[termID\]; Format (multiple terms): termLabel
        \[termID\]termLabel \[termID\]. Example: Annotating a fish
        swimming in the upper 100 m of the Atlantic Ocean, consider:
        ocean water \[ENVO:00002151\]. Example: Annotating a duck on a
        pond consider: pond water \[ENVO:00002228\]\|air ENVO_00002005.
        If needed, request new terms on the ENVO tracker, identified
        here: <http://www.obofoundry.org/ontology/envo.html>"
    -   Range: [ControlledTermValue](ControlledTermValue.md)
    -   in subsets: (environment)
-   [extreme_event](extreme_event.md) 0..1
    -   Description: Unusual physical events that may have affected
        microbial populations
    -   Range: [TimestampValue](TimestampValue.md)
-   [fao_class](fao_class.md) 0..1
    -   Description: Soil classification from the FAO World Reference
        Database for Soil Resources. The list can be found at
        <http://www.fao.org/nr/land/sols/soil/wrb-soil-maps/reference-groups>
    -   Range: [TextValue](TextValue.md)
-   [fire](fire.md) 0..1
    -   Description: Historical and/or physical evidence of fire
    -   Range: [TimestampValue](TimestampValue.md)
-   [flooding](flooding.md) 0..1
    -   Description: Historical and/or physical evidence of flooding
    -   Range: [TimestampValue](TimestampValue.md)
-   [geo_loc_name](geo_loc_name.md) 0..1
    -   Description: "The geographical origin of the sample as defined
        by the country or sea name followed by specific region name.
        Country or sea names should be chosen from the INSDC country
        list (<http://insdc.org/country.html>), or the GAZ ontology (v
        1.512) (<http://purl.bioontology.org/ontology/GAZ>)"
    -   Range: [TextValue](TextValue.md)
    -   in subsets: (environment)
-   [glucosidase_act](glucosidase_act.md) 0..1
    -   Description: Measurement of glucosidase activity
    -   Range: [QuantityValue](QuantityValue.md)
-   [heavy_metals](heavy_metals.md) 0..1
    -   Description: Heavy metals present and concentrationsany drug
        used by subject and the frequency of usage; can include multiple
        heavy metals and concentrations
    -   Range: [QuantityValue](QuantityValue.md)
-   [heavy_metals_meth](heavy_metals_meth.md) 0..1
    -   Description: Reference or method used in determining heavy
        metals
    -   Range: [TextValue](TextValue.md)
-   [horizon](horizon.md) 0..1
    -   Description: Specific layer in the land area which measures
        parallel to the soil surface and possesses physical
        characteristics which differ from the layers above and beneath
    -   Range: [TextValue](TextValue.md)
-   [horizon_meth](horizon_meth.md) 0..1
    -   Description: Reference or method used in determining the horizon
    -   Range: [TextValue](TextValue.md)
-   [biosample➞lat_lon](biosample_lat_lon.md) 0..1
    -   Description: This is currently a required field but it's not
        clear if this should be required for human hosts
    -   Range: [GeolocationValue](GeolocationValue.md)
    -   in subsets: (environment)
-   [link_addit_analys](link_addit_analys.md) 0..1
    -   Description: Link to additional analysis results performed on
        the sample
    -   Range: [TextValue](TextValue.md)
-   [link_class_info](link_class_info.md) 0..1
    -   Description: Link to digitized soil maps or other soil
        classification information
    -   Range: [TextValue](TextValue.md)
-   [link_climate_info](link_climate_info.md) 0..1
    -   Description: Link to climate resource
    -   Range: [TextValue](TextValue.md)
-   [local_class](local_class.md) 0..1
    -   Description: Soil classification based on local soil
        classification system
    -   Range: [TextValue](TextValue.md)
-   [local_class_meth](local_class_meth.md) 0..1
    -   Description: Reference or method used in determining the local
        soil classification
    -   Range: [TextValue](TextValue.md)
-   [magnesium](magnesium.md) 0..1
    -   Description: Concentration of magnesium in the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [mean_frict_vel](mean_frict_vel.md) 0..1
    -   Description: Measurement of mean friction velocity
    -   Range: [QuantityValue](QuantityValue.md)
-   [mean_peak_frict_vel](mean_peak_frict_vel.md) 0..1
    -   Description: Measurement of mean peak friction velocity
    -   Range: [QuantityValue](QuantityValue.md)
-   [microbial_biomass](microbial_biomass.md) 0..1
    -   Description: "The part of the organic matter in the soil that
        constitutes living microorganisms smaller than 5-10 micrometer.
        If you keep this, you would need to have correction factors used
        for conversion to the final units"
    -   Range: [QuantityValue](QuantityValue.md)
-   [microbial_biomass_meth](microbial_biomass_meth.md) 0..1
    -   Description: Reference or method used in determining microbial
        biomass
    -   Range: [TextValue](TextValue.md)
-   [misc_param](misc_param.md) 0..1
    -   Description: "Any other measurement performed or parameter
        collected, that is not listed here"
    -   Range: [QuantityValue](QuantityValue.md)
-   [n_alkanes](n_alkanes.md) 0..1
    -   Description: Concentration of n-alkanes; can include multiple
        n-alkanes
    -   Range: [QuantityValue](QuantityValue.md)
-   [nitrate](nitrate.md) 0..1
    -   Description: Concentration of nitrate in the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [nitrite](nitrite.md) 0..1
    -   Description: Concentration of nitrite in the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [org_matter](org_matter.md) 0..1
    -   Description: Concentration of organic matter
    -   Range: [QuantityValue](QuantityValue.md)
-   [org_nitro](org_nitro.md) 0..1
    -   Description: Concentration of organic nitrogen
    -   Range: [QuantityValue](QuantityValue.md)
-   [organism_count](organism_count.md) 0..1
    -   Description: "Total cell count of any organism (or group of
        organisms) per gram, volume or area of sample, should include
        name of organism followed by count. The method that was used for
        the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be
        provided. (example: total prokaryotes; 3.5e7 cells per ml;
        qpcr)"
    -   Range: [QuantityValue](QuantityValue.md)
-   [oxy_stat_samp](oxy_stat_samp.md) 0..1
    -   Description: Oxygenation status of sample
    -   Range: [TextValue](TextValue.md)
-   [part_org_carb](part_org_carb.md) 0..1
    -   Description: Concentration of particulate organic carbon
    -   Range: [QuantityValue](QuantityValue.md)
-   [perturbation](perturbation.md) 0..1
    -   Description: "Type of perturbation, e.g. chemical
        administration, physical disturbance, etc., coupled with
        perturbation regimen including how many times the perturbation
        was repeated, how long each perturbation lasted, and the start
        and end time of the entire perturbation period; can include
        multiple perturbation types"
    -   Range: [TextValue](TextValue.md)
-   [petroleum_hydrocarb](petroleum_hydrocarb.md) 0..1
    -   Description: Concentration of petroleum hydrocarbon
    -   Range: [QuantityValue](QuantityValue.md)
-   [ph](ph.md) 0..1
    -   Description: "Ph measurement of the sample, or liquid portion of
        sample, or aqueous phase of the fluid"
    -   Range: [QuantityValue](QuantityValue.md)
-   [ph_meth](ph_meth.md) 0..1
    -   Description: Reference or method used in determining ph
    -   Range: [TextValue](TextValue.md)
-   [phaeopigments](phaeopigments.md) 0..1
    -   Description: Concentration of phaeopigments; can include
        multiple phaeopigments
    -   Range: [QuantityValue](QuantityValue.md)
-   [phosplipid_fatt_acid](phosplipid_fatt_acid.md) 0..1
    -   Description: Concentration of phospholipid fatty acids; can
        include multiple values
    -   Range: [QuantityValue](QuantityValue.md)
-   [pool_dna_extracts](pool_dna_extracts.md) 0..1
    -   Description: "Indicate whether multiple DNA extractions were
        mixed. If the answer yes, the number of extracts that were
        pooled should be given"
    -   Range: [TextValue](TextValue.md)
-   [potassium](potassium.md) 0..1
    -   Description: Concentration of potassium in the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [pressure](pressure.md) 0..1
    -   Description: "Pressure to which the sample is subject to, in
        atmospheres"
    -   Range: [QuantityValue](QuantityValue.md)
-   [previous_land_use](previous_land_use.md) 0..1
    -   Description: Previous land use and dates
    -   Range: [TextValue](TextValue.md)
-   [previous_land_use_meth](previous_land_use_meth.md) 0..1
    -   Description: Reference or method used in determining previous
        land use and dates
    -   Range: [TextValue](TextValue.md)
-   [profile_position](profile_position.md) 0..1
    -   Description: Cross-sectional position in the hillslope where
        sample was collected.sample area position in relation to
        surrounding areas
    -   Range: [TextValue](TextValue.md)
-   [redox_potential](redox_potential.md) 0..1
    -   Description: "Redox potential, measured relative to a hydrogen
        cell, indicating oxidation or reduction potential"
    -   Range: [QuantityValue](QuantityValue.md)
-   [salinity](salinity.md) 0..1
    -   Description: "Salinity is the total concentration of all
        dissolved salts in a water sample. While salinity can be
        measured by a complete chemical analysis, this method is
        difficult and time consuming. More often, it is instead derived
        from the conductivity measurement. This is known as practical
        salinity. These derivations compare the specific conductance of
        the sample to a salinity standard such as seawater"
    -   Range: [QuantityValue](QuantityValue.md)
-   [salinity_meth](salinity_meth.md) 0..1
    -   Description: Reference or method used in determining salinity
    -   Range: [TextValue](TextValue.md)
-   [samp_collect_device](samp_collect_device.md) 0..1
    -   Description: The method or device employed for collecting the
        sample
    -   Range: [TextValue](TextValue.md)
    -   in subsets: (nucleic acid sequence source)
-   [samp_mat_process](samp_mat_process.md) 0..1
    -   Description: "Any processing applied to the sample during or
        after retrieving the sample from environment. This field accepts
        OBI, for a browser of OBI (v 2018-02-12) terms please see
        <http://purl.bioontology.org/ontology/OBI>"
    -   Range: [ControlledTermValue](ControlledTermValue.md)
    -   in subsets: (nucleic acid sequence source)
-   [samp_store_dur](samp_store_dur.md) 0..1
    -   Description: Duration for which the sample was stored
    -   Range: [TextValue](TextValue.md)
-   [samp_store_loc](samp_store_loc.md) 0..1
    -   Description: "Location at which sample was stored, usually name
        of a specific freezer/room"
    -   Range: [TextValue](TextValue.md)
-   [samp_store_temp](samp_store_temp.md) 0..1
    -   Description: "Temperature at which sample was stored, e.g. -80
        degree Celsius"
    -   Range: [QuantityValue](QuantityValue.md)
-   [samp_vol_we_dna_ext](samp_vol_we_dna_ext.md) 0..1
    -   Description: "Volume (ml), weight (g) of processed sample, or
        surface area swabbed from sample for DNA extraction"
    -   Range: [QuantityValue](QuantityValue.md)
-   [season_temp](season_temp.md) 0..1
    -   Description: Mean seasonal temperature
    -   Range: [QuantityValue](QuantityValue.md)
-   [season_precpt](season_precpt.md) 0..1
    -   Description: "The average of all seasonal precipitation values
        known, or an estimated equivalent value derived by such methods
        as regional indexes or Isohyetal maps."
    -   Range: [QuantityValue](QuantityValue.md)
-   [sieving](sieving.md) 0..1
    -   Description: Collection design of pooled samples and/or sieve
        size and amount of sample sieved
    -   Range: [QuantityValue](QuantityValue.md)
-   [size_frac_low](size_frac_low.md) 0..1
    -   Description: Refers to the mesh/pore size used to
        pre-filter/pre-sort the sample. Materials larger than the size
        threshold are excluded from the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [size_frac_up](size_frac_up.md) 0..1
    -   Description: Refers to the mesh/pore size used to retain the
        sample. Materials smaller than the size threshold are excluded
        from the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [slope_gradient](slope_gradient.md) 0..1
    -   Description: Commonly called 'slope'. The angle between ground
        surface and a horizontal line (in percent). This is the
        direction that overland water would flow. This measure is
        usually taken with a hand level meter or clinometer
    -   Range: [QuantityValue](QuantityValue.md)
-   [slope_aspect](slope_aspect.md) 0..1
    -   Description: "The direction a slope faces. While looking down a
        slope use a compass to record the direction you are facing
        (direction or degrees); e.g., nw or 315 degrees. This measure
        provides an indication of sun and wind exposure that will
        influence soil temperature and evapotranspiration."
    -   Range: [QuantityValue](QuantityValue.md)
-   [sodium](sodium.md) 0..1
    -   Description: Sodium concentration in the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [soil_type](soil_type.md) 0..1
    -   Description: Soil series name or other lower-level
        classification
    -   Range: [TextValue](TextValue.md)
-   [soil_type_meth](soil_type_meth.md) 0..1
    -   Description: Reference or method used in determining soil series
        name or other lower-level classification
    -   Range: [TextValue](TextValue.md)
-   [store_cond](store_cond.md) 0..1
    -   Description: Explain how and for how long the soil sample was
        stored before DNA extraction
    -   Range: [TextValue](TextValue.md)
-   [sulfate](sulfate.md) 0..1
    -   Description: Concentration of sulfate in the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [sulfide](sulfide.md) 0..1
    -   Description: Concentration of sulfide in the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [temp](temp.md) 0..1
    -   Description: Temperature of the sample at the time of sampling
    -   Range: [QuantityValue](QuantityValue.md)
-   [texture](texture.md) 0..1
    -   Description: "The relative proportion of different grain sizes
        of mineral particles in a soil, as described using a standard
        system; express as % sand (50 um to 2 mm), silt (2 um to 50 um),
        and clay (\<2 um) with textural name (e.g., silty clay loam)
        optional."
    -   Range: [QuantityValue](QuantityValue.md)
-   [texture_meth](texture_meth.md) 0..1
    -   Description: Reference or method used in determining soil
        texture
    -   Range: [TextValue](TextValue.md)
-   [tillage](tillage.md) 0..1
    -   Description: Note method(s) used for tilling
    -   Range: [TextValue](TextValue.md)
-   [tidal_stage](tidal_stage.md) 0..1
    -   Description: Stage of tide
    -   Range: [TextValue](TextValue.md)
-   [tot_carb](tot_carb.md) 0..1
    -   Description: Total carbon content
    -   Range: [QuantityValue](QuantityValue.md)
-   [tot_depth_water_col](tot_depth_water_col.md) 0..1
    -   Description: Measurement of total depth of water column
    -   Range: [QuantityValue](QuantityValue.md)
-   [tot_diss_nitro](tot_diss_nitro.md) 0..1
    -   Description: "Total dissolved nitrogen concentration, reported
        as nitrogen, measured by: total dissolved nitrogen = NH4 +
        NO3NO2 + dissolved organic nitrogen"
    -   Range: [QuantityValue](QuantityValue.md)
-   [tot_org_carb](tot_org_carb.md) 0..1
    -   Description: "Definition for soil: total organic carbon content
        of the soil, definition otherwise: total organic carbon content"
    -   Range: [QuantityValue](QuantityValue.md)
-   [tot_org_c\_meth](tot_org_c_meth.md) 0..1
    -   Description: Reference or method used in determining total
        organic carbon
    -   Range: [TextValue](TextValue.md)
-   [tot_nitro_content](tot_nitro_content.md) 0..1
    -   Description: Total nitrogen content of the sample
    -   Range: [QuantityValue](QuantityValue.md)
-   [tot_nitro_content_meth](tot_nitro_content_meth.md) 0..1
    -   Description: Reference or method used in determining the total
        nitrogen
    -   Range: [TextValue](TextValue.md)
-   [tot_phosp](tot_phosp.md) 0..1
    -   Description: "Total phosphorus concentration in the sample,
        calculated by: total phosphorus = total dissolved phosphorus +
        particulate phosphorus"
    -   Range: [QuantityValue](QuantityValue.md)
-   [water_content](water_content.md) 0..1
    -   Description: Water content measurement
    -   Range: [QuantityValue](QuantityValue.md)
-   [water_content_soil_meth](water_content_soil_meth.md) 0..1
    -   Description: Reference or method used in determining the water
        content of soil
    -   Range: [TextValue](TextValue.md)
-   [ecosystem](ecosystem.md) 0..1
    -   Description: TODO
    -   Range: [String](types/String.md)
-   [ecosystem_category](ecosystem_category.md) 0..1
    -   Description: TODO
    -   Range: [String](types/String.md)
-   [ecosystem_type](ecosystem_type.md) 0..1
    -   Description: TODO
    -   Range: [String](types/String.md)
-   [ecosystem_subtype](ecosystem_subtype.md) 0..1
    -   Description: TODO
    -   Range: [String](types/String.md)
-   [specific_ecosystem](specific_ecosystem.md) 0..1
    -   Description: TODO
    -   Range: [String](types/String.md)
-   [add_date](add_date.md) 0..1
    -   Description: The date on which the information was added to the
        database.
    -   Range: [String](types/String.md)
-   [community](community.md) 0..1
    -   Range: [String](types/String.md)
-   [depth2](depth2.md) 0..1
    -   Range: [QuantityValue](QuantityValue.md)
-   [habitat](habitat.md) 0..1
    -   Range: [String](types/String.md)
-   [host_name](host_name.md) 0..1
    -   Range: [String](types/String.md)
-   [identifier](identifier.md) 0..1
    -   Range: [String](types/String.md)
-   [location](location.md) 0..1
    -   Range: [String](types/String.md)
-   [mod_date](mod_date.md) 0..1
    -   Description: The last date on which the database information was
        modified.
    -   Range: [String](types/String.md)
-   [ncbi_taxonomy_name](ncbi_taxonomy_name.md) 0..1
    -   Range: [String](types/String.md)
-   [proport_woa_temperature](proport_woa_temperature.md) 0..1
    -   Range: [String](types/String.md)
-   [salinity_category](salinity_category.md) 0..1
    -   Description: Categorcial description of the sample's salinity.
        Examples: halophile, halotolerant, hypersaline, huryhaline
    -   Range: [String](types/String.md)
-   [sample_collection_site](sample_collection_site.md) 0..1
    -   Range: [String](types/String.md)
-   [soluble_iron_micromol](soluble_iron_micromol.md) 0..1
    -   Range: [String](types/String.md)
-   [subsurface_depth](subsurface_depth.md) 0..1
    -   Range: [QuantityValue](QuantityValue.md)
-   [subsurface_depth2](subsurface_depth2.md) 0..1
    -   Range: [QuantityValue](QuantityValue.md)
-   [GOLD sample identifiers](GOLD_sample_identifiers.md) 0..\*
    -   Description: \['identifiers for corresponding sample in GOLD'\]
    -   Range: [ExternalIdentifier](types/ExternalIdentifier.md)
    -   Example: <https://identifiers.org/gold:GbTODO> None
-   [INSDC biosample identifiers](INSDC_biosample_identifiers.md) 0..\*
    -   Description: \['identifiers for corresponding sample in INSDC'\]
    -   Range: [ExternalIdentifier](types/ExternalIdentifier.md)
    -   Example: <https://identifiers.org/biosample:SAMEA5989477> None
    -   Example: <https://identifiers.org/biosample:SAMD00212331>
        I13_N\_5-10 sample from Soil fungal diversity along elevational
        gradients
-   [INSDC secondary sample
    identifiers](INSDC_secondary_sample_identifiers.md) 0..\*
    -   Description: \['secondary identifiers for corresponding sample
        in INSDC'\]
    -   Range: [ExternalIdentifier](types/ExternalIdentifier.md)
    -   Example: <https://identifiers.org/insdc.sra:DRS166340>
        I13_N\_5-10 sample from Soil fungal diversity along elevational
        gradients

### Inherited from named thing:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)

## Other properties {#other-properties-2}

  ----------------------- ----------------------- -----------------------
  **Aliases:**                                    sample

                                                  material sample

                                                  specimen

                                                  biospecimen

  **Alt Descriptions:**                           A sample contains
                                                  information about the
                                                  sequenced source
                                                  material. Samples are
                                                  associated with
                                                  checklists, which
                                                  define the fields used
                                                  to annotate the
                                                  samples. Samples are
                                                  always associated with
                                                  a taxon. (embl.ena)

  **In Subsets:**                                 sample subset

  **Exact Mappings:**                             OBI:0000747

                                                  NCIT:C43412
  ----------------------- ----------------------- -----------------------

# Class: biosample processing

A process that takes one or more biosamples as inputs and generates one
or as outputs. Examples of outputs include samples cultivated from
another sample or data objects created by instruments runs.

URI:
[nmdc:BiosampleProcessing](https://microbiomedata/meta/BiosampleProcessing)

[![image4](https://yuml.me/diagram/nofunky;dir:TB/class/%5BOmicsProcessing%5D,%5BNamedThing%5D,%5BBiosample%5D%3Chas%20input%200..*-%20%5BBiosampleProcessing%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BBiosampleProcessing%5D%5E-%5BOmicsProcessing%5D,%5BNamedThing%5D%5E-%5BBiosampleProcessing%5D,%5BBiosample%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BOmicsProcessing%5D,%5BNamedThing%5D,%5BBiosample%5D%3Chas%20input%200..*-%20%5BBiosampleProcessing%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BBiosampleProcessing%5D%5E-%5BOmicsProcessing%5D,%5BNamedThing%5D%5E-%5BBiosampleProcessing%5D,%5BBiosample%5D)

## Parents

-   is_a: [NamedThing](NamedThing.md) - a databased entity or
    concept/class

## Children

-   [OmicsProcessing](OmicsProcessing.md) - The methods and processes
    used to generate omics data from a biosample or organism.

## Referenced by Class {#referenced-by-class-4}

## Attributes {#attributes-4}

### Own {#own-4}

-   [biosample processing➞has input](biosample_processing_has_input.md)
    0..\*
    -   Description: An input to a process.
    -   Range: [Biosample](Biosample.md)

### Inherited from named thing:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

## Other properties {#other-properties-3}

+---------------------+-----------------------+
| **Aliases:**        | > material processing |
+---------------------+-----------------------+
| **Broad Mappings:** | > OBI:0000094         |
+---------------------+-----------------------+

# Class: boolean value

A value that is a boolean

URI:
<https://microbiomedata/schema/workflow_execution_activity/BooleanValue>

[![image5](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BBooleanValue%7Chas_boolean_value:boolean%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BBooleanValue%7Chas_boolean_value:boolean%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)

## Parents

-   is_a: [AttributeValue](AttributeValue.md) - The value for any value
    of a attribute for a sample. This object can hold both the
    un-normalized atomic value and the structured value

## Attributes {#attributes-5}

### Own {#own-5}

-   [has boolean value](has_boolean_value.md) 0..1
    -   Description: Links a quantity value to a boolean
    -   Range: [Boolean](types/Boolean.md)

### Inherited from attribute value:

-   [has raw value](has_raw_value.md) 0..1
    -   Description: The value that was specified for an annotation in
        raw form, i.e. a string. E.g. "2 cm" or "2-4 cm"
    -   Range: [String](types/String.md)
-   [was generated by](was_generated_by.md) 0..1
    -   Range: [Activity](Activity.md)
-   [attribute value➞type](attribute_value_type.md) 0..1
    -   Description: An optional string that specified the type of
        object.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None

# Class: chemical entity

An atom or molecule that can be represented with a chemical formula.
Include lipids, glycans, natural products, drugs. There may be different
terms for distinct acid-base forms, protonation states

URI:
<https://microbiomedata/schema/workflow_execution_activity/ChemicalEntity>

[![image6](https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BMetaboliteQuantification%5D,%5BMetaboliteQuantification%5D-%20metabolite%20quantified%200..1%3E%5BChemicalEntity%7Cinchi:string%20%3F;inchi_key:string%20%3F;smiles:string%20*;chemical_formula:string%20%3F;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BOntologyClass%5D%5E-%5BChemicalEntity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BMetaboliteQuantification%5D,%5BMetaboliteQuantification%5D-%20metabolite%20quantified%200..1%3E%5BChemicalEntity%7Cinchi:string%20%3F;inchi_key:string%20%3F;smiles:string%20*;chemical_formula:string%20%3F;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BOntologyClass%5D%5E-%5BChemicalEntity%5D)

## Identifier prefixes

-   KEGG.COMPOUND
-   CHEBI
-   CHEMBL.COMPOUND
-   DRUGBANK
-   PUBCHEM.COMPOUND
-   CAS
-   HMDB
-   MESH

## Parents

-   is_a: [OntologyClass](OntologyClass.md)

## Referenced by Class {#referenced-by-class-5}

-   [MetaboliteQuantification](MetaboliteQuantification.md) [metabolite
    quantification➞metabolite
    quantified](metabolite_quantification_metabolite_quantified.md) 0..1
    [ChemicalEntity](ChemicalEntity.md)
-   **None** [metabolite quantified](metabolite_quantified.md) 0..1
    [ChemicalEntity](ChemicalEntity.md)

## Attributes {#attributes-6}

### Own {#own-6}

-   [chemical entity➞inchi](chemical_entity_inchi.md) 0..1
    -   Range: [String](types/String.md)
-   [chemical entity➞inchi key](chemical_entity_inchi_key.md) 0..1
    -   Range: [String](types/String.md)
-   [chemical entity➞smiles](chemical_entity_smiles.md) 0..\*
    -   Description: A string encoding of a molecular graph, no chiral
        or isotopic information. There are usually a large number of
        valid SMILES which represent a given structure. For example,
        CCO, OCC and C(O)C all specify the structure of ethanol.
    -   Range: [String](types/String.md)
-   [chemical entity➞chemical
    formula](chemical_entity_chemical_formula.md) 0..1
    -   Description: A generic grouping for miolecular formulae and
        empirican formulae
    -   Range: [String](types/String.md)

### Inherited from ontology class:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

## Other properties {#other-properties-4}

  ----------------------- ----------------------- -----------------------
  **Aliases:**                                    metabolite

                                                  chemical substance

                                                  chemical compound

                                                  chemical

  **See also:**                                   <https://biocon>
                                                  ductor.org/packages/d
                                                  evel/data/annotation/
                                                  vignettes/metaboliteI
                                                  Dmapping/inst/doc/met
                                                  aboliteIDmapping.html

  **Exact Mappings:**                             biol
                                                  ink:ChemicalSubstance
  ----------------------- ----------------------- -----------------------

# Class: controlled term value

A controlled term or class from an ontology

URI:
<https://microbiomedata/schema/workflow_execution_activity/ControlledTermValue>

[![image7](https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BOntologyClass%5D%3Cterm%200..1-++%5BControlledTermValue%7Chas_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D%5E-%5BControlledTermValue%5D,%5BAttributeValue%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BOntologyClass%5D%3Cterm%200..1-++%5BControlledTermValue%7Chas_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D%5E-%5BControlledTermValue%5D,%5BAttributeValue%5D,%5BActivity%5D)

## Parents

-   is_a: [AttributeValue](AttributeValue.md) - The value for any value
    of a attribute for a sample. This object can hold both the
    un-normalized atomic value and the structured value

## Referenced by Class {#referenced-by-class-6}

## Attributes {#attributes-7}

### Own {#own-7}

-   [term](term.md) 0..1
    -   Description: pointer to an ontology class
    -   Range: [OntologyClass](OntologyClass.md)

### Inherited from attribute value:

-   [has raw value](has_raw_value.md) 0..1
    -   Description: The value that was specified for an annotation in
        raw form, i.e. a string. E.g. "2 cm" or "2-4 cm"
    -   Range: [String](types/String.md)
-   [was generated by](was_generated_by.md) 0..1
    -   Range: [Activity](Activity.md)
-   [attribute value➞type](attribute_value_type.md) 0..1
    -   Description: An optional string that specified the type of
        object.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None

# Class: credit association

This class supports binding associated researchers to studies. There
will be at least a slot for a CRediT Contributor Role
(<https://casrai.org/credit/>) and for a person value Specifically see
the associated researchers tab on the
NMDC_SampleMetadata-V4_CommentsForUpdates at
<https://docs.google.com/spreadsheets/d/1INlBo5eoqn2efn4H2P2i8rwRBtnbDVTqXrochJEAPko/edit#gid=0>

URI:
[nmdc:CreditAssociation](https://microbiomedata/meta/CreditAssociation)

[![image8](https://yuml.me/diagram/nofunky;dir:TB/class/%5BStudy%5D,%5BPersonValue%5D,%5BPersonValue%5D%3Capplies%20to%20person%201..1-++%5BCreditAssociation%7Capplied_role:credit_enum;type:string%20%3F%5D,%5BStudy%5D++-%20has%20credit%20associations%200..*%3E%5BCreditAssociation%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BStudy%5D,%5BPersonValue%5D,%5BPersonValue%5D%3Capplies%20to%20person%201..1-++%5BCreditAssociation%7Capplied_role:credit_enum;type:string%20%3F%5D,%5BStudy%5D++-%20has%20credit%20associations%200..*%3E%5BCreditAssociation%5D)

## Referenced by Class {#referenced-by-class-7}

-   [Study](Study.md) [has credit
    associations](has_credit_associations.md) 0..\*
    [CreditAssociation](CreditAssociation.md)

## Attributes {#attributes-8}

### Own {#own-8}

-   [applies to person](applies_to_person.md) 1..1
    -   Range: [PersonValue](PersonValue.md)
-   [applied role](applied_role.md) 1..1
    -   Range: [credit enum](credit%20enum.md)
-   [type](type.md) 0..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None

## Other properties {#other-properties-5}

+---------------+--------------------------+
| **Aliases:**  | > study role             |
+---------------+--------------------------+
|               | > credit table           |
+---------------+--------------------------+
|               | > associated researchers |
+---------------+--------------------------+
| **Mappings:** | > prov:Association       |
+---------------+--------------------------+

# Class: data object

An object that primarily consists of symbols that represent information.
Files, records, and omics data are examples of data objects.

URI: [nmdc:DataObject](https://microbiomedata/meta/DataObject)

[![image9](https://yuml.me/diagram/nofunky;dir:TB/class/%5BNamedThing%5D,%5BDatabase%5D,%5BActivity%5D%3Cwas%20generated%20by%200..1-%20%5BDataObject%7Cfile_size_bytes:bytes%20%3F;md5_checksum:string%20%3F;data_object_type:file_type_enum%20%3F;compression_type:string%20%3F;url:string%20%3F;type:string%20%3F;name:string;description:string;id(i):string;alternative_identifiers(i):string%20*%5D,%5BDatabase%5D++-%20data%20object%20set%200..*%3E%5BDataObject%5D,%5BNamedThing%5D%5E-%5BDataObject%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BNamedThing%5D,%5BDatabase%5D,%5BActivity%5D%3Cwas%20generated%20by%200..1-%20%5BDataObject%7Cfile_size_bytes:bytes%20%3F;md5_checksum:string%20%3F;data_object_type:file_type_enum%20%3F;compression_type:string%20%3F;url:string%20%3F;type:string%20%3F;name:string;description:string;id(i):string;alternative_identifiers(i):string%20*%5D,%5BDatabase%5D++-%20data%20object%20set%200..*%3E%5BDataObject%5D,%5BNamedThing%5D%5E-%5BDataObject%5D,%5BActivity%5D)

## Identifier prefixes

-   GOLD

## Parents

-   is_a: [NamedThing](NamedThing.md) - a databased entity or
    concept/class

## Referenced by Class {#referenced-by-class-8}

-   [Database](Database.md) [data object set](data_object_set.md) 0..\*
    [DataObject](DataObject.md)

## Attributes {#attributes-9}

### Own {#own-9}

-   [file size bytes](file_size_bytes.md) 0..1
    -   Description: Size of the file in bytes
    -   Range: [Bytes](types/Bytes.md)
-   [md5 checksum](md5_checksum.md) 0..1
    -   Description: MD5 checksum of file (pre-compressed)
    -   Range: [String](types/String.md)
-   [data object type](data_object_type.md) 0..1
    -   Description: The type of file represented by the data object.
    -   Range: [file type enum](file%20type%20enum.md)
    -   Example: FT ICR-MS Analysis Results None
    -   Example: GC-MS Metabolomics Results None
-   [compression type](compression_type.md) 0..1
    -   Description: If provided, specifies the compression type
    -   Range: [String](types/String.md)
    -   Example: gzip None
-   [was generated by](was_generated_by.md) 0..1
    -   Range: [Activity](Activity.md)
-   [url](url.md) 0..1
    -   Range: [String](types/String.md)
-   [type](type.md) 0..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [data object➞name](data_object_name.md) 1..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [data object➞description](data_object_description.md) 1..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)

### Inherited from named thing:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

## Other properties {#other-properties-6}

  **In Subsets:**    data object subset
  ------------------ --------------------
                     
  Subset: data obj   ect subset

Subset consisting of the data objects that either inputs or outputs of
processes or workflows.

URI:
[nmdc:data_object_subset](https://microbiomedata/meta/data_object_subset)

## Classes

-   [DataObject](DataObject.md) - An object that primarily consists of
    symbols that represent information. Files, records, and omics data
    are examples of data objects.

## Mixins

## Slots

## Types

## Enums

# Class: database

An abstract holder for any set of metadata and data. It does not need to
correspond to an actual managed databse top level holder class. When
translated to JSON-Schema this is the 'root' object. It should contain
pointers to other objects of interest

URI: [nmdc:Database](https://microbiomedata/meta/Database)

[![image10](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BStudy%5D,%5BReadBasedAnalysisActivity%5D,%5BReadQCAnalysisActivity%5D,%5BOmicsProcessing%5D,%5BNomAnalysisActivity%5D,%5BMetatranscriptomeActivity%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BMetagenomeAssembly%5D,%5BMetagenomeAnnotationActivity%5D,%5BMetabolomicsAnalysisActivity%5D,%5BGenomeFeature%5D,%5BFunctionalAnnotation%5D,%5BGenomeFeature%5D%3Cgenome%20feature%20set%200..*-++%5BDatabase%7Cnmdc_schema_version:string%20%3F;date_created:string%20%3F;etl_software_version:string%20%3F%5D,%5BFunctionalAnnotation%5D%3Cfunctional%20annotation%20set%200..*-++%5BDatabase%5D,%5BOmicsProcessing%5D%3Comics%20processing%20set%200..*-++%5BDatabase%5D,%5BNomAnalysisActivity%5D%3Cnom%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BReadBasedAnalysisActivity%5D%3Cread%20based%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BReadQCAnalysisActivity%5D%3Cread%20QC%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BMetatranscriptomeActivity%5D%3Cmetatranscriptome%20activity%20set%200..*-++%5BDatabase%5D,%5BMetagenomeAssembly%5D%3Cmetagenome%20assembly%20set%200..*-++%5BDatabase%5D,%5BMetagenomeAnnotationActivity%5D%3Cmetagenome%20annotation%20activity%20set%200..*-++%5BDatabase%5D,%5BMetaproteomicsAnalysisActivity%5D%3Cmetaproteomics%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BMetabolomicsAnalysisActivity%5D%3Cmetabolomics%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BMAGsAnalysisActivity%5D%3Cmags%20activity%20set%200..*-++%5BDatabase%5D,%5BWorkflowExecutionActivity%5D%3Cactivity%20set%200..*-++%5BDatabase%5D,%5BDataObject%5D%3Cdata%20object%20set%200..*-++%5BDatabase%5D,%5BStudy%5D%3Cstudy%20set%200..*-++%5BDatabase%5D,%5BBiosample%5D%3Cbiosample%20set%200..*-++%5BDatabase%5D,%5BDataObject%5D,%5BBiosample%5D,%5BMAGsAnalysisActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BStudy%5D,%5BReadBasedAnalysisActivity%5D,%5BReadQCAnalysisActivity%5D,%5BOmicsProcessing%5D,%5BNomAnalysisActivity%5D,%5BMetatranscriptomeActivity%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BMetagenomeAssembly%5D,%5BMetagenomeAnnotationActivity%5D,%5BMetabolomicsAnalysisActivity%5D,%5BGenomeFeature%5D,%5BFunctionalAnnotation%5D,%5BGenomeFeature%5D%3Cgenome%20feature%20set%200..*-++%5BDatabase%7Cnmdc_schema_version:string%20%3F;date_created:string%20%3F;etl_software_version:string%20%3F%5D,%5BFunctionalAnnotation%5D%3Cfunctional%20annotation%20set%200..*-++%5BDatabase%5D,%5BOmicsProcessing%5D%3Comics%20processing%20set%200..*-++%5BDatabase%5D,%5BNomAnalysisActivity%5D%3Cnom%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BReadBasedAnalysisActivity%5D%3Cread%20based%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BReadQCAnalysisActivity%5D%3Cread%20QC%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BMetatranscriptomeActivity%5D%3Cmetatranscriptome%20activity%20set%200..*-++%5BDatabase%5D,%5BMetagenomeAssembly%5D%3Cmetagenome%20assembly%20set%200..*-++%5BDatabase%5D,%5BMetagenomeAnnotationActivity%5D%3Cmetagenome%20annotation%20activity%20set%200..*-++%5BDatabase%5D,%5BMetaproteomicsAnalysisActivity%5D%3Cmetaproteomics%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BMetabolomicsAnalysisActivity%5D%3Cmetabolomics%20analysis%20activity%20set%200..*-++%5BDatabase%5D,%5BMAGsAnalysisActivity%5D%3Cmags%20activity%20set%200..*-++%5BDatabase%5D,%5BWorkflowExecutionActivity%5D%3Cactivity%20set%200..*-++%5BDatabase%5D,%5BDataObject%5D%3Cdata%20object%20set%200..*-++%5BDatabase%5D,%5BStudy%5D%3Cstudy%20set%200..*-++%5BDatabase%5D,%5BBiosample%5D%3Cbiosample%20set%200..*-++%5BDatabase%5D,%5BDataObject%5D,%5BBiosample%5D,%5BMAGsAnalysisActivity%5D)

## Referenced by Class {#referenced-by-class-9}

## Attributes {#attributes-10}

### Own {#own-10}

-   [biosample set](biosample_set.md) 0..\*
    -   Description: This property links a database object to the set of
        samples within it.
    -   Range: [Biosample](Biosample.md)
-   [study set](study_set.md) 0..\*
    -   Description: This property links a database object to the set of
        studies within it.
    -   Range: [Study](Study.md)
-   [data object set](data_object_set.md) 0..\*
    -   Description: This property links a database object to the set of
        data objects within it.
    -   Range: [DataObject](DataObject.md)
-   [activity set](activity_set.md) 0..\*
    -   Description: This property links a database object to the set of
        workflow activities.
    -   Range: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
-   [mags activity set](mags_activity_set.md) 0..\*
    -   Description: This property links a database object to the set of
        MAGs analysis activities.
    -   Range: [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [metabolomics analysis activity
    set](metabolomics_analysis_activity_set.md) 0..\*
    -   Description: This property links a database object to the set of
        metabolomics analysis activities.
    -   Range:
        [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [metaproteomics analysis activity
    set](metaproteomics_analysis_activity_set.md) 0..\*
    -   Description: This property links a database object to the set of
        metaproteomics analysis activities.
    -   Range:
        [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [metagenome annotation activity
    set](metagenome_annotation_activity_set.md) 0..\*
    -   Description: This property links a database object to the set of
        metagenome annotation activities.
    -   Range:
        [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [metagenome assembly set](metagenome_assembly_set.md) 0..\*
    -   Description: This property links a database object to the set of
        metagenome assembly activities.
    -   Range: [MetagenomeAssembly](MetagenomeAssembly.md)
-   [metatranscriptome activity set](metatranscriptome_activity_set.md)
    0..\*
    -   Description: This property links a database object to the set of
        metatranscriptome analysis activities.
    -   Range: [MetatranscriptomeActivity](MetatranscriptomeActivity.md)
-   [read QC analysis activity set](read_QC_analysis_activity_set.md)
    0..\*
    -   Description: This property links a database object to the set of
        read QC analysis activities.
    -   Range: [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
-   [read based analysis activity
    set](read_based_analysis_activity_set.md) 0..\*
    -   Description: This property links a database object to the set of
        read based analysis activities.
    -   Range: [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [nom analysis activity set](nom_analysis_activity_set.md) 0..\*
    -   Description: This property links a database object to the set of
        natural organic matter (NOM) analysis activities.
    -   Range: [NomAnalysisActivity](NomAnalysisActivity.md)
-   [omics processing set](omics_processing_set.md) 0..\*
    -   Description: This property links a database object to the set of
        omics processings within it.
    -   Range: [OmicsProcessing](OmicsProcessing.md)
-   [functional annotation set](functional_annotation_set.md) 0..\*
    -   Description: This property links a database object to the set of
        all functional annotations
    -   Range: [FunctionalAnnotation](FunctionalAnnotation.md)
-   [genome feature set](genome_feature_set.md) 0..\*
    -   Description: This property links a database object to the set of
        all features
    -   Range: [GenomeFeature](GenomeFeature.md)
-   [database➞nmdc schema version](database_nmdc_schema_version.md) 0..1
    -   Description: TODO
    -   Range: [String](types/String.md)
-   [database➞date created](database_date_created.md) 0..1
    -   Description: TODO
    -   Range: [String](types/String.md)
-   [database➞etl software version](database_etl_software_version.md)
    0..1
    -   Description: TODO
    -   Range: [String](types/String.md)

## Other properties {#other-properties-7}

  **Aliases:**    NMDC metadata object
  --------------- ----------------------
                  
  Subset: envir   onment

URI:
<https://microbiomedata/schema/workflow_execution_activity/environment>

## Classes

## Mixins

## Slots

## Types

## Enums

# Class: environmental material term

URI:
<https://microbiomedata/schema/workflow_execution_activity/EnvironmentalMaterialTerm>

[![image11](https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BOntologyClass%5D%5E-%5BEnvironmentalMaterialTerm%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BOntologyClass%5D%5E-%5BEnvironmentalMaterialTerm%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D)

## Parents

-   is_a: [OntologyClass](OntologyClass.md)

## Attributes {#attributes-11}

### Inherited from ontology class:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

# Class: functional annotation

An assignment of a function term (e.g. reaction or pathway) that is
executed by a gene product, or which the gene product plays an active
role in. Functional annotations can be assigned manually by curators, or
automatically in workflows. In the context of NMDC, all function
annotation is performed automatically, typically using HMM or Blast type
methods

URI:
[nmdc:FunctionalAnnotation](https://microbiomedata/meta/FunctionalAnnotation)

[![image12](https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BMetagenomeAnnotationActivity%5D,%5BGeneProduct%5D,%5BOntologyClass%5D%3Ctype%200..1-%20%5BFunctionalAnnotation%7Chas_function:string%20%3F%5D,%5BGeneProduct%5D%3Csubject%200..1-%20%5BFunctionalAnnotation%5D,%5BMetagenomeAnnotationActivity%5D%3Cwas%20generated%20by%200..1-%20%5BFunctionalAnnotation%5D,%5BDatabase%5D++-%20functional%20annotation%20set%200..*%3E%5BFunctionalAnnotation%5D,%5BDatabase%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BMetagenomeAnnotationActivity%5D,%5BGeneProduct%5D,%5BOntologyClass%5D%3Ctype%200..1-%20%5BFunctionalAnnotation%7Chas_function:string%20%3F%5D,%5BGeneProduct%5D%3Csubject%200..1-%20%5BFunctionalAnnotation%5D,%5BMetagenomeAnnotationActivity%5D%3Cwas%20generated%20by%200..1-%20%5BFunctionalAnnotation%5D,%5BDatabase%5D++-%20functional%20annotation%20set%200..*%3E%5BFunctionalAnnotation%5D,%5BDatabase%5D)

## Referenced by Class {#referenced-by-class-10}

-   [Database](Database.md) [functional annotation
    set](functional_annotation_set.md) 0..\*
    [FunctionalAnnotation](FunctionalAnnotation.md)

## Attributes {#attributes-12}

### Own {#own-11}

-   [functional annotation➞was generated
    by](functional_annotation_was_generated_by.md) 0..1
    -   Description: provenance for the annotation.
    -   Range:
        [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [subject](subject.md) 0..1
    -   Range: [GeneProduct](GeneProduct.md)
-   [functional annotation➞has
    function](functional_annotation_has_function.md) 0..1
    -   Range: [String](types/String.md)
-   [functional annotation➞type](functional_annotation_type.md) 0..1
    -   Description: TODO
    -   Range: [OntologyClass](OntologyClass.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None

## Other properties {#other-properties-8}

  ----------------------- ----------------------- -----------------------
  **Comments:**                                   move id slot usage
                                                  patterns to
                                                  has_function slot
                                                  usage?

  **See also:**                                   <https://img>.
                                                  jgi.doe.gov/docs/func
                                                  tional-annotation.pdf

                                                  h ttps://github.com/mic
                                                  robiomedata/mg_annota
                                                  tion/blob/master/func
                                                  tional-annotation.wdl

  **Narrow Mappings:**                            biolink:Ge
                                                  neToGoTermAssociation
  ----------------------- ----------------------- -----------------------

# Class: functional annotation term

Abstract grouping class for any term/descriptor that can be applied to a
functional unit of a genome (protein, ncRNA, complex).

URI:
[nmdc:FunctionalAnnotationTerm](https://microbiomedata/meta/FunctionalAnnotationTerm)

[![image13](https://yuml.me/diagram/nofunky;dir:TB/class/%5BReaction%5D,%5BPathway%5D,%5BOrthologyGroup%5D,%5BOntologyClass%5D,%5BFunctionalAnnotationTerm%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D%5E-%5BReaction%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BPathway%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BOrthologyGroup%5D,%5BOntologyClass%5D%5E-%5BFunctionalAnnotationTerm%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BReaction%5D,%5BPathway%5D,%5BOrthologyGroup%5D,%5BOntologyClass%5D,%5BFunctionalAnnotationTerm%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D%5E-%5BReaction%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BPathway%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BOrthologyGroup%5D,%5BOntologyClass%5D%5E-%5BFunctionalAnnotationTerm%5D)

## Parents

-   is_a: [OntologyClass](OntologyClass.md)

## Children

-   [OrthologyGroup](OrthologyGroup.md) - A set of genes or gene
    products in which all members are orthologous
-   [Pathway](Pathway.md) - A pathway is a sequence of steps/reactions
    carried out by an organism or community of organisms
-   [Reaction](Reaction.md) - An individual biochemical transformation
    carried out by a functional unit of an organism, in which a
    collection of substrates are transformed into a collection of
    products. Can also represent transporters

## Referenced by Class {#referenced-by-class-11}

## Attributes {#attributes-13}

### Inherited from ontology class:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

## Other properties {#other-properties-9}

+--------------+-------------------------+
| **Aliases:** | > function              |
+--------------+-------------------------+
|              | > functional annotation |
+--------------+-------------------------+

# Slot: GOLD analysis project identifiers

\['identifiers for corresponding analysis project in GOLD'\]

URI:
<https://microbiomedata/schema/workflow_execution_activity/GOLD_analysis_project_identifiers>

## Domain and Range

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents

-   is_a: [analysis identifiers](analysis_identifiers.md)

## Children

## Used by

## Other properties {#other-properties-10}

  ----------------------- ----------------------- -------------------------
  **Examples:**                                   Example(va
                                                  lue='<https://identifi>
                                                  ers.org/gold:GaTODO',
                                                  description=None)

  ----------------------- ----------------------- -------------------------

# Slot: GOLD identifiers

URI:
<https://microbiomedata/schema/workflow_execution_activity/GOLD_identifiers>

## Domain and Range

None → 0..1 [String](types/String.md)

## Parents

## Children

## Used by

## Other properties {#other-properties-11}

  **See also:**    <https://gold.jgi.doe.gov/>
  ---------------- -----------------------------
                   
  Slot: GOLD sam   ple identifiers

\['identifiers for corresponding sample in GOLD'\]

URI:
<https://microbiomedata/schema/workflow_execution_activity/GOLD_sample_identifiers>

## Domain and Range

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents

-   is_a: [sample identifiers](sample_identifiers.md)

## Children

## Used by

## Other properties {#other-properties-12}

  ----------------------- ----------------------- -------------------------
  **Examples:**                                   Example(va
                                                  lue='<https://identifi>
                                                  ers.org/gold:GbTODO',
                                                  description=None)

  ----------------------- ----------------------- -------------------------

# Slot: GOLD sequencing project identifiers

\['identifiers for corresponding sequencing project in GOLD'\]

URI:
<https://microbiomedata/schema/workflow_execution_activity/GOLD_sequencing_project_identifiers>

## Domain and Range

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents

-   is_a: [external database
    identifiers](external_database_identifiers.md)

## Children

## Used by

## Other properties {#other-properties-13}

  ----------------------- ----------------------- -------------------------
  **Examples:**                                   Example(value
                                                  ='<https://identifiers>
                                                  .org/gold:Gp0108335',
                                                  description=None)

  ----------------------- ----------------------- -------------------------

# Slot: GOLD study identifiers

\['identifiers for corresponding project in GOLD'\]

URI:
<https://microbiomedata/schema/workflow_execution_activity/GOLD_study_identifiers>

## Domain and Range

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents

-   is_a: [study identifiers](study_identifiers.md)

## Children

## Used by

## Other properties {#other-properties-14}

  ----------------------- ----------------------- -------------------------
  **Examples:**                                   Example(value
                                                  ='<https://identifiers>
                                                  .org/gold:Gs0110115',
                                                  description=None)

  **See also:**                                   <https://gol>
                                                  d.jgi.doe.gov/studies
  ----------------------- ----------------------- -------------------------

# Class: gene product

A molecule encoded by a gene that has an evolved function

URI:
<https://microbiomedata/schema/workflow_execution_activity/GeneProduct>

[![image14](https://yuml.me/diagram/nofunky;dir:TB/class/%5BProteinQuantification%5D,%5BPeptideQuantification%5D,%5BNamedThing%5D,%5BPeptideQuantification%5D-%20all%20proteins%200..*%3E%5BGeneProduct%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BPeptideQuantification%5D-%20best%20protein%200..1%3E%5BGeneProduct%5D,%5BProteinQuantification%5D-%20all%20proteins%200..*%3E%5BGeneProduct%5D,%5BProteinQuantification%5D-%20best%20protein%200..1%3E%5BGeneProduct%5D,%5BNamedThing%5D%5E-%5BGeneProduct%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BProteinQuantification%5D,%5BPeptideQuantification%5D,%5BNamedThing%5D,%5BPeptideQuantification%5D-%20all%20proteins%200..*%3E%5BGeneProduct%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BPeptideQuantification%5D-%20best%20protein%200..1%3E%5BGeneProduct%5D,%5BProteinQuantification%5D-%20all%20proteins%200..*%3E%5BGeneProduct%5D,%5BProteinQuantification%5D-%20best%20protein%200..1%3E%5BGeneProduct%5D,%5BNamedThing%5D%5E-%5BGeneProduct%5D)

## Identifier prefixes

-   UniProtKB
-   gtpo
-   PR

## Parents

-   is_a: [NamedThing](NamedThing.md) - a databased entity or
    concept/class

## Referenced by Class {#referenced-by-class-12}

-   **None** [all proteins](all_proteins.md) 0..\*
    [GeneProduct](GeneProduct.md)
-   **None** [best protein](best_protein.md) 0..1
    [GeneProduct](GeneProduct.md)
-   [PeptideQuantification](PeptideQuantification.md) [peptide
    quantification➞all proteins](peptide_quantification_all_proteins.md)
    0..\* [GeneProduct](GeneProduct.md)
-   [PeptideQuantification](PeptideQuantification.md) [peptide
    quantification➞best protein](peptide_quantification_best_protein.md)
    0..1 [GeneProduct](GeneProduct.md)
-   [ProteinQuantification](ProteinQuantification.md) [protein
    quantification➞all proteins](protein_quantification_all_proteins.md)
    0..\* [GeneProduct](GeneProduct.md)
-   [ProteinQuantification](ProteinQuantification.md) [protein
    quantification➞best protein](protein_quantification_best_protein.md)
    0..1 [GeneProduct](GeneProduct.md)

## Attributes {#attributes-14}

### Inherited from named thing:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

## Other properties {#other-properties-15}

  ----------------------- ----------------------- -----------------------
  **Comments:**                                   we may include a more
                                                  general gene product
                                                  class in future to
                                                  allow for ncRNA
                                                  annotation

  **Exact Mappings:**                             biolink:GeneProduct
  ----------------------- ----------------------- -----------------------

# Class: genome feature

A feature localized to an interval along a genome

URI: [nmdc:GenomeFeature](https://microbiomedata/meta/GenomeFeature)

[![image15](https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BGeneProduct%5D%3Cencodes%200..1-%20%5BGenomeFeature%7Cseqid:string;start:integer;end:integer;strand:string%20%3F;phase:integer%20%3F;feature_type:string%20%3F%5D,%5BOntologyClass%5D%3Ctype%200..1-%20%5BGenomeFeature%5D,%5BDatabase%5D++-%20genome%20feature%20set%200..*%3E%5BGenomeFeature%5D,%5BGeneProduct%5D,%5BDatabase%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BOntologyClass%5D,%5BGeneProduct%5D%3Cencodes%200..1-%20%5BGenomeFeature%7Cseqid:string;start:integer;end:integer;strand:string%20%3F;phase:integer%20%3F;feature_type:string%20%3F%5D,%5BOntologyClass%5D%3Ctype%200..1-%20%5BGenomeFeature%5D,%5BDatabase%5D++-%20genome%20feature%20set%200..*%3E%5BGenomeFeature%5D,%5BGeneProduct%5D,%5BDatabase%5D)

## Referenced by Class {#referenced-by-class-13}

-   [Database](Database.md) [genome feature set](genome_feature_set.md)
    0..\* [GenomeFeature](GenomeFeature.md)

## Attributes {#attributes-15}

### Own {#own-12}

-   [genome feature➞seqid](genome_feature_seqid.md) 1..1
    -   Description: The ID of the landmark used to establish the
        coordinate system for the current feature.
    -   Range: [String](types/String.md)
-   [genome feature➞type](genome_feature_type.md) 0..1
    -   Description: A type from the sequence ontology
    -   Range: [OntologyClass](OntologyClass.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [genome feature➞start](genome_feature_start.md) 1..1
    -   Description: The start of the feature in positive 1-based
        integer coordinates
    -   Range: [Integer](types/Integer.md)
-   [genome feature➞end](genome_feature_end.md) 1..1
    -   Description: The end of the feature in positive 1-based integer
        coordinates
    -   Range: [Integer](types/Integer.md)
-   [genome feature➞strand](genome_feature_strand.md) 0..1
    -   Description: The strand on which a feature is located. Has a
        value of '+' (sense strand or forward strand) or '-' (anti-sense
        strand or reverse strand).
    -   Range: [String](types/String.md)
-   [genome feature➞phase](genome_feature_phase.md) 0..1
    -   Description: The phase for a coding sequence entity. For
        example, phase of a CDS as represented in a GFF3 with a value of
        0, 1 or 2.
    -   Range: [Integer](types/Integer.md)
-   [genome feature➞encodes](genome_feature_encodes.md) 0..1
    -   Description: The gene product encoded by this feature. Typically
        this is used for a CDS feature or gene feature which will encode
        a protein. It can also be used by a nc transcript ot gene
        feature that encoded a ncRNA
    -   Range: [GeneProduct](GeneProduct.md)
-   [genome feature➞feature type](genome_feature_feature_type.md) 0..1
    -   Description: TODO: Yuri to write
    -   Range: [String](types/String.md)

## Other properties {#other-properties-16}

  ----------------------- ----------------------- -----------------------
  **Comments:**                                   corresponds to an entry
                                                  in GFF3

  **See also:**                                   <https://gith>
                                                  ub.com/The-Sequence-O
                                                  ntology/Specification
                                                  s/blob/master/gff3.md
  ----------------------- ----------------------- -----------------------

# Class: geolocation value

A normalized value for a location on the earth's surface

URI:
<https://microbiomedata/schema/workflow_execution_activity/GeolocationValue>

[![image16](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BGeolocationValue%7Clatitude:decimal_degree%20%3F;longitude:decimal_degree%20%3F;has_raw_value:string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BGeolocationValue%7Clatitude:decimal_degree%20%3F;longitude:decimal_degree%20%3F;has_raw_value:string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)

## Parents

-   is_a: [AttributeValue](AttributeValue.md) - The value for any value
    of a attribute for a sample. This object can hold both the
    un-normalized atomic value and the structured value

## Referenced by Class {#referenced-by-class-14}

## Attributes {#attributes-16}

### Own {#own-13}

-   [latitude](latitude.md) 0..1
    -   Description: latitude
    -   Range: [DecimalDegree](types/DecimalDegree.md)
-   [longitude](longitude.md) 0..1
    -   Description: longitude
    -   Range: [DecimalDegree](types/DecimalDegree.md)
-   [geolocation value➞has raw
    value](geolocation_value_has_raw_value.md) 0..1
    -   Description: The raw value for a geolocation should follow {lat}
        {long}
    -   Range: [String](types/String.md)

### Inherited from attribute value:

-   [was generated by](was_generated_by.md) 0..1
    -   Range: [Activity](Activity.md)
-   [attribute value➞type](attribute_value_type.md) 0..1
    -   Description: An optional string that specified the type of
        object.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None

## Other properties {#other-properties-17}

  **Mappings:**    schema:GeoCoordinates
  ---------------- -------------------------
                   
  Slot: INSDC SR   A ENA study identifiers

\['identifiers for corresponding project in INSDC SRA / ENA'\]

URI:
<https://microbiomedata/schema/workflow_execution_activity/INSDC_SRA_ENA_study_identifiers>

## Domain and Range

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents

-   is_a: [study identifiers](study_identifiers.md)

## Children

## Used by

## Other properties {#other-properties-18}

  ----------------------- ----------------------- ------------------------
  **Aliases:**                                    EBI ENA study
                                                  identifiers

                                                  NCBI SRA identifiers

                                                  DDBJ SRA identifiers

  **Examples:**                                   Example(value='htt
                                                  ps://identifiers.org/
                                                  insdc.sra:SRP121659',
                                                  description='Avena fatua
                                                  rhizosphere microbial
                                                  communities -
                                                  H1_Rhizo_Litter_2
                                                  metatranscriptome')

  **See also:**                                   <https://git>
                                                  hub.com/bioregistry/b
                                                  ioregistry/issues/109

                                                  <https://trace.ncbi.n>
                                                  lm.nih.gov/Traces/sra
                                                  /sra.cgi?view=studies

                                                  <https://trace.ncbi.n>
                                                  lm.nih.gov/Traces/sra
                                                  /sra.cgi?view=studies
  ----------------------- ----------------------- ------------------------

# Slot: INSDC analysis identifiers

URI:
<https://microbiomedata/schema/workflow_execution_activity/INSDC_analysis_identifiers>

## Domain and Range

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents

-   is_a: [analysis identifiers](analysis_identifiers.md)

## Children

## Used by

## Other properties {#other-properties-19}

  ----------------------- ----------------------- -------------------------
  **Comments:**                                   in INSDC this is a run
                                                  but it corresponds to a
                                                  GOLD analysis

  **Examples:**                                   Exa
                                                  mple(value='<https://w>
                                                  ww.ebi.ac.uk/metageno
                                                  mics/runs/DRR218479',
                                                  description='Illumina
                                                  MiSeq paired end
                                                  sequencing of
                                                  SAMD00212331')

                                                  Ex
                                                  ample(value='<https://>
                                                  www.ebi.ac.uk/ena/bro
                                                  wser/view/ERR436051',
                                                  description=None)
  ----------------------- ----------------------- -------------------------

# Slot: INSDC assembly identifiers

URI:
<https://microbiomedata/schema/workflow_execution_activity/INSDC_assembly_identifiers>

## Domain and Range

None → 0..1 [String](types/String.md)

## Parents

-   is_a: [assembly identifiers](assembly_identifiers.md)

## Children

## Used by

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: INSDC bioproject identifiers

\['identifiers for corresponding project in INSDC Bioproject'\]

URI:
<https://microbiomedata/schema/workflow_execution_activity/INSDC_bioproject_identifiers>

## Domain and Range

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents

-   is_a: [study identifiers](study_identifiers.md)

## Children

## Used by

## Other properties {#other-properties-20}

  ----------------------- ----------------------- -----------------------
  **Aliases:**                                    NCBI bioproject
                                                  identifiers

                                                  DDBJ bioproject
                                                  identifiers

  **Comments:**                                   these are distinct IDs
                                                  from INSDC SRA/ENA
                                                  project identifiers,
                                                  but are usually(?) one
                                                  to one

  **Examples:**                                   Example(value='https:
                                                  //identifiers.org/bio
                                                  project:PRJNA366857',
                                                  description='Avena
                                                  fatua rhizosphere
                                                  microbial communities -
                                                  H1_Rhizo_Litter_2
                                                  metatranscriptome')

  **See also:**                                   <https://www.ncbi.nl>
                                                  m.nih.gov/bioproject/

                                                  <https://>
                                                  www.ddbj.nig.ac.jp/bi
                                                  oproject/index-e.html
  ----------------------- ----------------------- -----------------------

# Slot: INSDC biosample identifiers

\['identifiers for corresponding sample in INSDC'\]

URI:
<https://microbiomedata/schema/workflow_execution_activity/INSDC_biosample_identifiers>

## Domain and Range

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents

-   is_a: [sample identifiers](sample_identifiers.md)

## Children

## Used by

## Other properties {#other-properties-21}

  ----------------------- ----------------------- ------------------------
  **Aliases:**                                    EBI biosample
                                                  identifiers

                                                  NCBI biosample
                                                  identifiers

                                                  DDBJ biosample
                                                  identifiers

  **Examples:**                                   Example(value='https:
                                                  //identifiers.org/bio
                                                  sample:SAMEA5989477',
                                                  description=None)

                                                  Example(value='https:
                                                  //identifiers.org/bio
                                                  sample:SAMD00212331', de
                                                  scription='I13_N\_5-10
                                                  sample from Soil fungal
                                                  diversity along
                                                  elevational gradients')

  **See also:**                                   <https://git>
                                                  hub.com/bioregistry/b
                                                  ioregistry/issues/108

                                                  <https://www>.
                                                  ebi.ac.uk/biosamples/

                                                  <https://www.ncbi>.
                                                  nlm.nih.gov/biosample

                                                  <https:/>
                                                  /www.ddbj.nig.ac.jp/b
                                                  iosample/index-e.html
  ----------------------- ----------------------- ------------------------

# Slot: INSDC experiment identifiers

URI:
<https://microbiomedata/schema/workflow_execution_activity/INSDC_experiment_identifiers>

## Domain and Range

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents

-   is_a: [external database
    identifiers](external_database_identifiers.md)

## Children

## Used by

# Slot: INSDC identifiers

Any identifier covered by the International Nucleotide Sequence Database
Collaboration

URI:
<https://microbiomedata/schema/workflow_execution_activity/INSDC_identifiers>

## Domain and Range

None → 0..1 [String](types/String.md)

## Parents

## Children

## Used by

## Other properties {#other-properties-22}

  ----------------------- ----------------------- --------------------------------------------
  **Aliases:**                                    EBI identifiers

                                                  NCBI identifiers

                                                  DDBJ identifiers

  **Comments:**                                   note that we deliberately abstract over
                                                  which of the partner databases accepted the
                                                  initial submission

                                                  {'the first letter of the accession
                                                  indicates which partner accepted the initial
                                                  submission': 'E for ENA, D for DDBJ, or S or
                                                  N for NCBI.'}

  **See also:**                                   h ttps://www.insdc.org/

                                                  <https://ena-doc> s.readthedocs.io/en/l
                                                  atest/submit/general-guide/accessions.html
  ----------------------- ----------------------- --------------------------------------------

# Slot: INSDC secondary sample identifiers

\['secondary identifiers for corresponding sample in INSDC'\]

URI:
<https://microbiomedata/schema/workflow_execution_activity/INSDC_secondary_sample_identifiers>

## Domain and Range

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents

-   is_a: [sample identifiers](sample_identifiers.md)

## Children

## Used by

## Other properties {#other-properties-23}

  ----------------------- ----------------------- ------------------------
  **Comments:**                                   ENA redirects these to
                                                  primary IDs, e.g. https:
                                                  //www.ebi.ac.uk/ena/b
                                                  rowser/view/DRS166340
                                                  -\> SAMD00212331

                                                  MGnify uses these as
                                                  their primary sample IDs

  **Examples:**                                   Example(value='htt
                                                  ps://identifiers.org/
                                                  insdc.sra:DRS166340', de
                                                  scription='I13_N\_5-10
                                                  sample from Soil fungal
                                                  diversity along
                                                  elevational gradients')
  ----------------------- ----------------------- ------------------------

# Class: image value

An attribute value representing an image.

URI:
<https://microbiomedata/schema/workflow_execution_activity/ImageValue>

[![image17](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BImageValue%7Curl:string%20%3F;description:string%20%3F;display_order:string%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BImageValue%7Curl:string%20%3F;description:string%20%3F;display_order:string%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)

## Parents

-   is_a: [AttributeValue](AttributeValue.md) - The value for any value
    of a attribute for a sample. This object can hold both the
    un-normalized atomic value and the structured value

## Attributes {#attributes-17}

### Own {#own-14}

-   [url](url.md) 0..1
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [display order](display_order.md) 0..1
    -   Description: When rendering information, this attribute to
        specify the order in which the information should be rendered.
    -   Range: [String](types/String.md)

### Inherited from attribute value:

-   [has raw value](has_raw_value.md) 0..1
    -   Description: The value that was specified for an annotation in
        raw form, i.e. a string. E.g. "2 cm" or "2-4 cm"
    -   Range: [String](types/String.md)
-   [was generated by](was_generated_by.md) 0..1
    -   Range: [Activity](Activity.md)
-   [attribute value➞type](attribute_value_type.md) 0..1
    -   Description: An optional string that specified the type of
        object.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None

# Class: instrument

A material entity that is designed to perform a function in a scientific
investigation, but is not a reagent\[OBI\].

URI:
<https://microbiomedata/schema/workflow_execution_activity/Instrument>

[![image18](https://yuml.me/diagram/nofunky;dir:TB/class/%5BNomAnalysisActivity%5D,%5BNamedThing%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BMetabolomicsAnalysisActivity%5D,%5BMetabolomicsAnalysisActivity%5D-%20used%200..1%3E%5BInstrument%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BMetaproteomicsAnalysisActivity%5D-%20used%200..1%3E%5BInstrument%5D,%5BNomAnalysisActivity%5D-%20used%200..1%3E%5BInstrument%5D,%5BNamedThing%5D%5E-%5BInstrument%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BNomAnalysisActivity%5D,%5BNamedThing%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BMetabolomicsAnalysisActivity%5D,%5BMetabolomicsAnalysisActivity%5D-%20used%200..1%3E%5BInstrument%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BMetaproteomicsAnalysisActivity%5D-%20used%200..1%3E%5BInstrument%5D,%5BNomAnalysisActivity%5D-%20used%200..1%3E%5BInstrument%5D,%5BNamedThing%5D%5E-%5BInstrument%5D)

## Parents

-   is_a: [NamedThing](NamedThing.md) - a databased entity or
    concept/class

## Referenced by Class {#referenced-by-class-15}

-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
    [metabolomics analysis
    activity➞used](metabolomics_analysis_activity_used.md) 0..1
    [Instrument](Instrument.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
    [metaproteomics analysis
    activity➞used](metaproteomics_analysis_activity_used.md) 0..1
    [Instrument](Instrument.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md) [nom analysis
    activity➞used](nom_analysis_activity_used.md) 0..1
    [Instrument](Instrument.md)

## Attributes {#attributes-18}

### Inherited from named thing:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

## Other properties {#other-properties-24}

+---------------------+---------------+
| **Aliases:**        | > device      |
+---------------------+---------------+
| **Exact Mappings:** | > OBI:0000485 |
+---------------------+---------------+

# Class: integer value

A value that is an integer

URI:
<https://microbiomedata/schema/workflow_execution_activity/IntegerValue>

[![image19](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BIntegerValue%7Chas_numeric_value:float%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BIntegerValue%7Chas_numeric_value:float%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)

## Parents

-   is_a: [AttributeValue](AttributeValue.md) - The value for any value
    of a attribute for a sample. This object can hold both the
    un-normalized atomic value and the structured value

## Attributes {#attributes-19}

### Own {#own-15}

-   [has numeric value](has_numeric_value.md) 0..1
    -   Description: Links a quantity value to a number
    -   Range: [Float](types/Float.md)

### Inherited from attribute value:

-   [has raw value](has_raw_value.md) 0..1
    -   Description: The value that was specified for an annotation in
        raw form, i.e. a string. E.g. "2 cm" or "2-4 cm"
    -   Range: [String](types/String.md)
-   [was generated by](was_generated_by.md) 0..1
    -   Range: [Activity](Activity.md)
-   [attribute value➞type](attribute_value_type.md) 0..1
    -   Description: An optional string that specified the type of
        object.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None

# Subset: investigation

URI:
<https://microbiomedata/schema/workflow_execution_activity/investigation>

## Classes

## Mixins

## Slots

## Types

## Enums

# Class: MAG bin

URI: <https://microbiomedata/schema/workflow_execution_activity/MAGBin>

[![image20](https://yuml.me/diagram/nofunky;dir:TB/class/%5BMAGsAnalysisActivity%5D++-%20mags%20list%200..*%3E%5BMAGBin%7Ctype:string%20%3F;bin_name:string%20%3F;number_of_contig:integer%20%3F;completeness:float%20%3F;contamination:float%20%3F;gene_count:integer%20%3F;bin_quality:string%20%3F;num_16s:integer%20%3F;num_5s:integer%20%3F;num_23s:integer%20%3F;num_tRNA:integer%20%3F;gtdbtk_domain:string%20%3F;gtdbtk_phylum:string%20%3F;gtdbtk_class:string%20%3F;gtdbtk_order:string%20%3F;gtdbtk_family:string%20%3F;gtdbtk_genus:string%20%3F;gtdbtk_species:string%20%3F%5D,%5BMAGsAnalysisActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BMAGsAnalysisActivity%5D++-%20mags%20list%200..*%3E%5BMAGBin%7Ctype:string%20%3F;bin_name:string%20%3F;number_of_contig:integer%20%3F;completeness:float%20%3F;contamination:float%20%3F;gene_count:integer%20%3F;bin_quality:string%20%3F;num_16s:integer%20%3F;num_5s:integer%20%3F;num_23s:integer%20%3F;num_tRNA:integer%20%3F;gtdbtk_domain:string%20%3F;gtdbtk_phylum:string%20%3F;gtdbtk_class:string%20%3F;gtdbtk_order:string%20%3F;gtdbtk_family:string%20%3F;gtdbtk_genus:string%20%3F;gtdbtk_species:string%20%3F%5D,%5BMAGsAnalysisActivity%5D)

## Referenced by Class {#referenced-by-class-16}

-   **None** [mags list](mags_list.md) 0..\* [MAGBin](MAGBin.md)

## Attributes {#attributes-20}

### Own {#own-16}

-   [type](type.md) 0..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [➞bin name](mAGBin__bin_name.md) 0..1
    -   Range: [String](types/String.md)
-   [➞number of contig](mAGBin__number_of_contig.md) 0..1
    -   Range: [Integer](types/Integer.md)
-   [➞completeness](mAGBin__completeness.md) 0..1
    -   Range: [Float](types/Float.md)
-   [➞contamination](mAGBin__contamination.md) 0..1
    -   Range: [Float](types/Float.md)
-   [➞gene count](mAGBin__gene_count.md) 0..1
    -   Range: [Integer](types/Integer.md)
-   [➞bin quality](mAGBin__bin_quality.md) 0..1
    -   Range: [String](types/String.md)
-   [➞num 16s](mAGBin__num_16s.md) 0..1
    -   Range: [Integer](types/Integer.md)
-   [➞num 5s](mAGBin__num_5s.md) 0..1
    -   Range: [Integer](types/Integer.md)
-   [➞num 23s](mAGBin__num_23s.md) 0..1
    -   Range: [Integer](types/Integer.md)
-   [➞num tRNA](mAGBin__num_tRNA.md) 0..1
    -   Range: [Integer](types/Integer.md)
-   [➞gtdbtk domain](mAGBin__gtdbtk_domain.md) 0..1
    -   Range: [String](types/String.md)
-   [➞gtdbtk phylum](mAGBin__gtdbtk_phylum.md) 0..1
    -   Range: [String](types/String.md)
-   [➞gtdbtk class](mAGBin__gtdbtk_class.md) 0..1
    -   Range: [String](types/String.md)
-   [➞gtdbtk order](mAGBin__gtdbtk_order.md) 0..1
    -   Range: [String](types/String.md)
-   [➞gtdbtk family](mAGBin__gtdbtk_family.md) 0..1
    -   Range: [String](types/String.md)
-   [➞gtdbtk genus](mAGBin__gtdbtk_genus.md) 0..1
    -   Range: [String](types/String.md)
-   [➞gtdbtk species](mAGBin__gtdbtk_species.md) 0..1
    -   Range: [String](types/String.md)

# Class: MAGs analysis activity

URI:
<https://microbiomedata/schema/workflow_execution_activity/MAGsAnalysisActivity>

[![image21](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BActivity%5D,%5BMAGBin%5D%3Cmags%20list%200..*-++%5BMAGsAnalysisActivity%7Cinput_contig_num:integer%20%3F;binned_contig_num:integer%20%3F;too_short_contig_num:integer%20%3F;lowDepth_contig_num:integer%20%3F;unbinned_contig_num:integer%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMAGsAnalysisActivity%5D,%5BMAGBin%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BActivity%5D,%5BMAGBin%5D%3Cmags%20list%200..*-++%5BMAGsAnalysisActivity%7Cinput_contig_num:integer%20%3F;binned_contig_num:integer%20%3F;too_short_contig_num:integer%20%3F;lowDepth_contig_num:integer%20%3F;unbinned_contig_num:integer%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMAGsAnalysisActivity%5D,%5BMAGBin%5D)

## Parents

-   is_a: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
    -Represents an instance of an execution of a particular workflow

## Attributes {#attributes-21}

### Own {#own-17}

-   [input contig num](input_contig_num.md) 0..1
    -   Range: [Integer](types/Integer.md)
-   [binned contig num](binned_contig_num.md) 0..1
    -   Range: [Integer](types/Integer.md)
-   [too short contig num](too_short_contig_num.md) 0..1
    -   Range: [Integer](types/Integer.md)
-   [lowDepth contig num](lowDepth_contig_num.md) 0..1
    -   Range: [Integer](types/Integer.md)
-   [unbinned contig num](unbinned_contig_num.md) 0..1
    -   Range: [Integer](types/Integer.md)
-   [mags list](mags_list.md) 0..\*
    -   Range: [MAGBin](MAGBin.md)

### Inherited from workflow execution activity:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [used](used.md) 0..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞execution
    resource](workflow_execution_activity_execution_resource.md) 1..1
    -   Description: Example: NERSC-Cori
    -   Range: [String](types/String.md)
-   [workflow execution activity➞git
    url](workflow_execution_activity_git_url.md) 1..1
    -   Description: Example:
        <https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>
    -   Range: [String](types/String.md)
-   [workflow execution activity➞has
    input](workflow_execution_activity_has_input.md) 1..\*
    -   Description: An input to a process.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution activity➞has
    output](workflow_execution_activity_has_output.md) 1..\*
    -   Description: An output biosample to a processing step
    -   Range: [NamedThing](NamedThing.md)
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution
    activity➞type](workflow_execution_activity_type.md) 1..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [workflow execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md) 0..1
    -   Description: the agent/entity associated with the generation of
        the file
    -   Range: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
-   [workflow execution activity➞started at
    time](workflow_execution_activity_started_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞ended at
    time](workflow_execution_activity_ended_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md) 1..1
    -   Range: [Activity](Activity.md)

## Other properties {#other-properties-25}

  **In Subsets:**    workflow subset
  ------------------ -------------------
                     
  Slot: MGnify ana   lysis identifiers

URI:
<https://microbiomedata/schema/workflow_execution_activity/MGnify_analysis_identifiers>

## Domain and Range

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents

-   is_a: [analysis identifiers](analysis_identifiers.md)

## Children

## Used by

## Other properties {#other-properties-26}

  ----------------------- ----------------------- -------------------------
  **Examples:**                                   Example(va
                                                  lue='<https://www.ebi>.
                                                  ac.uk/metagenomics/an
                                                  alyses/MGYA00002270',
                                                  description='combined
                                                  analyses (taxonomic,
                                                  functional) of sample
                                                  ERS438107')

  ----------------------- ----------------------- -------------------------

# Slot: MGnify identifiers

URI:
<https://microbiomedata/schema/workflow_execution_activity/MGnify_identifiers>

## Domain and Range

None → 0..1 [String](types/String.md)

## Parents

## Children

## Used by

## Other properties {#other-properties-27}

  **See also:**    <https://www.ebi.ac.uk/metagenomics/>
  ---------------- ---------------------------------------
                   
  Slot: MGnify p   roject identifiers

\['identifiers for corresponding project in MGnify'\]

URI:
<https://microbiomedata/schema/workflow_execution_activity/MGnify_project_identifiers>

## Domain and Range

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents

-   is_a: [study identifiers](study_identifiers.md)

## Children

## Used by

## Other properties {#other-properties-28}

  ----------------------- ----------------------- -------------------------
  **Examples:**                                   Ex
                                                  ample(value='<https://>
                                                  identifiers.org/mgnif
                                                  y.proj:MGYS00005757',
                                                  description=None)

  ----------------------- ----------------------- -------------------------

# Class: metabolite quantification

This is used to link a metabolomics analysis workflow to a specific
metabolite

URI:
<https://microbiomedata/schema/workflow_execution_activity/MetaboliteQuantification>

[![image22](https://yuml.me/diagram/nofunky;dir:TB/class/%5BMetabolomicsAnalysisActivity%5D,%5BChemicalEntity%5D%3Cmetabolite%20quantified%200..1-%20%5BMetaboliteQuantification%7Calternative_identifiers:string%20*;highest_similarity_score:float%20%3F%5D,%5BMetabolomicsAnalysisActivity%5D++-%20has%20metabolite%20quantifications%200..*%3E%5BMetaboliteQuantification%5D,%5BChemicalEntity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BMetabolomicsAnalysisActivity%5D,%5BChemicalEntity%5D%3Cmetabolite%20quantified%200..1-%20%5BMetaboliteQuantification%7Calternative_identifiers:string%20*;highest_similarity_score:float%20%3F%5D,%5BMetabolomicsAnalysisActivity%5D++-%20has%20metabolite%20quantifications%200..*%3E%5BMetaboliteQuantification%5D,%5BChemicalEntity%5D)

## Referenced by Class {#referenced-by-class-17}

-   **None** [has metabolite
    quantifications](has_metabolite_quantifications.md) 0..\*
    [MetaboliteQuantification](MetaboliteQuantification.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
    [metabolomics analysis activity➞has metabolite
    quantifications](metabolomics_analysis_activity_has_metabolite_quantifications.md)
    0..\* [MetaboliteQuantification](MetaboliteQuantification.md)

## Attributes {#attributes-22}

### Own {#own-18}

-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)
-   [metabolite quantification➞metabolite
    quantified](metabolite_quantification_metabolite_quantified.md) 0..1
    -   Description: the specific metabolite identifier
    -   Range: [ChemicalEntity](ChemicalEntity.md)
-   [metabolite quantification➞highest similarity
    score](metabolite_quantification_highest_similarity_score.md) 0..1
    -   Description: TODO: Yuri to fill in
    -   Range: [Float](types/Float.md)

# Class: metabolomics analysis activity

URI:
<https://microbiomedata/schema/workflow_execution_activity/MetabolomicsAnalysisActivity>

[![image23](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BMetaboliteQuantification%5D%3Chas%20metabolite%20quantifications%200..*-++%5BMetabolomicsAnalysisActivity%7Chas_calibration:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F%5D,%5BInstrument%5D%3Cused%200..1-%20%5BMetabolomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetabolomicsAnalysisActivity%5D,%5BMetaboliteQuantification%5D,%5BInstrument%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BMetaboliteQuantification%5D%3Chas%20metabolite%20quantifications%200..*-++%5BMetabolomicsAnalysisActivity%7Chas_calibration:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F%5D,%5BInstrument%5D%3Cused%200..1-%20%5BMetabolomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetabolomicsAnalysisActivity%5D,%5BMetaboliteQuantification%5D,%5BInstrument%5D,%5BActivity%5D)

## Parents

-   is_a: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
    -Represents an instance of an execution of a particular workflow

## Referenced by Class {#referenced-by-class-18}

## Attributes {#attributes-23}

### Own {#own-19}

-   [metabolomics analysis
    activity➞used](metabolomics_analysis_activity_used.md) 0..1
    -   Description: The instrument used to collect the data used in the
        analysis
    -   Range: [Instrument](Instrument.md)
-   [metabolomics analysis activity➞has metabolite
    quantifications](metabolomics_analysis_activity_has_metabolite_quantifications.md)
    0..\*
    -   Range: [MetaboliteQuantification](MetaboliteQuantification.md)
-   [metabolomics analysis activity➞has
    calibration](metabolomics_analysis_activity_has_calibration.md) 0..1
    -   Description: TODO: Yuri to fill in
    -   Range: [String](types/String.md)

### Inherited from workflow execution activity:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [workflow execution activity➞execution
    resource](workflow_execution_activity_execution_resource.md) 1..1
    -   Description: Example: NERSC-Cori
    -   Range: [String](types/String.md)
-   [workflow execution activity➞git
    url](workflow_execution_activity_git_url.md) 1..1
    -   Description: Example:
        <https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>
    -   Range: [String](types/String.md)
-   [workflow execution activity➞has
    input](workflow_execution_activity_has_input.md) 1..\*
    -   Description: An input to a process.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution activity➞has
    output](workflow_execution_activity_has_output.md) 1..\*
    -   Description: An output biosample to a processing step
    -   Range: [NamedThing](NamedThing.md)
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution
    activity➞type](workflow_execution_activity_type.md) 1..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [workflow execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md) 0..1
    -   Description: the agent/entity associated with the generation of
        the file
    -   Range: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
-   [workflow execution activity➞started at
    time](workflow_execution_activity_started_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞ended at
    time](workflow_execution_activity_ended_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md) 1..1
    -   Range: [Activity](Activity.md)

## Other properties {#other-properties-29}

  **In Subsets:**    workflow subset
  ------------------ -----------------------
                     
  Class: metagenom   e annotation activity

URI:
<https://microbiomedata/schema/workflow_execution_activity/MetagenomeAnnotationActivity>

[![image24](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAnnotationActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAnnotationActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D)

## Parents

-   is_a: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
    -Represents an instance of an execution of a particular workflow

## Attributes {#attributes-24}

### Inherited from workflow execution activity:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [used](used.md) 0..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞execution
    resource](workflow_execution_activity_execution_resource.md) 1..1
    -   Description: Example: NERSC-Cori
    -   Range: [String](types/String.md)
-   [workflow execution activity➞git
    url](workflow_execution_activity_git_url.md) 1..1
    -   Description: Example:
        <https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>
    -   Range: [String](types/String.md)
-   [workflow execution activity➞has
    input](workflow_execution_activity_has_input.md) 1..\*
    -   Description: An input to a process.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution activity➞has
    output](workflow_execution_activity_has_output.md) 1..\*
    -   Description: An output biosample to a processing step
    -   Range: [NamedThing](NamedThing.md)
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution
    activity➞type](workflow_execution_activity_type.md) 1..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [workflow execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md) 0..1
    -   Description: the agent/entity associated with the generation of
        the file
    -   Range: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
-   [workflow execution activity➞started at
    time](workflow_execution_activity_started_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞ended at
    time](workflow_execution_activity_ended_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md) 1..1
    -   Range: [Activity](Activity.md)

## Other properties {#other-properties-30}

  **In Subsets:**    workflow subset
  ------------------ -----------------
                     
  Class: metagenom   e assembly

URI:
<https://microbiomedata/schema/workflow_execution_activity/MetagenomeAssembly>

[![image25](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAssembly%7Casm_score:float%20%3F;scaffolds:float%20%3F;scaf_logsum:float%20%3F;scaf_powsum:float%20%3F;scaf_max:float%20%3F;scaf_bp:float%20%3F;scaf_N50:float%20%3F;scaf_N90:float%20%3F;scaf_L50:float%20%3F;scaf_L90:float%20%3F;scaf_n_gt50K:float%20%3F;scaf_l_gt50K:float%20%3F;scaf_pct_gt50K:float%20%3F;contigs:float%20%3F;contig_bp:float%20%3F;ctg_N50:float%20%3F;ctg_L50:float%20%3F;ctg_N90:float%20%3F;ctg_L90:float%20%3F;ctg_logsum:float%20%3F;ctg_powsum:float%20%3F;ctg_max:float%20%3F;gap_pct:float%20%3F;gc_std:float%20%3F;gc_avg:float%20%3F;num_input_reads:float%20%3F;num_aligned_reads:float%20%3F;INSDC_assembly_identifiers:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAssembly%7Casm_score:float%20%3F;scaffolds:float%20%3F;scaf_logsum:float%20%3F;scaf_powsum:float%20%3F;scaf_max:float%20%3F;scaf_bp:float%20%3F;scaf_N50:float%20%3F;scaf_N90:float%20%3F;scaf_L50:float%20%3F;scaf_L90:float%20%3F;scaf_n_gt50K:float%20%3F;scaf_l_gt50K:float%20%3F;scaf_pct_gt50K:float%20%3F;contigs:float%20%3F;contig_bp:float%20%3F;ctg_N50:float%20%3F;ctg_L50:float%20%3F;ctg_N90:float%20%3F;ctg_L90:float%20%3F;ctg_logsum:float%20%3F;ctg_powsum:float%20%3F;ctg_max:float%20%3F;gap_pct:float%20%3F;gc_std:float%20%3F;gc_avg:float%20%3F;num_input_reads:float%20%3F;num_aligned_reads:float%20%3F;INSDC_assembly_identifiers:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D)

## Parents

-   is_a: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
    -Represents an instance of an execution of a particular workflow

## Attributes {#attributes-25}

### Own {#own-20}

-   [asm_score](asm_score.md) 0..1
    -   Description: A score for comparing metagenomic assembly quality
        from same sample.
    -   Range: [Float](types/Float.md)
-   [scaffolds](scaffolds.md) 0..1
    -   Description: Total sequence count of all scaffolds.
    -   Range: [Float](types/Float.md)
-   [scaf_logsum](scaf_logsum.md) 0..1
    -   Description: The sum of the (length\*log(length)) of all
        scaffolds, times some constant. Increase the contiguity, the
        score will increase
    -   Range: [Float](types/Float.md)
-   [scaf_powsum](scaf_powsum.md) 0..1
    -   Description: Powersum of all scaffolds is the same as logsum
        except that it uses the sum of (length\*(length\^P)) for some
        power P (default P=0.25).
    -   Range: [Float](types/Float.md)
-   [scaf_max](scaf_max.md) 0..1
    -   Description: Maximum scaffold length.
    -   Range: [Float](types/Float.md)
-   [scaf_bp](scaf_bp.md) 0..1
    -   Description: Total size in bp of all scaffolds.
    -   Range: [Float](types/Float.md)
-   [scaf_N50](scaf_N50.md) 0..1
    -   Description: Given a set of scaffolds, each with its own length,
        the N50 count is defined as the smallest number of scaffolds
        whose length sum makes up half of genome size.
    -   Range: [Float](types/Float.md)
-   [scaf_N90](scaf_N90.md) 0..1
    -   Description: Given a set of scaffolds, each with its own length,
        the N90 count is defined as the smallest number of scaffolds
        whose length sum makes up 90% of genome size.
    -   Range: [Float](types/Float.md)
-   [scaf_L50](scaf_L50.md) 0..1
    -   Description: Given a set of scaffolds, the L50 is defined as the
        sequence length of the shortest scaffold at 50% of the total
        genome length.
    -   Range: [Float](types/Float.md)
-   [scaf_L90](scaf_L90.md) 0..1
    -   Description: The L90 statistic is less than or equal to the L50
        statistic; it is the length for which the collection of all
        scaffolds of that length or longer contains at least 90% of the
        sum of the lengths of all scaffolds.
    -   Range: [Float](types/Float.md)
-   [scaf_n\_gt50K](scaf_n_gt50K.md) 0..1
    -   Description: Total sequence count of scaffolds greater than 50
        KB.
    -   Range: [Float](types/Float.md)
-   [scaf_l\_gt50K](scaf_l_gt50K.md) 0..1
    -   Description: Total size in bp of all scaffolds greater than 50
        KB.
    -   Range: [Float](types/Float.md)
-   [scaf_pct_gt50K](scaf_pct_gt50K.md) 0..1
    -   Description: Total sequence size percentage of scaffolds greater
        than 50 KB.
    -   Range: [Float](types/Float.md)
-   [contigs](contigs.md) 0..1
    -   Description: The sum of the (length\*log(length)) of all
        contigs, times some constant. Increase the contiguity, the score
        will increase
    -   Range: [Float](types/Float.md)
-   [contig_bp](contig_bp.md) 0..1
    -   Description: Total size in bp of all contigs.
    -   Range: [Float](types/Float.md)
-   [ctg_N50](ctg_N50.md) 0..1
    -   Description: Given a set of contigs, each with its own length,
        the N50 count is defined as the smallest number of contigs whose
        length sum makes up half of genome size.
    -   Range: [Float](types/Float.md)
-   [ctg_L50](ctg_L50.md) 0..1
    -   Description: Given a set of contigs, the L50 is defined as the
        sequence length of the shortest contig at 50% of the total
        genome length.
    -   Range: [Float](types/Float.md)
-   [ctg_N90](ctg_N90.md) 0..1
    -   Description: Given a set of contigs, each with its own length,
        the N90 count is defined as the smallest number of contigs whose
        length sum makes up 90% of genome size.
    -   Range: [Float](types/Float.md)
-   [ctg_L90](ctg_L90.md) 0..1
    -   Description: The L90 statistic is less than or equal to the L50
        statistic; it is the length for which the collection of all
        contigs of that length or longer contains at least 90% of the
        sum of the lengths of all contigs.
    -   Range: [Float](types/Float.md)
-   [ctg_logsum](ctg_logsum.md) 0..1
    -   Description: Maximum contig length.
    -   Range: [Float](types/Float.md)
-   [ctg_powsum](ctg_powsum.md) 0..1
    -   Description: Powersum of all contigs is the same as logsum
        except that it uses the sum of (length\*(length\^P)) for some
        power P (default P=0.25).
    -   Range: [Float](types/Float.md)
-   [ctg_max](ctg_max.md) 0..1
    -   Description: Maximum contig length.
    -   Range: [Float](types/Float.md)
-   [gap_pct](gap_pct.md) 0..1
    -   Description: The gap size percentage of all scaffolds.
    -   Range: [Float](types/Float.md)
-   [gc_std](gc_std.md) 0..1
    -   Description: Standard deviation of GC content of all contigs.
    -   Range: [Float](types/Float.md)
-   [gc_avg](gc_avg.md) 0..1
    -   Description: Average of GC content of all contigs.
    -   Range: [Float](types/Float.md)
-   [num_input_reads](num_input_reads.md) 0..1
    -   Description: The sequence count number of input reads for
        assembly.
    -   Range: [Float](types/Float.md)
-   [num_aligned_reads](num_aligned_reads.md) 0..1
    -   Description: The sequence count number of input reads aligned to
        assembled contigs.
    -   Range: [Float](types/Float.md)
-   [INSDC assembly identifiers](INSDC_assembly_identifiers.md) 0..1
    -   Range: [String](types/String.md)

### Inherited from workflow execution activity:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [used](used.md) 0..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞execution
    resource](workflow_execution_activity_execution_resource.md) 1..1
    -   Description: Example: NERSC-Cori
    -   Range: [String](types/String.md)
-   [workflow execution activity➞git
    url](workflow_execution_activity_git_url.md) 1..1
    -   Description: Example:
        <https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>
    -   Range: [String](types/String.md)
-   [workflow execution activity➞has
    input](workflow_execution_activity_has_input.md) 1..\*
    -   Description: An input to a process.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution activity➞has
    output](workflow_execution_activity_has_output.md) 1..\*
    -   Description: An output biosample to a processing step
    -   Range: [NamedThing](NamedThing.md)
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution
    activity➞type](workflow_execution_activity_type.md) 1..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [workflow execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md) 0..1
    -   Description: the agent/entity associated with the generation of
        the file
    -   Range: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
-   [workflow execution activity➞started at
    time](workflow_execution_activity_started_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞ended at
    time](workflow_execution_activity_ended_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md) 1..1
    -   Range: [Activity](Activity.md)

## Other properties {#other-properties-31}

  **In Subsets:**    workflow subset
  ------------------ -------------------------
                     
  Class: metaprote   omics analysis activity

URI:
<https://microbiomedata/schema/workflow_execution_activity/MetaproteomicsAnalysisActivity>

[![image26](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BPeptideQuantification%5D,%5BNamedThing%5D,%5BPeptideQuantification%5D%3Chas%20peptide%20quantifications%200..*-++%5BMetaproteomicsAnalysisActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F%5D,%5BInstrument%5D%3Cused%200..1-%20%5BMetaproteomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetaproteomicsAnalysisActivity%5D,%5BInstrument%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BPeptideQuantification%5D,%5BNamedThing%5D,%5BPeptideQuantification%5D%3Chas%20peptide%20quantifications%200..*-++%5BMetaproteomicsAnalysisActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F%5D,%5BInstrument%5D%3Cused%200..1-%20%5BMetaproteomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetaproteomicsAnalysisActivity%5D,%5BInstrument%5D,%5BActivity%5D)

## Parents

-   is_a: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
    -Represents an instance of an execution of a particular workflow

## Referenced by Class {#referenced-by-class-19}

## Attributes {#attributes-26}

### Own {#own-21}

-   [metaproteomics analysis
    activity➞used](metaproteomics_analysis_activity_used.md) 0..1
    -   Description: The instrument used to collect the data used in the
        analysis
    -   Range: [Instrument](Instrument.md)
-   [metaproteomics analysis activity➞has peptide
    quantifications](metaproteomics_analysis_activity_has_peptide_quantifications.md)
    0..\*
    -   Range: [PeptideQuantification](PeptideQuantification.md)

### Inherited from workflow execution activity:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [workflow execution activity➞execution
    resource](workflow_execution_activity_execution_resource.md) 1..1
    -   Description: Example: NERSC-Cori
    -   Range: [String](types/String.md)
-   [workflow execution activity➞git
    url](workflow_execution_activity_git_url.md) 1..1
    -   Description: Example:
        <https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>
    -   Range: [String](types/String.md)
-   [workflow execution activity➞has
    input](workflow_execution_activity_has_input.md) 1..\*
    -   Description: An input to a process.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution activity➞has
    output](workflow_execution_activity_has_output.md) 1..\*
    -   Description: An output biosample to a processing step
    -   Range: [NamedThing](NamedThing.md)
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution
    activity➞type](workflow_execution_activity_type.md) 1..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [workflow execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md) 0..1
    -   Description: the agent/entity associated with the generation of
        the file
    -   Range: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
-   [workflow execution activity➞started at
    time](workflow_execution_activity_started_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞ended at
    time](workflow_execution_activity_ended_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md) 1..1
    -   Range: [Activity](Activity.md)

## Other properties {#other-properties-32}

  **In Subsets:**    workflow subset
  ------------------ -------------------
                     
  Class: metatrans   criptome activity

A metatranscriptome activity that e.g. pools assembly and annotation
activity.

URI:
<https://microbiomedata/schema/workflow_execution_activity/MetatranscriptomeActivity>

[![image27](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D)

## Parents

-   is_a: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
    -Represents an instance of an execution of a particular workflow

## Attributes {#attributes-27}

### Inherited from workflow execution activity:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [used](used.md) 0..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞execution
    resource](workflow_execution_activity_execution_resource.md) 1..1
    -   Description: Example: NERSC-Cori
    -   Range: [String](types/String.md)
-   [workflow execution activity➞git
    url](workflow_execution_activity_git_url.md) 1..1
    -   Description: Example:
        <https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>
    -   Range: [String](types/String.md)
-   [workflow execution activity➞has
    input](workflow_execution_activity_has_input.md) 1..\*
    -   Description: An input to a process.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution activity➞has
    output](workflow_execution_activity_has_output.md) 1..\*
    -   Description: An output biosample to a processing step
    -   Range: [NamedThing](NamedThing.md)
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution
    activity➞type](workflow_execution_activity_type.md) 1..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [workflow execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md) 0..1
    -   Description: the agent/entity associated with the generation of
        the file
    -   Range: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
-   [workflow execution activity➞started at
    time](workflow_execution_activity_started_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞ended at
    time](workflow_execution_activity_ended_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md) 1..1
    -   Range: [Activity](Activity.md)

## Other properties {#other-properties-33}

  **In Subsets:**    workflow subset
  ------------------ ------------------------------
                     
  Class: metatrans   criptome annotation activity

URI:
<https://microbiomedata/schema/workflow_execution_activity/MetatranscriptomeAnnotationActivity>

[![image28](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAnnotationActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAnnotationActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D)

## Parents

-   is_a: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
    -Represents an instance of an execution of a particular workflow

## Attributes {#attributes-28}

### Inherited from workflow execution activity:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [used](used.md) 0..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞execution
    resource](workflow_execution_activity_execution_resource.md) 1..1
    -   Description: Example: NERSC-Cori
    -   Range: [String](types/String.md)
-   [workflow execution activity➞git
    url](workflow_execution_activity_git_url.md) 1..1
    -   Description: Example:
        <https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>
    -   Range: [String](types/String.md)
-   [workflow execution activity➞has
    input](workflow_execution_activity_has_input.md) 1..\*
    -   Description: An input to a process.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution activity➞has
    output](workflow_execution_activity_has_output.md) 1..\*
    -   Description: An output biosample to a processing step
    -   Range: [NamedThing](NamedThing.md)
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution
    activity➞type](workflow_execution_activity_type.md) 1..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [workflow execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md) 0..1
    -   Description: the agent/entity associated with the generation of
        the file
    -   Range: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
-   [workflow execution activity➞started at
    time](workflow_execution_activity_started_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞ended at
    time](workflow_execution_activity_ended_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md) 1..1
    -   Range: [Activity](Activity.md)

## Other properties {#other-properties-34}

  **In Subsets:**    workflow subset
  ------------------ -------------------
                     
  Class: metatrans   criptome assembly

URI:
<https://microbiomedata/schema/workflow_execution_activity/MetatranscriptomeAssembly>

[![image29](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAssembly%7Casm_score:float%20%3F;scaffolds:float%20%3F;scaf_logsum:float%20%3F;scaf_powsum:float%20%3F;scaf_max:float%20%3F;scaf_bp:float%20%3F;scaf_N50:float%20%3F;scaf_N90:float%20%3F;scaf_L50:float%20%3F;scaf_L90:float%20%3F;scaf_n_gt50K:float%20%3F;scaf_l_gt50K:float%20%3F;scaf_pct_gt50K:float%20%3F;contigs:float%20%3F;contig_bp:float%20%3F;ctg_N50:float%20%3F;ctg_L50:float%20%3F;ctg_N90:float%20%3F;ctg_L90:float%20%3F;ctg_logsum:float%20%3F;ctg_powsum:float%20%3F;ctg_max:float%20%3F;gap_pct:float%20%3F;gc_std:float%20%3F;gc_avg:float%20%3F;num_input_reads:float%20%3F;num_aligned_reads:float%20%3F;INSDC_assembly_identifiers:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAssembly%7Casm_score:float%20%3F;scaffolds:float%20%3F;scaf_logsum:float%20%3F;scaf_powsum:float%20%3F;scaf_max:float%20%3F;scaf_bp:float%20%3F;scaf_N50:float%20%3F;scaf_N90:float%20%3F;scaf_L50:float%20%3F;scaf_L90:float%20%3F;scaf_n_gt50K:float%20%3F;scaf_l_gt50K:float%20%3F;scaf_pct_gt50K:float%20%3F;contigs:float%20%3F;contig_bp:float%20%3F;ctg_N50:float%20%3F;ctg_L50:float%20%3F;ctg_N90:float%20%3F;ctg_L90:float%20%3F;ctg_logsum:float%20%3F;ctg_powsum:float%20%3F;ctg_max:float%20%3F;gap_pct:float%20%3F;gc_std:float%20%3F;gc_avg:float%20%3F;num_input_reads:float%20%3F;num_aligned_reads:float%20%3F;INSDC_assembly_identifiers:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BActivity%5D)

## Parents

-   is_a: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
    -Represents an instance of an execution of a particular workflow

## Attributes {#attributes-29}

### Own {#own-22}

-   [asm_score](asm_score.md) 0..1
    -   Description: A score for comparing metagenomic assembly quality
        from same sample.
    -   Range: [Float](types/Float.md)
-   [scaffolds](scaffolds.md) 0..1
    -   Description: Total sequence count of all scaffolds.
    -   Range: [Float](types/Float.md)
-   [scaf_logsum](scaf_logsum.md) 0..1
    -   Description: The sum of the (length\*log(length)) of all
        scaffolds, times some constant. Increase the contiguity, the
        score will increase
    -   Range: [Float](types/Float.md)
-   [scaf_powsum](scaf_powsum.md) 0..1
    -   Description: Powersum of all scaffolds is the same as logsum
        except that it uses the sum of (length\*(length\^P)) for some
        power P (default P=0.25).
    -   Range: [Float](types/Float.md)
-   [scaf_max](scaf_max.md) 0..1
    -   Description: Maximum scaffold length.
    -   Range: [Float](types/Float.md)
-   [scaf_bp](scaf_bp.md) 0..1
    -   Description: Total size in bp of all scaffolds.
    -   Range: [Float](types/Float.md)
-   [scaf_N50](scaf_N50.md) 0..1
    -   Description: Given a set of scaffolds, each with its own length,
        the N50 count is defined as the smallest number of scaffolds
        whose length sum makes up half of genome size.
    -   Range: [Float](types/Float.md)
-   [scaf_N90](scaf_N90.md) 0..1
    -   Description: Given a set of scaffolds, each with its own length,
        the N90 count is defined as the smallest number of scaffolds
        whose length sum makes up 90% of genome size.
    -   Range: [Float](types/Float.md)
-   [scaf_L50](scaf_L50.md) 0..1
    -   Description: Given a set of scaffolds, the L50 is defined as the
        sequence length of the shortest scaffold at 50% of the total
        genome length.
    -   Range: [Float](types/Float.md)
-   [scaf_L90](scaf_L90.md) 0..1
    -   Description: The L90 statistic is less than or equal to the L50
        statistic; it is the length for which the collection of all
        scaffolds of that length or longer contains at least 90% of the
        sum of the lengths of all scaffolds.
    -   Range: [Float](types/Float.md)
-   [scaf_n\_gt50K](scaf_n_gt50K.md) 0..1
    -   Description: Total sequence count of scaffolds greater than 50
        KB.
    -   Range: [Float](types/Float.md)
-   [scaf_l\_gt50K](scaf_l_gt50K.md) 0..1
    -   Description: Total size in bp of all scaffolds greater than 50
        KB.
    -   Range: [Float](types/Float.md)
-   [scaf_pct_gt50K](scaf_pct_gt50K.md) 0..1
    -   Description: Total sequence size percentage of scaffolds greater
        than 50 KB.
    -   Range: [Float](types/Float.md)
-   [contigs](contigs.md) 0..1
    -   Description: The sum of the (length\*log(length)) of all
        contigs, times some constant. Increase the contiguity, the score
        will increase
    -   Range: [Float](types/Float.md)
-   [contig_bp](contig_bp.md) 0..1
    -   Description: Total size in bp of all contigs.
    -   Range: [Float](types/Float.md)
-   [ctg_N50](ctg_N50.md) 0..1
    -   Description: Given a set of contigs, each with its own length,
        the N50 count is defined as the smallest number of contigs whose
        length sum makes up half of genome size.
    -   Range: [Float](types/Float.md)
-   [ctg_L50](ctg_L50.md) 0..1
    -   Description: Given a set of contigs, the L50 is defined as the
        sequence length of the shortest contig at 50% of the total
        genome length.
    -   Range: [Float](types/Float.md)
-   [ctg_N90](ctg_N90.md) 0..1
    -   Description: Given a set of contigs, each with its own length,
        the N90 count is defined as the smallest number of contigs whose
        length sum makes up 90% of genome size.
    -   Range: [Float](types/Float.md)
-   [ctg_L90](ctg_L90.md) 0..1
    -   Description: The L90 statistic is less than or equal to the L50
        statistic; it is the length for which the collection of all
        contigs of that length or longer contains at least 90% of the
        sum of the lengths of all contigs.
    -   Range: [Float](types/Float.md)
-   [ctg_logsum](ctg_logsum.md) 0..1
    -   Description: Maximum contig length.
    -   Range: [Float](types/Float.md)
-   [ctg_powsum](ctg_powsum.md) 0..1
    -   Description: Powersum of all contigs is the same as logsum
        except that it uses the sum of (length\*(length\^P)) for some
        power P (default P=0.25).
    -   Range: [Float](types/Float.md)
-   [ctg_max](ctg_max.md) 0..1
    -   Description: Maximum contig length.
    -   Range: [Float](types/Float.md)
-   [gap_pct](gap_pct.md) 0..1
    -   Description: The gap size percentage of all scaffolds.
    -   Range: [Float](types/Float.md)
-   [gc_std](gc_std.md) 0..1
    -   Description: Standard deviation of GC content of all contigs.
    -   Range: [Float](types/Float.md)
-   [gc_avg](gc_avg.md) 0..1
    -   Description: Average of GC content of all contigs.
    -   Range: [Float](types/Float.md)
-   [num_input_reads](num_input_reads.md) 0..1
    -   Description: The sequence count number of input reads for
        assembly.
    -   Range: [Float](types/Float.md)
-   [num_aligned_reads](num_aligned_reads.md) 0..1
    -   Description: The sequence count number of input reads aligned to
        assembled contigs.
    -   Range: [Float](types/Float.md)
-   [INSDC assembly identifiers](INSDC_assembly_identifiers.md) 0..1
    -   Range: [String](types/String.md)

### Inherited from workflow execution activity:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [used](used.md) 0..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞execution
    resource](workflow_execution_activity_execution_resource.md) 1..1
    -   Description: Example: NERSC-Cori
    -   Range: [String](types/String.md)
-   [workflow execution activity➞git
    url](workflow_execution_activity_git_url.md) 1..1
    -   Description: Example:
        <https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>
    -   Range: [String](types/String.md)
-   [workflow execution activity➞has
    input](workflow_execution_activity_has_input.md) 1..\*
    -   Description: An input to a process.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution activity➞has
    output](workflow_execution_activity_has_output.md) 1..\*
    -   Description: An output biosample to a processing step
    -   Range: [NamedThing](NamedThing.md)
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution
    activity➞type](workflow_execution_activity_type.md) 1..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [workflow execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md) 0..1
    -   Description: the agent/entity associated with the generation of
        the file
    -   Range: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
-   [workflow execution activity➞started at
    time](workflow_execution_activity_started_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞ended at
    time](workflow_execution_activity_ended_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md) 1..1
    -   Range: [Activity](Activity.md)

## Other properties {#other-properties-35}

  **In Subsets:**    workflow subset
  ------------------ -----------------
                     
  Subset: mixs ext   ension

URI:
<https://microbiomedata/schema/workflow_execution_activity/mixs_extension>

## Classes

## Mixins

## Slots

## Types

## Enums

# Class: named thing

a databased entity or concept/class

URI:
<https://microbiomedata/schema/workflow_execution_activity/NamedThing>

[![image30](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BReadQCAnalysisActivity%5D,%5BPerson%5D,%5BOntologyClass%5D,%5BWorkflowExecutionActivity%5D-%20has%20input(i)%200..*%3E%5BNamedThing%7Cid:string;name:string%20%3F;description:string%20%3F;alternative_identifiers:string%20*%5D,%5BWorkflowExecutionActivity%5D-%20has%20output(i)%200..*%3E%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D-%20part%20of%200..*%3E%5BNamedThing%5D,%5BReadQCAnalysisActivity%5D-%20has%20input%201..*%3E%5BNamedThing%5D,%5BReadQCAnalysisActivity%5D-%20has%20output%201..*%3E%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D-%20has%20input%201..*%3E%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D-%20has%20output%201..*%3E%5BNamedThing%5D,%5BNamedThing%5D%5E-%5BPerson%5D,%5BNamedThing%5D%5E-%5BOntologyClass%5D,%5BNamedThing%5D%5E-%5BInstrument%5D,%5BNamedThing%5D%5E-%5BGeneProduct%5D,%5BInstrument%5D,%5BGeneProduct%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BReadQCAnalysisActivity%5D,%5BPerson%5D,%5BOntologyClass%5D,%5BWorkflowExecutionActivity%5D-%20has%20input(i)%200..*%3E%5BNamedThing%7Cid:string;name:string%20%3F;description:string%20%3F;alternative_identifiers:string%20*%5D,%5BWorkflowExecutionActivity%5D-%20has%20output(i)%200..*%3E%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D-%20part%20of%200..*%3E%5BNamedThing%5D,%5BReadQCAnalysisActivity%5D-%20has%20input%201..*%3E%5BNamedThing%5D,%5BReadQCAnalysisActivity%5D-%20has%20output%201..*%3E%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D-%20has%20input%201..*%3E%5BNamedThing%5D,%5BWorkflowExecutionActivity%5D-%20has%20output%201..*%3E%5BNamedThing%5D,%5BNamedThing%5D%5E-%5BPerson%5D,%5BNamedThing%5D%5E-%5BOntologyClass%5D,%5BNamedThing%5D%5E-%5BInstrument%5D,%5BNamedThing%5D%5E-%5BGeneProduct%5D,%5BInstrument%5D,%5BGeneProduct%5D)

## Children

-   [GeneProduct](GeneProduct.md) - A molecule encoded by a gene that
    has an evolved function
-   [Instrument](Instrument.md) - A material entity that is designed to
    perform a function in a scientific investigation, but is not a
    reagent\[OBI\].
-   [OntologyClass](OntologyClass.md)
-   [Person](Person.md) - represents a person, such as a researcher

## Referenced by Class {#referenced-by-class-20}

-   [NamedThing](NamedThing.md) [has input](has_input.md) 0..\*
    [NamedThing](NamedThing.md)
-   [NamedThing](NamedThing.md) [has output](has_output.md) 0..\*
    [NamedThing](NamedThing.md)
-   [NamedThing](NamedThing.md) [part of](part_of.md) 0..\*
    [NamedThing](NamedThing.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md) [read QC
    analysis activity➞has input](read_QC_analysis_activity_has_input.md)
    1..\* [NamedThing](NamedThing.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md) [read QC
    analysis activity➞has
    output](read_QC_analysis_activity_has_output.md) 1..\*
    [NamedThing](NamedThing.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md) [workflow
    execution activity➞has
    input](workflow_execution_activity_has_input.md) 1..\*
    [NamedThing](NamedThing.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md) [workflow
    execution activity➞has
    output](workflow_execution_activity_has_output.md) 1..\*
    [NamedThing](NamedThing.md)

## Attributes {#attributes-30}

### Own {#own-23}

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

# Class: nom analysis activity

URI:
<https://microbiomedata/schema/workflow_execution_activity/NomAnalysisActivity>

[![image31](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BInstrument%5D%3Cused%200..1-%20%5BNomAnalysisActivity%7Chas_calibration:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F%5D,%5BWorkflowExecutionActivity%5D%5E-%5BNomAnalysisActivity%5D,%5BNamedThing%5D,%5BInstrument%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BInstrument%5D%3Cused%200..1-%20%5BNomAnalysisActivity%7Chas_calibration:string%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F%5D,%5BWorkflowExecutionActivity%5D%5E-%5BNomAnalysisActivity%5D,%5BNamedThing%5D,%5BInstrument%5D,%5BActivity%5D)

## Parents

-   is_a: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
    -Represents an instance of an execution of a particular workflow

## Referenced by Class {#referenced-by-class-21}

## Attributes {#attributes-31}

### Own {#own-24}

-   [nom analysis activity➞used](nom_analysis_activity_used.md) 0..1
    -   Description: The instrument used to collect the data used in the
        analysis
    -   Range: [Instrument](Instrument.md)
-   [nom analysis activity➞has
    calibration](nom_analysis_activity_has_calibration.md) 0..1
    -   Description: A reference to a file that holds calibration
        information.
    -   Range: [String](types/String.md)

### Inherited from workflow execution activity:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [workflow execution activity➞execution
    resource](workflow_execution_activity_execution_resource.md) 1..1
    -   Description: Example: NERSC-Cori
    -   Range: [String](types/String.md)
-   [workflow execution activity➞git
    url](workflow_execution_activity_git_url.md) 1..1
    -   Description: Example:
        <https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>
    -   Range: [String](types/String.md)
-   [workflow execution activity➞has
    input](workflow_execution_activity_has_input.md) 1..\*
    -   Description: An input to a process.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution activity➞has
    output](workflow_execution_activity_has_output.md) 1..\*
    -   Description: An output biosample to a processing step
    -   Range: [NamedThing](NamedThing.md)
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution
    activity➞type](workflow_execution_activity_type.md) 1..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [workflow execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md) 0..1
    -   Description: the agent/entity associated with the generation of
        the file
    -   Range: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
-   [workflow execution activity➞started at
    time](workflow_execution_activity_started_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞ended at
    time](workflow_execution_activity_ended_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md) 1..1
    -   Range: [Activity](Activity.md)

## Other properties {#other-properties-36}

  **In Subsets:**   workflow subset
  ----------------- ----------------------
                    
  Subset: nucleic   acid sequence source

URI:
<https://microbiomedata/schema/workflow_execution_activity/nucleic_acid_sequence_source>

## Classes

## Mixins

## Slots

## Types

## Enums

# Class: omics processing

The methods and processes used to generate omics data from a biosample
or organism.

URI: [nmdc:OmicsProcessing](https://microbiomedata/meta/OmicsProcessing)

[![image32](https://yuml.me/diagram/nofunky;dir:TB/class/%5BTextValue%5D,%5BQuantityValue%5D,%5BPersonValue%5D,%5BTextValue%5D%3Cchimera_check%200..1-++%5BOmicsProcessing%7Cadd_date:string%20%3F;mod_date:string%20%3F;instrument_name:string%20%3F;ncbi_project_name:string%20%3F;processing_institution:string%20%3F;type:string%20%3F;GOLD_sequencing_project_identifiers:external_identifier%20*;INSDC_experiment_identifiers:external_identifier%20*;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BTextValue%5D%3Cseq_quality_check%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cseq_meth%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cpcr_cond%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cpcr_primers%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Ctarget_subfragment%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Ctarget_gene%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cnucl_acid_amp%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cnucl_acid_ext%200..1-++%5BOmicsProcessing%5D,%5BQuantityValue%5D%3Csamp_vol_we_dna_ext%200..1-++%5BOmicsProcessing%5D,%5BPersonValue%5D%3Cprincipal%20investigator%200..1-++%5BOmicsProcessing%5D,%5BNamedThing%5D%3Cpart%20of%200..*-%20%5BOmicsProcessing%5D,%5BControlledTermValue%5D%3Comics%20type%200..1-++%5BOmicsProcessing%5D,%5BNamedThing%5D%3Chas%20output%200..*-%20%5BOmicsProcessing%5D,%5BNamedThing%5D%3Chas%20input%200..*-%20%5BOmicsProcessing%5D,%5BDatabase%5D++-%20omics%20processing%20set%200..*%3E%5BOmicsProcessing%5D,%5BBiosampleProcessing%5D%5E-%5BOmicsProcessing%5D,%5BNamedThing%5D,%5BDatabase%5D,%5BControlledTermValue%5D,%5BBiosampleProcessing%5D,%5BBiosample%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BTextValue%5D,%5BQuantityValue%5D,%5BPersonValue%5D,%5BTextValue%5D%3Cchimera_check%200..1-++%5BOmicsProcessing%7Cadd_date:string%20%3F;mod_date:string%20%3F;instrument_name:string%20%3F;ncbi_project_name:string%20%3F;processing_institution:string%20%3F;type:string%20%3F;GOLD_sequencing_project_identifiers:external_identifier%20*;INSDC_experiment_identifiers:external_identifier%20*;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BTextValue%5D%3Cseq_quality_check%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cseq_meth%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cpcr_cond%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cpcr_primers%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Ctarget_subfragment%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Ctarget_gene%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cnucl_acid_amp%200..1-++%5BOmicsProcessing%5D,%5BTextValue%5D%3Cnucl_acid_ext%200..1-++%5BOmicsProcessing%5D,%5BQuantityValue%5D%3Csamp_vol_we_dna_ext%200..1-++%5BOmicsProcessing%5D,%5BPersonValue%5D%3Cprincipal%20investigator%200..1-++%5BOmicsProcessing%5D,%5BNamedThing%5D%3Cpart%20of%200..*-%20%5BOmicsProcessing%5D,%5BControlledTermValue%5D%3Comics%20type%200..1-++%5BOmicsProcessing%5D,%5BNamedThing%5D%3Chas%20output%200..*-%20%5BOmicsProcessing%5D,%5BNamedThing%5D%3Chas%20input%200..*-%20%5BOmicsProcessing%5D,%5BDatabase%5D++-%20omics%20processing%20set%200..*%3E%5BOmicsProcessing%5D,%5BBiosampleProcessing%5D%5E-%5BOmicsProcessing%5D,%5BNamedThing%5D,%5BDatabase%5D,%5BControlledTermValue%5D,%5BBiosampleProcessing%5D,%5BBiosample%5D)

## Parents

-   is_a: [BiosampleProcessing](BiosampleProcessing.md) - A process that
    takes one or more biosamples as inputs and generates one or as
    outputs. Examples of outputs include samples cultivated from another
    sample or data objects created by instruments runs.

## Referenced by Class {#referenced-by-class-22}

-   [Database](Database.md) [omics processing
    set](omics_processing_set.md) 0..\*
    [OmicsProcessing](OmicsProcessing.md)

## Attributes {#attributes-32}

### Own {#own-25}

-   [add_date](add_date.md) 0..1
    -   Description: The date on which the information was added to the
        database.
    -   Range: [String](types/String.md)
-   [mod_date](mod_date.md) 0..1
    -   Description: The last date on which the database information was
        modified.
    -   Range: [String](types/String.md)
-   [has input](has_input.md) 0..\*
    -   Description: An input to a process.
    -   Range: [NamedThing](NamedThing.md)
-   [has output](has_output.md) 0..\*
    -   Description: An output biosample to a processing step
    -   Range: [NamedThing](NamedThing.md)
-   [instrument_name](instrument_name.md) 0..1
    -   Description: The name of the instrument that was used for
        processing the sample.
    -   Range: [String](types/String.md)
-   [ncbi_project_name](ncbi_project_name.md) 0..1
    -   Range: [String](types/String.md)
-   [omics type](omics_type.md) 0..1
    -   Description: The type of omics data
    -   Range: [ControlledTermValue](ControlledTermValue.md)
    -   Example: metatranscriptome None
    -   Example: metagenome None
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [principal investigator](principal_investigator.md) 0..1
    -   Description: represents the PI
    -   Range: [PersonValue](PersonValue.md)
-   [processing_institution](processing_institution.md) 0..1
    -   Description: The organization that processed the sample.
    -   Range: [String](types/String.md)
-   [type](type.md) 0..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [GOLD sequencing project
    identifiers](GOLD_sequencing_project_identifiers.md) 0..\*
    -   Description: \['identifiers for corresponding sequencing project
        in GOLD'\]
    -   Range: [ExternalIdentifier](types/ExternalIdentifier.md)
    -   Example: <https://identifiers.org/gold:Gp0108335> None
-   [INSDC experiment identifiers](INSDC_experiment_identifiers.md)
    0..\*
    -   Range: [ExternalIdentifier](types/ExternalIdentifier.md)
-   [samp_vol_we_dna_ext](samp_vol_we_dna_ext.md) 0..1
    -   Description: "Volume (ml), weight (g) of processed sample, or
        surface area swabbed from sample for DNA extraction"
    -   Range: [QuantityValue](QuantityValue.md)
-   [nucl_acid_ext](nucl_acid_ext.md) 0..1
    -   Description: "A link to a literature reference, electronic
        resource or a standard operating procedure (SOP), that describes
        the material separation to recover the nucleic acid fraction
        from a sample"
    -   Range: [TextValue](TextValue.md)
    -   in subsets: (sequencing)
-   [nucl_acid_amp](nucl_acid_amp.md) 0..1
    -   Description: "A link to a literature reference, electronic
        resource or a standard operating procedure (SOP), that describes
        the enzymatic amplification (PCR, TMA, NASBA) of specific
        nucleic acids"
    -   Range: [TextValue](TextValue.md)
    -   in subsets: (sequencing)
-   [target_gene](target_gene.md) 0..1
    -   Description: Targeted gene or locus name for marker gene studies
    -   Range: [TextValue](TextValue.md)
    -   in subsets: (sequencing)
-   [target_subfragment](target_subfragment.md) 0..1
    -   Description: Name of subfragment of a gene or locus. Important
        to e.g. identify special regions on marker genes like V6 on 16S
        rRNA
    -   Range: [TextValue](TextValue.md)
    -   in subsets: (sequencing)
-   [pcr_primers](pcr_primers.md) 0..1
    -   Description: "PCR primers that were used to amplify the sequence
        of the targeted gene, locus or subfragment. This field should
        contain all the primers used for a single PCR reaction if
        multiple forward or reverse primers are present in a single PCR
        reaction. The primer sequence should be reported in uppercase
        letters"
    -   Range: [TextValue](TextValue.md)
    -   in subsets: (sequencing)
-   [pcr_cond](pcr_cond.md) 0..1
    -   Description: Description of reaction conditions and components
        of PCR in the form of 'initial denaturation:94degC_1.5min;
        annealing=...'
    -   Range: [TextValue](TextValue.md)
    -   in subsets: (sequencing)
-   [seq_meth](seq_meth.md) 0..1
    -   Description: "Sequencing method used; e.g. Sanger,
        pyrosequencing, ABI-solid"
    -   Range: [TextValue](TextValue.md)
    -   in subsets: (sequencing)
-   [seq_quality_check](seq_quality_check.md) 0..1
    -   Description: "Indicate if the sequence has been called by
        automatic systems (none) or undergone a manual editing procedure
        (e.g. by inspecting the raw data or chromatograms). Applied only
        for sequences that are not submitted to SRA,ENA or DRA"
    -   Range: [TextValue](TextValue.md)
    -   in subsets: (sequencing)
-   [chimera_check](chimera_check.md) 0..1
    -   Description: "A chimeric sequence, or chimera for short, is a
        sequence comprised of two or more phylogenetically distinct
        parent sequences. Chimeras are usually PCR artifacts thought to
        occur when a prematurely terminated amplicon reanneals to a
        foreign DNA strand and is copied to completion in the following
        PCR cycles. The point at which the chimeric sequence changes
        from one parent to the next is called the breakpoint or
        conversion point"
    -   Range: [TextValue](TextValue.md)
    -   in subsets: (sequencing)
-   [omics processing➞has input](omics_processing_has_input.md) 1..\*
    -   Description: An input to a process.
    -   Range: [Biosample](Biosample.md)

### Inherited from biosample processing:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

## Other properties {#other-properties-37}

  ----------------------- ----------------------- -----------------------
  **Aliases:**                                    omics assay

                                                  sequencing project

                                                  experiment

  **Alt Descriptions:**                           An experiment contains
                                                  information about a
                                                  sequencing experiment
                                                  including library and
                                                  instrument details.
                                                  (embl.ena)

  **Comments:**                                   The IDs for objects
                                                  coming from GOLD will
                                                  have prefixes
                                                  gold:GpNNNN

  **In Subsets:**                                 sample subset

  **Broad Mappings:**                             OBI:0000070

                                                  ISA:Assay
  ----------------------- ----------------------- -----------------------

# Class: ontology class

URI:
<https://microbiomedata/schema/workflow_execution_activity/OntologyClass>

[![image33](https://yuml.me/diagram/nofunky;dir:TB/class/%5BControlledTermValue%5D++-%20term%200..1%3E%5BOntologyClass%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BOntologyClass%5D%5E-%5BEnvironmentalMaterialTerm%5D,%5BOntologyClass%5D%5E-%5BChemicalEntity%5D,%5BNamedThing%5D%5E-%5BOntologyClass%5D,%5BNamedThing%5D,%5BEnvironmentalMaterialTerm%5D,%5BControlledTermValue%5D,%5BChemicalEntity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BControlledTermValue%5D++-%20term%200..1%3E%5BOntologyClass%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BOntologyClass%5D%5E-%5BEnvironmentalMaterialTerm%5D,%5BOntologyClass%5D%5E-%5BChemicalEntity%5D,%5BNamedThing%5D%5E-%5BOntologyClass%5D,%5BNamedThing%5D,%5BEnvironmentalMaterialTerm%5D,%5BControlledTermValue%5D,%5BChemicalEntity%5D)

## Parents

-   is_a: [NamedThing](NamedThing.md) - a databased entity or
    concept/class

## Children

-   [ChemicalEntity](ChemicalEntity.md) - An atom or molecule that can
    be represented with a chemical formula. Include lipids, glycans,
    natural products, drugs. There may be different terms for distinct
    acid-base forms, protonation states
-   [EnvironmentalMaterialTerm](EnvironmentalMaterialTerm.md)

## Referenced by Class {#referenced-by-class-23}

-   [ControlledTermValue](ControlledTermValue.md) [term](term.md) 0..1
    [OntologyClass](OntologyClass.md)

## Attributes {#attributes-33}

### Inherited from named thing:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

# Class: orthology group

A set of genes or gene products in which all members are orthologous

URI: [nmdc:OrthologyGroup](https://microbiomedata/meta/OrthologyGroup)

[![image34](https://yuml.me/diagram/nofunky;dir:TB/class/%5BFunctionalAnnotationTerm%5D%5E-%5BOrthologyGroup%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BFunctionalAnnotationTerm%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BFunctionalAnnotationTerm%5D%5E-%5BOrthologyGroup%7Cid(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BFunctionalAnnotationTerm%5D)

## Identifier prefixes

-   KEGG.ORTHOLOGY
-   EGGNOG
-   PFAM
-   TIGRFAM
-   SUPFAM
-   CATH
-   PANTHER.FAMILY

## Parents

-   is_a: [FunctionalAnnotationTerm](FunctionalAnnotationTerm.md)
    -Abstract grouping class for any term/descriptor that can be applied
    to a functional unit of a genome (protein, ncRNA, complex).

## Attributes {#attributes-34}

### Inherited from functional annotation term:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

## Other properties {#other-properties-38}

+----------------------------------+----------------------------------+
| **Exact Mappings:**              | biolink:GeneFamily               |
+==================================+==================================+
| Class: pathway ==============    |                                  |
+----------------------------------+----------------------------------+
| A pathway is a seque             | nce of steps/reactions carried   |
|                                  | out by an organism or            |
+----------------------------------+----------------------------------+
| community of organis             | ms                               |
+----------------------------------+----------------------------------+
| URI: \`nmdc:Pathway \<           | <https://micr                    |
|                                  | obiomedata/meta/Pathway>\>\`\_\_ |
| [![image35](https://             |                                  |
| yuml.me/diagram/nofunky;dir:TB/c |                                  |
| lass/%5BReaction%5D,%5BReaction% |                                  |
| 5D%3Chas_part%200..*-%20%5BPathw |                                  |
| ay%7Cid(i):string;name(i):string |                                  |
| %20%3F;description(i):string%20% |                                  |
| 3F;alternative_identifiers(i):st |                                  |
| ring%20*%5D,%5BFunctionalAnnotat |                                  |
| ionTerm%5D%5E-%5BPathway%5D,%5BF |                                  |
| unctionalAnnotationTerm%5D)](htt |                                  |
| ps://yuml.me/diagram/nofunky;dir |                                  |
| :TB/class/%5BReaction%5D,%5BReac |                                  |
| tion%5D%3Chas_part%200..*-%20%5B |                                  |
| Pathway%7Cid(i):string;name(i):s |                                  |
| tring%20%3F;description(i):strin |                                  |
| g%20%3F;alternative_identifiers( |                                  |
| i):string%20*%5D,%5BFunctionalAn |                                  |
| notationTerm%5D%5E-%5BPathway%5D |                                  |
| ,%5BFunctionalAnnotationTerm%5D) |                                  |
+----------------------------------+----------------------------------+
| ## Identifier p                  | xes-5:                           |
| refixes {#identifier-prefixes-5} |                                  |
|                                  |                                  |
| -   KEGG.PATHWAY                 |                                  |
| -   COG                          |                                  |
|                                  |                                  |
| ## Parents {#parents-41}         |                                  |
+----------------------------------+----------------------------------+
| \- is_a: \`Functional            | AnnotationTerm                   |
|                                  | \<Fun                            |
|                                  | ctionalAnnotationTerm.md\>\`\_\_ |
|                                  | -                                |
+----------------------------------+----------------------------------+
| > Abstract grouping              | > class for any term/descriptor  |
|                                  | > that can be applied            |
+----------------------------------+----------------------------------+
| > to a functional u              | nit of a genome (protein, ncRNA, |
|                                  | complex).                        |
+----------------------------------+----------------------------------+
| ## Referenced by                 | ass-24:                          |
|  Class {#referenced-by-class-23} |                                  |
|                                  |                                  |
| ## Attributes {#attributes-35}   |                                  |
|                                  |                                  |
| ### Own {#own-26}                |                                  |
+----------------------------------+----------------------------------+
| -   \`pathway➞has_par            | t \<pathway_has_part.md\>\`\_\_  |
|                                  | 0..\*                            |
+----------------------------------+----------------------------------+
| > -   Description: A step        | > pathway can be broken down to  |
|                                  | > a series of reaction           |
+----------------------------------+----------------------------------+
| > -   Range: \`Reacti            | on \<Reaction.md\>\`\_\_         |
+----------------------------------+----------------------------------+
|                                  | unctional-annotation-term-1:     |
+----------------------------------+----------------------------------+
| Inherited from funct             | ional annotation term:           |
+----------------------------------+----------------------------------+
| ------------------------------   | ------------------------------   |
+----------------------------------+----------------------------------+
| -   [id](id.md) 1.               | .1                               |
+----------------------------------+----------------------------------+
| > -   Description: A             | > unique identifier for a thing. |
|                                  | > Must be either a               |
+----------------------------------+----------------------------------+
| > CURIE shorthan                 | d for a URI or a complete URI    |
+----------------------------------+----------------------------------+
| > -   Range: \`String            | > \<types/String.md\>\`\_\_      |
+----------------------------------+----------------------------------+
| -   [name](name.md)              | \_ 0..1                          |
+----------------------------------+----------------------------------+
| > -   Description: A             | > human readable label for an    |
|                                  | > entity                         |
+----------------------------------+----------------------------------+
| > -   Range: \`String            | > \<types/String.md\>\`\_\_      |
+----------------------------------+----------------------------------+
| -   \`description \<des          | cription.md\>\`\_\_ 0..1         |
+----------------------------------+----------------------------------+
| > -   Description: a             | > human-readable description of  |
|                                  | > a thing                        |
+----------------------------------+----------------------------------+
| > -   Range: \`String            | > \<types/String.md\>\`\_\_      |
+----------------------------------+----------------------------------+
| -   \`alternative iden           | tifiers                          |
|                                  | \<al                             |
|                                  | ternative_identifiers.md\>\`\_\_ |
|                                  | 0..\*                            |
+----------------------------------+----------------------------------+
| > -   Description: A             | > list of alternative            |
|                                  | > identifiers for the entity.    |
+----------------------------------+----------------------------------+
| > -   Range: \`String            | > \<types/String.md\>\`\_\_      |
+----------------------------------+----------------------------------+
| ## Other p                       | -39:                             |
| roperties {#other-properties-38} |                                  |
+----------------------------------+----------------------------------+
| ===================              | ==================               |
+----------------------------------+----------------------------------+
| **Aliases:**                     | > biological process             |
+----------------------------------+----------------------------------+
|                                  | > metabolic pathway              |
+----------------------------------+----------------------------------+
|                                  | > signaling pathway              |
+----------------------------------+----------------------------------+
| **Exact Mappings:**              | > biolink:Pathway                |
+----------------------------------+----------------------------------+

# Class: peptide quantification

This is used to link a metaproteomics analysis workflow to a specific
peptide sequence and related information

URI:
<https://microbiomedata/schema/workflow_execution_activity/PeptideQuantification>

[![image36](https://yuml.me/diagram/nofunky;dir:TB/class/%5BGeneProduct%5D%3Call%20proteins%200..*-%20%5BPeptideQuantification%7Cpeptide_sequence:string%20%3F;min_q_value:float%20%3F;peptide_spectral_count:integer%20%3F;peptide_sum_masic_abundance:integer%20%3F%5D,%5BGeneProduct%5D%3Cbest%20protein%200..1-%20%5BPeptideQuantification%5D,%5BMetaproteomicsAnalysisActivity%5D++-%20has%20peptide%20quantifications%200..*%3E%5BPeptideQuantification%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BGeneProduct%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BGeneProduct%5D%3Call%20proteins%200..*-%20%5BPeptideQuantification%7Cpeptide_sequence:string%20%3F;min_q_value:float%20%3F;peptide_spectral_count:integer%20%3F;peptide_sum_masic_abundance:integer%20%3F%5D,%5BGeneProduct%5D%3Cbest%20protein%200..1-%20%5BPeptideQuantification%5D,%5BMetaproteomicsAnalysisActivity%5D++-%20has%20peptide%20quantifications%200..*%3E%5BPeptideQuantification%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BGeneProduct%5D)

## Referenced by Class

-   **None** [has peptide
    quantifications](has_peptide_quantifications.md) 0..\*
    [PeptideQuantification](PeptideQuantification.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
    [metaproteomics analysis activity➞has peptide
    quantifications](metaproteomics_analysis_activity_has_peptide_quantifications.md)
    0..\* [PeptideQuantification](PeptideQuantification.md)

## Attributes {#attributes-36}

### Own {#own-27}

-   [peptide quantification➞peptide
    sequence](peptide_quantification_peptide_sequence.md) 0..1
    -   Range: [String](types/String.md)
-   [peptide quantification➞best
    protein](peptide_quantification_best_protein.md) 0..1
    -   Description: the specific protein identifier most correctly
        associated with the peptide sequence
    -   Range: [GeneProduct](GeneProduct.md)
-   [peptide quantification➞all
    proteins](peptide_quantification_all_proteins.md) 0..\*
    -   Description: the list of protein identifiers that are associated
        with the peptide sequence
    -   Range: [GeneProduct](GeneProduct.md)
-   [peptide
    quantification➞min_q\_value](peptide_quantification_min_q_value.md)
    0..1
    -   Description: smallest Q-Value associated with the peptide
        sequence as provided by MSGFPlus tool
    -   Range: [Float](types/Float.md)
-   [peptide
    quantification➞peptide_spectral_count](peptide_quantification_peptide_spectral_count.md)
    0..1
    -   Description: sum of filter passing MS2 spectra associated with
        the peptide sequence within a given LC-MS/MS data file
    -   Range: [Integer](types/Integer.md)
-   [peptide
    quantification➞peptide_sum_masic_abundance](peptide_quantification_peptide_sum_masic_abundance.md)
    0..1
    -   Description: combined MS1 extracted ion chromatograms derived
        from MS2 spectra associated with the peptide sequence from a
        given LC-MS/MS data file using the MASIC tool
    -   Range: [Integer](types/Integer.md)

# Class: person

represents a person, such as a researcher

URI: <https://microbiomedata/schema/workflow_execution_activity/Person>

[![image37](https://yuml.me/diagram/nofunky;dir:TB/class/%5BNamedThing%5D%5E-%5BPerson%7Cid:string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BNamedThing%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BNamedThing%5D%5E-%5BPerson%7Cid:string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BNamedThing%5D)

## Parents {#parents-42}

-   is_a: [NamedThing](NamedThing.md) - a databased entity or
    concept/class

## Referenced by Class

## Attributes {#attributes-37}

### Own {#own-28}

-   [person➞id](person_id.md) 1..1
    -   Description: Should be an ORCID. Specify in CURIE format. E.g
        ORCID:0000-1111-...
    -   Range: [String](types/String.md)

### Inherited from named thing:

-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

## Other properties

  **Comments:**   not yet in use
  --------------- ----------------
                  
  Class: person   value

An attribute value representing a person

URI:
<https://microbiomedata/schema/workflow_execution_activity/PersonValue>

[![image38](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BPersonValue%7Corcid:string%20%3F;profile_image_url:string%20%3F;email:string%20%3F;name:string%20%3F;websites:string%20*;has_raw_value:string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BPersonValue%7Corcid:string%20%3F;profile_image_url:string%20%3F;email:string%20%3F;name:string%20%3F;websites:string%20*;has_raw_value:string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)

## Parents {#parents-43}

-   is_a: [AttributeValue](AttributeValue.md) - The value for any value
    of a attribute for a sample. This object can hold both the
    un-normalized atomic value and the structured value

## Referenced by Class

## Attributes {#attributes-38}

### Own {#own-29}

-   [orcid](orcid.md) 0..1
    -   Description: The ORICD of a person.
    -   Range: [String](types/String.md)
-   [profile image url](profile_image_url.md) 0..1
    -   Description: A url that points to an image of a person.
    -   Range: [String](types/String.md)
-   [email](email.md) 0..1
    -   Description: An email address for an entity such as a person.
        This should be the primarly email address used.
    -   Range: [String](types/String.md)
-   [person value➞name](person_value_name.md) 0..1
    -   Description: The full name of the Investgator. It should follow
        the format FIRST \[MIDDLE NAME\| MIDDLE INITIAL\] LAST, where
        MIDDLE NAME\| MIDDLE INITIAL is optional.
    -   Range: [String](types/String.md)
-   [websites](websites.md) 0..\*
    -   Description: A list of websites that are assocatiated with the
        entity.
    -   Range: [String](types/String.md)
-   [person value➞has raw value](person_value_has_raw_value.md) 0..1
    -   Description: The full name of the Investgator in format FIRST
        LAST.
    -   Range: [String](types/String.md)

### Inherited from attribute value:

-   [was generated by](was_generated_by.md) 0..1
    -   Range: [Activity](Activity.md)
-   [attribute value➞type](attribute_value_type.md) 0..1
    -   Description: An optional string that specified the type of
        object.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None

# Class: protein quantification

This is used to link a metaproteomics analysis workflow to a specific
protein

URI:
<https://microbiomedata/schema/workflow_execution_activity/ProteinQuantification>

[![image39](https://yuml.me/diagram/nofunky;dir:TB/class/%5BGeneProduct%5D%3Call%20proteins%200..*-%20%5BProteinQuantification%7Cpeptide_sequence_count:integer%20%3F;protein_spectral_count:integer%20%3F;protein_sum_masic_abundance:integer%20%3F%5D,%5BGeneProduct%5D%3Cbest%20protein%200..1-%20%5BProteinQuantification%5D,%5BGeneProduct%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BGeneProduct%5D%3Call%20proteins%200..*-%20%5BProteinQuantification%7Cpeptide_sequence_count:integer%20%3F;protein_spectral_count:integer%20%3F;protein_sum_masic_abundance:integer%20%3F%5D,%5BGeneProduct%5D%3Cbest%20protein%200..1-%20%5BProteinQuantification%5D,%5BGeneProduct%5D)

## Referenced by Class

## Attributes {#attributes-39}

### Own {#own-30}

-   [protein quantification➞best
    protein](protein_quantification_best_protein.md) 0..1
    -   Description: the specific protein identifier most correctly
        grouped to its associated peptide sequences
    -   Range: [GeneProduct](GeneProduct.md)
-   [protein quantification➞all
    proteins](protein_quantification_all_proteins.md) 0..\*
    -   Description: the grouped list of protein identifiers associated
        with the peptide sequences that were grouped to a best protein
    -   Range: [GeneProduct](GeneProduct.md)
-   [protein
    quantification➞peptide_sequence_count](protein_quantification_peptide_sequence_count.md)
    0..1
    -   Description: count of peptide sequences grouped to the
        best_protein
    -   Range: [Integer](types/Integer.md)
-   [protein
    quantification➞protein_spectral_count](protein_quantification_protein_spectral_count.md)
    0..1
    -   Description: sum of filter passing MS2 spectra associated with
        the best protein within a given LC-MS/MS data file
    -   Range: [Integer](types/Integer.md)
-   [protein
    quantification➞protein_sum_masic_abundance](protein_quantification_protein_sum_masic_abundance.md)
    0..1
    -   Description: combined MS1 extracted ion chromatograms derived
        from MS2 spectra associated with the best protein from a given
        LC-MS/MS data file using the MASIC tool
    -   Range: [Integer](types/Integer.md)

# Class: quantity value

A simple quantity, e.g. 2cm

URI:
<https://microbiomedata/schema/workflow_execution_activity/QuantityValue>

[![image40](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BQuantityValue%7Chas_unit:unit%20%3F;has_numeric_value:double%20%3F;has_minimum_numeric_value:float%20%3F;has_maximum_numeric_value:float%20%3F;has_raw_value:string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BQuantityValue%7Chas_unit:unit%20%3F;has_numeric_value:double%20%3F;has_minimum_numeric_value:float%20%3F;has_maximum_numeric_value:float%20%3F;has_raw_value:string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)

## Parents {#parents-44}

-   is_a: [AttributeValue](AttributeValue.md) - The value for any value
    of a attribute for a sample. This object can hold both the
    un-normalized atomic value and the structured value

## Referenced by Class

## Attributes {#attributes-40}

### Own {#own-31}

-   [quantity value➞has unit](quantity_value_has_unit.md) 0..1
    -   Description: The unit of the quantity
    -   Range: [Unit](types/Unit.md)
-   [quantity value➞has numeric
    value](quantity_value_has_numeric_value.md) 0..1
    -   Description: The number part of the quantity
    -   Range: [Double](types/Double.md)
-   [has minimum numeric value](has_minimum_numeric_value.md) 0..1
    -   Description: The minimum value part, expressed as number, of the
        quantity value when the value covers a range.
    -   Range: [Float](types/Float.md)
-   [has maximum numeric value](has_maximum_numeric_value.md) 0..1
    -   Description: The maximum value part, expressed as number, of the
        quantity value when the value covers a range.
    -   Range: [Float](types/Float.md)
-   [quantity value➞has raw value](quantity_value_has_raw_value.md) 0..1
    -   Description: Unnormalized atomic string representation, should
        in syntax {number} {unit}
    -   Range: [String](types/String.md)

### Inherited from attribute value:

-   [was generated by](was_generated_by.md) 0..1
    -   Range: [Activity](Activity.md)
-   [attribute value➞type](attribute_value_type.md) 0..1
    -   Description: An optional string that specified the type of
        object.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None

## Other properties

  **Mappings:**    schema:QuantityValue
  ---------------- ----------------------
                   
  Class: reactio   n

An individual biochemical transformation carried out by a functional
unit of an organism, in which a collection of substrates are transformed
into a collection of products. Can also represent transporters

URI: [nmdc:Reaction](https://microbiomedata/meta/Reaction)

[![image41](https://yuml.me/diagram/nofunky;dir:TB/class/%5BReactionParticipant%5D,%5BReactionParticipant%5D%3Cright%20participants%200..*-++%5BReaction%7Cdirection:string%20%3F;smarts_string:string%20%3F;is_diastereoselective:boolean%20%3F;is_stereo:boolean%20%3F;is_balanced:boolean%20%3F;is_transport:boolean%20%3F;is_fully_characterized:boolean%20%3F;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BReactionParticipant%5D%3Cleft%20participants%200..*-++%5BReaction%5D,%5BPathway%5D-%20has_part%200..*%3E%5BReaction%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BReaction%5D,%5BPathway%5D,%5BFunctionalAnnotationTerm%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BReactionParticipant%5D,%5BReactionParticipant%5D%3Cright%20participants%200..*-++%5BReaction%7Cdirection:string%20%3F;smarts_string:string%20%3F;is_diastereoselective:boolean%20%3F;is_stereo:boolean%20%3F;is_balanced:boolean%20%3F;is_transport:boolean%20%3F;is_fully_characterized:boolean%20%3F;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BReactionParticipant%5D%3Cleft%20participants%200..*-++%5BReaction%5D,%5BPathway%5D-%20has_part%200..*%3E%5BReaction%5D,%5BFunctionalAnnotationTerm%5D%5E-%5BReaction%5D,%5BPathway%5D,%5BFunctionalAnnotationTerm%5D)

## Identifier prefixes {#identifier-prefixes-6}

-   KEGG.REACTION
-   RHEA
-   MetaCyc
-   EC
-   GO
-   MetaNetX
-   SEED
-   RetroRules

## Parents {#parents-45}

-   is_a: [FunctionalAnnotationTerm](FunctionalAnnotationTerm.md)
    -Abstract grouping class for any term/descriptor that can be applied
    to a functional unit of a genome (protein, ncRNA, complex).

## Referenced by Class

-   **None** [has_part](has_part.md) 0..\* [Reaction](Reaction.md)
-   [Pathway](Pathway.md) [pathway➞has_part](pathway_has_part.md) 0..\*
    [Reaction](Reaction.md)

## Attributes {#attributes-41}

### Own {#own-32}

-   [reaction➞left participants](reaction_left_participants.md) 0..\*
    -   Range: [ReactionParticipant](ReactionParticipant.md)
-   [reaction➞right participants](reaction_right_participants.md) 0..\*
    -   Range: [ReactionParticipant](ReactionParticipant.md)
-   [reaction➞direction](reaction_direction.md) 0..1
    -   Description: One of l-\>r, r-\>l, bidirectional, neutral
    -   Range: [String](types/String.md)
-   [reaction➞smarts string](reaction_smarts_string.md) 0..1
    -   Range: [String](types/String.md)
-   [reaction➞is diastereoselective](reaction_is_diastereoselective.md)
    0..1
    -   Range: [Boolean](types/Boolean.md)
-   [reaction➞is stereo](reaction_is_stereo.md) 0..1
    -   Range: [Boolean](types/Boolean.md)
-   [reaction➞is balanced](reaction_is_balanced.md) 0..1
    -   Range: [Boolean](types/Boolean.md)
-   [reaction➞is transport](reaction_is_transport.md) 0..1
    -   Range: [Boolean](types/Boolean.md)
-   [reaction➞is fully
    characterized](reaction_is_fully_characterized.md) 0..1
    -   Description: False if includes R-groups
    -   Range: [Boolean](types/Boolean.md)

### Inherited from functional annotation term: {#inherited-from-functional-annotation-term-2}

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

## Other properties

  **Exact Mappings:**    biolink:MolecularActivity
  ---------------------- ---------------------------
                         
  Class: reaction part   icipant

Instances of this link a reaction to a chemical entity participant

URI:
[nmdc:ReactionParticipant](https://microbiomedata/meta/ReactionParticipant)

[![image42](https://yuml.me/diagram/nofunky;dir:TB/class/%5BChemicalEntity%5D%3Cchemical%200..1-%20%5BReactionParticipant%7Cstoichiometry:integer%20%3F%5D,%5BReaction%5D++-%20left%20participants%200..*%3E%5BReactionParticipant%5D,%5BReaction%5D++-%20right%20participants%200..*%3E%5BReactionParticipant%5D,%5BReaction%5D,%5BChemicalEntity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BChemicalEntity%5D%3Cchemical%200..1-%20%5BReactionParticipant%7Cstoichiometry:integer%20%3F%5D,%5BReaction%5D++-%20left%20participants%200..*%3E%5BReactionParticipant%5D,%5BReaction%5D++-%20right%20participants%200..*%3E%5BReactionParticipant%5D,%5BReaction%5D,%5BChemicalEntity%5D)

## Referenced by Class

-   **None** [left participants](left_participants.md) 0..\*
    [ReactionParticipant](ReactionParticipant.md)
-   [Reaction](Reaction.md) [reaction➞left
    participants](reaction_left_participants.md) 0..\*
    [ReactionParticipant](ReactionParticipant.md)
-   [Reaction](Reaction.md) [reaction➞right
    participants](reaction_right_participants.md) 0..\*
    [ReactionParticipant](ReactionParticipant.md)
-   **None** [right participants](right_participants.md) 0..\*
    [ReactionParticipant](ReactionParticipant.md)

## Attributes {#attributes-42}

### Own {#own-33}

-   [reaction participant➞chemical](reaction_participant_chemical.md)
    0..1
    -   Range: [ChemicalEntity](ChemicalEntity.md)
-   [reaction
    participant➞stoichiometry](reaction_participant_stoichiometry.md)
    0..1
    -   Range: [Integer](types/Integer.md)

# Class: read based analysis activity

URI:
<https://microbiomedata/schema/workflow_execution_activity/ReadBasedAnalysisActivity>

[![image43](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadBasedAnalysisActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BNamedThing%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadBasedAnalysisActivity%7Cexecution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BNamedThing%5D,%5BActivity%5D)

## Parents {#parents-46}

-   is_a: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
    -Represents an instance of an execution of a particular workflow

## Attributes {#attributes-43}

### Inherited from workflow execution activity:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [used](used.md) 0..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞execution
    resource](workflow_execution_activity_execution_resource.md) 1..1
    -   Description: Example: NERSC-Cori
    -   Range: [String](types/String.md)
-   [workflow execution activity➞git
    url](workflow_execution_activity_git_url.md) 1..1
    -   Description: Example:
        <https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>
    -   Range: [String](types/String.md)
-   [workflow execution activity➞has
    input](workflow_execution_activity_has_input.md) 1..\*
    -   Description: An input to a process.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution activity➞has
    output](workflow_execution_activity_has_output.md) 1..\*
    -   Description: An output biosample to a processing step
    -   Range: [NamedThing](NamedThing.md)
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution
    activity➞type](workflow_execution_activity_type.md) 1..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [workflow execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md) 0..1
    -   Description: the agent/entity associated with the generation of
        the file
    -   Range: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
-   [workflow execution activity➞started at
    time](workflow_execution_activity_started_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞ended at
    time](workflow_execution_activity_ended_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md) 1..1
    -   Range: [Activity](Activity.md)

## Other properties

  **In Subsets:**    workflow subset
  ------------------ ------------------
                     
  Class: read QC a   nalysis activity

URI:
<https://microbiomedata/schema/workflow_execution_activity/ReadQCAnalysisActivity>

[![image44](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Chas%20output%201..*-%20%5BReadQCAnalysisActivity%7Cinput_read_count:float%20%3F;input_base_count:float%20%3F;output_read_count:float%20%3F;output_base_count:float%20%3F;input_read_bases:float%20%3F;output_read_bases:float%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BNamedThing%5D%3Chas%20input%201..*-%20%5BReadQCAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadQCAnalysisActivity%5D,%5BNamedThing%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Chas%20output%201..*-%20%5BReadQCAnalysisActivity%7Cinput_read_count:float%20%3F;input_base_count:float%20%3F;output_read_count:float%20%3F;output_base_count:float%20%3F;input_read_bases:float%20%3F;output_read_bases:float%20%3F;execution_resource(i):string;git_url(i):string;type(i):string;started_at_time(i):string;ended_at_time(i):string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BNamedThing%5D%3Chas%20input%201..*-%20%5BReadQCAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadQCAnalysisActivity%5D,%5BNamedThing%5D,%5BActivity%5D)

## Parents {#parents-47}

-   is_a: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
    -Represents an instance of an execution of a particular workflow

## Referenced by Class

## Attributes {#attributes-44}

### Own {#own-34}

-   [input read count](input_read_count.md) 0..1
    -   Description: The sequence count number of input reads for QC
        analysis.
    -   Range: [Float](types/Float.md)
-   [input base count](input_base_count.md) 0..1
    -   Description: The nucleotide base count number of input reads for
        QC analysis.
    -   Range: [Float](types/Float.md)
-   [output read count](output_read_count.md) 0..1
    -   Description: After QC analysis sequence count number.
    -   Range: [Float](types/Float.md)
-   [output base count](output_base_count.md) 0..1
    -   Description: After QC analysis nucleotide base count number.
    -   Range: [Float](types/Float.md)
-   [read QC analysis
    activity➞input_read_bases](read_QC_analysis_activity_input_read_bases.md)
    0..1
    -   Description: TODO
    -   Range: [Float](types/Float.md)
-   [read QC analysis
    activity➞output_read_bases](read_QC_analysis_activity_output_read_bases.md)
    0..1
    -   Description: TODO
    -   Range: [Float](types/Float.md)
-   [read QC analysis activity➞has
    input](read_QC_analysis_activity_has_input.md) 1..\*
    -   Description: An input to a process.
    -   Range: [NamedThing](NamedThing.md)
-   [read QC analysis activity➞has
    output](read_QC_analysis_activity_has_output.md) 1..\*
    -   Description: An output biosample to a processing step
    -   Range: [NamedThing](NamedThing.md)

### Inherited from workflow execution activity:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [used](used.md) 0..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞execution
    resource](workflow_execution_activity_execution_resource.md) 1..1
    -   Description: Example: NERSC-Cori
    -   Range: [String](types/String.md)
-   [workflow execution activity➞git
    url](workflow_execution_activity_git_url.md) 1..1
    -   Description: Example:
        <https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>
    -   Range: [String](types/String.md)
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution
    activity➞type](workflow_execution_activity_type.md) 1..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [workflow execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md) 0..1
    -   Description: the agent/entity associated with the generation of
        the file
    -   Range: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
-   [workflow execution activity➞started at
    time](workflow_execution_activity_started_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞ended at
    time](workflow_execution_activity_ended_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md) 1..1
    -   Range: [Activity](Activity.md)

## Other properties

  **In Subsets:**    workflow subset
  ------------------ -----------------
                     
  Subset: sample s   ubset

Subset consisting of entities linked to the processing of samples.
Currently, this subset consists of study, omics process, and biosample.

URI: [nmdc:sample_subset](https://microbiomedata/meta/sample_subset)

## Classes

-   [Biosample](Biosample.md) - A material sample. It may be
    environmental (encompassing many organisms) or isolate or tissue. An
    environmental sample containing genetic material from multiple
    individuals is commonly referred to as a biosample.
-   [OmicsProcessing](OmicsProcessing.md) - The methods and processes
    used to generate omics data from a biosample or organism.
-   [Study](Study.md) - A study summarizes the overall goal of a
    research initiative and outlines the key objective of its underlying
    projects.

## Mixins

## Slots

## Types

## Enums

# Subset: sequencing

URI:
<https://microbiomedata/schema/workflow_execution_activity/sequencing>

## Classes

## Mixins

## Slots

## Types

## Enums

# Subset: soil

URI: <https://microbiomedata/schema/workflow_execution_activity/soil>

## Classes

## Mixins

## Slots

## Types

## Enums

# Class: study

A study summarizes the overall goal of a research initiative and
outlines the key objective of its underlying projects.

URI: [nmdc:Study](https://microbiomedata/meta/Study)

[![image45](https://yuml.me/diagram/nofunky;dir:TB/class/%5BImageValue%5D%3Cstudy%20image%200..*-++%5BStudy%7Cecosystem:string%20%3F;ecosystem_category:string%20%3F;ecosystem_type:string%20%3F;ecosystem_subtype:string%20%3F;specific_ecosystem:string%20%3F;title:string%20%3F;alternative_titles:string%20*;alternative_descriptions:string%20*;alternative_names:string%20*;abstract:string%20%3F;objective:string%20%3F;websites:string%20*;publications:string%20*;ess_dive_datasets:string%20*;type:string%20%3F;relevant_protocols:string%20*;funding_sources:string%20*;INSDC_bioproject_identifiers:external_identifier%20*;INSDC_SRA_ENA_study_identifiers:external_identifier%20*;GOLD_study_identifiers:external_identifier%20*;MGnify_project_identifiers:external_identifier%20*;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BCreditAssociation%5D%3Chas%20credit%20associations%200..*-++%5BStudy%5D,%5BAttributeValue%5D%3Cdoi%200..1-++%5BStudy%5D,%5BPersonValue%5D%3Cprincipal%20investigator%200..1-++%5BStudy%5D,%5BDatabase%5D++-%20study%20set%200..*%3E%5BStudy%5D,%5BNamedThing%5D%5E-%5BStudy%5D,%5BPersonValue%5D,%5BNamedThing%5D,%5BImageValue%5D,%5BDatabase%5D,%5BCreditAssociation%5D,%5BAttributeValue%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BImageValue%5D%3Cstudy%20image%200..*-++%5BStudy%7Cecosystem:string%20%3F;ecosystem_category:string%20%3F;ecosystem_type:string%20%3F;ecosystem_subtype:string%20%3F;specific_ecosystem:string%20%3F;title:string%20%3F;alternative_titles:string%20*;alternative_descriptions:string%20*;alternative_names:string%20*;abstract:string%20%3F;objective:string%20%3F;websites:string%20*;publications:string%20*;ess_dive_datasets:string%20*;type:string%20%3F;relevant_protocols:string%20*;funding_sources:string%20*;INSDC_bioproject_identifiers:external_identifier%20*;INSDC_SRA_ENA_study_identifiers:external_identifier%20*;GOLD_study_identifiers:external_identifier%20*;MGnify_project_identifiers:external_identifier%20*;id(i):string;name(i):string%20%3F;description(i):string%20%3F;alternative_identifiers(i):string%20*%5D,%5BCreditAssociation%5D%3Chas%20credit%20associations%200..*-++%5BStudy%5D,%5BAttributeValue%5D%3Cdoi%200..1-++%5BStudy%5D,%5BPersonValue%5D%3Cprincipal%20investigator%200..1-++%5BStudy%5D,%5BDatabase%5D++-%20study%20set%200..*%3E%5BStudy%5D,%5BNamedThing%5D%5E-%5BStudy%5D,%5BPersonValue%5D,%5BNamedThing%5D,%5BImageValue%5D,%5BDatabase%5D,%5BCreditAssociation%5D,%5BAttributeValue%5D)

## Identifier prefixes {#identifier-prefixes-7}

-   GOLD
-   insdc.srs
-   mgnify

## Parents {#parents-48}

-   is_a: [NamedThing](NamedThing.md) - a databased entity or
    concept/class

## Referenced by Class

-   [Database](Database.md) [study set](study_set.md) 0..\*
    [Study](Study.md)

## Attributes {#attributes-45}

### Own {#own-35}

-   [ecosystem](ecosystem.md) 0..1
    -   Description: TODO
    -   Range: [String](types/String.md)
-   [ecosystem_category](ecosystem_category.md) 0..1
    -   Description: TODO
    -   Range: [String](types/String.md)
-   [ecosystem_type](ecosystem_type.md) 0..1
    -   Description: TODO
    -   Range: [String](types/String.md)
-   [ecosystem_subtype](ecosystem_subtype.md) 0..1
    -   Description: TODO
    -   Range: [String](types/String.md)
-   [specific_ecosystem](specific_ecosystem.md) 0..1
    -   Description: TODO
    -   Range: [String](types/String.md)
-   [principal investigator](principal_investigator.md) 0..1
    -   Description: represents the PI
    -   Range: [PersonValue](PersonValue.md)
-   [study➞doi](study_doi.md) 0..1
    -   Description: The dataset citation for this study
    -   Range: [AttributeValue](AttributeValue.md)
-   [title](title.md) 0..1
    -   Description: A name given to the entity that differs from the
        name/label programatically assigned to it. For example, when
        extracting study information for GOLD, the GOLD system has
        assigned a name/label. However, for display purposes, we may
        also wish the capture the title of the proposal that was used to
        fund the study.
    -   Range: [String](types/String.md)
-   [alternative titles](alternative_titles.md) 0..\*
    -   Description: A list of alternative titles for the entity. The
        distinction between title and alternative titles is
        application-specific.
    -   Range: [String](types/String.md)
-   [alternative descriptions](alternative_descriptions.md) 0..\*
    -   Description: A list of alternative descriptions for the entity.
        The distinction between desciption and alternative descriptions
        is application-specific.
    -   Range: [String](types/String.md)
-   [alternative names](alternative_names.md) 0..\*
    -   Description: A list of alternative names used to refer to the
        entity. The distinction between name and alternative names is
        application-specific.
    -   Range: [String](types/String.md)
-   [abstract](abstract.md) 0..1
    -   Description: The abstract of manuscript/grant associated with
        the entity; i.e., a summary of the resource.
    -   Range: [String](types/String.md)
-   [objective](objective.md) 0..1
    -   Description: The scientific objectives associated with the
        entity. It SHOULD correspond to scientific norms for objectives
        field in a structured abstract.
    -   Range: [String](types/String.md)
-   [websites](websites.md) 0..\*
    -   Description: A list of websites that are assocatiated with the
        entity.
    -   Range: [String](types/String.md)
-   [publications](publications.md) 0..\*
    -   Description: A list of publications that are assocatiated with
        the entity. The publicatons SHOULD be given using an identifier,
        such as a DOI or Pubmed ID, if possible.
    -   Range: [String](types/String.md)
-   [ess dive datasets](ess_dive_datasets.md) 0..\*
    -   Description: List of ESS-DIVE dataset DOIs
    -   Range: [String](types/String.md)
-   [type](type.md) 0..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [relevant protocols](relevant_protocols.md) 0..\*
    -   Range: [String](types/String.md)
-   [funding sources](funding_sources.md) 0..\*
    -   Range: [String](types/String.md)
-   [INSDC bioproject identifiers](INSDC_bioproject_identifiers.md)
    0..\*
    -   Description: \['identifiers for corresponding project in INSDC
        Bioproject'\]
    -   Range: [ExternalIdentifier](types/ExternalIdentifier.md)
    -   Example: <https://identifiers.org/bioproject:PRJNA366857> Avena
        fatua rhizosphere microbial communities - H1_Rhizo_Litter_2
        metatranscriptome
-   [INSDC SRA ENA study
    identifiers](INSDC_SRA_ENA_study_identifiers.md) 0..\*
    -   Description: \['identifiers for corresponding project in INSDC
        SRA / ENA'\]
    -   Range: [ExternalIdentifier](types/ExternalIdentifier.md)
    -   Example: <https://identifiers.org/insdc.sra:SRP121659> Avena
        fatua rhizosphere microbial communities - H1_Rhizo_Litter_2
        metatranscriptome
-   [GOLD study identifiers](GOLD_study_identifiers.md) 0..\*
    -   Description: \['identifiers for corresponding project in GOLD'\]
    -   Range: [ExternalIdentifier](types/ExternalIdentifier.md)
    -   Example: <https://identifiers.org/gold:Gs0110115> None
-   [MGnify project identifiers](MGnify_project_identifiers.md) 0..\*
    -   Description: \['identifiers for corresponding project in
        MGnify'\]
    -   Range: [ExternalIdentifier](types/ExternalIdentifier.md)
    -   Example: <https://identifiers.org/mgnify.proj:MGYS00005757> None
-   [has credit associations](has_credit_associations.md) 0..\*
    -   Description: This slot links a study to a credit association.
        The credit association will be linked to a person value and to a
        CRediT Contributor Roles term. Overall semantics: person should
        get credit X for their participation in the study
    -   Range: [CreditAssociation](CreditAssociation.md)
-   [study image](study_image.md) 0..\*
    -   Description: Links a study to one or more images.
    -   Range: [ImageValue](ImageValue.md)

### Inherited from named thing:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [description](description.md) 0..1
    -   Description: a human-readable description of a thing
    -   Range: [String](types/String.md)
-   [alternative identifiers](alternative_identifiers.md) 0..\*
    -   Description: A list of alternative identifiers for the entity.
    -   Range: [String](types/String.md)

## Other properties

  ----------------------- ----------------------- -----------------------
  **Aliases:**                                    proposal

                                                  research proposal

                                                  research study

                                                  investigation

                                                  project

                                                  umbrella project

  **Alt Descriptions:**                           A study (project)
                                                  groups together data
                                                  submitted to the
                                                  archive and controls
                                                  its release date. A
                                                  study accession is
                                                  typically used when
                                                  citing data submitted
                                                  to ENA (embl.ena)

  **In Subsets:**                                 sample subset

  **Exact Mappings:**                             SIO:001066

                                                  NCIT:C63536

                                                  OBI:0000066

                                                  ISA:Study

  **Close Mappings:**                             NCIT:C41198

                                                  ISA:Investigation

  **Broad Mappings:**                             prov:Activity
  ----------------------- ----------------------- -----------------------

# Class: text value

A basic string value

URI:
<https://microbiomedata/schema/workflow_execution_activity/TextValue>

[![image46](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BTextValue%7Clanguage:language_code%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BTextValue%7Clanguage:language_code%20%3F;has_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)

## Parents {#parents-49}

-   is_a: [AttributeValue](AttributeValue.md) - The value for any value
    of a attribute for a sample. This object can hold both the
    un-normalized atomic value and the structured value

## Attributes {#attributes-46}

### Own {#own-36}

-   [language](language.md) 0..1
    -   Description: Should use ISO 639-1 code e.g. "en", "fr"
    -   Range: [LanguageCode](types/LanguageCode.md)

### Inherited from attribute value:

-   [has raw value](has_raw_value.md) 0..1
    -   Description: The value that was specified for an annotation in
        raw form, i.e. a string. E.g. "2 cm" or "2-4 cm"
    -   Range: [String](types/String.md)
-   [was generated by](was_generated_by.md) 0..1
    -   Range: [Activity](Activity.md)
-   [attribute value➞type](attribute_value_type.md) 0..1
    -   Description: An optional string that specified the type of
        object.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None

# Class: timestamp value

A value that is a timestamp. The range should be ISO-8601

URI:
<https://microbiomedata/schema/workflow_execution_activity/TimestampValue>

[![image47](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BTimestampValue%7Chas_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BTimestampValue%7Chas_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)

## Parents {#parents-50}

-   is_a: [AttributeValue](AttributeValue.md) - The value for any value
    of a attribute for a sample. This object can hold both the
    un-normalized atomic value and the structured value

## Attributes {#attributes-47}

### Inherited from attribute value:

-   [has raw value](has_raw_value.md) 0..1
    -   Description: The value that was specified for an annotation in
        raw form, i.e. a string. E.g. "2 cm" or "2-4 cm"
    -   Range: [String](types/String.md)
-   [was generated by](was_generated_by.md) 0..1
    -   Range: [Activity](Activity.md)
-   [attribute value➞type](attribute_value_type.md) 0..1
    -   Description: An optional string that specified the type of
        object.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None

# Class: url value

A value that is a string that conforms to URL syntax

URI:
<https://microbiomedata/schema/workflow_execution_activity/UrlValue>

[![image48](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BUrlValue%7Chas_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BAttributeValue%5D%5E-%5BUrlValue%7Chas_raw_value(i):string%20%3F;type(i):string%20%3F%5D,%5BAttributeValue%5D,%5BActivity%5D)

## Parents {#parents-51}

-   is_a: [AttributeValue](AttributeValue.md) - The value for any value
    of a attribute for a sample. This object can hold both the
    un-normalized atomic value and the structured value

## Attributes {#attributes-48}

### Inherited from attribute value:

-   [has raw value](has_raw_value.md) 0..1
    -   Description: The value that was specified for an annotation in
        raw form, i.e. a string. E.g. "2 cm" or "2-4 cm"
    -   Range: [String](types/String.md)
-   [was generated by](was_generated_by.md) 0..1
    -   Range: [Activity](Activity.md)
-   [attribute value➞type](attribute_value_type.md) 0..1
    -   Description: An optional string that specified the type of
        object.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None

# Class: workflow execution activity

Represents an instance of an execution of a particular workflow

URI:
<https://microbiomedata/schema/workflow_execution_activity/WorkflowExecutionActivity>

[![image49](https://yuml.me/diagram/nofunky;dir:TB/class/%5BActivity%5D%3Cwas%20informed%20by%201..1-%20%5BWorkflowExecutionActivity%7Cexecution_resource:string;git_url:string;type:string;started_at_time:string;ended_at_time:string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BWorkflowExecutionActivity%5D%3Cwas%20associated%20with%200..1-%20%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Cpart%20of%200..*-%20%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Chas%20output%201..*-%20%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Chas%20input%201..*-%20%5BWorkflowExecutionActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadBasedAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadQCAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BNomAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAssembly%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAnnotationActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetaproteomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAssembly%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAnnotationActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetabolomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMAGsAnalysisActivity%5D,%5BActivity%5D%5E-%5BWorkflowExecutionActivity%5D,%5BReadBasedAnalysisActivity%5D,%5BReadQCAnalysisActivity%5D,%5BNomAnalysisActivity%5D,%5BNamedThing%5D,%5BMetatranscriptomeAssembly%5D,%5BMetatranscriptomeAnnotationActivity%5D,%5BMetatranscriptomeActivity%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BMetagenomeAssembly%5D,%5BMetagenomeAnnotationActivity%5D,%5BMetabolomicsAnalysisActivity%5D,%5BActivity%5D,%5BMAGsAnalysisActivity%5D)](https://yuml.me/diagram/nofunky;dir:TB/class/%5BActivity%5D%3Cwas%20informed%20by%201..1-%20%5BWorkflowExecutionActivity%7Cexecution_resource:string;git_url:string;type:string;started_at_time:string;ended_at_time:string;id(i):string;name(i):string%20%3F;used(i):string%20%3F%5D,%5BWorkflowExecutionActivity%5D%3Cwas%20associated%20with%200..1-%20%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Cpart%20of%200..*-%20%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Chas%20output%201..*-%20%5BWorkflowExecutionActivity%5D,%5BNamedThing%5D%3Chas%20input%201..*-%20%5BWorkflowExecutionActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadBasedAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BReadQCAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BNomAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAssembly%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeAnnotationActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetatranscriptomeActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetaproteomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAssembly%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetagenomeAnnotationActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMetabolomicsAnalysisActivity%5D,%5BWorkflowExecutionActivity%5D%5E-%5BMAGsAnalysisActivity%5D,%5BActivity%5D%5E-%5BWorkflowExecutionActivity%5D,%5BReadBasedAnalysisActivity%5D,%5BReadQCAnalysisActivity%5D,%5BNomAnalysisActivity%5D,%5BNamedThing%5D,%5BMetatranscriptomeAssembly%5D,%5BMetatranscriptomeAnnotationActivity%5D,%5BMetatranscriptomeActivity%5D,%5BMetaproteomicsAnalysisActivity%5D,%5BMetagenomeAssembly%5D,%5BMetagenomeAnnotationActivity%5D,%5BMetabolomicsAnalysisActivity%5D,%5BActivity%5D,%5BMAGsAnalysisActivity%5D)

## Parents {#parents-52}

-   is_a: [Activity](Activity.md) - a provence-generating activity

## Children

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md) - A
    metatranscriptome activity that e.g. pools assembly and annotation
    activity.
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)

## Referenced by Class

-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md) [workflow
    execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md) 0..1
    [WorkflowExecutionActivity](WorkflowExecutionActivity.md)

## Attributes {#attributes-49}

### Own {#own-37}

-   [workflow execution activity➞execution
    resource](workflow_execution_activity_execution_resource.md) 1..1
    -   Description: Example: NERSC-Cori
    -   Range: [String](types/String.md)
-   [workflow execution activity➞git
    url](workflow_execution_activity_git_url.md) 1..1
    -   Description: Example:
        <https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>
    -   Range: [String](types/String.md)
-   [workflow execution activity➞has
    input](workflow_execution_activity_has_input.md) 1..\*
    -   Description: An input to a process.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution activity➞has
    output](workflow_execution_activity_has_output.md) 1..\*
    -   Description: An output biosample to a processing step
    -   Range: [NamedThing](NamedThing.md)
-   [part of](part_of.md) 0..\*
    -   Description: Links a resource to another resource that either
        logically or physically includes it.
    -   Range: [NamedThing](NamedThing.md)
-   [workflow execution
    activity➞type](workflow_execution_activity_type.md) 1..1
    -   Description: An optional string that specifies the type object.
        This is used to allow for searches for different kinds of
        objects.
    -   Range: [String](types/String.md)
    -   Example: nmdc:Biosample None
    -   Example: nmdc:Study None
-   [workflow execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md) 0..1
    -   Description: the agent/entity associated with the generation of
        the file
    -   Range: [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
-   [workflow execution activity➞started at
    time](workflow_execution_activity_started_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞ended at
    time](workflow_execution_activity_ended_at_time.md) 1..1
    -   Range: [String](types/String.md)
-   [workflow execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md) 1..1
    -   Range: [Activity](Activity.md)

### Inherited from activity:

-   [id](id.md) 1..1
    -   Description: A unique identifier for a thing. Must be either a
        CURIE shorthand for a URI or a complete URI
    -   Range: [String](types/String.md)
-   [name](name.md) 0..1
    -   Description: A human readable label for an entity
    -   Range: [String](types/String.md)
-   [used](used.md) 0..1
    -   Range: [String](types/String.md)

## Other properties

  ----------------------- ----------------------- -----------------------
  **Aliases:**                                    analysis

  **Alt Descriptions:**                           An analysis contains
                                                  secondary analysis
                                                  results derived from
                                                  sequence reads (e.g. a
                                                  genome assembly)
                                                  (embl.ena)

  **In Subsets:**                                 workflow subset
  ----------------------- ----------------------- -----------------------

# Subset: workflow subset

Subset consisting of just the workflow exectution activities

URI:
<https://microbiomedata/schema/workflow_execution_activity/workflow_subset>

## Classes

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md) - A
    metatranscriptome activity that e.g. pools assembly and annotation
    activity.
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
    -Represents an instance of an execution of a particular workflow

## Mixins

## Slots

## Types

## Enums

# Slot: \_16s_recover

Can a 16S gene be recovered from the submitted SAG or MAG?

URI: \[nmdc:\_16s_recover\](<https://microbiomedata/meta/_16s_recover>)

## Domain and Range

None → 0..1 [BooleanValue](BooleanValue.md)

## Parents {#parents-53}

-   is_a: [attribute](attribute.md)

## Children

## Used by

## Other properties

+-----------------+----------------------+
| **Aliases:**    | > 16S recovered      |
+-----------------+----------------------+
| **Mappings:**   | > mixs:\_16s_recover |
+-----------------+----------------------+
| **In Subsets:** | > sequencing         |
+-----------------+----------------------+

# Slot: \_16s_recover_software

Tools used for 16S rRNA gene extraction

URI:
\[nmdc:\_16s_recover_software\](<https://microbiomedata/meta/_16s_recover_software>)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-54}

-   is_a: [attribute](attribute.md)

## Children

## Used by

## Other properties

+-----------------+-------------------------------+
| **Aliases:**    | > 16S recovery software       |
+-----------------+-------------------------------+
| **Mappings:**   | > mixs:\_16s_recover_software |
+-----------------+-------------------------------+
| **In Subsets:** | > sequencing                  |
+-----------------+-------------------------------+

# Slot: abs_air_humidity

Actual mass of water vapor - mh20 - present in the air water vapor
mixture

URI:
[nmdc:abs_air_humidity](https://microbiomedata/meta/abs_air_humidity)

## Domain and Range

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-55}

-   is_a: [attribute](attribute.md)

## Children

## Used by

## Other properties

+---------------+-------------------------+
| **Aliases:**  | > absolute air humidity |
+---------------+-------------------------+
| **Mappings:** | > mixs:abs_air_humidity |
+---------------+-------------------------+

# Slot: abstract

The abstract of manuscript/grant associated with the entity; i.e., a
summary of the resource.

URI:
<https://microbiomedata/schema/workflow_execution_activity/abstract>

## Domain and Range

None → 0..1 [String](types/String.md)

## Parents {#parents-56}

## Children

## Used by

## Other properties

  **Exact Mappings:**    dcterms:abstract
  ---------------------- ------------------
                         
  Slot: acted on behal   f of

URI:
<https://microbiomedata/schema/workflow_execution_activity/acted_on_behalf_of>

## Domain and Range

None → 0..1 [Agent](Agent.md)

## Parents {#parents-57}

## Children

## Used by

-   [Agent](Agent.md)

## Other properties

+----------------+----------------------+
| **Mappings:**  | prov:actedOnBehalfOf |
+================+======================+
|                |                      |
+----------------+----------------------+
| Slot: activity | > set                |
+----------------+----------------------+

This property links a database object to the set of workflow activities.

URI: [nmdc:activity_set](https://microbiomedata/meta/activity_set)

## Domain and Range

[Database](Database.md) → 0..\*
[WorkflowExecutionActivity](WorkflowExecutionActivity.md)

## Parents {#parents-58}

## Children

## Used by

-   [Database](Database.md)

# Slot: adapters

Adapters provide priming sequences for both amplification and sequencing
of the sample-library fragments. Both adapters should be reported; in
uppercase letters

URI: [nmdc:adapters](https://microbiomedata/meta/adapters)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-59}

-   is_a: [attribute](attribute.md)

## Children

## Used by

## Other properties

+-----------------+-----------------+
| **Aliases:**    | > adapters      |
+-----------------+-----------------+
| **Mappings:**   | > mixs:adapters |
+-----------------+-----------------+
| **In Subsets:** | > sequencing    |
+-----------------+-----------------+

# Slot: add_date

The date on which the information was added to the database.

URI: [nmdc:add_date](https://microbiomedata/meta/add_date)

## Domain and Range

None → 0..1 [String](types/String.md)

## Parents {#parents-60}

## Children

## Used by

-   [Biosample](Biosample.md)
-   [OmicsProcessing](OmicsProcessing.md)

# Slot: add_recov_method

"Additional (i.e. Secondary, tertiary, etc.) recovery methods deployed
for increase of hydrocarbon recovery from resource and start date for
each one of them. If""other"" is specified, please propose entry in
""additional info"" field"

URI:
[nmdc:add_recov_method](https://microbiomedata/meta/add_recov_method)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-61}

-   is_a: [attribute](attribute.md)

## Children

## Used by

## Other properties

+---------------+----------------------------------------------------------+
| **Aliases:**  | > secondary and tertiary recovery methods and start date |
+---------------+----------------------------------------------------------+
| **Mappings:** | > mixs:add_recov_method                                  |
+---------------+----------------------------------------------------------+

# Slot: additional_info

Information that doesn't fit anywhere else. Can also be used to propose
new entries for fields with controlled vocabulary

URI: [nmdc:additional_info](https://microbiomedata/meta/additional_info)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-62}

-   is_a: [attribute](attribute.md)

## Children

## Used by

## Other properties

+---------------+------------------------+
| **Aliases:**  | > additional info      |
+---------------+------------------------+
| **Mappings:** | > mixs:additional_info |
+---------------+------------------------+

# Slot: address

The street name and building number where the sampling occurred.

URI: [nmdc:address](https://microbiomedata/meta/address)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-63}

-   is_a: [attribute](attribute.md)

## Children

## Used by

## Other properties

+---------------+----------------+
| **Aliases:**  | > address      |
+---------------+----------------+
| **Mappings:** | > mixs:address |
+---------------+----------------+

# Slot: adj_room

"List of rooms (room number, room name) immediately adjacent to the
sampling room"

URI: [nmdc:adj_room](https://microbiomedata/meta/adj_room)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-64}

-   is_a: [attribute](attribute.md)

## Children

## Used by

## Other properties

+---------------+------------------+
| **Aliases:**  | > adjacent rooms |
+---------------+------------------+
| **Mappings:** | > mixs:adj_room  |
+---------------+------------------+

# Slot: aero_struc

"Aerospace structures typically consist of thin plates with stiffeners
for the external surfaces, bulkheads and frames to support the shape and
fasteners such as welds, rivets, screws and bolts to hold the components
together"

URI: [nmdc:aero_struc](https://microbiomedata/meta/aero_struc)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-65}

-   is_a: [attribute](attribute.md)

## Children

## Used by

## Other properties

+---------------+-----------------------+
| **Aliases:**  | > aerospace structure |
+---------------+-----------------------+
| **Mappings:** | > mixs:aero_struc     |
+---------------+-----------------------+

# Slot: agrochem_addition

"Addition of fertilizers, pesticides, etc. - amount and time of
applications"

URI:
[nmdc:agrochem_addition](https://microbiomedata/meta/agrochem_addition)

## Domain and Range

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-66}

-   is_a: [attribute](attribute.md)

## Children

## Used by

-   [Biosample](Biosample.md)

## Other properties

+---------------+----------------------------------+
| **Aliases:**  | > history/agrochemical additions |
+---------------+----------------------------------+
| **Mappings:** | > mixs:agrochem_addition         |
+---------------+----------------------------------+

# Slot: air_temp

Temperature of the air at the time of sampling

URI: [nmdc:air_temp](https://microbiomedata/meta/air_temp)

## Domain and Range

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-67}

-   is_a: [attribute](attribute.md)

## Children

## Used by

## Other properties

+---------------+-------------------+
| **Aliases:**  | > air temperature |
+---------------+-------------------+
| **Mappings:** | > mixs:air_temp   |
+---------------+-------------------+

# Slot: air_temp_regm

"Information about treatment involving an exposure to varying
temperatures; should include the temperature, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include different temperature regimens"

URI: [nmdc:air_temp_regm](https://microbiomedata/meta/air_temp_regm)

## Domain and Range

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-68}

-   is_a: [attribute](attribute.md)

## Children

## Used by

## Other properties

+---------------+---------------------------+
| **Aliases:**  | > air temperature regimen |
+---------------+---------------------------+
| **Mappings:** | > mixs:air_temp_regm      |
+---------------+---------------------------+

# Slot: al_sat

Aluminum saturation (esp. For tropical soils)

URI: [nmdc:al_sat](https://microbiomedata/meta/al_sat)

## Domain and Range

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-69}

-   is_a: [attribute](attribute.md)

## Children

## Used by

-   [Biosample](Biosample.md)

## Other properties

+---------------+--------------------------------------------+
| **Aliases:**  | > extreme_unusual_properties/Al saturation |
+---------------+--------------------------------------------+
| **Mappings:** | > mixs:al_sat                              |
+---------------+--------------------------------------------+

# Slot: al_sat_meth

Reference or method used in determining Al saturation

URI: [nmdc:al_sat_meth](https://microbiomedata/meta/al_sat_meth)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-70}

-   is_a: [attribute](attribute.md)

## Children

## Used by

-   [Biosample](Biosample.md)

## Other properties

+---------------+---------------------------------------------------+
| **Aliases:**  | > extreme_unusual_properties/Al saturation method |
+---------------+---------------------------------------------------+
| **Mappings:** | > mixs:al_sat_meth                                |
+---------------+---------------------------------------------------+

# Slot: alkalinity

"Alkalinity, the ability of a solution to neutralize acids to the
equivalence point of carbonate or bicarbonate"

URI: [nmdc:alkalinity](https://microbiomedata/meta/alkalinity)

## Domain and Range

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-71}

-   is_a: [attribute](attribute.md)

## Children

## Used by

-   [Biosample](Biosample.md)

## Other properties

+---------------+-------------------+
| **Aliases:**  | > alkalinity      |
+---------------+-------------------+
| **Mappings:** | > mixs:alkalinity |
+---------------+-------------------+

# Slot: alkalinity_method

Method used for alkalinity measurement

URI:
[nmdc:alkalinity_method](https://microbiomedata/meta/alkalinity_method)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-72}

-   is_a: [attribute](attribute.md)

## Children

## Used by

-   [Biosample](Biosample.md)

## Other properties

+---------------+--------------------------+
| **Aliases:**  | > alkalinity method      |
+---------------+--------------------------+
| **Mappings:** | > mixs:alkalinity_method |
+---------------+--------------------------+

# Slot: alkyl_diethers

Concentration of alkyl diethers

URI: [nmdc:alkyl_diethers](https://microbiomedata/meta/alkyl_diethers)

## Domain and Range

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-73}

-   is_a: [attribute](attribute.md)

## Children

## Used by

-   [Biosample](Biosample.md)

## Other properties

+---------------+-----------------------+
| **Aliases:**  | > alkyl diethers      |
+---------------+-----------------------+
| **Mappings:** | > mixs:alkyl_diethers |
+---------------+-----------------------+

# Slot: all proteins

the list of protein identifiers that are associated with the peptide
sequence

URI:
<https://microbiomedata/schema/workflow_execution_activity/all_proteins>

## Domain and Range

None → 0..\* [GeneProduct](GeneProduct.md)

## Parents {#parents-74}

## Children

-   [peptide quantification➞all
    proteins](peptide_quantification_all_proteins.md)
-   [protein quantification➞all
    proteins](protein_quantification_all_proteins.md)

## Used by

# Slot: alt

"Altitude is a term used to identify heights of objects such as
airplanes, space shuttles, rockets, atmospheric balloons and heights of
places such as atmospheric layers and clouds. It is used to measure the
height of an object which is above the earthbs surface. In this context,
the altitude measurement is the vertical distance between the earth's
surface above sea level and the sampled position in the air"

URI: [nmdc:alt](https://microbiomedata/meta/alt)

## Domain and Range

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-75}

-   is_a: [attribute](attribute.md)

## Children

## Used by

-   [Biosample](Biosample.md)

## Other properties

+-----------------+---------------+
| **Aliases:**    | > altitude    |
+-----------------+---------------+
| **Mappings:**   | > mixs:alt    |
+-----------------+---------------+
| **In Subsets:** | > environment |
+-----------------+---------------+

# Slot: alternate emails

One or more other email addresses for an entity.

URI:
<https://microbiomedata/schema/workflow_execution_activity/alternate_emails>

## Domain and Range

None → 0..1 [String](types/String.md)

## Parents {#parents-76}

## Children

## Used by

# Slot: alternative descriptions

A list of alternative descriptions for the entity. The distinction
between desciption and alternative descriptions is application-specific.

URI:
<https://microbiomedata/schema/workflow_execution_activity/alternative_descriptions>

## Domain and Range

None → 0..\* [String](types/String.md)

## Parents {#parents-77}

## Children

## Used by

# Slot: alternative identifiers

A list of alternative identifiers for the entity.

URI:
<https://microbiomedata/schema/workflow_execution_activity/alternative_identifiers>

## Domain and Range

None → 0..\* [String](types/String.md)

## Parents {#parents-78}

## Children

-   [external database identifiers](external_database_identifiers.md)

## Used by

-   [ChemicalEntity](ChemicalEntity.md)
-   [EnvironmentalMaterialTerm](EnvironmentalMaterialTerm.md)
-   [GeneProduct](GeneProduct.md)
-   [Instrument](Instrument.md)
-   [MetaboliteQuantification](MetaboliteQuantification.md)
-   [NamedThing](NamedThing.md)
-   [OntologyClass](OntologyClass.md)
-   [Person](Person.md)

# Slot: alternative names

A list of alternative names used to refer to the entity. The distinction
between name and alternative names is application-specific.

URI:
<https://microbiomedata/schema/workflow_execution_activity/alternative_names>

## Domain and Range

None → 0..\* [String](types/String.md)

## Parents {#parents-79}

## Children

## Used by

## Other properties

+---------------------+-----------------------+
| **Exact Mappings:** | > dcterms:alternative |
+---------------------+-----------------------+
|                     | > skos:altLabel       |
+---------------------+-----------------------+

# Slot: alternative titles

A list of alternative titles for the entity. The distinction between
title and alternative titles is application-specific.

URI:
<https://microbiomedata/schema/workflow_execution_activity/alternative_titles>

## Domain and Range

None → 0..\* [String](types/String.md)

## Parents {#parents-80}

## Children

## Used by

## Other properties

+----------------------------------+----------------------------------+
| **Exact Mappings:**              | dcterms:alternative              |
+==================================+==================================+
| Slot: aminopept_act              |                                  |
| ===================              |                                  |
+----------------------------------+----------------------------------+
| Measurement of amino             | peptidase activity               |
+----------------------------------+----------------------------------+
| URI: \`nmdc:aminopept            | \_act                            |
|                                  | \<<https://microbiome            |
|                                  | data/meta/aminopept_act>\>\`\_\_ |
+----------------------------------+----------------------------------+
| ## Domain                        | -44:                             |
| and Range {#domain-and-range-44} |                                  |
+----------------------------------+----------------------------------+
| None → 0..1 \`Quantit            | yValue                           |
|                                  | \<QuantityValue.md\>\`\_\_       |
| ## Parents {#parents-81}         |                                  |
+----------------------------------+----------------------------------+
| -   is_a: \`attribute            | \<attribute.md\>\`\_\_           |
|                                  |                                  |
| ## Children {#children-51}       |                                  |
|                                  |                                  |
| ## Used by {#used-by-44}         |                                  |
+----------------------------------+----------------------------------+
| -   \`Biosample \<Biosa          | mple.md\>\`\_\_                  |
+----------------------------------+----------------------------------+
| ## Other p                       | -69:                             |
| roperties {#other-properties-68} |                                  |
+----------------------------------+----------------------------------+
| ============= ======             | =================                |
+----------------------------------+----------------------------------+
| **Aliases:** amino               | peptidase activity               |
+----------------------------------+----------------------------------+
| **Mappings:** mixs:              | aminopept_act                    |
+----------------------------------+----------------------------------+

# Slot: ammonium

Concentration of ammonium in the sample

URI: [nmdc:ammonium](https://microbiomedata/meta/ammonium)

## Domain and Range {#domain-and-range-45}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-82}

-   is_a: [attribute](attribute.md)

## Children {#children-52}

## Used by {#used-by-45}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-70}

+---------------+-----------------+
| **Aliases:**  | > ammonium      |
+---------------+-----------------+
| **Mappings:** | > mixs:ammonium |
+---------------+-----------------+

# Slot: amniotic_fluid_color

Specification of the color of the amniotic fluid sample

URI:
[nmdc:amniotic_fluid_color](https://microbiomedata/meta/amniotic_fluid_color)

## Domain and Range {#domain-and-range-46}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-83}

-   is_a: [attribute](attribute.md)

## Children {#children-53}

## Used by {#used-by-46}

## Other properties {#other-properties-71}

+---------------+-----------------------------+
| **Aliases:**  | > amniotic fluid/color      |
+---------------+-----------------------------+
| **Mappings:** | > mixs:amniotic_fluid_color |
+---------------+-----------------------------+

# Slot: amount_light

"The unit of illuminance and luminous emittance, measuring luminous flux
per unit area"

URI: [nmdc:amount_light](https://microbiomedata/meta/amount_light)

## Domain and Range {#domain-and-range-47}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-84}

-   is_a: [attribute](attribute.md)

## Children {#children-54}

## Used by {#used-by-47}

## Other properties {#other-properties-72}

+---------------+---------------------+
| **Aliases:**  | > amount of light   |
+---------------+---------------------+
| **Mappings:** | > mixs:amount_light |
+---------------+---------------------+

# Slot: analysis identifiers

URI:
<https://microbiomedata/schema/workflow_execution_activity/analysis_identifiers>

## Domain and Range {#domain-and-range-48}

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents {#parents-85}

-   is_a: [external database
    identifiers](external_database_identifiers.md)

## Children {#children-55}

-   [GOLD analysis project
    identifiers](GOLD_analysis_project_identifiers.md)
-   [INSDC analysis identifiers](INSDC_analysis_identifiers.md)
-   [MGnify analysis identifiers](MGnify_analysis_identifiers.md)

## Used by {#used-by-48}

# Slot: ances_data

"Information about either pedigree or other ancestral information
description (e.g. parental variety in case of mutant or selection),
e.g. A/3\*B (meaning \[(A x B) x B\] x B)"

URI: [nmdc:ances_data](https://microbiomedata/meta/ances_data)

## Domain and Range {#domain-and-range-49}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-86}

-   is_a: [attribute](attribute.md)

## Children {#children-56}

## Used by {#used-by-49}

## Other properties {#other-properties-73}

+---------------+-------------------+
| **Aliases:**  | > ancestral data  |
+---------------+-------------------+
| **Mappings:** | > mixs:ances_data |
+---------------+-------------------+

# Slot: annot

"Tool used for annotation, or for cases where annotation was provided by
a community jamboree or model organism database rather than by a
specific submitter"

URI: [nmdc:annot](https://microbiomedata/meta/annot)

## Domain and Range {#domain-and-range-50}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-87}

-   is_a: [attribute](attribute.md)

## Children {#children-57}

## Used by {#used-by-50}

## Other properties {#other-properties-74}

+-----------------+--------------+
| **Aliases:**    | > annotation |
+-----------------+--------------+
| **Mappings:**   | > mixs:annot |
+-----------------+--------------+
| **In Subsets:** | > sequencing |
+-----------------+--------------+

# Slot: annual_precpt

"The average of all annual precipitation values known, or an estimated
equivalent value derived by such methods as regional indexes or
Isohyetal maps."

URI: [nmdc:annual_precpt](https://microbiomedata/meta/annual_precpt)

## Domain and Range {#domain-and-range-51}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-88}

-   is_a: [attribute](attribute.md)

## Children {#children-58}

## Used by {#used-by-51}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-75}

+---------------+-----------------------------+
| **Aliases:**  | > mean annual precipitation |
+---------------+-----------------------------+
| **Mappings:** | > mixs:annual_precpt        |
+---------------+-----------------------------+

# Slot: annual_temp

Mean annual temperature

URI: [nmdc:annual_temp](https://microbiomedata/meta/annual_temp)

## Domain and Range {#domain-and-range-52}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-89}

-   is_a: [attribute](attribute.md)

## Children {#children-59}

## Used by {#used-by-52}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-76}

+---------------+---------------------------+
| **Aliases:**  | > mean annual temperature |
+---------------+---------------------------+
| **Mappings:** | > mixs:annual_temp        |
+---------------+---------------------------+

# Slot: antibiotic_regm

"Information about treatment involving antibiotic administration; should
include the name of antibiotic, amount administered, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include multiple antibiotic regimens"

URI: [nmdc:antibiotic_regm](https://microbiomedata/meta/antibiotic_regm)

## Domain and Range {#domain-and-range-53}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-90}

-   is_a: [attribute](attribute.md)

## Children {#children-60}

## Used by {#used-by-53}

## Other properties {#other-properties-77}

+---------------+------------------------+
| **Aliases:**  | > antibiotic regimen   |
+---------------+------------------------+
| **Mappings:** | > mixs:antibiotic_regm |
+---------------+------------------------+

# Slot: api

API gravity is a measure of how heavy or light a petroleum liquid is
compared to water (source: <https://en.wikipedia.org/wiki/API_gravity>)
(e.g. 31.1B0 API)

URI: [nmdc:api](https://microbiomedata/meta/api)

## Domain and Range {#domain-and-range-54}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-91}

-   is_a: [attribute](attribute.md)

## Children {#children-61}

## Used by {#used-by-54}

## Other properties {#other-properties-78}

+---------------+---------------+
| **Aliases:**  | > API gravity |
+---------------+---------------+
| **Mappings:** | > mixs:api    |
+---------------+---------------+

# Slot: applied role

URI: [nmdc:applied_role](https://microbiomedata/meta/applied_role)

## Domain and Range {#domain-and-range-55}

[CreditAssociation](CreditAssociation.md) → 1..1 [credit
enum](credit%20enum.md)

## Parents {#parents-92}

## Children {#children-62}

## Used by {#used-by-55}

-   [CreditAssociation](CreditAssociation.md)

## Other properties {#other-properties-79}

  **Mappings:**   prov:hadRole
  --------------- --------------
                  
  Slot: applies   to person

URI:
[nmdc:applies_to_person](https://microbiomedata/meta/applies_to_person)

## Domain and Range {#domain-and-range-56}

[CreditAssociation](CreditAssociation.md) → 1..1
[PersonValue](PersonValue.md)

## Parents {#parents-93}

## Children {#children-63}

## Used by {#used-by-56}

-   [CreditAssociation](CreditAssociation.md)

## Other properties {#other-properties-80}

  **Mappings:**    prov:agent
  ---------------- ------------
                   
  Slot: arch_str   uc

"An architectural structure is a human-made, free-standing, immobile
outdoor construction"

URI: [nmdc:arch_struc](https://microbiomedata/meta/arch_struc)

## Domain and Range {#domain-and-range-57}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-94}

-   is_a: [attribute](attribute.md)

## Children {#children-64}

## Used by {#used-by-57}

## Other properties {#other-properties-81}

+---------------+---------------------------+
| **Aliases:**  | > architectural structure |
+---------------+---------------------------+
| **Mappings:** | > mixs:arch_struc         |
+---------------+---------------------------+

# Slot: aromatics_pc

"Saturate, Aromatic, Resin and AsphalteneB (SARA) is an analysis method
that dividesB crude oilB components according to their polarizability
and polarity. There are three main methods to obtain SARA results. The
most popular one is known as the Iatroscan TLC-FID and is referred to as
IP-143 (source:
<https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene>)"

URI: [nmdc:aromatics_pc](https://microbiomedata/meta/aromatics_pc)

## Domain and Range {#domain-and-range-58}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-95}

-   is_a: [attribute](attribute.md)

## Children {#children-65}

## Used by {#used-by-58}

## Other properties {#other-properties-82}

+---------------+---------------------+
| **Aliases:**  | > aromatics wt%     |
+---------------+---------------------+
| **Mappings:** | > mixs:aromatics_pc |
+---------------+---------------------+

# Slot: asm_score

A score for comparing metagenomic assembly quality from same sample.

URI:
<https://microbiomedata/schema/workflow_execution_activity/asm_score>

## Domain and Range {#domain-and-range-59}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-96}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-66}

## Used by {#used-by-59}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: asphaltenes_pc

"Saturate, Aromatic, Resin and AsphalteneB (SARA) is an analysis method
that dividesB crude oilB components according to their polarizability
and polarity. There are three main methods to obtain SARA results. The
most popular one is known as the Iatroscan TLC-FID and is referred to as
IP-143 (source:
<https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene>)"

URI: [nmdc:asphaltenes_pc](https://microbiomedata/meta/asphaltenes_pc)

## Domain and Range {#domain-and-range-60}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-97}

-   is_a: [attribute](attribute.md)

## Children {#children-67}

## Used by {#used-by-60}

## Other properties {#other-properties-83}

+---------------+-----------------------+
| **Aliases:**  | > asphaltenes wt%     |
+---------------+-----------------------+
| **Mappings:** | > mixs:asphaltenes_pc |
+---------------+-----------------------+

# Slot: assembly identifiers

URI:
<https://microbiomedata/schema/workflow_execution_activity/assembly_identifiers>

## Domain and Range {#domain-and-range-61}

None → 0..1 [String](types/String.md)

## Parents {#parents-98}

## Children {#children-68}

-   [INSDC assembly identifiers](INSDC_assembly_identifiers.md)

## Used by {#used-by-61}

# Slot: assembly_name

Name/version of the assembly provided by the submitter that is used in
the genome browsers and in the community

URI: [nmdc:assembly_name](https://microbiomedata/meta/assembly_name)

## Domain and Range {#domain-and-range-62}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-99}

-   is_a: [attribute](attribute.md)

## Children {#children-69}

## Used by {#used-by-62}

## Other properties {#other-properties-84}

+-----------------+----------------------+
| **Aliases:**    | > assembly name      |
+-----------------+----------------------+
| **Mappings:**   | > mixs:assembly_name |
+-----------------+----------------------+
| **In Subsets:** | > sequencing         |
+-----------------+----------------------+

# Slot: assembly_qual

"The assembly quality category is based on sets of criteria outlined for
each assembly quality category. For MISAG/MIMAG; Finished: Single,
validated, contiguous sequence per replicon without gaps or ambiguities
with a consensus error rate equivalent to Q50 or better. High Quality
Draft:Multiple fragments where gaps span repetitive regions. Presence of
the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality
Draft:Many fragments with little to no review of assembly other than
reporting of standard assembly statistics. Low Quality Draft:Many
fragments with little to no review of assembly other than reporting of
standard assembly statistics. Assembly statistics include, but are not
limited to total assembly size, number of contigs, contig N50/L50, and
maximum contig length. For MIUVIG; Finished: Single, validated,
contiguous sequence per replicon without gaps or ambiguities, with
extensive manual review and editing to annotate putative gene functions
and transcriptional units. High-quality draft genome: One or multiple
fragments, totaling 3 90% of the expected genome or replicon sequence or
predicted complete. Genome fragment(s): One or multiple fragments,
totalling \< 90% of the expected genome or replicon sequence, or for
which no genome size could be estimated"

URI: [nmdc:assembly_qual](https://microbiomedata/meta/assembly_qual)

## Domain and Range {#domain-and-range-63}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-100}

-   is_a: [attribute](attribute.md)

## Children {#children-70}

## Used by {#used-by-63}

## Other properties {#other-properties-85}

+-----------------+----------------------+
| **Aliases:**    | > assembly quality   |
+-----------------+----------------------+
| **Mappings:**   | > mixs:assembly_qual |
+-----------------+----------------------+
| **In Subsets:** | > sequencing         |
+-----------------+----------------------+

# Slot: assembly_software

"Tool(s) used for assembly, including version number and parameters"

URI:
[nmdc:assembly_software](https://microbiomedata/meta/assembly_software)

## Domain and Range {#domain-and-range-64}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-101}

-   is_a: [attribute](attribute.md)

## Children {#children-71}

## Used by {#used-by-64}

## Other properties {#other-properties-86}

+-----------------+--------------------------+
| **Aliases:**    | > assembly software      |
+-----------------+--------------------------+
| **Mappings:**   | > mixs:assembly_software |
+-----------------+--------------------------+
| **In Subsets:** | > sequencing             |
+-----------------+--------------------------+

# Slot: atmospheric_data

Measurement of atmospheric data; can include multiple data

URI:
[nmdc:atmospheric_data](https://microbiomedata/meta/atmospheric_data)

## Domain and Range {#domain-and-range-65}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-102}

-   is_a: [attribute](attribute.md)

## Children {#children-72}

## Used by {#used-by-65}

## Other properties {#other-properties-87}

+---------------+-------------------------+
| **Aliases:**  | > atmospheric data      |
+---------------+-------------------------+
| **Mappings:** | > mixs:atmospheric_data |
+---------------+-------------------------+

# Slot: attribute

A attribute of a biosample. Examples: depth, habitat, material. For
NMDC, attributes SHOULD be mapped to terms within a MIxS template

URI:
<https://microbiomedata/schema/workflow_execution_activity/attribute>

## Domain and Range {#domain-and-range-66}

None → 0..1 [String](types/String.md)

## Parents {#parents-103}

## Children {#children-73}

-   [display order](display_order.md)
-   [execution resource](execution_resource.md)
-   [file size bytes](file_size_bytes.md)
-   [git url](git_url.md)
-   [md5 checksum](md5_checksum.md)
-   [url](url.md)

## Used by {#used-by-66}

## Other properties {#other-properties-88}

+--------------+------------------+
| **Aliases:** | > field          |
+--------------+------------------+
|              | > property       |
+--------------+------------------+
|              | > template field |
+--------------+------------------+
|              | > key            |
+--------------+------------------+
|              | > characteristic |
+--------------+------------------+

# Slot: type

An optional string that specified the type of object.

URI:
<https://microbiomedata/schema/workflow_execution_activity/attribute_value_type>

## Domain and Range {#domain-and-range-67}

[AttributeValue](AttributeValue.md) → 0..1 [String](types/String.md)

## Parents {#parents-104}

-   is_a: [type](type.md)

## Children {#children-74}

## Used by {#used-by-67}

-   [AttributeValue](AttributeValue.md)
-   [BooleanValue](BooleanValue.md)
-   [ControlledTermValue](ControlledTermValue.md)
-   [GeolocationValue](GeolocationValue.md)
-   [ImageValue](ImageValue.md)
-   [IntegerValue](IntegerValue.md)
-   [PersonValue](PersonValue.md)
-   [QuantityValue](QuantityValue.md)
-   [TextValue](TextValue.md)
-   [TimestampValue](TimestampValue.md)
-   [UrlValue](UrlValue.md)

## Other properties {#other-properties-89}

+---------------+-----------------------------------------------------+
| **Examples:** | > Example(value='nmdc:Biosample', description=None) |
+---------------+-----------------------------------------------------+
|               | > Example(value='nmdc:Study', description=None)     |
+---------------+-----------------------------------------------------+

# Slot: avg_dew_point

The average of dew point measures taken at the beginning of every hour
over a 24 hour period on the sampling day

URI: [nmdc:avg_dew_point](https://microbiomedata/meta/avg_dew_point)

## Domain and Range {#domain-and-range-68}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-105}

-   is_a: [attribute](attribute.md)

## Children {#children-75}

## Used by {#used-by-68}

## Other properties {#other-properties-90}

+---------------+----------------------+
| **Aliases:**  | > average dew point  |
+---------------+----------------------+
| **Mappings:** | > mixs:avg_dew_point |
+---------------+----------------------+

# Slot: avg_occup

Daily average occupancy of room

URI: [nmdc:avg_occup](https://microbiomedata/meta/avg_occup)

## Domain and Range {#domain-and-range-69}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-106}

-   is_a: [attribute](attribute.md)

## Children {#children-76}

## Used by {#used-by-69}

## Other properties {#other-properties-91}

+---------------+---------------------------+
| **Aliases:**  | > average daily occupancy |
+---------------+---------------------------+
| **Mappings:** | > mixs:avg_occup          |
+---------------+---------------------------+

# Slot: avg_temp

The average of temperatures taken at the beginning of every hour over a
24 hour period on the sampling day

URI: [nmdc:avg_temp](https://microbiomedata/meta/avg_temp)

## Domain and Range {#domain-and-range-70}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-107}

-   is_a: [attribute](attribute.md)

## Children {#children-77}

## Used by {#used-by-70}

## Other properties {#other-properties-92}

+---------------+-----------------------+
| **Aliases:**  | > average temperature |
+---------------+-----------------------+
| **Mappings:** | > mixs:avg_temp       |
+---------------+-----------------------+

# Slot: bac_prod

Bacterial production in the water column measured by isotope uptake

URI: [nmdc:bac_prod](https://microbiomedata/meta/bac_prod)

## Domain and Range {#domain-and-range-71}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-108}

-   is_a: [attribute](attribute.md)

## Children {#children-78}

## Used by {#used-by-71}

## Other properties {#other-properties-93}

+---------------+------------------------+
| **Aliases:**  | > bacterial production |
+---------------+------------------------+
| **Mappings:** | > mixs:bac_prod        |
+---------------+------------------------+

# Slot: bac_resp

Measurement of bacterial respiration in the water column

URI: [nmdc:bac_resp](https://microbiomedata/meta/bac_resp)

## Domain and Range {#domain-and-range-72}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-109}

-   is_a: [attribute](attribute.md)

## Children {#children-79}

## Used by {#used-by-72}

## Other properties {#other-properties-94}

+---------------+-------------------------+
| **Aliases:**  | > bacterial respiration |
+---------------+-------------------------+
| **Mappings:** | > mixs:bac_resp         |
+---------------+-------------------------+

# Slot: bacteria_carb_prod

Measurement of bacterial carbon production

URI:
[nmdc:bacteria_carb_prod](https://microbiomedata/meta/bacteria_carb_prod)

## Domain and Range {#domain-and-range-73}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-110}

-   is_a: [attribute](attribute.md)

## Children {#children-80}

## Used by {#used-by-73}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-95}

+---------------+-------------------------------+
| **Aliases:**  | > bacterial carbon production |
+---------------+-------------------------------+
| **Mappings:** | > mixs:bacteria_carb_prod     |
+---------------+-------------------------------+

# Slot: barometric_press

Force per unit area exerted against a surface by the weight of air above
that surface

URI:
[nmdc:barometric_press](https://microbiomedata/meta/barometric_press)

## Domain and Range {#domain-and-range-74}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-111}

-   is_a: [attribute](attribute.md)

## Children {#children-81}

## Used by {#used-by-74}

## Other properties {#other-properties-96}

+---------------+-------------------------+
| **Aliases:**  | > barometric pressure   |
+---------------+-------------------------+
| **Mappings:** | > mixs:barometric_press |
+---------------+-------------------------+

# Slot: basin

Name of the basin (e.g. Campos)

URI: [nmdc:basin](https://microbiomedata/meta/basin)

## Domain and Range {#domain-and-range-75}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-112}

-   is_a: [attribute](attribute.md)

## Children {#children-82}

## Used by {#used-by-75}

## Other properties {#other-properties-97}

+---------------+--------------+
| **Aliases:**  | > basin name |
+---------------+--------------+
| **Mappings:** | > mixs:basin |
+---------------+--------------+

# Slot: bathroom_count

The number of bathrooms in the building

URI: [nmdc:bathroom_count](https://microbiomedata/meta/bathroom_count)

## Domain and Range {#domain-and-range-76}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-113}

-   is_a: [attribute](attribute.md)

## Children {#children-83}

## Used by {#used-by-76}

## Other properties {#other-properties-98}

+---------------+-----------------------+
| **Aliases:**  | > bathroom count      |
+---------------+-----------------------+
| **Mappings:** | > mixs:bathroom_count |
+---------------+-----------------------+

# Slot: bedroom_count

The number of bedrooms in the building

URI: [nmdc:bedroom_count](https://microbiomedata/meta/bedroom_count)

## Domain and Range {#domain-and-range-77}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-114}

-   is_a: [attribute](attribute.md)

## Children {#children-84}

## Used by {#used-by-77}

## Other properties {#other-properties-99}

+---------------+----------------------+
| **Aliases:**  | > bedroom count      |
+---------------+----------------------+
| **Mappings:** | > mixs:bedroom_count |
+---------------+----------------------+

# Slot: benzene

Concentration of benzene in the sample

URI: [nmdc:benzene](https://microbiomedata/meta/benzene)

## Domain and Range {#domain-and-range-78}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-115}

-   is_a: [attribute](attribute.md)

## Children {#children-85}

## Used by {#used-by-78}

## Other properties {#other-properties-100}

+---------------+----------------+
| **Aliases:**  | > benzene      |
+---------------+----------------+
| **Mappings:** | > mixs:benzene |
+---------------+----------------+

# Slot: best protein

the specific protein identifier most correctly associated with the
peptide sequence

URI:
<https://microbiomedata/schema/workflow_execution_activity/best_protein>

## Domain and Range {#domain-and-range-79}

None → 0..1 [GeneProduct](GeneProduct.md)

## Parents {#parents-116}

## Children {#children-86}

-   [peptide quantification➞best
    protein](peptide_quantification_best_protein.md)
-   [protein quantification➞best
    protein](protein_quantification_best_protein.md)

## Used by {#used-by-79}

# Slot: bin_param

The parameters that have been applied during the extraction of genomes
from metagenomic datasets

URI: [nmdc:bin_param](https://microbiomedata/meta/bin_param)

## Domain and Range {#domain-and-range-80}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-117}

-   is_a: [attribute](attribute.md)

## Children {#children-87}

## Used by {#used-by-80}

## Other properties {#other-properties-101}

+-----------------+----------------------+
| **Aliases:**    | > binning parameters |
+-----------------+----------------------+
| **Mappings:**   | > mixs:bin_param     |
+-----------------+----------------------+
| **In Subsets:** | > sequencing         |
+-----------------+----------------------+

# Slot: bin_software

Tool(s) used for the extraction of genomes from metagenomic datasets

URI: [nmdc:bin_software](https://microbiomedata/meta/bin_software)

## Domain and Range {#domain-and-range-81}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-118}

-   is_a: [attribute](attribute.md)

## Children {#children-88}

## Used by {#used-by-81}

## Other properties {#other-properties-102}

+-----------------+---------------------+
| **Aliases:**    | > binning software  |
+-----------------+---------------------+
| **Mappings:**   | > mixs:bin_software |
+-----------------+---------------------+
| **In Subsets:** | > sequencing        |
+-----------------+---------------------+

# Slot: binned contig num

URI:
<https://microbiomedata/schema/workflow_execution_activity/binned_contig_num>

## Domain and Range {#domain-and-range-82}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-119}

## Children {#children-89}

## Used by {#used-by-82}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)

# Slot: biochem_oxygen_dem

Amount of dissolved oxygen needed by aerobic biological organisms in a
body of water to break down organic material present in a given water
sample at certain temperature over a specific time period

URI:
[nmdc:biochem_oxygen_dem](https://microbiomedata/meta/biochem_oxygen_dem)

## Domain and Range {#domain-and-range-83}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-120}

-   is_a: [attribute](attribute.md)

## Children {#children-90}

## Used by {#used-by-83}

## Other properties {#other-properties-103}

+---------------+-----------------------------+
| **Aliases:**  | > biochemical oxygen demand |
+---------------+-----------------------------+
| **Mappings:** | > mixs:biochem_oxygen_dem   |
+---------------+-----------------------------+

# Slot: biocide

List of biocides (commercial name of product and supplier) and date of
administration

URI: [nmdc:biocide](https://microbiomedata/meta/biocide)

## Domain and Range {#domain-and-range-84}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-121}

-   is_a: [attribute](attribute.md)

## Children {#children-91}

## Used by {#used-by-84}

## Other properties {#other-properties-104}

+---------------+--------------------------+
| **Aliases:**  | > biocide administration |
+---------------+--------------------------+
| **Mappings:** | > mixs:biocide           |
+---------------+--------------------------+

# Slot: biocide_admin_method

"Method of biocide administration (dose, frequency, duration, time
elapsed between last biociding and sampling) (e.g. 150 mg/l; weekly; 4
hr; 3 days)"

URI:
[nmdc:biocide_admin_method](https://microbiomedata/meta/biocide_admin_method)

## Domain and Range {#domain-and-range-85}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-122}

-   is_a: [attribute](attribute.md)

## Children {#children-92}

## Used by {#used-by-85}

## Other properties {#other-properties-105}

+---------------+---------------------------------+
| **Aliases:**  | > biocide administration method |
+---------------+---------------------------------+
| **Mappings:** | > mixs:biocide_admin_method     |
+---------------+---------------------------------+

# Slot: biogas_retention_time

URI:
[nmdc:biogas_retention_time](https://microbiomedata/meta/biogas_retention_time)

## Domain and Range {#domain-and-range-86}

None → 0..1 [String](types/String.md)

## Parents {#parents-123}

## Children {#children-93}

## Used by {#used-by-86}

# Slot: biogas_temperature

URI:
[nmdc:biogas_temperature](https://microbiomedata/meta/biogas_temperature)

## Domain and Range {#domain-and-range-87}

None → 0..1 [String](types/String.md)

## Parents {#parents-124}

## Children {#children-94}

## Used by {#used-by-87}

# Slot: biol_stat

The level of genome modification

URI: [nmdc:biol_stat](https://microbiomedata/meta/biol_stat)

## Domain and Range {#domain-and-range-88}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-125}

-   is_a: [attribute](attribute.md)

## Children {#children-95}

## Used by {#used-by-88}

## Other properties {#other-properties-106}

+---------------+---------------------+
| **Aliases:**  | > biological status |
+---------------+---------------------+
| **Mappings:** | > mixs:biol_stat    |
+---------------+---------------------+

# Slot: biomass

"Amount of biomass; should include the name for the part of biomass
measured, e.g. Microbial, total. Can include multiple measurements"

URI: [nmdc:biomass](https://microbiomedata/meta/biomass)

## Domain and Range {#domain-and-range-89}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-126}

-   is_a: [attribute](attribute.md)

## Children {#children-96}

## Used by {#used-by-89}

## Other properties {#other-properties-107}

+---------------+----------------+
| **Aliases:**  | > biomass      |
+---------------+----------------+
| **Mappings:** | > mixs:biomass |
+---------------+----------------+

# Slot: env_broad_scale

"In this field, report which major environmental system your sample or
specimen came from. The systems identified should have a coarse spatial
grain, to provide the general environmental context of where the
sampling was done (e.g. were you in the desert or a rainforest?). We
recommend using subclasses of ENVOUs biome class:
<http://purl.obolibrary.org/obo/ENVO_00000428>. Format (one term):
termLabel \[termID\], Format (multiple terms): termLabel
\[termID\]termLabel \[termID\]. Example: Annotating a water sample from
the photic zone in middle of the Atlantic Ocean, consider: oceanic
epipelagic zone biome \[ENVO:01000033\]. Example: Annotating a sample
from the Amazon rainforest consider: tropical moist broadleaf forest
biome \[ENVO:01000228\]. If needed, request new terms on the ENVO
tracker, identified here:
<http://www.obofoundry.org/ontology/envo.html>"

URI:
[nmdc:biosample_env_broad_scale](https://microbiomedata/meta/biosample_env_broad_scale)

## Domain and Range {#domain-and-range-90}

[Biosample](Biosample.md) → 1..1
[ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-127}

-   is_a: [env_broad_scale](env_broad_scale.md)

## Children {#children-97}

## Used by {#used-by-90}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-108}

+-----------------+-------------------------------------+
| **Aliases:**    | > broad-scale environmental context |
+-----------------+-------------------------------------+
| **Mappings:**   | > mixs:env_broad_scale              |
+-----------------+-------------------------------------+
| **In Subsets:** | > environment                       |
+-----------------+-------------------------------------+

# Slot: env_local_scale

"In this field, report the entity or entities which are in your sample
or specimenUs local vicinity and which you believe have significant
causal influences on your sample or specimen. Please use terms that are
present in ENVO and which are of smaller spatial grain than your entry
for env_broad_scale. Format (one term): termLabel \[termID\]; Format
(multiple terms): termLabel \[termID\]termLabel \[termID\]. Example:
Annotating a pooled sample taken from various vegetation layers in a
forest consider: canopy \[ENVO:00000047\]litter layer
\[ENVO:01000338\]shrub layer \[ENVO:01000336\]. If needed, request new
terms on the ENVO tracker, identified here:
<http://www.obofoundry.org/ontology/envo.html>"

URI:
[nmdc:biosample_env_local_scale](https://microbiomedata/meta/biosample_env_local_scale)

## Domain and Range {#domain-and-range-91}

[Biosample](Biosample.md) → 1..1
[ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-128}

-   is_a: [env_local_scale](env_local_scale.md)

## Children {#children-98}

## Used by {#used-by-91}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-109}

+-----------------+-------------------------------+
| **Aliases:**    | > local environmental context |
+-----------------+-------------------------------+
| **Mappings:**   | > mixs:env_local_scale        |
+-----------------+-------------------------------+
| **In Subsets:** | > environment                 |
+-----------------+-------------------------------+

# Slot: env_medium

"In this field, report which environmental material or materials (pipe
separated) immediately surrounded your sample or specimen prior to
sampling, using one or more subclasses of ENVOUs environmental material
class: <http://purl.obolibrary.org/obo/ENVO_00010483>. Format (one
term): termLabel \[termID\]; Format (multiple terms): termLabel
\[termID\]termLabel \[termID\]. Example: Annotating a fish swimming in
the upper 100 m of the Atlantic Ocean, consider: ocean water
\[ENVO:00002151\]. Example: Annotating a duck on a pond consider: pond
water \[ENVO:00002228\]\|air ENVO_00002005. If needed, request new terms
on the ENVO tracker, identified here:
<http://www.obofoundry.org/ontology/envo.html>"

URI:
[nmdc:biosample_env_medium](https://microbiomedata/meta/biosample_env_medium)

## Domain and Range {#domain-and-range-92}

[Biosample](Biosample.md) → 1..1
[ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-129}

-   is_a: [env_medium](env_medium.md)

## Children {#children-99}

## Used by {#used-by-92}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-110}

+-----------------+------------------------+
| **Aliases:**    | > environmental medium |
+-----------------+------------------------+
| **Mappings:**   | > mixs:env_medium      |
+-----------------+------------------------+
| **In Subsets:** | > environment          |
+-----------------+------------------------+

# Slot: lat_lon

This is currently a required field but it's not clear if this should be
required for human hosts

URI:
[nmdc:biosample_lat_lon](https://microbiomedata/meta/biosample_lat_lon)

## Domain and Range {#domain-and-range-93}

[Biosample](Biosample.md) → 0..1 [GeolocationValue](GeolocationValue.md)

## Parents {#parents-130}

-   is_a: [lat_lon](lat_lon.md)

## Children {#children-100}

## Used by {#used-by-93}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-111}

+-----------------+------------------------------------------------+
| **Aliases:**    | > geographic location (latitude and longitude) |
+-----------------+------------------------------------------------+
| **Mappings:**   | > mixs:lat_lon                                 |
+-----------------+------------------------------------------------+
| **In Subsets:** | > environment                                  |
+-----------------+------------------------------------------------+

# Slot: has input

An input to a process.

URI:
[nmdc:biosample_processing_has_input](https://microbiomedata/meta/biosample_processing_has_input)

## Domain and Range {#domain-and-range-94}

[BiosampleProcessing](BiosampleProcessing.md) → 0..\*
[Biosample](Biosample.md)

## Parents {#parents-131}

-   is_a: [has input](has_input.md)

## Children {#children-101}

-   [omics processing➞has input](omics_processing_has_input.md)

## Used by {#used-by-94}

-   [BiosampleProcessing](BiosampleProcessing.md)

## Other properties {#other-properties-112}

  **Aliases:**    input
  --------------- --------
                  
  Slot: biosamp   le set

This property links a database object to the set of samples within it.

URI: [nmdc:biosample_set](https://microbiomedata/meta/biosample_set)

## Domain and Range {#domain-and-range-95}

[Database](Database.md) → 0..\* [Biosample](Biosample.md)

## Parents {#parents-132}

## Children {#children-102}

## Used by {#used-by-95}

-   [Database](Database.md)

# Slot: biotic_regm

"Information about treatment(s) involving use of biotic factors, such as
bacteria, viruses or fungi."

URI: [nmdc:biotic_regm](https://microbiomedata/meta/biotic_regm)

## Domain and Range {#domain-and-range-96}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-133}

-   is_a: [attribute](attribute.md)

## Children {#children-103}

## Used by {#used-by-96}

## Other properties {#other-properties-113}

+---------------+--------------------+
| **Aliases:**  | > biotic regimen   |
+---------------+--------------------+
| **Mappings:** | > mixs:biotic_regm |
+---------------+--------------------+

# Slot: biotic_relationship

"Description of relationship(s) between the subject organism and other
organism(s) it is associated with. E.g., parasite on species X;
mutualist with species Y. The target organism is the subject of the
relationship, and the other organism(s) is the object"

URI:
[nmdc:biotic_relationship](https://microbiomedata/meta/biotic_relationship)

## Domain and Range {#domain-and-range-97}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-134}

-   is_a: [attribute](attribute.md)

## Children {#children-104}

## Used by {#used-by-97}

## Other properties {#other-properties-114}

+-----------------+--------------------------------+
| **Aliases:**    | > observed biotic relationship |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:biotic_relationship     |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: birth_control

Specification of birth control medication used

URI: [nmdc:birth_control](https://microbiomedata/meta/birth_control)

## Domain and Range {#domain-and-range-98}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-135}

-   is_a: [attribute](attribute.md)

## Children {#children-105}

## Used by {#used-by-98}

## Other properties {#other-properties-115}

+---------------+----------------------+
| **Aliases:**  | > birth control      |
+---------------+----------------------+
| **Mappings:** | > mixs:birth_control |
+---------------+----------------------+

# Slot: bishomohopanol

Concentration of bishomohopanol

URI: [nmdc:bishomohopanol](https://microbiomedata/meta/bishomohopanol)

## Domain and Range {#domain-and-range-99}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-136}

-   is_a: [attribute](attribute.md)

## Children {#children-106}

## Used by {#used-by-99}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-116}

+---------------+-----------------------+
| **Aliases:**  | > bishomohopanol      |
+---------------+-----------------------+
| **Mappings:** | > mixs:bishomohopanol |
+---------------+-----------------------+

# Slot: blood_blood_disord

History of blood disorders; can include multiple disorders

URI:
[nmdc:blood_blood_disord](https://microbiomedata/meta/blood_blood_disord)

## Domain and Range {#domain-and-range-100}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-137}

-   is_a: [attribute](attribute.md)

## Children {#children-107}

## Used by {#used-by-100}

## Other properties {#other-properties-117}

+---------------+---------------------------+
| **Aliases:**  | > blood/blood disorder    |
+---------------+---------------------------+
| **Mappings:** | > mixs:blood_blood_disord |
+---------------+---------------------------+

# Slot: bromide

Concentration of bromide

URI: [nmdc:bromide](https://microbiomedata/meta/bromide)

## Domain and Range {#domain-and-range-101}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-138}

-   is_a: [attribute](attribute.md)

## Children {#children-108}

## Used by {#used-by-101}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-118}

+---------------+----------------+
| **Aliases:**  | > bromide      |
+---------------+----------------+
| **Mappings:** | > mixs:bromide |
+---------------+----------------+

# Slot: build_docs

"The building design, construction and operation documents"

URI: [nmdc:build_docs](https://microbiomedata/meta/build_docs)

## Domain and Range {#domain-and-range-102}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-139}

-   is_a: [attribute](attribute.md)

## Children {#children-109}

## Used by {#used-by-102}

## Other properties {#other-properties-119}

+---------------+-------------------------------------------------+
| **Aliases:**  | > design, construction, and operation documents |
+---------------+-------------------------------------------------+
| **Mappings:** | > mixs:build_docs                               |
+---------------+-------------------------------------------------+

# Slot: build_occup_type

The primary function for which a building or discrete part of a building
is intended to be used

URI:
[nmdc:build_occup_type](https://microbiomedata/meta/build_occup_type)

## Domain and Range {#domain-and-range-103}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-140}

-   is_a: [attribute](attribute.md)

## Children {#children-110}

## Used by {#used-by-103}

## Other properties {#other-properties-120}

+---------------+---------------------------+
| **Aliases:**  | > building occupancy type |
+---------------+---------------------------+
| **Mappings:** | > mixs:build_occup_type   |
+---------------+---------------------------+

# Slot: building_setting

A location (geography) where a building is set

URI:
[nmdc:building_setting](https://microbiomedata/meta/building_setting)

## Domain and Range {#domain-and-range-104}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-141}

-   is_a: [attribute](attribute.md)

## Children {#children-111}

## Used by {#used-by-104}

## Other properties {#other-properties-121}

+---------------+-------------------------+
| **Aliases:**  | > building setting      |
+---------------+-------------------------+
| **Mappings:** | > mixs:building_setting |
+---------------+-------------------------+

# Slot: built_struc_age

The age of the built structure since construction

URI: [nmdc:built_struc_age](https://microbiomedata/meta/built_struc_age)

## Domain and Range {#domain-and-range-105}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-142}

-   is_a: [attribute](attribute.md)

## Children {#children-112}

## Used by {#used-by-105}

## Other properties {#other-properties-122}

+---------------+------------------------+
| **Aliases:**  | > built structure age  |
+---------------+------------------------+
| **Mappings:** | > mixs:built_struc_age |
+---------------+------------------------+

# Slot: built_struc_set

The characterization of the location of the built structure as high or
low human density

URI: [nmdc:built_struc_set](https://microbiomedata/meta/built_struc_set)

## Domain and Range {#domain-and-range-106}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-143}

-   is_a: [attribute](attribute.md)

## Children {#children-113}

## Used by {#used-by-106}

## Other properties {#other-properties-123}

+---------------+---------------------------+
| **Aliases:**  | > built structure setting |
+---------------+---------------------------+
| **Mappings:** | > mixs:built_struc_set    |
+---------------+---------------------------+

# Slot: built_struc_type

A physical structure that is a body or assemblage of bodies in space to
form a system capable of supporting loads

URI:
[nmdc:built_struc_type](https://microbiomedata/meta/built_struc_type)

## Domain and Range {#domain-and-range-107}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-144}

-   is_a: [attribute](attribute.md)

## Children {#children-114}

## Used by {#used-by-107}

## Other properties {#other-properties-124}

+---------------+-------------------------+
| **Aliases:**  | > built structure type  |
+---------------+-------------------------+
| **Mappings:** | > mixs:built_struc_type |
+---------------+-------------------------+

# Slot: calcium

Concentration of calcium in the sample

URI: [nmdc:calcium](https://microbiomedata/meta/calcium)

## Domain and Range {#domain-and-range-108}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-145}

-   is_a: [attribute](attribute.md)

## Children {#children-115}

## Used by {#used-by-108}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-125}

+---------------+----------------+
| **Aliases:**  | > calcium      |
+---------------+----------------+
| **Mappings:** | > mixs:calcium |
+---------------+----------------+

# Slot: carb_dioxide

Carbon dioxide (gas) amount or concentration at the time of sampling

URI: [nmdc:carb_dioxide](https://microbiomedata/meta/carb_dioxide)

## Domain and Range {#domain-and-range-109}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-146}

-   is_a: [attribute](attribute.md)

## Children {#children-116}

## Used by {#used-by-109}

## Other properties {#other-properties-126}

+---------------+---------------------+
| **Aliases:**  | > carbon dioxide    |
+---------------+---------------------+
| **Mappings:** | > mixs:carb_dioxide |
+---------------+---------------------+

# Slot: carb_monoxide

Carbon monoxide (gas) amount or concentration at the time of sampling

URI: [nmdc:carb_monoxide](https://microbiomedata/meta/carb_monoxide)

## Domain and Range {#domain-and-range-110}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-147}

-   is_a: [attribute](attribute.md)

## Children {#children-117}

## Used by {#used-by-110}

## Other properties {#other-properties-127}

+---------------+----------------------+
| **Aliases:**  | > carbon monoxide    |
+---------------+----------------------+
| **Mappings:** | > mixs:carb_monoxide |
+---------------+----------------------+

# Slot: carb_nitro_ratio

Ratio of amount or concentrations of carbon to nitrogen

URI:
[nmdc:carb_nitro_ratio](https://microbiomedata/meta/carb_nitro_ratio)

## Domain and Range {#domain-and-range-111}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-148}

-   is_a: [attribute](attribute.md)

## Children {#children-118}

## Used by {#used-by-111}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-128}

+---------------+-------------------------+
| **Aliases:**  | > carbon/nitrogen ratio |
+---------------+-------------------------+
| **Mappings:** | > mixs:carb_nitro_ratio |
+---------------+-------------------------+

# Slot: ceil_area

The area of the ceiling space within the room

URI: [nmdc:ceil_area](https://microbiomedata/meta/ceil_area)

## Domain and Range {#domain-and-range-112}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-149}

-   is_a: [attribute](attribute.md)

## Children {#children-119}

## Used by {#used-by-112}

## Other properties {#other-properties-129}

+---------------+------------------+
| **Aliases:**  | > ceiling area   |
+---------------+------------------+
| **Mappings:** | > mixs:ceil_area |
+---------------+------------------+

# Slot: ceil_cond

The physical condition of the ceiling at the time of sampling; photos or
video preferred; use drawings to indicate location of damaged areas

URI: [nmdc:ceil_cond](https://microbiomedata/meta/ceil_cond)

## Domain and Range {#domain-and-range-113}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-150}

-   is_a: [attribute](attribute.md)

## Children {#children-120}

## Used by {#used-by-113}

## Other properties {#other-properties-130}

+---------------+---------------------+
| **Aliases:**  | > ceiling condition |
+---------------+---------------------+
| **Mappings:** | > mixs:ceil_cond    |
+---------------+---------------------+

# Slot: ceil_finish_mat

The type of material used to finish a ceiling

URI: [nmdc:ceil_finish_mat](https://microbiomedata/meta/ceil_finish_mat)

## Domain and Range {#domain-and-range-114}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-151}

-   is_a: [attribute](attribute.md)

## Children {#children-121}

## Used by {#used-by-114}

## Other properties {#other-properties-131}

+---------------+---------------------------+
| **Aliases:**  | > ceiling finish material |
+---------------+---------------------------+
| **Mappings:** | > mixs:ceil_finish_mat    |
+---------------+---------------------------+

# Slot: ceil_struc

The construction format of the ceiling

URI: [nmdc:ceil_struc](https://microbiomedata/meta/ceil_struc)

## Domain and Range {#domain-and-range-115}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-152}

-   is_a: [attribute](attribute.md)

## Children {#children-122}

## Used by {#used-by-115}

## Other properties {#other-properties-132}

+---------------+---------------------+
| **Aliases:**  | > ceiling structure |
+---------------+---------------------+
| **Mappings:** | > mixs:ceil_struc   |
+---------------+---------------------+

# Slot: ceil_texture

"The feel, appearance, or consistency of a ceiling surface"

URI: [nmdc:ceil_texture](https://microbiomedata/meta/ceil_texture)

## Domain and Range {#domain-and-range-116}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-153}

-   is_a: [attribute](attribute.md)

## Children {#children-123}

## Used by {#used-by-116}

## Other properties {#other-properties-133}

+---------------+---------------------+
| **Aliases:**  | > ceiling texture   |
+---------------+---------------------+
| **Mappings:** | > mixs:ceil_texture |
+---------------+---------------------+

# Slot: ceil_thermal_mass

The ability of the ceiling to provide inertia against temperature
fluctuations. Generally this means concrete that is exposed. A metal
deck that supports a concrete slab will act thermally as long as it is
exposed to room air flow

URI:
[nmdc:ceil_thermal_mass](https://microbiomedata/meta/ceil_thermal_mass)

## Domain and Range {#domain-and-range-117}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-154}

-   is_a: [attribute](attribute.md)

## Children {#children-124}

## Used by {#used-by-117}

## Other properties {#other-properties-134}

+---------------+--------------------------+
| **Aliases:**  | > ceiling thermal mass   |
+---------------+--------------------------+
| **Mappings:** | > mixs:ceil_thermal_mass |
+---------------+--------------------------+

# Slot: ceil_type

The type of ceiling according to the ceiling's appearance or
construction

URI: [nmdc:ceil_type](https://microbiomedata/meta/ceil_type)

## Domain and Range {#domain-and-range-118}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-155}

-   is_a: [attribute](attribute.md)

## Children {#children-125}

## Used by {#used-by-118}

## Other properties {#other-properties-135}

+---------------+------------------+
| **Aliases:**  | > ceiling type   |
+---------------+------------------+
| **Mappings:** | > mixs:ceil_type |
+---------------+------------------+

# Slot: ceil_water_mold

Signs of the presence of mold or mildew on the ceiling

URI: [nmdc:ceil_water_mold](https://microbiomedata/meta/ceil_water_mold)

## Domain and Range {#domain-and-range-119}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-156}

-   is_a: [attribute](attribute.md)

## Children {#children-126}

## Used by {#used-by-119}

## Other properties {#other-properties-136}

+---------------+-------------------------------+
| **Aliases:**  | > ceiling signs of water/mold |
+---------------+-------------------------------+
| **Mappings:** | > mixs:ceil_water_mold        |
+---------------+-------------------------------+

# Slot: chem_administration

"List of chemical compounds administered to the host or site where
sampling occurred, and when (e.g. Antibiotics, n fertilizer, air
filter); can include multiple compounds. For chemical entities of
biological interest ontology (chebi) (v 163),
<http://purl.bioontology.org/ontology/chebi>"

URI:
[nmdc:chem_administration](https://microbiomedata/meta/chem_administration)

## Domain and Range {#domain-and-range-120}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-157}

-   is_a: [attribute](attribute.md)

## Children {#children-127}

## Used by {#used-by-120}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-137}

+---------------+----------------------------+
| **Aliases:**  | > chemical administration  |
+---------------+----------------------------+
| **Mappings:** | > mixs:chem_administration |
+---------------+----------------------------+

# Slot: chem_mutagen

"Treatment involving use of mutagens; should include the name of
mutagen, amount administered, treatment regimen including how many times
the treatment was repeated, how long each treatment lasted, and the
start and end time of the entire treatment; can include multiple mutagen
regimens"

URI: [nmdc:chem_mutagen](https://microbiomedata/meta/chem_mutagen)

## Domain and Range {#domain-and-range-121}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-158}

-   is_a: [attribute](attribute.md)

## Children {#children-128}

## Used by {#used-by-121}

## Other properties {#other-properties-138}

+---------------+---------------------+
| **Aliases:**  | > chemical mutagen  |
+---------------+---------------------+
| **Mappings:** | > mixs:chem_mutagen |
+---------------+---------------------+

# Slot: chem_oxygen_dem

A measure of the capacity of water to consume oxygen during the
decomposition of organic matter and the oxidation of inorganic chemicals
such as ammonia and nitrite

URI: [nmdc:chem_oxygen_dem](https://microbiomedata/meta/chem_oxygen_dem)

## Domain and Range {#domain-and-range-122}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-159}

-   is_a: [attribute](attribute.md)

## Children {#children-129}

## Used by {#used-by-122}

## Other properties {#other-properties-139}

+---------------+--------------------------+
| **Aliases:**  | > chemical oxygen demand |
+---------------+--------------------------+
| **Mappings:** | > mixs:chem_oxygen_dem   |
+---------------+--------------------------+

# Slot: chem_treatment

"List of chemical compounds administered upstream the sampling location
where sampling occurred (e.g. Glycols, H2S scavenger, corrosion and
scale inhibitors, demulsifiers, and other production chemicals etc.).
The commercial name of the product and name of the supplier should be
provided. The date of administration should also be included"

URI: [nmdc:chem_treatment](https://microbiomedata/meta/chem_treatment)

## Domain and Range {#domain-and-range-123}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-160}

-   is_a: [attribute](attribute.md)

## Children {#children-130}

## Used by {#used-by-123}

## Other properties {#other-properties-140}

+---------------+-----------------------+
| **Aliases:**  | > chemical treatment  |
+---------------+-----------------------+
| **Mappings:** | > mixs:chem_treatment |
+---------------+-----------------------+

# Slot: chem_treatment_method

"Method of chemical administration(dose, frequency, duration, time
elapsed between administration and sampling) (e.g. 50 mg/l; twice a
week; 1 hr; 0 days)"

URI:
[nmdc:chem_treatment_method](https://microbiomedata/meta/chem_treatment_method)

## Domain and Range {#domain-and-range-124}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-161}

-   is_a: [attribute](attribute.md)

## Children {#children-131}

## Used by {#used-by-124}

## Other properties {#other-properties-141}

+---------------+------------------------------+
| **Aliases:**  | > chemical treatment method  |
+---------------+------------------------------+
| **Mappings:** | > mixs:chem_treatment_method |
+---------------+------------------------------+

# Slot: chemical

URI: [nmdc:chemical](https://microbiomedata/meta/chemical)

## Domain and Range {#domain-and-range-125}

None → 0..1 [ChemicalEntity](ChemicalEntity.md)

## Parents {#parents-162}

## Children {#children-132}

-   [reaction participant➞chemical](reaction_participant_chemical.md)

## Used by {#used-by-125}

# Slot: chemical formula

A generic grouping for miolecular formulae and empirican formulae

URI:
<https://microbiomedata/schema/workflow_execution_activity/chemical_entity_chemical_formula>

## Domain and Range {#domain-and-range-126}

[ChemicalEntity](ChemicalEntity.md) → 0..1 [String](types/String.md)

## Parents {#parents-163}

-   is_a: [chemical formula](chemical_formula.md)

## Children {#children-133}

## Used by {#used-by-126}

-   [ChemicalEntity](ChemicalEntity.md)

# Slot: inchi

URI:
<https://microbiomedata/schema/workflow_execution_activity/chemical_entity_inchi>

## Domain and Range {#domain-and-range-127}

[ChemicalEntity](ChemicalEntity.md) → 0..1 [String](types/String.md)

## Parents {#parents-164}

-   is_a: [inchi](inchi.md)

## Children {#children-134}

## Used by {#used-by-127}

-   [ChemicalEntity](ChemicalEntity.md)

# Slot: inchi key

URI:
<https://microbiomedata/schema/workflow_execution_activity/chemical_entity_inchi_key>

## Domain and Range {#domain-and-range-128}

[ChemicalEntity](ChemicalEntity.md) → 0..1 [String](types/String.md)

## Parents {#parents-165}

-   is_a: [inchi key](inchi_key.md)

## Children {#children-135}

## Used by {#used-by-128}

-   [ChemicalEntity](ChemicalEntity.md)

# Slot: smiles

A string encoding of a molecular graph, no chiral or isotopic
information. There are usually a large number of valid SMILES which
represent a given structure. For example, CCO, OCC and C(O)C all specify
the structure of ethanol.

URI:
<https://microbiomedata/schema/workflow_execution_activity/chemical_entity_smiles>

## Domain and Range {#domain-and-range-129}

[ChemicalEntity](ChemicalEntity.md) → 0..\* [String](types/String.md)

## Parents {#parents-166}

-   is_a: [smiles](smiles.md)

## Children {#children-136}

## Used by {#used-by-129}

-   [ChemicalEntity](ChemicalEntity.md)

# Slot: chemical formula

A generic grouping for miolecular formulae and empirican formulae

URI:
<https://microbiomedata/schema/workflow_execution_activity/chemical_formula>

## Domain and Range {#domain-and-range-130}

None → 0..1 [String](types/String.md)

## Parents {#parents-167}

## Children {#children-137}

-   [chemical entity➞chemical
    formula](chemical_entity_chemical_formula.md)

## Used by {#used-by-130}

# Slot: chimera_check

"A chimeric sequence, or chimera for short, is a sequence comprised of
two or more phylogenetically distinct parent sequences. Chimeras are
usually PCR artifacts thought to occur when a prematurely terminated
amplicon reanneals to a foreign DNA strand and is copied to completion
in the following PCR cycles. The point at which the chimeric sequence
changes from one parent to the next is called the breakpoint or
conversion point"

URI: [nmdc:chimera_check](https://microbiomedata/meta/chimera_check)

## Domain and Range {#domain-and-range-131}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-168}

-   is_a: [attribute](attribute.md)

## Children {#children-138}

## Used by {#used-by-131}

-   [OmicsProcessing](OmicsProcessing.md)

## Other properties {#other-properties-142}

+-----------------+----------------------+
| **Aliases:**    | > chimera check      |
+-----------------+----------------------+
| **Mappings:**   | > mixs:chimera_check |
+-----------------+----------------------+
| **In Subsets:** | > sequencing         |
+-----------------+----------------------+

# Slot: chloride

Concentration of chloride in the sample

URI: [nmdc:chloride](https://microbiomedata/meta/chloride)

## Domain and Range {#domain-and-range-132}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-169}

-   is_a: [attribute](attribute.md)

## Children {#children-139}

## Used by {#used-by-132}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-143}

+---------------+-----------------+
| **Aliases:**  | > chloride      |
+---------------+-----------------+
| **Mappings:** | > mixs:chloride |
+---------------+-----------------+

# Slot: chlorophyll

Concentration of chlorophyll

URI: [nmdc:chlorophyll](https://microbiomedata/meta/chlorophyll)

## Domain and Range {#domain-and-range-133}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-170}

-   is_a: [attribute](attribute.md)

## Children {#children-140}

## Used by {#used-by-133}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-144}

+---------------+--------------------+
| **Aliases:**  | > chlorophyll      |
+---------------+--------------------+
| **Mappings:** | > mixs:chlorophyll |
+---------------+--------------------+

# Slot: climate_environment

"Treatment involving an exposure to a particular climate; treatment
regimen including how many times the treatment was repeated, how long
each treatment lasted, and the start and end time of the entire
treatment; can include multiple climates"

URI:
[nmdc:climate_environment](https://microbiomedata/meta/climate_environment)

## Domain and Range {#domain-and-range-134}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-171}

-   is_a: [attribute](attribute.md)

## Children {#children-141}

## Used by {#used-by-134}

## Other properties {#other-properties-145}

+---------------+----------------------------+
| **Aliases:**  | > climate environment      |
+---------------+----------------------------+
| **Mappings:** | > mixs:climate_environment |
+---------------+----------------------------+

# Slot: collection_date

"The time of sampling, either as an instance (single point in time) or
interval. In case no exact time is available, the date/time can be right
truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00;
2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008
all are ISO8601 compliant"

URI: [nmdc:collection_date](https://microbiomedata/meta/collection_date)

## Domain and Range {#domain-and-range-135}

None → 0..1 [TimestampValue](TimestampValue.md)

## Parents {#parents-172}

-   is_a: [attribute](attribute.md)

## Children {#children-142}

## Used by {#used-by-135}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-146}

+-----------------+------------------------+
| **Aliases:**    | > collection date      |
+-----------------+------------------------+
| **Mappings:**   | > mixs:collection_date |
+-----------------+------------------------+
| **In Subsets:** | > environment          |
+-----------------+------------------------+

# Slot: community

URI: [nmdc:community](https://microbiomedata/meta/community)

## Domain and Range {#domain-and-range-136}

None → 0..1 [String](types/String.md)

## Parents {#parents-173}

## Children {#children-143}

## Used by {#used-by-136}

-   [Biosample](Biosample.md)

# Slot: compl_appr

"The approach used to determine the completeness of a given SAG or MAG,
which would typically make use of a set of conserved marker genes or a
closely related reference genome. For UViG completeness, include
reference genome or group used, and contig feature suggesting a complete
genome"

URI: [nmdc:compl_appr](https://microbiomedata/meta/compl_appr)

## Domain and Range {#domain-and-range-137}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-174}

-   is_a: [attribute](attribute.md)

## Children {#children-144}

## Used by {#used-by-137}

## Other properties {#other-properties-147}

+-----------------+-------------------------+
| **Aliases:**    | > completeness approach |
+-----------------+-------------------------+
| **Mappings:**   | > mixs:compl_appr       |
+-----------------+-------------------------+
| **In Subsets:** | > sequencing            |
+-----------------+-------------------------+

# Slot: compl_score

"Completeness score is typically based on either the fraction of markers
found as compared to a database or the percent of a genome found as
compared to a closely related reference genome. High Quality Draft:
\>90%, Medium Quality Draft: \>50%, and Low Quality Draft: \< 50% should
have the indicated completeness scores"

URI: [nmdc:compl_score](https://microbiomedata/meta/compl_score)

## Domain and Range {#domain-and-range-138}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-175}

-   is_a: [attribute](attribute.md)

## Children {#children-145}

## Used by {#used-by-138}

## Other properties {#other-properties-148}

+-----------------+----------------------+
| **Aliases:**    | > completeness score |
+-----------------+----------------------+
| **Mappings:**   | > mixs:compl_score   |
+-----------------+----------------------+
| **In Subsets:** | > sequencing         |
+-----------------+----------------------+

# Slot: compl_software

"Tools used for completion estimate, i.e. checkm, anvi'o, busco"

URI: [nmdc:compl_software](https://microbiomedata/meta/compl_software)

## Domain and Range {#domain-and-range-139}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-176}

-   is_a: [attribute](attribute.md)

## Children {#children-146}

## Used by {#used-by-139}

## Other properties {#other-properties-149}

+-----------------+-------------------------+
| **Aliases:**    | > completeness software |
+-----------------+-------------------------+
| **Mappings:**   | > mixs:compl_software   |
+-----------------+-------------------------+
| **In Subsets:** | > sequencing            |
+-----------------+-------------------------+

# Slot: completion_date

URI: [nmdc:completion_date](https://microbiomedata/meta/completion_date)

## Domain and Range {#domain-and-range-140}

None → 0..1 [String](types/String.md)

## Parents {#parents-177}

## Children {#children-147}

## Used by {#used-by-140}

# Slot: compression type

If provided, specifies the compression type

URI:
[nmdc:compression_type](https://microbiomedata/meta/compression_type)

## Domain and Range {#domain-and-range-141}

None → 0..1 [String](types/String.md)

## Parents {#parents-178}

## Children {#children-148}

## Used by {#used-by-141}

-   [DataObject](DataObject.md)

## Other properties {#other-properties-150}

+----------------------------------+----------------------------------+
| **Examples:**                    | Example(value='gzip',            |
|                                  | description=None)                |
+==================================+==================================+
| Slot: conduc ============        |                                  |
+----------------------------------+----------------------------------+
| Electrical con                   | ductivity of water               |
+----------------------------------+----------------------------------+
| URI: \`nmdc:con                  | duc                              |
|                                  | \<<https://mic                   |
|                                  | robiomedata/meta/conduc>\>\`\_\_ |
+----------------------------------+----------------------------------+
|                                  | -range-142:                      |
+----------------------------------+----------------------------------+
| Domain and Ran                   | ge                               |
+----------------------------------+----------------------------------+
| ------------------------------   | \--                              |
+----------------------------------+----------------------------------+
| None → 0..1 \`Q                  | uantityValue                     |
|                                  | \<QuantityValue.md\>\`\_\_       |
+----------------------------------+----------------------------------+
| ## Parents {#parents-179}        | 9:                               |
+----------------------------------+----------------------------------+
| -   is_a: \`attr                 | ibute \<attribute.md\>\`\_\_     |
+----------------------------------+----------------------------------+
| ## Children {#children-149}      | 49:                              |
+----------------------------------+----------------------------------+
| ## Used by {#used-by-142}        | 2:                               |
+----------------------------------+----------------------------------+
|                                  | erties-151:                      |
+----------------------------------+----------------------------------+
| Other properti                   | es                               |
+----------------------------------+----------------------------------+
| ------------------------------   | \--                              |
+----------------------------------+----------------------------------+
| =============                    | ============                     |
+----------------------------------+----------------------------------+
| **Aliases:**                     | > conductivity                   |
+----------------------------------+----------------------------------+
| **Mappings:**                    | > mixs:conduc                    |
+----------------------------------+----------------------------------+

# Slot: contam_score

"The contamination score is based on the fraction of single-copy genes
that are observed more than once in a query genome. The following scores
are acceptable for; High Quality Draft: \< 5%, Medium Quality Draft: \<
10%, Low Quality Draft: \< 10%. Contamination must be below 5% for a SAG
or MAG to be deposited into any of the public databases"

URI: [nmdc:contam_score](https://microbiomedata/meta/contam_score)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-180}

-   is_a: [attribute](attribute.md)

## Children {#children-150}

## Used by {#used-by-143}

## Other properties

+-----------------+-----------------------+
| **Aliases:**    | > contamination score |
+-----------------+-----------------------+
| **Mappings:**   | > mixs:contam_score   |
+-----------------+-----------------------+
| **In Subsets:** | > sequencing          |
+-----------------+-----------------------+

# Slot: contam_screen_input

The type of sequence data used as input

URI:
[nmdc:contam_screen_input](https://microbiomedata/meta/contam_screen_input)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-181}

-   is_a: [attribute](attribute.md)

## Children {#children-151}

## Used by {#used-by-144}

## Other properties

+-----------------+---------------------------------+
| **Aliases:**    | > contamination screening input |
+-----------------+---------------------------------+
| **Mappings:**   | > mixs:contam_screen_input      |
+-----------------+---------------------------------+
| **In Subsets:** | > sequencing                    |
+-----------------+---------------------------------+

# Slot: contam_screen_param

"Specific parameters used in the decontamination sofware, such as
reference database, coverage, and kmers. Combinations of these
parameters may also be used, i.e. kmer and coverage, or reference
database and kmer"

URI:
[nmdc:contam_screen_param](https://microbiomedata/meta/contam_screen_param)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-182}

-   is_a: [attribute](attribute.md)

## Children {#children-152}

## Used by {#used-by-145}

## Other properties

+-----------------+--------------------------------------+
| **Aliases:**    | > contamination screening parameters |
+-----------------+--------------------------------------+
| **Mappings:**   | > mixs:contam_screen_param           |
+-----------------+--------------------------------------+
| **In Subsets:** | > sequencing                         |
+-----------------+--------------------------------------+

# Slot: contig_bp

Total size in bp of all contigs.

URI:
<https://microbiomedata/schema/workflow_execution_activity/contig_bp>

## Domain and Range

None → 0..1 [Float](types/Float.md)

## Parents {#parents-183}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-153}

## Used by {#used-by-146}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: contigs

The sum of the (length\*log(length)) of all contigs, times some
constant. Increase the contiguity, the score will increase

URI: <https://microbiomedata/schema/workflow_execution_activity/contigs>

## Domain and Range

None → 0..1 [Float](types/Float.md)

## Parents {#parents-184}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-154}

## Used by {#used-by-147}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: cool_syst_id

The cooling system identifier

URI: [nmdc:cool_syst_id](https://microbiomedata/meta/cool_syst_id)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-185}

-   is_a: [attribute](attribute.md)

## Children {#children-155}

## Used by {#used-by-148}

## Other properties

+---------------+-----------------------------+
| **Aliases:**  | > cooling system identifier |
+---------------+-----------------------------+
| **Mappings:** | > mixs:cool_syst_id         |
+---------------+-----------------------------+

# Enum: credit enum

URI: [nmdc:credit_enum](https://microbiomedata/meta/credit_enum)

## Other properties

## Permissible Values

  -------------------------------------------------------------------
  Text              Description       Meaning       Other Information
  ----------------- ----------------- ------------- -----------------
  Co                Co                              
  nceptualization   nceptualization                 

  Data curation     Data curation                   

  Formal Analysis   Formal Analysis                 

  Funding           Funding                         
  acquisition       acquisition                     

  Investigation     Investigation                   

  Methodology       Methodology                     

  Project           Project                         
  administration    administration                  

  Resources         Resources                       

  Software          Software                        

  Supervision       Supervision                     

  Validation        Validation                      

  Visualization     Visualization                   

  Writing original  Writing --                      
  draft             original draft                  

  Writing review    Writing -- review               
  and editing       & editing                       

  Principal         principal         OBI:0000103   
  Investigator      investigator role               
  -------------------------------------------------------------------

# Slot: crop_rotation

"Whether or not crop is rotated, and if yes, rotation schedule"

URI: [nmdc:crop_rotation](https://microbiomedata/meta/crop_rotation)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-186}

-   is_a: [attribute](attribute.md)

## Children {#children-156}

## Used by {#used-by-149}

-   [Biosample](Biosample.md)

## Other properties

+---------------+-------------------------+
| **Aliases:**  | > history/crop rotation |
+---------------+-------------------------+
| **Mappings:** | > mixs:crop_rotation    |
+---------------+-------------------------+

# Slot: ctg_L50

Given a set of contigs, the L50 is defined as the sequence length of the
shortest contig at 50% of the total genome length.

URI: <https://microbiomedata/schema/workflow_execution_activity/ctg_L50>

## Domain and Range

None → 0..1 [Float](types/Float.md)

## Parents {#parents-187}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-157}

## Used by {#used-by-150}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: ctg_L90

The L90 statistic is less than or equal to the L50 statistic; it is the
length for which the collection of all contigs of that length or longer
contains at least 90% of the sum of the lengths of all contigs.

URI: <https://microbiomedata/schema/workflow_execution_activity/ctg_L90>

## Domain and Range

None → 0..1 [Float](types/Float.md)

## Parents {#parents-188}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-158}

## Used by {#used-by-151}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: ctg_N50

Given a set of contigs, each with its own length, the N50 count is
defined as the smallest number of contigs whose length sum makes up half
of genome size.

URI: <https://microbiomedata/schema/workflow_execution_activity/ctg_N50>

## Domain and Range

None → 0..1 [Float](types/Float.md)

## Parents {#parents-189}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-159}

## Used by {#used-by-152}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: ctg_N90

Given a set of contigs, each with its own length, the N90 count is
defined as the smallest number of contigs whose length sum makes up 90%
of genome size.

URI: <https://microbiomedata/schema/workflow_execution_activity/ctg_N90>

## Domain and Range

None → 0..1 [Float](types/Float.md)

## Parents {#parents-190}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-160}

## Used by {#used-by-153}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: ctg_logsum

Maximum contig length.

URI:
<https://microbiomedata/schema/workflow_execution_activity/ctg_logsum>

## Domain and Range

None → 0..1 [Float](types/Float.md)

## Parents {#parents-191}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-161}

## Used by {#used-by-154}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: ctg_max

Maximum contig length.

URI: <https://microbiomedata/schema/workflow_execution_activity/ctg_max>

## Domain and Range

None → 0..1 [Float](types/Float.md)

## Parents {#parents-192}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-162}

## Used by {#used-by-155}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: ctg_powsum

Powersum of all contigs is the same as logsum except that it uses the
sum of (length\*(length\^P)) for some power P (default P=0.25).

URI:
<https://microbiomedata/schema/workflow_execution_activity/ctg_powsum>

## Domain and Range

None → 0..1 [Float](types/Float.md)

## Parents {#parents-193}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-163}

## Used by {#used-by-156}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: cult_root_med

"Name or reference for the hydroponic or in vitro culture rooting
medium; can be the name of a commonly used medium or reference to a
specific medium, e.g. Murashige and Skoog medium. If the medium has not
been formally published, use the rooting medium descriptors."

URI: [nmdc:cult_root_med](https://microbiomedata/meta/cult_root_med)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-194}

-   is_a: [attribute](attribute.md)

## Children {#children-164}

## Used by {#used-by-157}

## Other properties

+---------------+--------------------------+
| **Aliases:**  | > culture rooting medium |
+---------------+--------------------------+
| **Mappings:** | > mixs:cult_root_med     |
+---------------+--------------------------+

# Slot: cur_land_use

Present state of sample site

URI: [nmdc:cur_land_use](https://microbiomedata/meta/cur_land_use)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-195}

-   is_a: [attribute](attribute.md)

## Children {#children-165}

## Used by {#used-by-158}

-   [Biosample](Biosample.md)

## Other properties

+---------------+---------------------+
| **Aliases:**  | > current land use  |
+---------------+---------------------+
| **Mappings:** | > mixs:cur_land_use |
+---------------+---------------------+

# Slot: cur_vegetation

"Vegetation classification from one or more standard classification
systems, or agricultural crop"

URI: [nmdc:cur_vegetation](https://microbiomedata/meta/cur_vegetation)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-196}

-   is_a: [attribute](attribute.md)

## Children {#children-166}

## Used by {#used-by-159}

-   [Biosample](Biosample.md)

## Other properties

+---------------+-----------------------+
| **Aliases:**  | > current vegetation  |
+---------------+-----------------------+
| **Mappings:** | > mixs:cur_vegetation |
+---------------+-----------------------+

# Slot: cur_vegetation_meth

Reference or method used in vegetation classification

URI:
[nmdc:cur_vegetation_meth](https://microbiomedata/meta/cur_vegetation_meth)

## Domain and Range

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-197}

-   is_a: [attribute](attribute.md)

## Children {#children-167}

## Used by {#used-by-160}

-   [Biosample](Biosample.md)

## Other properties

+---------------+-----------------------------+
| **Aliases:**  | > current vegetation method |
+---------------+-----------------------------+
| **Mappings:** | > mixs:cur_vegetation_meth  |
+---------------+-----------------------------+

# Slot: description

a human-readable description of a thing

URI:
[nmdc:data_object_description](https://microbiomedata/meta/data_object_description)

## Domain and Range

[DataObject](DataObject.md) → 1..1 [String](types/String.md)

## Parents {#parents-198}

-   is_a: [description](description.md)

## Children {#children-168}

## Used by {#used-by-161}

-   [DataObject](DataObject.md)

## Other properties

+----------------------------------+----------------------------------+
| **Mappings:**                    | dcterms:description              |
+==================================+==================================+
| Slot: name ==========            |                                  |
+----------------------------------+----------------------------------+
| A human readab                   | le label for an entity           |
|                                  |                                  |
| URI:                             |                                  |
+----------------------------------+----------------------------------+
| \`nmdc:data_obj                  | ect_name                         |
|                                  | \<<https://microbiomedat         |
|                                  | a/meta/data_object_name>\>\`\_\_ |
+----------------------------------+----------------------------------+
|                                  | -range-162:                      |
+----------------------------------+----------------------------------+
| Domain and Ran                   | ge                               |
+----------------------------------+----------------------------------+
| ------------------------------   | \--                              |
+----------------------------------+----------------------------------+
| \`DataObject \<D                 | ataObject.md\>[\_\_ → 1..1       |
|                                  | \`String                         |
|                                  | \<ty                             |
|                                  | pes/String.md\>]{.title-ref}\_\_ |
+----------------------------------+----------------------------------+
| ## Parents {#parents-199}        | 9:                               |
+----------------------------------+----------------------------------+
| -   is_a: \`name                 | > \<name.md\>\`\_\_              |
+----------------------------------+----------------------------------+
| ## Children {#children-169}      | 69:                              |
+----------------------------------+----------------------------------+
| ## Used by {#used-by-162}        | 2:                               |
+----------------------------------+----------------------------------+
| -   \`DataObject                 | > \<DataObject.md\>\`\_\_        |
+----------------------------------+----------------------------------+
| Slot: data obj                   | ect set                          |
+----------------------------------+----------------------------------+

This property links a database object to the set of data objects within
it.

URI: [nmdc:data_object_set](https://microbiomedata/meta/data_object_set)

## Domain and Range {#domain-and-range-163}

[Database](Database.md) → 0..\* [DataObject](DataObject.md)

## Parents {#parents-200}

## Children {#children-170}

## Used by {#used-by-163}

-   [Database](Database.md)

# Slot: data object type

The type of file represented by the data object.

URI:
[nmdc:data_object_type](https://microbiomedata/meta/data_object_type)

## Domain and Range {#domain-and-range-164}

None → 0..1 [file type enum](file%20type%20enum.md)

## Parents {#parents-201}

## Children {#children-171}

## Used by {#used-by-164}

-   [DataObject](DataObject.md)

## Other properties

  --------------- --- ------------------------------------------------
  **Examples:**       Example(value='FT ICR-MS Analysis Results',
                      description=None)

                      Example(value='GC-MS Metabolomics Results',
                      description=None)
  --------------- --- ------------------------------------------------

# Slot: date created

TODO

URI:
[nmdc:database_date_created](https://microbiomedata/meta/database_date_created)

## Domain and Range {#domain-and-range-165}

[Database](Database.md) → 0..1 [String](types/String.md)

## Parents {#parents-202}

-   is_a: [date created](date_created.md)

## Children {#children-172}

## Used by {#used-by-165}

-   [Database](Database.md)

# Slot: etl software version

TODO

URI:
[nmdc:database_etl_software_version](https://microbiomedata/meta/database_etl_software_version)

## Domain and Range {#domain-and-range-166}

[Database](Database.md) → 0..1 [String](types/String.md)

## Parents {#parents-203}

-   is_a: [etl software version](etl_software_version.md)

## Children {#children-173}

## Used by {#used-by-166}

-   [Database](Database.md)

# Slot: nmdc schema version

TODO

URI:
[nmdc:database_nmdc_schema_version](https://microbiomedata/meta/database_nmdc_schema_version)

## Domain and Range {#domain-and-range-167}

[Database](Database.md) → 0..1 [String](types/String.md)

## Parents {#parents-204}

-   is_a: [nmdc schema version](nmdc_schema_version.md)

## Children {#children-174}

## Used by {#used-by-167}

-   [Database](Database.md)

# Slot: date created

TODO

URI: [nmdc:date_created](https://microbiomedata/meta/date_created)

## Domain and Range {#domain-and-range-168}

None → 0..1 [String](types/String.md)

## Parents {#parents-205}

## Children {#children-175}

-   [database➞date created](database_date_created.md)

## Used by {#used-by-168}

# Slot: date_last_rain

The date of the last time it rained

URI: [nmdc:date_last_rain](https://microbiomedata/meta/date_last_rain)

## Domain and Range {#domain-and-range-169}

None → 0..1 [TimestampValue](TimestampValue.md)

## Parents {#parents-206}

-   is_a: [attribute](attribute.md)

## Children {#children-176}

## Used by {#used-by-169}

## Other properties

+---------------+-----------------------+
| **Aliases:**  | > date last rain      |
+---------------+-----------------------+
| **Mappings:** | > mixs:date_last_rain |
+---------------+-----------------------+

# Slot: decontam_software

Tool(s) used in contamination screening

URI:
[nmdc:decontam_software](https://microbiomedata/meta/decontam_software)

## Domain and Range {#domain-and-range-170}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-207}

-   is_a: [attribute](attribute.md)

## Children {#children-177}

## Used by {#used-by-170}

## Other properties

+-----------------+----------------------------+
| **Aliases:**    | > decontamination software |
+-----------------+----------------------------+
| **Mappings:**   | > mixs:decontam_software   |
+-----------------+----------------------------+
| **In Subsets:** | > sequencing               |
+-----------------+----------------------------+

# Slot: density

"Density of the sample, which is its mass per unit volume (aka
volumetric mass density)"

URI: [nmdc:density](https://microbiomedata/meta/density)

## Domain and Range {#domain-and-range-171}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-208}

-   is_a: [attribute](attribute.md)

## Children {#children-178}

## Used by {#used-by-171}

-   [Biosample](Biosample.md)

## Other properties

+---------------+----------------+
| **Aliases:**  | > density      |
+---------------+----------------+
| **Mappings:** | > mixs:density |
+---------------+----------------+

# Slot: depos_env

"Main depositional environment
(<https://en.wikipedia.org/wiki/Depositional_environment>). If""other""
is specified, please propose entry in ""additional info"" field"

URI: [nmdc:depos_env](https://microbiomedata/meta/depos_env)

## Domain and Range {#domain-and-range-172}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-209}

-   is_a: [attribute](attribute.md)

## Children {#children-179}

## Used by {#used-by-172}

## Other properties

+---------------+----------------------------+
| **Aliases:**  | > depositional environment |
+---------------+----------------------------+
| **Mappings:** | > mixs:depos_env           |
+---------------+----------------------------+

# Slot: depth

Please refer to the definitions of depth in the environmental packages

URI: [nmdc:depth](https://microbiomedata/meta/depth)

## Domain and Range {#domain-and-range-173}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-210}

-   is_a: [attribute](attribute.md)

## Children {#children-180}

## Used by {#used-by-173}

-   [Biosample](Biosample.md)

## Other properties

+-----------------+-------------------------------+
| **Aliases:**    | > geographic location (depth) |
+-----------------+-------------------------------+
| **Mappings:**   | > mixs:depth                  |
+-----------------+-------------------------------+
| **In Subsets:** | > environment                 |
+-----------------+-------------------------------+

# Slot: depth2

URI: [nmdc:depth2](https://microbiomedata/meta/depth2)

## Domain and Range {#domain-and-range-174}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-211}

## Children {#children-181}

## Used by {#used-by-174}

-   [Biosample](Biosample.md)

# Slot: dermatology_disord

History of dermatology disorders; can include multiple disorders

URI:
[nmdc:dermatology_disord](https://microbiomedata/meta/dermatology_disord)

## Domain and Range {#domain-and-range-175}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-212}

-   is_a: [attribute](attribute.md)

## Children {#children-182}

## Used by {#used-by-175}

## Other properties

+---------------+---------------------------+
| **Aliases:**  | > dermatology disorder    |
+---------------+---------------------------+
| **Mappings:** | > mixs:dermatology_disord |
+---------------+---------------------------+

# Slot: description

a human-readable description of a thing

URI:
<https://microbiomedata/schema/workflow_execution_activity/description>

## Domain and Range {#domain-and-range-176}

None → 0..1 [String](types/String.md)

## Parents {#parents-213}

## Children {#children-183}

## Used by {#used-by-176}

-   [ChemicalEntity](ChemicalEntity.md)
-   [EnvironmentalMaterialTerm](EnvironmentalMaterialTerm.md)
-   [GeneProduct](GeneProduct.md)
-   [ImageValue](ImageValue.md)
-   [Instrument](Instrument.md)
-   [NamedThing](NamedThing.md)
-   [OntologyClass](OntologyClass.md)
-   [Person](Person.md)

## Other properties

  **Mappings:**    dcterms:description
  ---------------- ---------------------
                   
  Slot: detec_ty   pe

Type of UViG detection

URI: [nmdc:detec_type](https://microbiomedata/meta/detec_type)

## Domain and Range {#domain-and-range-177}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-214}

-   is_a: [attribute](attribute.md)

## Children {#children-184}

## Used by {#used-by-177}

## Other properties

+-----------------+-------------------+
| **Aliases:**    | > detection type  |
+-----------------+-------------------+
| **Mappings:**   | > mixs:detec_type |
+-----------------+-------------------+
| **In Subsets:** | > sequencing      |
+-----------------+-------------------+

# Slot: dew_point

"The temperature to which a given parcel of humid air must be cooled, at
constant barometric pressure, for water vapor to condense into water."

URI: [nmdc:dew_point](https://microbiomedata/meta/dew_point)

## Domain and Range {#domain-and-range-178}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-215}

-   is_a: [attribute](attribute.md)

## Children {#children-185}

## Used by {#used-by-178}

## Other properties

+---------------+------------------+
| **Aliases:**  | > dew point      |
+---------------+------------------+
| **Mappings:** | > mixs:dew_point |
+---------------+------------------+

# Slot: diet_last_six_month

"Specification of major diet changes in the last six months, if yes the
change should be specified"

URI:
[nmdc:diet_last_six_month](https://microbiomedata/meta/diet_last_six_month)

## Domain and Range {#domain-and-range-179}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-216}

-   is_a: [attribute](attribute.md)

## Children {#children-186}

## Used by {#used-by-179}

## Other properties

+---------------+----------------------------------------+
| **Aliases:**  | > major diet change in last six months |
+---------------+----------------------------------------+
| **Mappings:** | > mixs:diet_last_six_month             |
+---------------+----------------------------------------+

# Slot: diether_lipids

Concentration of diether lipids; can include multiple types of diether
lipids

URI: [nmdc:diether_lipids](https://microbiomedata/meta/diether_lipids)

## Domain and Range {#domain-and-range-180}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-217}

-   is_a: [attribute](attribute.md)

## Children {#children-187}

## Used by {#used-by-180}

## Other properties

+---------------+-----------------------+
| **Aliases:**  | > diether lipids      |
+---------------+-----------------------+
| **Mappings:** | > mixs:diether_lipids |
+---------------+-----------------------+

# Slot: direction

One of l-\>r, r-\>l, bidirectional, neutral

URI: [nmdc:direction](https://microbiomedata/meta/direction)

## Domain and Range {#domain-and-range-181}

None → 0..1 [String](types/String.md)

## Parents {#parents-218}

## Children {#children-188}

-   [reaction➞direction](reaction_direction.md)

## Used by {#used-by-181}

# Slot: display order

When rendering information, this attribute to specify the order in which
the information should be rendered.

URI:
<https://microbiomedata/schema/workflow_execution_activity/display_order>

## Domain and Range {#domain-and-range-182}

None → 0..1 [String](types/String.md)

## Parents {#parents-219}

-   is_a: [attribute](attribute.md)

## Children {#children-189}

## Used by {#used-by-182}

-   [ImageValue](ImageValue.md)

# Slot: diss_carb_dioxide

Concentration of dissolved carbon dioxide in the sample or liquid
portion of the sample

URI:
[nmdc:diss_carb_dioxide](https://microbiomedata/meta/diss_carb_dioxide)

## Domain and Range {#domain-and-range-183}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-220}

-   is_a: [attribute](attribute.md)

## Children {#children-190}

## Used by {#used-by-183}

-   [Biosample](Biosample.md)

## Other properties

+---------------+----------------------------+
| **Aliases:**  | > dissolved carbon dioxide |
+---------------+----------------------------+
| **Mappings:** | > mixs:diss_carb_dioxide   |
+---------------+----------------------------+

# Slot: diss_hydrogen

Concentration of dissolved hydrogen

URI: [nmdc:diss_hydrogen](https://microbiomedata/meta/diss_hydrogen)

## Domain and Range {#domain-and-range-184}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-221}

-   is_a: [attribute](attribute.md)

## Children {#children-191}

## Used by {#used-by-184}

-   [Biosample](Biosample.md)

## Other properties

+---------------+----------------------+
| **Aliases:**  | > dissolved hydrogen |
+---------------+----------------------+
| **Mappings:** | > mixs:diss_hydrogen |
+---------------+----------------------+

# Slot: diss_inorg_carb

"Dissolved inorganic carbon concentration in the sample, typically
measured after filtering the sample using a 0.45 micrometer filter"

URI: [nmdc:diss_inorg_carb](https://microbiomedata/meta/diss_inorg_carb)

## Domain and Range {#domain-and-range-185}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-222}

-   is_a: [attribute](attribute.md)

## Children {#children-192}

## Used by {#used-by-185}

-   [Biosample](Biosample.md)

## Other properties

+---------------+------------------------------+
| **Aliases:**  | > dissolved inorganic carbon |
+---------------+------------------------------+
| **Mappings:** | > mixs:diss_inorg_carb       |
+---------------+------------------------------+

# Slot: diss_inorg_nitro

Concentration of dissolved inorganic nitrogen

URI:
[nmdc:diss_inorg_nitro](https://microbiomedata/meta/diss_inorg_nitro)

## Domain and Range {#domain-and-range-186}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-223}

-   is_a: [attribute](attribute.md)

## Children {#children-193}

## Used by {#used-by-186}

## Other properties

+---------------+--------------------------------+
| **Aliases:**  | > dissolved inorganic nitrogen |
+---------------+--------------------------------+
| **Mappings:** | > mixs:diss_inorg_nitro        |
+---------------+--------------------------------+

# Slot: diss_inorg_phosp

Concentration of dissolved inorganic phosphorus in the sample

URI:
[nmdc:diss_inorg_phosp](https://microbiomedata/meta/diss_inorg_phosp)

## Domain and Range {#domain-and-range-187}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-224}

-   is_a: [attribute](attribute.md)

## Children {#children-194}

## Used by {#used-by-187}

-   [Biosample](Biosample.md)

## Other properties

+---------------+----------------------------------+
| **Aliases:**  | > dissolved inorganic phosphorus |
+---------------+----------------------------------+
| **Mappings:** | > mixs:diss_inorg_phosp          |
+---------------+----------------------------------+

# Slot: diss_iron

Concentration of dissolved iron in the sample

URI: [nmdc:diss_iron](https://microbiomedata/meta/diss_iron)

## Domain and Range {#domain-and-range-188}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-225}

-   is_a: [attribute](attribute.md)

## Children {#children-195}

## Used by {#used-by-188}

## Other properties

+---------------+------------------+
| **Aliases:**  | > dissolved iron |
+---------------+------------------+
| **Mappings:** | > mixs:diss_iron |
+---------------+------------------+

# Slot: diss_org_carb

"Concentration of dissolved organic carbon in the sample, liquid portion
of the sample, or aqueous phase of the fluid"

URI: [nmdc:diss_org_carb](https://microbiomedata/meta/diss_org_carb)

## Domain and Range {#domain-and-range-189}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-226}

-   is_a: [attribute](attribute.md)

## Children {#children-196}

## Used by {#used-by-189}

-   [Biosample](Biosample.md)

## Other properties

+---------------+----------------------------+
| **Aliases:**  | > dissolved organic carbon |
+---------------+----------------------------+
| **Mappings:** | > mixs:diss_org_carb       |
+---------------+----------------------------+

# Slot: diss_org_nitro

Dissolved organic nitrogen concentration measured as; total dissolved
nitrogen - NH4 - NO3 - NO2

URI: [nmdc:diss_org_nitro](https://microbiomedata/meta/diss_org_nitro)

## Domain and Range {#domain-and-range-190}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-227}

-   is_a: [attribute](attribute.md)

## Children {#children-197}

## Used by {#used-by-190}

-   [Biosample](Biosample.md)

## Other properties

+---------------+------------------------------+
| **Aliases:**  | > dissolved organic nitrogen |
+---------------+------------------------------+
| **Mappings:** | > mixs:diss_org_nitro        |
+---------------+------------------------------+

# Slot: diss_oxygen

Concentration of dissolved oxygen

URI: [nmdc:diss_oxygen](https://microbiomedata/meta/diss_oxygen)

## Domain and Range {#domain-and-range-191}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-228}

-   is_a: [attribute](attribute.md)

## Children {#children-198}

## Used by {#used-by-191}

-   [Biosample](Biosample.md)

## Other properties

+---------------+--------------------+
| **Aliases:**  | > dissolved oxygen |
+---------------+--------------------+
| **Mappings:** | > mixs:diss_oxygen |
+---------------+--------------------+

# Slot: diss_oxygen_fluid

Concentration of dissolved oxygen in the oil field produced fluids as it
contributes to oxgen-corrosion and microbial activity (e.g. Mic).

URI:
[nmdc:diss_oxygen_fluid](https://microbiomedata/meta/diss_oxygen_fluid)

## Domain and Range {#domain-and-range-192}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-229}

-   is_a: [attribute](attribute.md)

## Children {#children-199}

## Used by {#used-by-192}

## Other properties

+---------------+------------------------------+
| **Aliases:**  | > dissolved oxygen in fluids |
+---------------+------------------------------+
| **Mappings:** | > mixs:diss_oxygen_fluid     |
+---------------+------------------------------+

# Slot: doi

URI: [nmdc:doi](https://microbiomedata/meta/doi)

## Domain and Range {#domain-and-range-193}

None → 0..1 [AttributeValue](AttributeValue.md)

## Parents {#parents-230}

-   is_a: [attribute](attribute.md)

## Children {#children-200}

-   [study➞doi](study_doi.md)

## Used by {#used-by-193}

# Slot: dominant_hand

Dominant hand of the subject

URI: [nmdc:dominant_hand](https://microbiomedata/meta/dominant_hand)

## Domain and Range {#domain-and-range-194}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-231}

-   is_a: [attribute](attribute.md)

## Children {#children-201}

## Used by {#used-by-194}

## Other properties

+---------------+----------------------+
| **Aliases:**  | > dominant hand      |
+---------------+----------------------+
| **Mappings:** | > mixs:dominant_hand |
+---------------+----------------------+

# Slot: door_comp_type

The composite type of the door

URI: [nmdc:door_comp_type](https://microbiomedata/meta/door_comp_type)

## Domain and Range {#domain-and-range-195}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-232}

-   is_a: [attribute](attribute.md)

## Children {#children-202}

## Used by {#used-by-195}

## Other properties

+---------------+------------------------+
| **Aliases:**  | > door type, composite |
+---------------+------------------------+
| **Mappings:** | > mixs:door_comp_type  |
+---------------+------------------------+

# Slot: door_cond

The phsical condition of the door

URI: [nmdc:door_cond](https://microbiomedata/meta/door_cond)

## Domain and Range {#domain-and-range-196}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-233}

-   is_a: [attribute](attribute.md)

## Children {#children-203}

## Used by {#used-by-196}

## Other properties

+---------------+------------------+
| **Aliases:**  | > door condition |
+---------------+------------------+
| **Mappings:** | > mixs:door_cond |
+---------------+------------------+

# Slot: door_direct

The direction the door opens

URI: [nmdc:door_direct](https://microbiomedata/meta/door_direct)

## Domain and Range {#domain-and-range-197}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-234}

-   is_a: [attribute](attribute.md)

## Children {#children-204}

## Used by {#used-by-197}

## Other properties

+---------------+-----------------------------+
| **Aliases:**  | > door direction of opening |
+---------------+-----------------------------+
| **Mappings:** | > mixs:door_direct          |
+---------------+-----------------------------+

# Slot: door_loc

The relative location of the door in the room

URI: [nmdc:door_loc](https://microbiomedata/meta/door_loc)

## Domain and Range {#domain-and-range-198}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-235}

-   is_a: [attribute](attribute.md)

## Children {#children-205}

## Used by {#used-by-198}

## Other properties

+---------------+-----------------+
| **Aliases:**  | > door location |
+---------------+-----------------+
| **Mappings:** | > mixs:door_loc |
+---------------+-----------------+

# Slot: door_mat

The material the door is composed of

URI: [nmdc:door_mat](https://microbiomedata/meta/door_mat)

## Domain and Range {#domain-and-range-199}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-236}

-   is_a: [attribute](attribute.md)

## Children {#children-206}

## Used by {#used-by-199}

## Other properties

+---------------+-----------------+
| **Aliases:**  | > door material |
+---------------+-----------------+
| **Mappings:** | > mixs:door_mat |
+---------------+-----------------+

# Slot: door_move

The type of movement of the door

URI: [nmdc:door_move](https://microbiomedata/meta/door_move)

## Domain and Range {#domain-and-range-200}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-237}

-   is_a: [attribute](attribute.md)

## Children {#children-207}

## Used by {#used-by-200}

## Other properties

+---------------+------------------+
| **Aliases:**  | > door movement  |
+---------------+------------------+
| **Mappings:** | > mixs:door_move |
+---------------+------------------+

# Slot: door_size

The size of the door

URI: [nmdc:door_size](https://microbiomedata/meta/door_size)

## Domain and Range {#domain-and-range-201}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-238}

-   is_a: [attribute](attribute.md)

## Children {#children-208}

## Used by {#used-by-201}

## Other properties

+---------------+---------------------+
| **Aliases:**  | > door area or size |
+---------------+---------------------+
| **Mappings:** | > mixs:door_size    |
+---------------+---------------------+

# Slot: door_type

The type of door material

URI: [nmdc:door_type](https://microbiomedata/meta/door_type)

## Domain and Range {#domain-and-range-202}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-239}

-   is_a: [attribute](attribute.md)

## Children {#children-209}

## Used by {#used-by-202}

## Other properties

+---------------+------------------+
| **Aliases:**  | > door type      |
+---------------+------------------+
| **Mappings:** | > mixs:door_type |
+---------------+------------------+

# Slot: door_type_metal

The type of metal door

URI: [nmdc:door_type_metal](https://microbiomedata/meta/door_type_metal)

## Domain and Range {#domain-and-range-203}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-240}

-   is_a: [attribute](attribute.md)

## Children {#children-210}

## Used by {#used-by-203}

## Other properties

+---------------+------------------------+
| **Aliases:**  | > door type, metal     |
+---------------+------------------------+
| **Mappings:** | > mixs:door_type_metal |
+---------------+------------------------+

# Slot: door_type_wood

The type of wood door

URI: [nmdc:door_type_wood](https://microbiomedata/meta/door_type_wood)

## Domain and Range {#domain-and-range-204}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-241}

-   is_a: [attribute](attribute.md)

## Children {#children-211}

## Used by {#used-by-204}

## Other properties

+---------------+-----------------------+
| **Aliases:**  | > door type, wood     |
+---------------+-----------------------+
| **Mappings:** | > mixs:door_type_wood |
+---------------+-----------------------+

# Slot: door_water_mold

Signs of the presence of mold or mildew on a door

URI: [nmdc:door_water_mold](https://microbiomedata/meta/door_water_mold)

## Domain and Range {#domain-and-range-205}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-242}

-   is_a: [attribute](attribute.md)

## Children {#children-212}

## Used by {#used-by-205}

## Other properties

+---------------+----------------------------+
| **Aliases:**  | > door signs of water/mold |
+---------------+----------------------------+
| **Mappings:** | > mixs:door_water_mold     |
+---------------+----------------------------+

# Slot: douche

Date of most recent douche

URI: [nmdc:douche](https://microbiomedata/meta/douche)

## Domain and Range {#domain-and-range-206}

None → 0..1 [TimestampValue](TimestampValue.md)

## Parents {#parents-243}

-   is_a: [attribute](attribute.md)

## Children {#children-213}

## Used by {#used-by-206}

## Other properties

+---------------+---------------+
| **Aliases:**  | > douche      |
+---------------+---------------+
| **Mappings:** | > mixs:douche |
+---------------+---------------+

# Slot: down_par

Visible waveband radiance and irradiance measurements in the water
column

URI: [nmdc:down_par](https://microbiomedata/meta/down_par)

## Domain and Range {#domain-and-range-207}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-244}

-   is_a: [attribute](attribute.md)

## Children {#children-214}

## Used by {#used-by-207}

## Other properties

+---------------+-----------------+
| **Aliases:**  | > downward PAR  |
+---------------+-----------------+
| **Mappings:** | > mixs:down_par |
+---------------+-----------------+

# Slot: drainage_class

Drainage classification from a standard system such as the USDA system

URI: [nmdc:drainage_class](https://microbiomedata/meta/drainage_class)

## Domain and Range {#domain-and-range-208}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-245}

-   is_a: [attribute](attribute.md)

## Children {#children-215}

## Used by {#used-by-208}

-   [Biosample](Biosample.md)

## Other properties

+---------------+---------------------------+
| **Aliases:**  | > drainage classification |
+---------------+---------------------------+
| **Mappings:** | > mixs:drainage_class     |
+---------------+---------------------------+

# Slot: drawings

"The buildings architectural drawings; if design is chosen, indicate
phase-conceptual, schematic, design development, and construction
documents"

URI: [nmdc:drawings](https://microbiomedata/meta/drawings)

## Domain and Range {#domain-and-range-209}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-246}

-   is_a: [attribute](attribute.md)

## Children {#children-216}

## Used by {#used-by-209}

## Other properties

+---------------+-----------------+
| **Aliases:**  | > drawings      |
+---------------+-----------------+
| **Mappings:** | > mixs:drawings |
+---------------+-----------------+

# Slot: drug_usage

Any drug used by subject and the frequency of usage; can include
multiple drugs used

URI: [nmdc:drug_usage](https://microbiomedata/meta/drug_usage)

## Domain and Range {#domain-and-range-210}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-247}

-   is_a: [attribute](attribute.md)

## Children {#children-217}

## Used by {#used-by-210}

## Other properties

+---------------+-------------------+
| **Aliases:**  | > drug usage      |
+---------------+-------------------+
| **Mappings:** | > mixs:drug_usage |
+---------------+-------------------+

# Slot: ecosystem

TODO

URI: [nmdc:ecosystem](https://microbiomedata/meta/ecosystem)

## Domain and Range {#domain-and-range-211}

None → 0..1 [String](types/String.md)

## Parents {#parents-248}

-   is_a: [gold_path_field](gold_path_field.md)

## Children {#children-218}

## Used by {#used-by-211}

-   [Biosample](Biosample.md)
-   [Study](Study.md)

# Slot: ecosystem_category

TODO

URI:
[nmdc:ecosystem_category](https://microbiomedata/meta/ecosystem_category)

## Domain and Range {#domain-and-range-212}

None → 0..1 [String](types/String.md)

## Parents {#parents-249}

-   is_a: [gold_path_field](gold_path_field.md)

## Children {#children-219}

## Used by {#used-by-212}

-   [Biosample](Biosample.md)
-   [Study](Study.md)

# Slot: ecosystem_path_id

A unique id representing the GOLD classifers associated with a sample.

URI:
[nmdc:ecosystem_path_id](https://microbiomedata/meta/ecosystem_path_id)

## Domain and Range {#domain-and-range-213}

None → 0..1 [String](types/String.md)

## Parents {#parents-250}

## Children {#children-220}

## Used by {#used-by-213}

# Slot: ecosystem_subtype

TODO

URI:
[nmdc:ecosystem_subtype](https://microbiomedata/meta/ecosystem_subtype)

## Domain and Range {#domain-and-range-214}

None → 0..1 [String](types/String.md)

## Parents {#parents-251}

-   is_a: [gold_path_field](gold_path_field.md)

## Children {#children-221}

## Used by {#used-by-214}

-   [Biosample](Biosample.md)
-   [Study](Study.md)

# Slot: ecosystem_type

TODO

URI: [nmdc:ecosystem_type](https://microbiomedata/meta/ecosystem_type)

## Domain and Range {#domain-and-range-215}

None → 0..1 [String](types/String.md)

## Parents {#parents-252}

-   is_a: [gold_path_field](gold_path_field.md)

## Children {#children-222}

## Used by {#used-by-215}

-   [Biosample](Biosample.md)
-   [Study](Study.md)

# Slot: efficiency_percent

Percentage of volatile solids removed from the anaerobic digestor

URI:
[nmdc:efficiency_percent](https://microbiomedata/meta/efficiency_percent)

## Domain and Range {#domain-and-range-216}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-253}

-   is_a: [attribute](attribute.md)

## Children {#children-223}

## Used by {#used-by-216}

## Other properties

+---------------+---------------------------+
| **Aliases:**  | > efficiency percent      |
+---------------+---------------------------+
| **Mappings:** | > mixs:efficiency_percent |
+---------------+---------------------------+

# Slot: elev

"Elevation of the sampling site is its height above a fixed reference
point, most commonly the mean sea level. Elevation is mainly used when
referring to points on the earth's surface, while altitude is used for
points above the surface, such as an aircraft in flight or a spacecraft
in orbit"

URI: [nmdc:elev](https://microbiomedata/meta/elev)

## Domain and Range {#domain-and-range-217}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-254}

-   is_a: [attribute](attribute.md)

## Children {#children-224}

## Used by {#used-by-217}

-   [Biosample](Biosample.md)

## Other properties

+-----------------+---------------+
| **Aliases:**    | > elevation   |
+-----------------+---------------+
| **Mappings:**   | > mixs:elev   |
+-----------------+---------------+
| **In Subsets:** | > environment |
+-----------------+---------------+

# Slot: elevator

The number of elevators within the built structure

URI: [nmdc:elevator](https://microbiomedata/meta/elevator)

## Domain and Range {#domain-and-range-218}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-255}

-   is_a: [attribute](attribute.md)

## Children {#children-225}

## Used by {#used-by-218}

## Other properties

+---------------+------------------+
| **Aliases:**  | > elevator count |
+---------------+------------------+
| **Mappings:** | > mixs:elevator  |
+---------------+------------------+

# Slot: email

An email address for an entity such as a person. This should be the
primarly email address used.

URI: <https://microbiomedata/schema/workflow_execution_activity/email>

## Domain and Range {#domain-and-range-219}

None → 0..1 [String](types/String.md)

## Parents {#parents-256}

## Children {#children-226}

## Used by {#used-by-219}

-   [PersonValue](PersonValue.md)

## Other properties

  **Mappings:**    schema:email
  ---------------- --------------
                   
  Slot: emulsion   s

"Amount or concentration of substances such as paints, adhesives,
mayonnaise, hair colorants, emulsified oils, etc.; can include multiple
emulsion types"

URI: [nmdc:emulsions](https://microbiomedata/meta/emulsions)

## Domain and Range {#domain-and-range-220}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-257}

-   is_a: [attribute](attribute.md)

## Children {#children-227}

## Used by {#used-by-220}

## Other properties

+---------------+------------------+
| **Aliases:**  | > emulsions      |
+---------------+------------------+
| **Mappings:** | > mixs:emulsions |
+---------------+------------------+

# Slot: encoded_traits

"Should include key traits like antibiotic resistance or xenobiotic
degradation phenotypes for plasmids, converting genes for phage"

URI: [nmdc:encoded_traits](https://microbiomedata/meta/encoded_traits)

## Domain and Range {#domain-and-range-221}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-258}

-   is_a: [attribute](attribute.md)

## Children {#children-228}

## Used by {#used-by-221}

## Other properties

+-----------------+--------------------------------+
| **Aliases:**    | > encoded traits               |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:encoded_traits          |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: encodes

The gene product encoded by this feature. Typically this is used for a
CDS feature or gene feature which will encode a protein. It can also be
used by a nc transcript ot gene feature that encoded a ncRNA

URI: [nmdc:encodes](https://microbiomedata/meta/encodes)

## Domain and Range {#domain-and-range-222}

None → 0..1 [GeneProduct](GeneProduct.md)

## Parents {#parents-259}

## Children {#children-229}

-   [genome feature➞encodes](genome_feature_encodes.md)

## Used by {#used-by-222}

# Slot: end

The end of the feature in positive 1-based integer coordinates

URI: [nmdc:end](https://microbiomedata/meta/end)

## Domain and Range {#domain-and-range-223}

None → 1..1 [Integer](types/Integer.md)

## Parents {#parents-260}

-   is_a: [gff coordinate](gff_coordinate.md)

## Children {#children-230}

## Used by {#used-by-223}

## Other properties

  ----------------------- ----------------------- ------------------------
  **Comments:**                                   \- "constraint: end \>
                                                  start" - "For features
                                                  that cross the origin of
                                                  a circular feature, end
                                                  = the position of the
                                                  end + the length of the
                                                  landmark feature."

  **Close Mappings:**                             biolink:end
                                                  \_interbase_coordinate
  ----------------------- ----------------------- ------------------------

# Slot: ended at time

URI:
<https://microbiomedata/schema/workflow_execution_activity/ended_at_time>

## Domain and Range {#domain-and-range-224}

None → 0..1 [String](types/String.md)

## Parents {#parents-261}

## Children {#children-231}

-   [workflow execution activity➞ended at
    time](workflow_execution_activity_ended_at_time.md)

## Used by {#used-by-224}

-   [Activity](Activity.md)

## Other properties

  **Mappings:**    prov:endedAtTime
  ---------------- ------------------
                   
                   road_scale-1:
  Slot: env_broa   d_scale

"In this field, report which major environmental system your sample or
specimen came from. The systems identified should have a coarse spatial
grain, to provide the general environmental context of where the
sampling was done (e.g. were you in the desert or a rainforest?). We
recommend using subclasses of ENVOUs biome class:
<http://purl.obolibrary.org/obo/ENVO_00000428>. Format (one term):
termLabel \[termID\], Format (multiple terms): termLabel
\[termID\]termLabel \[termID\]. Example: Annotating a water sample from
the photic zone in middle of the Atlantic Ocean, consider: oceanic
epipelagic zone biome \[ENVO:01000033\]. Example: Annotating a sample
from the Amazon rainforest consider: tropical moist broadleaf forest
biome \[ENVO:01000228\]. If needed, request new terms on the ENVO
tracker, identified here:
<http://www.obofoundry.org/ontology/envo.html>"

URI: [nmdc:env_broad_scale](https://microbiomedata/meta/env_broad_scale)

## Domain and Range {#domain-and-range-225}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-262}

-   is_a: [attribute](attribute.md)

## Children {#children-232}

-   [biosample➞env_broad_scale](biosample_env_broad_scale.md)

## Used by {#used-by-225}

## Other properties

+-----------------+-------------------------------------+
| **Aliases:**    | > broad-scale environmental context |
+-----------------+-------------------------------------+
| **Mappings:**   | > mixs:env_broad_scale              |
+-----------------+-------------------------------------+
| **In Subsets:** | > environment                       |
+-----------------+-------------------------------------+

# Slot: env_local_scale

"In this field, report the entity or entities which are in your sample
or specimenUs local vicinity and which you believe have significant
causal influences on your sample or specimen. Please use terms that are
present in ENVO and which are of smaller spatial grain than your entry
for env_broad_scale. Format (one term): termLabel \[termID\]; Format
(multiple terms): termLabel \[termID\]termLabel \[termID\]. Example:
Annotating a pooled sample taken from various vegetation layers in a
forest consider: canopy \[ENVO:00000047\]litter layer
\[ENVO:01000338\]shrub layer \[ENVO:01000336\]. If needed, request new
terms on the ENVO tracker, identified here:
<http://www.obofoundry.org/ontology/envo.html>"

URI: [nmdc:env_local_scale](https://microbiomedata/meta/env_local_scale)

## Domain and Range {#domain-and-range-226}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-263}

-   is_a: [attribute](attribute.md)

## Children {#children-233}

-   [biosample➞env_local_scale](biosample_env_local_scale.md)

## Used by {#used-by-226}

## Other properties

+-----------------+-------------------------------+
| **Aliases:**    | > local environmental context |
+-----------------+-------------------------------+
| **Mappings:**   | > mixs:env_local_scale        |
+-----------------+-------------------------------+
| **In Subsets:** | > environment                 |
+-----------------+-------------------------------+

# Slot: env_medium

"In this field, report which environmental material or materials (pipe
separated) immediately surrounded your sample or specimen prior to
sampling, using one or more subclasses of ENVOUs environmental material
class: <http://purl.obolibrary.org/obo/ENVO_00010483>. Format (one
term): termLabel \[termID\]; Format (multiple terms): termLabel
\[termID\]termLabel \[termID\]. Example: Annotating a fish swimming in
the upper 100 m of the Atlantic Ocean, consider: ocean water
\[ENVO:00002151\]. Example: Annotating a duck on a pond consider: pond
water \[ENVO:00002228\]\|air ENVO_00002005. If needed, request new terms
on the ENVO tracker, identified here:
<http://www.obofoundry.org/ontology/envo.html>"

URI: [nmdc:env_medium](https://microbiomedata/meta/env_medium)

## Domain and Range {#domain-and-range-227}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-264}

-   is_a: [attribute](attribute.md)

## Children {#children-234}

-   [biosample➞env_medium](biosample_env_medium.md)

## Used by {#used-by-227}

## Other properties

+-----------------+------------------------+
| **Aliases:**    | > environmental medium |
+-----------------+------------------------+
| **Mappings:**   | > mixs:env_medium      |
+-----------------+------------------------+
| **In Subsets:** | > environment          |
+-----------------+------------------------+

# Slot: env_package

"MIxS extension for reporting of measurements and observations obtained
from one or more of the environments where the sample was obtained. All
environmental packages listed here are further defined in separate
subtables. By giving the name of the environmental package, a selection
of fields can be made from the subtables and can be reported"

URI: [nmdc:env_package](https://microbiomedata/meta/env_package)

## Domain and Range {#domain-and-range-228}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-265}

-   is_a: [attribute](attribute.md)

## Children {#children-235}

## Used by {#used-by-228}

-   [Biosample](Biosample.md)

## Other properties

+-----------------+-------------------------+
| **Aliases:**    | > environmental package |
+-----------------+-------------------------+
| **Mappings:**   | > mixs:env_package      |
+-----------------+-------------------------+
| **In Subsets:** | > mixs extension        |
+-----------------+-------------------------+

# Slot: escalator

The number of escalators within the built structure

URI: [nmdc:escalator](https://microbiomedata/meta/escalator)

## Domain and Range {#domain-and-range-229}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-266}

-   is_a: [attribute](attribute.md)

## Children {#children-236}

## Used by {#used-by-229}

## Other properties

+---------------+-------------------+
| **Aliases:**  | > escalator count |
+---------------+-------------------+
| **Mappings:** | > mixs:escalator  |
+---------------+-------------------+

# Slot: ess dive datasets

List of ESS-DIVE dataset DOIs

URI:
[nmdc:ess_dive_datasets](https://microbiomedata/meta/ess_dive_datasets)

## Domain and Range {#domain-and-range-230}

None → 0..\* [String](types/String.md)

## Parents {#parents-267}

## Children {#children-237}

## Used by {#used-by-230}

-   [Study](Study.md)

# Slot: estimated_size

The estimated size of the genome prior to sequencing. Of particular
importance in the sequencing of (eukaryotic) genome which could remain
in draft form for a long or unspecified period.

URI: [nmdc:estimated_size](https://microbiomedata/meta/estimated_size)

## Domain and Range {#domain-and-range-231}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-268}

-   is_a: [attribute](attribute.md)

## Children {#children-238}

## Used by {#used-by-231}

## Other properties

+-----------------+--------------------------------+
| **Aliases:**    | > estimated size               |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:estimated_size          |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: ethylbenzene

Concentration of ethylbenzene in the sample

URI: [nmdc:ethylbenzene](https://microbiomedata/meta/ethylbenzene)

## Domain and Range {#domain-and-range-232}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-269}

-   is_a: [attribute](attribute.md)

## Children {#children-239}

## Used by {#used-by-232}

## Other properties

+---------------+---------------------+
| **Aliases:**  | > ethylbenzene      |
+---------------+---------------------+
| **Mappings:** | > mixs:ethylbenzene |
+---------------+---------------------+

# Slot: etl software version

TODO

URI:
[nmdc:etl_software_version](https://microbiomedata/meta/etl_software_version)

## Domain and Range {#domain-and-range-233}

None → 0..1 [String](types/String.md)

## Parents {#parents-270}

## Children {#children-240}

-   [database➞etl software version](database_etl_software_version.md)

## Used by {#used-by-233}

# Slot: execution resource

Example: NERSC-Cori

URI:
<https://microbiomedata/schema/workflow_execution_activity/execution_resource>

## Domain and Range {#domain-and-range-234}

None → 0..1 [String](types/String.md)

## Parents {#parents-271}

-   is_a: [attribute](attribute.md)

## Children {#children-241}

-   [workflow execution activity➞execution
    resource](workflow_execution_activity_execution_resource.md)

## Used by {#used-by-234}

# Slot: exp_duct

The amount of exposed ductwork in the room

URI: [nmdc:exp_duct](https://microbiomedata/meta/exp_duct)

## Domain and Range {#domain-and-range-235}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-272}

-   is_a: [attribute](attribute.md)

## Children {#children-242}

## Used by {#used-by-235}

## Other properties

+---------------+--------------------+
| **Aliases:**  | > exposed ductwork |
+---------------+--------------------+
| **Mappings:** | > mixs:exp_duct    |
+---------------+--------------------+

# Slot: exp_pipe

The number of exposed pipes in the room

URI: [nmdc:exp_pipe](https://microbiomedata/meta/exp_pipe)

## Domain and Range {#domain-and-range-236}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-273}

-   is_a: [attribute](attribute.md)

## Children {#children-243}

## Used by {#used-by-236}

## Other properties

+---------------+-----------------+
| **Aliases:**  | > exposed pipes |
+---------------+-----------------+
| **Mappings:** | > mixs:exp_pipe |
+---------------+-----------------+

# Slot: experimental_factor

"Experimental factors are essentially the variable aspects of an
experiment design which can be used to describe an experiment, or set of
experiments, in an increasingly detailed manner. This field accepts
ontology terms from Experimental Factor Ontology (EFO) and/or Ontology
for Biomedical Investigations (OBI). For a browser of EFO (v 2.95)
terms, please see <http://purl.bioontology.org/ontology/EFO>; for a
browser of OBI (v 2018-02-12) terms please see
<http://purl.bioontology.org/ontology/OBI>"

URI:
[nmdc:experimental_factor](https://microbiomedata/meta/experimental_factor)

## Domain and Range {#domain-and-range-237}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-274}

-   is_a: [attribute](attribute.md)

## Children {#children-244}

## Used by {#used-by-237}

## Other properties

+-----------------+----------------------------+
| **Aliases:**    | > experimental factor      |
+-----------------+----------------------------+
| **Mappings:**   | > mixs:experimental_factor |
+-----------------+----------------------------+
| **In Subsets:** | > investigation            |
+-----------------+----------------------------+

# Slot: ext_door

The number of exterior doors in the built structure

URI: [nmdc:ext_door](https://microbiomedata/meta/ext_door)

## Domain and Range {#domain-and-range-238}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-275}

-   is_a: [attribute](attribute.md)

## Children {#children-245}

## Used by {#used-by-238}

## Other properties

+---------------+-----------------------+
| **Aliases:**  | > exterior door count |
+---------------+-----------------------+
| **Mappings:** | > mixs:ext_door       |
+---------------+-----------------------+

# Slot: ext_wall_orient

The orientation of the exterior wall

URI: [nmdc:ext_wall_orient](https://microbiomedata/meta/ext_wall_orient)

## Domain and Range {#domain-and-range-239}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-276}

-   is_a: [attribute](attribute.md)

## Children {#children-246}

## Used by {#used-by-239}

## Other properties

+---------------+---------------------------------+
| **Aliases:**  | > orientations of exterior wall |
+---------------+---------------------------------+
| **Mappings:** | > mixs:ext_wall_orient          |
+---------------+---------------------------------+

# Slot: ext_window_orient

The compass direction the exterior window of the room is facing

URI:
[nmdc:ext_window_orient](https://microbiomedata/meta/ext_window_orient)

## Domain and Range {#domain-and-range-240}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-277}

-   is_a: [attribute](attribute.md)

## Children {#children-247}

## Used by {#used-by-240}

## Other properties

+---------------+-----------------------------------+
| **Aliases:**  | > orientations of exterior window |
+---------------+-----------------------------------+
| **Mappings:** | > mixs:ext_window_orient          |
+---------------+-----------------------------------+

# Slot: external database identifiers

Link to corresponding identifier in external database

URI:
<https://microbiomedata/schema/workflow_execution_activity/external_database_identifiers>

## Domain and Range {#domain-and-range-241}

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents {#parents-278}

-   is_a: [alternative identifiers](alternative_identifiers.md)

## Children {#children-248}

-   [GOLD sequencing project
    identifiers](GOLD_sequencing_project_identifiers.md)
-   [INSDC experiment identifiers](INSDC_experiment_identifiers.md)
-   [analysis identifiers](analysis_identifiers.md)
-   [omics processing identifiers](omics_processing_identifiers.md)
-   [sample identifiers](sample_identifiers.md)
-   [study identifiers](study_identifiers.md)

## Used by {#used-by-241}

## Other properties

  ------------------- --- ------------------------------------------------
  **Comments:**           The value of this field is always a registered
                          CURIE

  **Close Mappings:**     skos:closeMatch
  ------------------- --- ------------------------------------------------

# Slot: extrachrom_elements

Do plasmids exist of significant phenotypic consequence (e.g. ones that
determine virulence or antibiotic resistance). Megaplasmids? Other
plasmids (borrelia has 15+ plasmids)

URI:
[nmdc:extrachrom_elements](https://microbiomedata/meta/extrachrom_elements)

## Domain and Range {#domain-and-range-242}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-279}

-   is_a: [attribute](attribute.md)

## Children {#children-249}

## Used by {#used-by-242}

## Other properties

+-----------------+--------------------------------+
| **Aliases:**    | > extrachromosomal elements    |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:extrachrom_elements     |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: extreme_event

Unusual physical events that may have affected microbial populations

URI: [nmdc:extreme_event](https://microbiomedata/meta/extreme_event)

## Domain and Range {#domain-and-range-243}

None → 0..1 [TimestampValue](TimestampValue.md)

## Parents {#parents-280}

-   is_a: [attribute](attribute.md)

## Children {#children-250}

## Used by {#used-by-243}

-   [Biosample](Biosample.md)

## Other properties

+---------------+--------------------------+
| **Aliases:**  | > history/extreme events |
+---------------+--------------------------+
| **Mappings:** | > mixs:extreme_event     |
+---------------+--------------------------+

# Slot: extreme_salinity

Measured salinity

URI:
[nmdc:extreme_salinity](https://microbiomedata/meta/extreme_salinity)

## Domain and Range {#domain-and-range-244}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-281}

-   is_a: [attribute](attribute.md)

## Children {#children-251}

## Used by {#used-by-244}

## Other properties

+---------------+---------------------------------------+
| **Aliases:**  | > extreme_unusual_properties/salinity |
+---------------+---------------------------------------+
| **Mappings:** | > mixs:extreme_salinity               |
+---------------+---------------------------------------+

# Slot: fao_class

Soil classification from the FAO World Reference Database for Soil
Resources. The list can be found at
<http://www.fao.org/nr/land/sols/soil/wrb-soil-maps/reference-groups>

URI: [nmdc:fao_class](https://microbiomedata/meta/fao_class)

## Domain and Range {#domain-and-range-245}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-282}

-   is_a: [attribute](attribute.md)

## Children {#children-252}

## Used by {#used-by-245}

-   [Biosample](Biosample.md)

## Other properties

+---------------+-------------------------------------+
| **Aliases:**  | > soil_taxonomic/FAO classification |
+---------------+-------------------------------------+
| **Mappings:** | > mixs:fao_class                    |
+---------------+-------------------------------------+

# Slot: feat_pred

"Method used to predict UViGs features such as ORFs, integration site,
etc."

URI: [nmdc:feat_pred](https://microbiomedata/meta/feat_pred)

## Domain and Range {#domain-and-range-246}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-283}

-   is_a: [attribute](attribute.md)

## Children {#children-253}

## Used by {#used-by-246}

## Other properties

+-----------------+----------------------+
| **Aliases:**    | > feature prediction |
+-----------------+----------------------+
| **Mappings:**   | > mixs:feat_pred     |
+-----------------+----------------------+
| **In Subsets:** | > sequencing         |
+-----------------+----------------------+

# Slot: feature type

TODO: Yuri to write

URI: [nmdc:feature_type](https://microbiomedata/meta/feature_type)

## Domain and Range {#domain-and-range-247}

None → 0..1 [String](types/String.md)

## Parents {#parents-284}

## Children {#children-254}

-   [genome feature➞feature type](genome_feature_feature_type.md)

## Used by {#used-by-247}

# Slot: fertilizer_regm

"Information about treatment involving the use of fertilizers; should
include the name of fertilizer, amount administered, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include multiple fertilizer regimens"

URI: [nmdc:fertilizer_regm](https://microbiomedata/meta/fertilizer_regm)

## Domain and Range {#domain-and-range-248}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-285}

-   is_a: [attribute](attribute.md)

## Children {#children-255}

## Used by {#used-by-248}

## Other properties

+---------------+------------------------+
| **Aliases:**  | > fertilizer regimen   |
+---------------+------------------------+
| **Mappings:** | > mixs:fertilizer_regm |
+---------------+------------------------+

# Slot: field

Name of the hydrocarbon field (e.g. Albacora)

URI: [nmdc:field](https://microbiomedata/meta/field)

## Domain and Range {#domain-and-range-249}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-286}

-   is_a: [attribute](attribute.md)

## Children {#children-256}

## Used by {#used-by-249}

## Other properties

+---------------+--------------+
| **Aliases:**  | > field name |
+---------------+--------------+
| **Mappings:** | > mixs:field |
+---------------+--------------+

# Slot: file size bytes

Size of the file in bytes

URI:
<https://microbiomedata/schema/workflow_execution_activity/file_size_bytes>

## Domain and Range {#domain-and-range-250}

None → 0..1 [Bytes](types/Bytes.md)

## Parents {#parents-287}

-   is_a: [attribute](attribute.md)

## Children {#children-257}

## Used by {#used-by-250}

# Enum: file type enum

URI: [nmdc:file_type_enum](https://microbiomedata/meta/file_type_enum)

## Other properties

## Permissible Values

  ---------------------------------------------------------------------------------------
  Text            Description         Meaning             Other Information
  --------------- ------------------- ------------------- -------------------------------
  FT ICR-MS       FT ICR-MS-based                         
  Analysis        metabolite                              
  Results         assignment results                      
                  table                                   

  GC-MS           GC-MS-based                             
  Metabolomics    metabolite                              
  Results         assignment results                      
                  table                                   

  M etaproteomics Aggregate workflow                      
  Workflow        statistics file                         
  Statistics                                              

  Protein Report  Filtered protein                        
                  report file                             

  Peptide Report  Filtered peptide                        
                  report file                             

  Unfiltered M    MSGFjobs and MASIC                      
  etaproteomics   output file                             
  Results                                                 

  Read Count and  Annotation read                         
  RPKM            count and RPKM per                      
                  feature JSON                            

  QC non-rRNA R2  QC removed rRNA                         
                  reads (R2) fastq                        

  QC non-rRNA R1  QC removed rRNA                         
                  reads (R1) fastq                        

  Metagenome Bins Metagenome bin                          
                  contigs fasta                           

  CheckM          CheckM statistics                       
  Statistics      report                                  

  GOTTCHA2 Krona  GOTTCHA2 krona plot                     
  Plot            HTML file                               

  Kraken2 Krona   Kraken2 krona plot                      
  Plot            HTML file                               

  Centrifuge      Centrifug krona                         
  Krona Plot      plot HTML file                          

  Kraken2 C       Kraken2 output                          
  lassification   report file                             
  Report                                                  

  Kraken2         Kraken2 output read                     
  Taxonomic C     classification file                     
  lassification                                           

  Centrifuge C    Centrifuge output                       
  lassification   report file                             
  Report                                                  

  Centrifuge      Centrifuge output                       
  Taxonomic C     read classification                     
  lassification   file                                    

  Structural      GFF3 format file                        
  Annotation GFF  with structural                         
                  annotations                             

  Functional      GFF3 format file                        
  Annotation GFF  with functional                         
                  annotations                             

  Annotation      FASTA amino acid                        
  Amino Acid      file for annotated                      
  FASTA           proteins                                

  Annotation      Tab delimited file                      
  Enzyme          for EC annotation                       
  Commission                                              

  Annotation KEGG Tab delimited file                      
  Orthology       for KO annotation                       

  Assembly        Sorted bam file of                      
  Coverage BAM    reads mapping back                      
                  to the final                            
                  assembly                                

  Assembly AGP    An AGP format file                      
                  that describes the                      
                  assembly                                

  Assembly        Final assembly                          
  Scaffolds       scaffolds fasta                         

  Assembly        Final assembly                          
  Contigs         contigs fasta                           

  Assembly        Assembled contigs                       
  Coverage Stats  coverage                                
                  information                             

  Filtered        Reads QC result                         
  Sequencing      fastq (clean data)                      
  Reads                                                   

  QC Statistics   Reads QC summary                        
                  statistics                              

  TIGRFam         GFF3 format file                        {' annotations': {'file_n
  Annotation GFF  with TIGRfam                            ame_pattern': Annotatio
                                                          n(tag='file_n ame_pattern',
                                                          valu e='[\[GOLD-AP\]]()
                                                          tigrfam.gff', e xtensions={},
                                                          anno tations={})}}

  Clusters of     GFF3 format file                        {' annotations': {'file_n
  Orthologous     with COGs                               ame_pattern': Annotatio
  Groups (COG)                                            n(tag='file_n ame_pattern',
  Annotation GFF                                          value='\[GOLD-AP\]\_cog.gff', e
                                                          xtensions={}, anno
                                                          tations={})}}

  CATH FunFams    GFF3 format file                        {' annotations': {'file_n
  (Functional     with CATH FunFams                       ame_pattern': Annotatio
  Families)                                               n(tag='file_n ame_pattern',
  Annotation GFF                                          value='\[ GOLD-AP\]\_cath
                                                          \_funfam.gff', e xtensions={},
                                                          anno tations={})}}

  SUPERFam        GFF3 format file                        {' annotations': {'file_n
  Annotation GFF  with SUPERFam                           ame_pattern': Annotatio
                                                          n(tag='file_n ame_pattern', val
                                                          ue='\[GOLD-AP\] \_supfam.gff',
                                                          e xtensions={}, anno
                                                          tations={})}}

  SMART           GFF3 format file                        {' annotations': {'file_n
  Annotation GFF  with SMART                              ame_pattern': Annotatio
                                                          n(tag='file_n ame_pattern', va
                                                          lue='\[GOLD-AP \]\_smart.gff',
                                                          e xtensions={}, anno
                                                          tations={})}}

  Pfam Annotation GFF3 format file                        {' annotations': {'file_n
  GFF             with Pfam                               ame_pattern': Annotatio
                                                          n(tag='file_n ame_pattern', v
                                                          alue='\[GOLD-A P\]\_pfam.gff',
                                                          e xtensions={}, anno
                                                          tations={})}}
  ---------------------------------------------------------------------------------------

# Slot: filter_type

A device which removes solid particulates or airborne molecular
contaminants

URI: [nmdc:filter_type](https://microbiomedata/meta/filter_type)

## Domain and Range {#domain-and-range-251}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-288}

-   is_a: [attribute](attribute.md)

## Children {#children-258}

## Used by {#used-by-251}

## Other properties

+---------------+--------------------+
| **Aliases:**  | > filter type      |
+---------------+--------------------+
| **Mappings:** | > mixs:filter_type |
+---------------+--------------------+

# Slot: fire

Historical and/or physical evidence of fire

URI: [nmdc:fire](https://microbiomedata/meta/fire)

## Domain and Range {#domain-and-range-252}

None → 0..1 [TimestampValue](TimestampValue.md)

## Parents {#parents-289}

-   is_a: [attribute](attribute.md)

## Children {#children-259}

## Used by {#used-by-252}

-   [Biosample](Biosample.md)

## Other properties

+---------------+----------------+
| **Aliases:**  | > history/fire |
+---------------+----------------+
| **Mappings:** | > mixs:fire    |
+---------------+----------------+

# Slot: fireplace_type

A firebox with chimney

URI: [nmdc:fireplace_type](https://microbiomedata/meta/fireplace_type)

## Domain and Range {#domain-and-range-253}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-290}

-   is_a: [attribute](attribute.md)

## Children {#children-260}

## Used by {#used-by-253}

## Other properties

+---------------+-----------------------+
| **Aliases:**  | > fireplace type      |
+---------------+-----------------------+
| **Mappings:** | > mixs:fireplace_type |
+---------------+-----------------------+

# Slot: flooding

Historical and/or physical evidence of flooding

URI: [nmdc:flooding](https://microbiomedata/meta/flooding)

## Domain and Range {#domain-and-range-254}

None → 0..1 [TimestampValue](TimestampValue.md)

## Parents {#parents-291}

-   is_a: [attribute](attribute.md)

## Children {#children-261}

## Used by {#used-by-254}

-   [Biosample](Biosample.md)

## Other properties

+---------------+--------------------+
| **Aliases:**  | > history/flooding |
+---------------+--------------------+
| **Mappings:** | > mixs:flooding    |
+---------------+--------------------+

# Slot: floor_age

The time period since installment of the carpet or flooring

URI: [nmdc:floor_age](https://microbiomedata/meta/floor_age)

## Domain and Range {#domain-and-range-255}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-292}

-   is_a: [attribute](attribute.md)

## Children {#children-262}

## Used by {#used-by-255}

## Other properties

+---------------+------------------+
| **Aliases:**  | > floor age      |
+---------------+------------------+
| **Mappings:** | > mixs:floor_age |
+---------------+------------------+

# Slot: floor_area

The area of the floor space within the room

URI: [nmdc:floor_area](https://microbiomedata/meta/floor_area)

## Domain and Range {#domain-and-range-256}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-293}

-   is_a: [attribute](attribute.md)

## Children {#children-263}

## Used by {#used-by-256}

## Other properties

+---------------+-------------------+
| **Aliases:**  | > floor area      |
+---------------+-------------------+
| **Mappings:** | > mixs:floor_area |
+---------------+-------------------+

# Slot: floor_cond

The physical condition of the floor at the time of sampling; photos or
video preferred; use drawings to indicate location of damaged areas

URI: [nmdc:floor_cond](https://microbiomedata/meta/floor_cond)

## Domain and Range {#domain-and-range-257}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-294}

-   is_a: [attribute](attribute.md)

## Children {#children-264}

## Used by {#used-by-257}

## Other properties

+---------------+-------------------+
| **Aliases:**  | > floor condition |
+---------------+-------------------+
| **Mappings:** | > mixs:floor_cond |
+---------------+-------------------+

# Slot: floor_count

"The number of floors in the building, including basements and
mechanical penthouse"

URI: [nmdc:floor_count](https://microbiomedata/meta/floor_count)

## Domain and Range {#domain-and-range-258}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-295}

-   is_a: [attribute](attribute.md)

## Children {#children-265}

## Used by {#used-by-258}

## Other properties

+---------------+--------------------+
| **Aliases:**  | > floor count      |
+---------------+--------------------+
| **Mappings:** | > mixs:floor_count |
+---------------+--------------------+

# Slot: floor_finish_mat

The floor covering type; the finished surface that is walked on

URI:
[nmdc:floor_finish_mat](https://microbiomedata/meta/floor_finish_mat)

## Domain and Range {#domain-and-range-259}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-296}

-   is_a: [attribute](attribute.md)

## Children {#children-266}

## Used by {#used-by-259}

## Other properties

+---------------+-------------------------+
| **Aliases:**  | > floor finish material |
+---------------+-------------------------+
| **Mappings:** | > mixs:floor_finish_mat |
+---------------+-------------------------+

# Slot: floor_struc

Refers to the structural elements and subfloor upon which the finish
flooring is installed

URI: [nmdc:floor_struc](https://microbiomedata/meta/floor_struc)

## Domain and Range {#domain-and-range-260}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-297}

-   is_a: [attribute](attribute.md)

## Children {#children-267}

## Used by {#used-by-260}

## Other properties

+---------------+--------------------+
| **Aliases:**  | > floor structure  |
+---------------+--------------------+
| **Mappings:** | > mixs:floor_struc |
+---------------+--------------------+

# Slot: floor_thermal_mass

The ability of the floor to provide inertia against temperature
fluctuations

URI:
[nmdc:floor_thermal_mass](https://microbiomedata/meta/floor_thermal_mass)

## Domain and Range {#domain-and-range-261}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-298}

-   is_a: [attribute](attribute.md)

## Children {#children-268}

## Used by {#used-by-261}

## Other properties

+---------------+---------------------------+
| **Aliases:**  | > floor thermal mass      |
+---------------+---------------------------+
| **Mappings:** | > mixs:floor_thermal_mass |
+---------------+---------------------------+

# Slot: floor_water_mold

Signs of the presence of mold or mildew in a room

URI:
[nmdc:floor_water_mold](https://microbiomedata/meta/floor_water_mold)

## Domain and Range {#domain-and-range-262}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-299}

-   is_a: [attribute](attribute.md)

## Children {#children-269}

## Used by {#used-by-262}

## Other properties

+---------------+-----------------------------+
| **Aliases:**  | > floor signs of water/mold |
+---------------+-----------------------------+
| **Mappings:** | > mixs:floor_water_mold     |
+---------------+-----------------------------+

# Slot: fluor

Raw or converted fluorescence of water

URI: [nmdc:fluor](https://microbiomedata/meta/fluor)

## Domain and Range {#domain-and-range-263}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-300}

-   is_a: [attribute](attribute.md)

## Children {#children-270}

## Used by {#used-by-263}

## Other properties

+---------------+----------------+
| **Aliases:**  | > fluorescence |
+---------------+----------------+
| **Mappings:** | > mixs:fluor   |
+---------------+----------------+

# Slot: foetal_health_stat

"Specification of foetal health status, should also include abortion"

URI:
[nmdc:foetal_health_stat](https://microbiomedata/meta/foetal_health_stat)

## Domain and Range {#domain-and-range-264}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-301}

-   is_a: [attribute](attribute.md)

## Children {#children-271}

## Used by {#used-by-264}

## Other properties

+---------------+---------------------------------------+
| **Aliases:**  | > amniotic fluid/foetal health status |
+---------------+---------------------------------------+
| **Mappings:** | > mixs:foetal_health_stat             |
+---------------+---------------------------------------+

# Slot: freq_clean

The number of times the building is cleaned per week

URI: [nmdc:freq_clean](https://microbiomedata/meta/freq_clean)

## Domain and Range {#domain-and-range-265}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-302}

-   is_a: [attribute](attribute.md)

## Children {#children-272}

## Used by {#used-by-265}

## Other properties

+---------------+-------------------------+
| **Aliases:**  | > frequency of cleaning |
+---------------+-------------------------+
| **Mappings:** | > mixs:freq_clean       |
+---------------+-------------------------+

# Slot: freq_cook

The number of times a meal is cooked per week

URI: [nmdc:freq_cook](https://microbiomedata/meta/freq_cook)

## Domain and Range {#domain-and-range-266}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-303}

-   is_a: [attribute](attribute.md)

## Children {#children-273}

## Used by {#used-by-266}

## Other properties

+---------------+------------------------+
| **Aliases:**  | > frequency of cooking |
+---------------+------------------------+
| **Mappings:** | > mixs:freq_cook       |
+---------------+------------------------+

# Slot: has function

URI:
[nmdc:functional_annotation_has_function](https://microbiomedata/meta/functional_annotation_has_function)

## Domain and Range {#domain-and-range-267}

[FunctionalAnnotation](FunctionalAnnotation.md) → 0..1
[String](types/String.md)

## Parents {#parents-304}

-   is_a: [has function](has_function.md)

## Children {#children-274}

## Used by {#used-by-267}

-   [FunctionalAnnotation](FunctionalAnnotation.md)

## Other properties

  ----------------------- ----------------------- -----------------------
  **Comments:**                                   Still missing patterns
                                                  for COG and RetroRules.

                                                  These patterns aren't
                                                  tied to the listed
                                                  prefixes. A discussion
                                                  about that possibility
                                                  had been started,
                                                  including the question
                                                  of whether these lists
                                                  are intended to be open
                                                  examples or closed
  ----------------------- ----------------------- -----------------------

# Slot: functional annotation set

This property links a database object to the set of all functional
annotations

URI:
[nmdc:functional_annotation_set](https://microbiomedata/meta/functional_annotation_set)

## Domain and Range {#domain-and-range-268}

[Database](Database.md) → 0..\*
[FunctionalAnnotation](FunctionalAnnotation.md)

## Parents {#parents-305}

## Children {#children-275}

## Used by {#used-by-268}

-   [Database](Database.md)

# Slot: type

TODO

URI:
[nmdc:functional_annotation_type](https://microbiomedata/meta/functional_annotation_type)

## Domain and Range {#domain-and-range-269}

[FunctionalAnnotation](FunctionalAnnotation.md) → 0..1
[OntologyClass](OntologyClass.md)

## Parents {#parents-306}

-   is_a: [type](type.md)

## Children {#children-276}

## Used by {#used-by-269}

-   [FunctionalAnnotation](FunctionalAnnotation.md)

## Other properties

+---------------+-----------------------------------------------------+
| **Examples:** | > Example(value='nmdc:Biosample', description=None) |
+---------------+-----------------------------------------------------+
|               | > Example(value='nmdc:Study', description=None)     |
+---------------+-----------------------------------------------------+

# Slot: was generated by

provenance for the annotation.

URI:
[nmdc:functional_annotation_was_generated_by](https://microbiomedata/meta/functional_annotation_was_generated_by)

## Domain and Range {#domain-and-range-270}

[FunctionalAnnotation](FunctionalAnnotation.md) → 0..1
[MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)

## Parents {#parents-307}

-   is_a: [was generated by](was_generated_by.md)

## Children {#children-277}

## Used by {#used-by-270}

-   [FunctionalAnnotation](FunctionalAnnotation.md)

## Other properties

  **Mappings:**   prov:wasGeneratedBy
  --------------- ---------------------
                  
  Slot: funding   sources

URI: [nmdc:funding_sources](https://microbiomedata/meta/funding_sources)

## Domain and Range {#domain-and-range-271}

None → 0..\* [String](types/String.md)

## Parents {#parents-308}

## Children {#children-278}

## Used by {#used-by-271}

-   [Study](Study.md)

# Slot: fungicide_regm

"Information about treatment involving use of fungicides; should include
the name of fungicide, amount administered, treatment regimen including
how many times the treatment was repeated, how long each treatment
lasted, and the start and end time of the entire treatment; can include
multiple fungicide regimens"

URI: [nmdc:fungicide_regm](https://microbiomedata/meta/fungicide_regm)

## Domain and Range {#domain-and-range-272}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-309}

-   is_a: [attribute](attribute.md)

## Children {#children-279}

## Used by {#used-by-272}

## Other properties

+---------------+-----------------------+
| **Aliases:**  | > fungicide regimen   |
+---------------+-----------------------+
| **Mappings:** | > mixs:fungicide_regm |
+---------------+-----------------------+

# Slot: furniture

The types of furniture present in the sampled room

URI: [nmdc:furniture](https://microbiomedata/meta/furniture)

## Domain and Range {#domain-and-range-273}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-310}

-   is_a: [attribute](attribute.md)

## Children {#children-280}

## Used by {#used-by-273}

## Other properties

+---------------+------------------+
| **Aliases:**  | > furniture      |
+---------------+------------------+
| **Mappings:** | > mixs:furniture |
+---------------+------------------+

# Slot: gap_pct

The gap size percentage of all scaffolds.

URI: <https://microbiomedata/schema/workflow_execution_activity/gap_pct>

## Domain and Range {#domain-and-range-274}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-311}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-281}

## Used by {#used-by-274}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: gaseous_environment

"Use of conditions with differing gaseous environments; should include
the name of gaseous compound, amount administered, treatment duration,
interval and total experimental duration; can include multiple gaseous
environment regimens"

URI:
[nmdc:gaseous_environment](https://microbiomedata/meta/gaseous_environment)

## Domain and Range {#domain-and-range-275}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-312}

-   is_a: [attribute](attribute.md)

## Children {#children-282}

## Used by {#used-by-275}

## Other properties

+---------------+----------------------------+
| **Aliases:**  | > gaseous environment      |
+---------------+----------------------------+
| **Mappings:** | > mixs:gaseous_environment |
+---------------+----------------------------+

# Slot: gaseous_substances

"Amount or concentration of substances such as hydrogen sulfide, carbon
dioxide, methane, etc.; can include multiple substances"

URI:
[nmdc:gaseous_substances](https://microbiomedata/meta/gaseous_substances)

## Domain and Range {#domain-and-range-276}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-313}

-   is_a: [attribute](attribute.md)

## Children {#children-283}

## Used by {#used-by-276}

## Other properties

+---------------+---------------------------+
| **Aliases:**  | > gaseous substances      |
+---------------+---------------------------+
| **Mappings:** | > mixs:gaseous_substances |
+---------------+---------------------------+

# Slot: gastrointest_disord

History of gastrointestinal tract disorders; can include multiple
disorders

URI:
[nmdc:gastrointest_disord](https://microbiomedata/meta/gastrointest_disord)

## Domain and Range {#domain-and-range-277}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-314}

-   is_a: [attribute](attribute.md)

## Children {#children-284}

## Used by {#used-by-277}

## Other properties

+---------------+-----------------------------------+
| **Aliases:**  | > gastrointestinal tract disorder |
+---------------+-----------------------------------+
| **Mappings:** | > mixs:gastrointest_disord        |
+---------------+-----------------------------------+

# Slot: gc_avg

Average of GC content of all contigs.

URI: <https://microbiomedata/schema/workflow_execution_activity/gc_avg>

## Domain and Range {#domain-and-range-278}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-315}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-285}

## Used by {#used-by-278}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: gc_std

Standard deviation of GC content of all contigs.

URI: <https://microbiomedata/schema/workflow_execution_activity/gc_std>

## Domain and Range {#domain-and-range-279}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-316}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-286}

## Used by {#used-by-279}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: gender_restroom

The gender type of the restroom

URI: [nmdc:gender_restroom](https://microbiomedata/meta/gender_restroom)

## Domain and Range {#domain-and-range-280}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-317}

-   is_a: [attribute](attribute.md)

## Children {#children-287}

## Used by {#used-by-280}

## Other properties

+---------------+------------------------+
| **Aliases:**  | > gender of restroom   |
+---------------+------------------------+
| **Mappings:** | > mixs:gender_restroom |
+---------------+------------------------+

# Slot: genetic_mod

"Genetic modifications of the genome of an organism, which may occur
naturally by spontaneous mutation, or be introduced by some experimental
means, e.g. specification of a transgene or the gene knocked-out or
details of transient transfection"

URI: [nmdc:genetic_mod](https://microbiomedata/meta/genetic_mod)

## Domain and Range {#domain-and-range-281}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-318}

-   is_a: [attribute](attribute.md)

## Children {#children-288}

## Used by {#used-by-281}

## Other properties

+---------------+------------------------+
| **Aliases:**  | > genetic modification |
+---------------+------------------------+
| **Mappings:** | > mixs:genetic_mod     |
+---------------+------------------------+

# Slot: encodes

The gene product encoded by this feature. Typically this is used for a
CDS feature or gene feature which will encode a protein. It can also be
used by a nc transcript ot gene feature that encoded a ncRNA

URI:
[nmdc:genome_feature_encodes](https://microbiomedata/meta/genome_feature_encodes)

## Domain and Range {#domain-and-range-282}

[GenomeFeature](GenomeFeature.md) → 0..1 [GeneProduct](GeneProduct.md)

## Parents {#parents-319}

-   is_a: [encodes](encodes.md)

## Children {#children-289}

## Used by {#used-by-282}

-   [GenomeFeature](GenomeFeature.md)

# Slot: end

The end of the feature in positive 1-based integer coordinates

URI:
[nmdc:genome_feature_end](https://microbiomedata/meta/genome_feature_end)

## Domain and Range {#domain-and-range-283}

[GenomeFeature](GenomeFeature.md) → 1..1 [Integer](types/Integer.md)

## Parents {#parents-320}

-   is_a: [gff coordinate](gff_coordinate.md)

## Children {#children-290}

## Used by {#used-by-283}

-   [GenomeFeature](GenomeFeature.md)

## Other properties

  ----------------------- ----------------------- ------------------------
  **Comments:**                                   \- "constraint: end \>
                                                  start" - "For features
                                                  that cross the origin of
                                                  a circular feature, end
                                                  = the position of the
                                                  end + the length of the
                                                  landmark feature."

  **Close Mappings:**                             biolink:end
                                                  \_interbase_coordinate
  ----------------------- ----------------------- ------------------------

# Slot: feature type

TODO: Yuri to write

URI:
[nmdc:genome_feature_feature_type](https://microbiomedata/meta/genome_feature_feature_type)

## Domain and Range {#domain-and-range-284}

[GenomeFeature](GenomeFeature.md) → 0..1 [String](types/String.md)

## Parents {#parents-321}

-   is_a: [feature type](feature_type.md)

## Children {#children-291}

## Used by {#used-by-284}

-   [GenomeFeature](GenomeFeature.md)

# Slot: phase

The phase for a coding sequence entity. For example, phase of a CDS as
represented in a GFF3 with a value of 0, 1 or 2.

URI:
[nmdc:genome_feature_phase](https://microbiomedata/meta/genome_feature_phase)

## Domain and Range {#domain-and-range-285}

[GenomeFeature](GenomeFeature.md) → 0..1 [Integer](types/Integer.md)

## Parents {#parents-322}

-   is_a: [phase](phase.md)

## Children {#children-292}

## Used by {#used-by-285}

-   [GenomeFeature](GenomeFeature.md)

## Other properties

+----------------------------------+----------------------------------+
| **Exact Mappings:**              | biolink:phase                    |
+==================================+==================================+
| Slot: seqid ===========          |                                  |
+----------------------------------+----------------------------------+
| The ID of the landma current     | rk used to establish the         |
| feature.                         | coordinate system for the        |
|                                  |                                  |
| URI:                             |                                  |
+----------------------------------+----------------------------------+
| \`nmdc:genome_feature            | \_seqid                          |
|                                  | \<<https://microbiomedata/me     |
|                                  | ta/genome_feature_seqid>\>\`\_\_ |
+----------------------------------+----------------------------------+
| ## Domain a                      | -286:                            |
| nd Range {#domain-and-range-284} |                                  |
+----------------------------------+----------------------------------+
| \`GenomeFeature \<Geno           | meFeature.md\>[\_\_ → 1..1       |
|                                  | \`String                         |
| ## Parents {#parents-323}        | \<ty                             |
|                                  | pes/String.md\>]{.title-ref}\_\_ |
+----------------------------------+----------------------------------+
| -   is_a: \`seqid \<seq          | id.md\>\`\_\_                    |
|                                  |                                  |
| ## Children {#children-293}      |                                  |
|                                  |                                  |
| ## Used by {#used-by-286}        |                                  |
+----------------------------------+----------------------------------+
| -   \`GenomeFeature \<G          | enomeFeature.md\>\`\_\_          |
+----------------------------------+----------------------------------+
| Slot: genome feature             | > set                            |
+----------------------------------+----------------------------------+

This property links a database object to the set of all features

URI:
[nmdc:genome_feature_set](https://microbiomedata/meta/genome_feature_set)

## Domain and Range

[Database](Database.md) → 0..\* [GenomeFeature](GenomeFeature.md)

## Parents {#parents-324}

## Children {#children-294}

## Used by {#used-by-287}

-   [Database](Database.md)

# Slot: start

The start of the feature in positive 1-based integer coordinates

URI:
[nmdc:genome_feature_start](https://microbiomedata/meta/genome_feature_start)

## Domain and Range

[GenomeFeature](GenomeFeature.md) → 1..1 [Integer](types/Integer.md)

## Parents {#parents-325}

-   is_a: [gff coordinate](gff_coordinate.md)

## Children {#children-295}

## Used by {#used-by-288}

-   [GenomeFeature](GenomeFeature.md)

## Other properties

+----------------------------------+----------------------------------+
| **Close Mappings:**              | bi                               |
|                                  | olink:start_interbase_coordinate |
+==================================+==================================+
| Slot: strand ============        |                                  |
+----------------------------------+----------------------------------+
| The strand on which              | a feature is located. Has a      |
|                                  | value of '+' (sense              |
+----------------------------------+----------------------------------+
| strand or forward st             | rand) or '-' (anti-sense strand  |
|                                  | or reverse strand).              |
| URI:                             |                                  |
+----------------------------------+----------------------------------+
| \`nmdc:genome_feature            | \_strand                         |
|                                  | \<<https://microbiomedata/met    |
|                                  | a/genome_feature_strand>\>\`\_\_ |
+----------------------------------+----------------------------------+
| ## Domain a                      | -289:                            |
| nd Range {#domain-and-range-287} |                                  |
+----------------------------------+----------------------------------+
| \`GenomeFeature \<Geno           | meFeature.md\>[\_\_ → 0..1       |
|                                  | \`String                         |
| ## Parents {#parents-326}        | \<ty                             |
|                                  | pes/String.md\>]{.title-ref}\_\_ |
+----------------------------------+----------------------------------+
| -   is_a: \`strand \<st          | rand.md\>\`\_\_                  |
|                                  |                                  |
| ## Children {#children-296}      |                                  |
|                                  |                                  |
| ## Used by {#used-by-289}        |                                  |
+----------------------------------+----------------------------------+
| -   \`GenomeFeature \<G          | enomeFeature.md\>\`\_\_          |
+----------------------------------+----------------------------------+
| ## Other pr                      | -261:                            |
| operties {#other-properties-259} |                                  |
+----------------------------------+----------------------------------+
| ===================              | ==============                   |
+----------------------------------+----------------------------------+
| **Exact Mappings:**              | > biolink:strand                 |
+----------------------------------+----------------------------------+

# Slot: type

A type from the sequence ontology

URI:
[nmdc:genome_feature_type](https://microbiomedata/meta/genome_feature_type)

## Domain and Range {#domain-and-range-290}

[GenomeFeature](GenomeFeature.md) → 0..1
[OntologyClass](OntologyClass.md)

## Parents {#parents-327}

-   is_a: [type](type.md)

## Children {#children-297}

## Used by {#used-by-290}

-   [GenomeFeature](GenomeFeature.md)

## Other properties {#other-properties-262}

+---------------+-----------------------------------------------------+
| **Mappings:** | > rdf:type                                          |
+---------------+-----------------------------------------------------+
| **Examples:** | > Example(value='nmdc:Biosample', description=None) |
+---------------+-----------------------------------------------------+
|               | > Example(value='nmdc:Study', description=None)     |
+---------------+-----------------------------------------------------+

# Slot: geo_loc_name

"The geographical origin of the sample as defined by the country or sea
name followed by specific region name. Country or sea names should be
chosen from the INSDC country list (<http://insdc.org/country.html>), or
the GAZ ontology (v 1.512) (<http://purl.bioontology.org/ontology/GAZ>)"

URI: [nmdc:geo_loc_name](https://microbiomedata/meta/geo_loc_name)

## Domain and Range {#domain-and-range-291}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-328}

-   is_a: [attribute](attribute.md)

## Children {#children-298}

## Used by {#used-by-291}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-263}

+-----------------+---------------------------------------------------+
| **Aliases:**    | > geographic location (country and/or sea,region) |
+-----------------+---------------------------------------------------+
| **Mappings:**   | > mixs:geo_loc_name                               |
+-----------------+---------------------------------------------------+
| **In Subsets:** | > environment                                     |
+-----------------+---------------------------------------------------+

# Slot: has raw value

The raw value for a geolocation should follow {lat} {long}

URI:
<https://microbiomedata/schema/workflow_execution_activity/geolocation_value_has_raw_value>

## Domain and Range {#domain-and-range-292}

[GeolocationValue](GeolocationValue.md) → 0..1 [String](types/String.md)

## Parents {#parents-329}

-   is_a: [has raw value](has_raw_value.md)

## Children {#children-299}

## Used by {#used-by-292}

-   [GeolocationValue](GeolocationValue.md)

# Slot: gestation_state

Specification of the gestation state

URI: [nmdc:gestation_state](https://microbiomedata/meta/gestation_state)

## Domain and Range {#domain-and-range-293}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-330}

-   is_a: [attribute](attribute.md)

## Children {#children-300}

## Used by {#used-by-293}

## Other properties {#other-properties-264}

+---------------+----------------------------------+
| **Aliases:**  | > amniotic fluid/gestation state |
+---------------+----------------------------------+
| **Mappings:** | > mixs:gestation_state           |
+---------------+----------------------------------+

# Slot: gff coordinate

A positive 1-based integer coordinate indicating start or end

URI: [nmdc:gff_coordinate](https://microbiomedata/meta/gff_coordinate)

## Domain and Range {#domain-and-range-294}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-331}

## Children {#children-301}

-   [end](end.md)
-   [genome feature➞end](genome_feature_end.md)
-   [genome feature➞start](genome_feature_start.md)
-   [start](start.md)

## Used by {#used-by-294}

## Other properties {#other-properties-265}

  ----------------------- ----------------------- -----------------------
  **Comments:**                                   For features that cross
                                                  the origin of a
                                                  circular feature
                                                  (e.g. most bacterial
                                                  genomes, plasmids, and
                                                  some viral genomes),
                                                  the requirement for
                                                  start to be less than
                                                  or equal to end is
                                                  satisfied by making end
                                                  = the position of the
                                                  end + the length of the
                                                  landmark feature.

  ----------------------- ----------------------- -----------------------

# Slot: git url

Example:
<https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>

URI: <https://microbiomedata/schema/workflow_execution_activity/git_url>

## Domain and Range {#domain-and-range-295}

None → 0..1 [String](types/String.md)

## Parents {#parents-332}

-   is_a: [attribute](attribute.md)

## Children {#children-302}

-   [workflow execution activity➞git
    url](workflow_execution_activity_git_url.md)

## Used by {#used-by-295}

# Slot: glucosidase_act

Measurement of glucosidase activity

URI: [nmdc:glucosidase_act](https://microbiomedata/meta/glucosidase_act)

## Domain and Range {#domain-and-range-296}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-333}

-   is_a: [attribute](attribute.md)

## Children {#children-303}

## Used by {#used-by-296}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-266}

+---------------+------------------------+
| **Aliases:**  | > glucosidase activity |
+---------------+------------------------+
| **Mappings:** | > mixs:glucosidase_act |
+---------------+------------------------+

# Slot: gold_path_field

This is a grouping for any of the gold path fields

URI: [nmdc:gold_path_field](https://microbiomedata/meta/gold_path_field)

## Domain and Range {#domain-and-range-297}

None → 0..1 [String](types/String.md)

## Parents {#parents-334}

-   is_a: [attribute](attribute.md)

## Children {#children-304}

-   [ecosystem](ecosystem.md)
-   [ecosystem_category](ecosystem_category.md)
-   [ecosystem_subtype](ecosystem_subtype.md)
-   [ecosystem_type](ecosystem_type.md)
-   [specific_ecosystem](specific_ecosystem.md)

## Used by {#used-by-297}

# Slot: gravidity

"Whether or not subject is gravid, and if yes date due or date
post-conception, specifying which is used"

URI: [nmdc:gravidity](https://microbiomedata/meta/gravidity)

## Domain and Range {#domain-and-range-298}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-335}

-   is_a: [attribute](attribute.md)

## Children {#children-305}

## Used by {#used-by-298}

## Other properties {#other-properties-267}

+---------------+------------------+
| **Aliases:**  | > gravidity      |
+---------------+------------------+
| **Mappings:** | > mixs:gravidity |
+---------------+------------------+

# Slot: gravity

"Information about treatment involving use of gravity factor to study
various types of responses in presence, absence or modified levels of
gravity; treatment regimen including how many times the treatment was
repeated, how long each treatment lasted, and the start and end time of
the entire treatment; can include multiple treatments"

URI: [nmdc:gravity](https://microbiomedata/meta/gravity)

## Domain and Range {#domain-and-range-299}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-336}

-   is_a: [attribute](attribute.md)

## Children {#children-306}

## Used by {#used-by-299}

## Other properties {#other-properties-268}

+---------------+----------------+
| **Aliases:**  | > gravity      |
+---------------+----------------+
| **Mappings:** | > mixs:gravity |
+---------------+----------------+

# Slot: growth_facil

"Type of facility where the sampled plant was grown; controlled
vocabulary: growth chamber, open top chamber, glasshouse, experimental
garden, field. Alternatively use Crop Ontology (CO) terms, see
<http://www.cropontology.org/ontology/CO_715/Crop%20Research>"

URI: [nmdc:growth_facil](https://microbiomedata/meta/growth_facil)

## Domain and Range {#domain-and-range-300}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-337}

-   is_a: [attribute](attribute.md)

## Children {#children-307}

## Used by {#used-by-300}

## Other properties {#other-properties-269}

+---------------+---------------------+
| **Aliases:**  | > growth facility   |
+---------------+---------------------+
| **Mappings:** | > mixs:growth_facil |
+---------------+---------------------+

# Slot: growth_habit

"Characteristic shape, appearance or growth form of a plant species"

URI: [nmdc:growth_habit](https://microbiomedata/meta/growth_habit)

## Domain and Range {#domain-and-range-301}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-338}

-   is_a: [attribute](attribute.md)

## Children {#children-308}

## Used by {#used-by-301}

## Other properties {#other-properties-270}

+---------------+---------------------+
| **Aliases:**  | > growth habit      |
+---------------+---------------------+
| **Mappings:** | > mixs:growth_habit |
+---------------+---------------------+

# Slot: growth_hormone_regm

"Information about treatment involving use of growth hormones; should
include the name of growth hormone, amount administered, treatment
regimen including how many times the treatment was repeated, how long
each treatment lasted, and the start and end time of the entire
treatment; can include multiple growth hormone regimens"

URI:
[nmdc:growth_hormone_regm](https://microbiomedata/meta/growth_hormone_regm)

## Domain and Range {#domain-and-range-302}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-339}

-   is_a: [attribute](attribute.md)

## Children {#children-309}

## Used by {#used-by-302}

## Other properties {#other-properties-271}

+---------------+----------------------------+
| **Aliases:**  | > growth hormone regimen   |
+---------------+----------------------------+
| **Mappings:** | > mixs:growth_hormone_regm |
+---------------+----------------------------+

# Slot: gynecologic_disord

History of gynecological disorders; can include multiple disorders

URI:
[nmdc:gynecologic_disord](https://microbiomedata/meta/gynecologic_disord)

## Domain and Range {#domain-and-range-303}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-340}

-   is_a: [attribute](attribute.md)

## Children {#children-310}

## Used by {#used-by-303}

## Other properties {#other-properties-272}

+---------------+---------------------------+
| **Aliases:**  | > gynecological disorder  |
+---------------+---------------------------+
| **Mappings:** | > mixs:gynecologic_disord |
+---------------+---------------------------+

# Slot: habitat

URI: [nmdc:habitat](https://microbiomedata/meta/habitat)

## Domain and Range {#domain-and-range-304}

None → 0..1 [String](types/String.md)

## Parents {#parents-341}

## Children {#children-311}

## Used by {#used-by-304}

-   [Biosample](Biosample.md)

# Slot: hall_count

The total count of hallways and cooridors in the built structure

URI: [nmdc:hall_count](https://microbiomedata/meta/hall_count)

## Domain and Range {#domain-and-range-305}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-342}

-   is_a: [attribute](attribute.md)

## Children {#children-312}

## Used by {#used-by-305}

## Other properties {#other-properties-273}

+---------------+--------------------------+
| **Aliases:**  | > hallway/corridor count |
+---------------+--------------------------+
| **Mappings:** | > mixs:hall_count        |
+---------------+--------------------------+

# Slot: handidness

The handidness of the individual sampled

URI: [nmdc:handidness](https://microbiomedata/meta/handidness)

## Domain and Range {#domain-and-range-306}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-343}

-   is_a: [attribute](attribute.md)

## Children {#children-313}

## Used by {#used-by-306}

## Other properties {#other-properties-274}

+---------------+-------------------+
| **Aliases:**  | > handidness      |
+---------------+-------------------+
| **Mappings:** | > mixs:handidness |
+---------------+-------------------+

# Slot: has boolean value

Links a quantity value to a boolean

URI:
<https://microbiomedata/schema/workflow_execution_activity/has_boolean_value>

## Domain and Range {#domain-and-range-307}

None → 0..1 [Boolean](types/Boolean.md)

## Parents {#parents-344}

## Children {#children-314}

## Used by {#used-by-307}

-   [BooleanValue](BooleanValue.md)

# Slot: has calibration

TODO: Yuri to fill in

URI:
<https://microbiomedata/schema/workflow_execution_activity/has_calibration>

## Domain and Range {#domain-and-range-308}

None → 0..1 [String](types/String.md)

## Parents {#parents-345}

## Children {#children-315}

-   [metabolomics analysis activity➞has
    calibration](metabolomics_analysis_activity_has_calibration.md)
-   [nom analysis activity➞has
    calibration](nom_analysis_activity_has_calibration.md)

## Used by {#used-by-308}

# Slot: has credit associations

This slot links a study to a credit association. The credit association
will be linked to a person value and to a CRediT Contributor Roles term.
Overall semantics: person should get credit X for their participation in
the study

URI:
[nmdc:has_credit_associations](https://microbiomedata/meta/has_credit_associations)

## Domain and Range {#domain-and-range-309}

[Study](Study.md) → 0..\* [CreditAssociation](CreditAssociation.md)

## Parents {#parents-346}

## Children {#children-316}

## Used by {#used-by-309}

-   [Study](Study.md)

## Other properties {#other-properties-275}

  **Mappings:**    prov:qualifiedAssociation
  ---------------- ---------------------------
                   
                   unction-1:
  Slot: has func   tion

URI: [nmdc:has_function](https://microbiomedata/meta/has_function)

## Domain and Range {#domain-and-range-310}

None → 0..1 [String](types/String.md)

## Parents {#parents-347}

## Children {#children-317}

-   [functional annotation➞has
    function](functional_annotation_has_function.md)

## Used by {#used-by-310}

## Other properties {#other-properties-276}

  ----------------------- ----------------------- -----------------------
  **Comments:**                                   the range for
                                                  has_function was
                                                  asserted as functi
                                                  onal_annotation_term,

                                                  but is actually taking
                                                  string arguments in the
                                                  Polyneme MongoDB,

                                                  and those are
                                                  frequently fulltext,
                                                  not CURIEs. MAM
                                                  2021-06-23
  ----------------------- ----------------------- -----------------------

# Slot: has input

An input to a process.

URI:
<https://microbiomedata/schema/workflow_execution_activity/has_input>

## Domain and Range {#domain-and-range-311}

[NamedThing](NamedThing.md) → 0..\* [NamedThing](NamedThing.md)

## Parents {#parents-348}

## Children {#children-318}

-   [workflow execution activity➞has
    input](workflow_execution_activity_has_input.md)

## Used by {#used-by-311}

## Other properties {#other-properties-277}

  **Aliases:**    input
  --------------- --------------------
                  
  Slot: has max   imum numeric value

The maximum value part, expressed as number, of the quantity value when
the value covers a range.

URI:
<https://microbiomedata/schema/workflow_execution_activity/has_maximum_numeric_value>

## Domain and Range {#domain-and-range-312}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-349}

-   is_a: [has numeric value](has_numeric_value.md)

## Children {#children-319}

## Used by {#used-by-312}

-   [QuantityValue](QuantityValue.md)

# Slot: has metabolite quantifications

URI:
<https://microbiomedata/schema/workflow_execution_activity/has_metabolite_quantifications>

## Domain and Range {#domain-and-range-313}

None → 0..\* [MetaboliteQuantification](MetaboliteQuantification.md)

## Parents {#parents-350}

## Children {#children-320}

-   [metabolomics analysis activity➞has metabolite
    quantifications](metabolomics_analysis_activity_has_metabolite_quantifications.md)

## Used by {#used-by-313}

# Slot: has minimum numeric value

The minimum value part, expressed as number, of the quantity value when
the value covers a range.

URI:
<https://microbiomedata/schema/workflow_execution_activity/has_minimum_numeric_value>

## Domain and Range {#domain-and-range-314}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-351}

-   is_a: [has numeric value](has_numeric_value.md)

## Children {#children-321}

## Used by {#used-by-314}

-   [QuantityValue](QuantityValue.md)

# Slot: has numeric value

Links a quantity value to a number

URI:
<https://microbiomedata/schema/workflow_execution_activity/has_numeric_value>

## Domain and Range {#domain-and-range-315}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-352}

## Children {#children-322}

-   [has maximum numeric value](has_maximum_numeric_value.md)
-   [has minimum numeric value](has_minimum_numeric_value.md)
-   [quantity value➞has numeric
    value](quantity_value_has_numeric_value.md)

## Used by {#used-by-315}

-   [IntegerValue](IntegerValue.md)

## Other properties {#other-properties-278}

+---------------+---------------------+
| **Mappings:** | > qud:quantityValue |
+---------------+---------------------+
|               | > schema:value      |
+---------------+---------------------+

# Slot: has output

An output biosample to a processing step

URI:
<https://microbiomedata/schema/workflow_execution_activity/has_output>

## Domain and Range {#domain-and-range-316}

[NamedThing](NamedThing.md) → 0..\* [NamedThing](NamedThing.md)

## Parents {#parents-353}

## Children {#children-323}

-   [workflow execution activity➞has
    output](workflow_execution_activity_has_output.md)

## Used by {#used-by-316}

## Other properties {#other-properties-279}

  **Aliases:**    output
  --------------- --------
                  
  Slot: has_par   t

A pathway can be broken down to a series of reaction step

URI: [nmdc:has_part](https://microbiomedata/meta/has_part)

## Domain and Range {#domain-and-range-317}

None → 0..\* [Reaction](Reaction.md)

## Parents {#parents-354}

## Children {#children-324}

-   [pathway➞has_part](pathway_has_part.md)

## Used by {#used-by-317}

# Slot: has participants

URI:
[nmdc:has_participants](https://microbiomedata/meta/has_participants)

## Domain and Range {#domain-and-range-318}

None → 0..1 [String](types/String.md)

## Parents {#parents-355}

## Children {#children-325}

-   [left participants](left_participants.md)
-   [reaction➞left participants](reaction_left_participants.md)
-   [reaction➞right participants](reaction_right_participants.md)
-   [right participants](right_participants.md)

## Used by {#used-by-318}

## Other properties {#other-properties-280}

  **Exact Mappings:**    RO:0000057
  ---------------------- ---------------
                         
  Slot: has peptide qu   antifications

URI:
<https://microbiomedata/schema/workflow_execution_activity/has_peptide_quantifications>

## Domain and Range {#domain-and-range-319}

None → 0..\* [PeptideQuantification](PeptideQuantification.md)

## Parents {#parents-356}

## Children {#children-326}

-   [metaproteomics analysis activity➞has peptide
    quantifications](metaproteomics_analysis_activity_has_peptide_quantifications.md)

## Used by {#used-by-319}

# Slot: has raw value

The value that was specified for an annotation in raw form, i.e. a
string. E.g. "2 cm" or "2-4 cm"

URI:
<https://microbiomedata/schema/workflow_execution_activity/has_raw_value>

## Domain and Range {#domain-and-range-320}

[AttributeValue](AttributeValue.md) → 0..1 [String](types/String.md)

## Parents {#parents-357}

## Children {#children-327}

-   [geolocation value➞has raw
    value](geolocation_value_has_raw_value.md)
-   [person value➞has raw value](person_value_has_raw_value.md)
-   [quantity value➞has raw value](quantity_value_has_raw_value.md)

## Used by {#used-by-320}

-   [AttributeValue](AttributeValue.md)
-   [BooleanValue](BooleanValue.md)
-   [ControlledTermValue](ControlledTermValue.md)
-   [ImageValue](ImageValue.md)
-   [IntegerValue](IntegerValue.md)
-   [TextValue](TextValue.md)
-   [TimestampValue](TimestampValue.md)
-   [UrlValue](UrlValue.md)

# Slot: has unit

Links a quantity value to a unit

URI:
<https://microbiomedata/schema/workflow_execution_activity/has_unit>

## Domain and Range {#domain-and-range-321}

None → 0..1 [Unit](types/Unit.md)

## Parents {#parents-358}

## Children {#children-328}

-   [quantity value➞has unit](quantity_value_has_unit.md)

## Used by {#used-by-321}

## Other properties {#other-properties-281}

+---------------+-------------------+
| **Aliases:**  | > scale           |
+---------------+-------------------+
| **Mappings:** | > qud:unit        |
+---------------+-------------------+
|               | > schema:unitCode |
+---------------+-------------------+

# Slot: hc_produced

"Main hydrocarbon type produced from resource (i.e. Oil, gas,
condensate, etc). If""other"" is specified, please propose entry in
""additional info"" field"

URI: [nmdc:hc_produced](https://microbiomedata/meta/hc_produced)

## Domain and Range {#domain-and-range-322}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-359}

-   is_a: [attribute](attribute.md)

## Children {#children-329}

## Used by {#used-by-322}

## Other properties {#other-properties-282}

+---------------+-----------------------------+
| **Aliases:**  | > hydrocarbon type produced |
+---------------+-----------------------------+
| **Mappings:** | > mixs:hc_produced          |
+---------------+-----------------------------+

# Slot: hcr

"Main Hydrocarbon Resource type. The term""Hydrocarbon Resource"" HCR
defined as a natural environmental feature containing large amounts of
hydrocarbons at high concentrations potentially suitable for commercial
exploitation. This term should not be confused with the Hydrocarbon
Occurrence term which also includes hydrocarbon-rich environments with
currently limited commercial interest such as seeps, outcrops, gas
hydrates etc. If ""other"" is specified, please propose entry in
""additional info"" field"

URI: [nmdc:hcr](https://microbiomedata/meta/hcr)

## Domain and Range {#domain-and-range-323}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-360}

-   is_a: [attribute](attribute.md)

## Children {#children-330}

## Used by {#used-by-323}

## Other properties {#other-properties-283}

+---------------+-----------------------------+
| **Aliases:**  | > hydrocarbon resource type |
+---------------+-----------------------------+
| **Mappings:** | > mixs:hcr                  |
+---------------+-----------------------------+

# Slot: hcr_fw_salinity

Original formation water salinity (prior to secondary recovery
e.g. Waterflooding) expressed as TDS

URI: [nmdc:hcr_fw_salinity](https://microbiomedata/meta/hcr_fw_salinity)

## Domain and Range {#domain-and-range-324}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-361}

-   is_a: [attribute](attribute.md)

## Children {#children-331}

## Used by {#used-by-324}

## Other properties {#other-properties-284}

+---------------+----------------------------+
| **Aliases:**  | > formation water salinity |
+---------------+----------------------------+
| **Mappings:** | > mixs:hcr_fw_salinity     |
+---------------+----------------------------+

# Slot: hcr_geol_age

"Geological age of hydrocarbon resource (Additional info:
<https://en.wikipedia.org/wiki/Period_(geology)>). If""other"" is
specified, please propose entry in ""additional info"" field"

URI: [nmdc:hcr_geol_age](https://microbiomedata/meta/hcr_geol_age)

## Domain and Range {#domain-and-range-325}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-362}

-   is_a: [attribute](attribute.md)

## Children {#children-332}

## Used by {#used-by-325}

## Other properties {#other-properties-285}

+---------------+---------------------------------------+
| **Aliases:**  | > hydrocarbon resource geological age |
+---------------+---------------------------------------+
| **Mappings:** | > mixs:hcr_geol_age                   |
+---------------+---------------------------------------+

# Slot: hcr_pressure

Original pressure of the hydrocarbon resource

URI: [nmdc:hcr_pressure](https://microbiomedata/meta/hcr_pressure)

## Domain and Range {#domain-and-range-326}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-363}

-   is_a: [attribute](attribute.md)

## Children {#children-333}

## Used by {#used-by-326}

## Other properties {#other-properties-286}

+---------------+------------------------------------------+
| **Aliases:**  | > hydrocarbon resource original pressure |
+---------------+------------------------------------------+
| **Mappings:** | > mixs:hcr_pressure                      |
+---------------+------------------------------------------+

# Slot: hcr_temp

Original temperature of the hydrocarbon resource

URI: [nmdc:hcr_temp](https://microbiomedata/meta/hcr_temp)

## Domain and Range {#domain-and-range-327}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-364}

-   is_a: [attribute](attribute.md)

## Children {#children-334}

## Used by {#used-by-327}

## Other properties {#other-properties-287}

+---------------+---------------------------------------------+
| **Aliases:**  | > hydrocarbon resource original temperature |
+---------------+---------------------------------------------+
| **Mappings:** | > mixs:hcr_temp                             |
+---------------+---------------------------------------------+

# Slot: health_disease_stat

Health or disease status of specific host at time of collection

URI:
[nmdc:health_disease_stat](https://microbiomedata/meta/health_disease_stat)

## Domain and Range {#domain-and-range-328}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-365}

-   is_a: [attribute](attribute.md)

## Children {#children-335}

## Used by {#used-by-328}

## Other properties {#other-properties-288}

  ----------------- --- ----------------------------------------------
  **Aliases:**          health or disease status of specific host at
                        time of collection

  **Mappings:**         mixs:health_disease_stat

  **In Subsets:**       nucleic acid sequence source
  ----------------- --- ----------------------------------------------

# Slot: heat_cool_type

Methods of conditioning or heating a room or building

URI: [nmdc:heat_cool_type](https://microbiomedata/meta/heat_cool_type)

## Domain and Range {#domain-and-range-329}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-366}

-   is_a: [attribute](attribute.md)

## Children {#children-336}

## Used by {#used-by-329}

## Other properties {#other-properties-289}

+---------------+-----------------------------------+
| **Aliases:**  | > heating and cooling system type |
+---------------+-----------------------------------+
| **Mappings:** | > mixs:heat_cool_type             |
+---------------+-----------------------------------+

# Slot: heat_deliv_loc

The location of heat delivery within the room

URI: [nmdc:heat_deliv_loc](https://microbiomedata/meta/heat_deliv_loc)

## Domain and Range {#domain-and-range-330}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-367}

-   is_a: [attribute](attribute.md)

## Children {#children-337}

## Used by {#used-by-330}

## Other properties {#other-properties-290}

+---------------+------------------------------+
| **Aliases:**  | > heating delivery locations |
+---------------+------------------------------+
| **Mappings:** | > mixs:heat_deliv_loc        |
+---------------+------------------------------+

# Slot: heat_system_deliv_meth

The method by which the heat is delivered through the system

URI:
[nmdc:heat_system_deliv_meth](https://microbiomedata/meta/heat_system_deliv_meth)

## Domain and Range {#domain-and-range-331}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-368}

-   is_a: [attribute](attribute.md)

## Children {#children-338}

## Used by {#used-by-331}

## Other properties {#other-properties-291}

+---------------+----------------------------------+
| **Aliases:**  | > heating system delivery method |
+---------------+----------------------------------+
| **Mappings:** | > mixs:heat_system_deliv_meth    |
+---------------+----------------------------------+

# Slot: heat_system_id

The heating system identifier

URI: [nmdc:heat_system_id](https://microbiomedata/meta/heat_system_id)

## Domain and Range {#domain-and-range-332}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-369}

-   is_a: [attribute](attribute.md)

## Children {#children-339}

## Used by {#used-by-332}

## Other properties {#other-properties-292}

+---------------+-----------------------------+
| **Aliases:**  | > heating system identifier |
+---------------+-----------------------------+
| **Mappings:** | > mixs:heat_system_id       |
+---------------+-----------------------------+

# Slot: heavy_metals

Heavy metals present and concentrationsany drug used by subject and the
frequency of usage; can include multiple heavy metals and concentrations

URI: [nmdc:heavy_metals](https://microbiomedata/meta/heavy_metals)

## Domain and Range {#domain-and-range-333}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-370}

-   is_a: [attribute](attribute.md)

## Children {#children-340}

## Used by {#used-by-333}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-293}

+---------------+-------------------------------------------+
| **Aliases:**  | > extreme_unusual_properties/heavy metals |
+---------------+-------------------------------------------+
| **Mappings:** | > mixs:heavy_metals                       |
+---------------+-------------------------------------------+

# Slot: heavy_metals_meth

Reference or method used in determining heavy metals

URI:
[nmdc:heavy_metals_meth](https://microbiomedata/meta/heavy_metals_meth)

## Domain and Range {#domain-and-range-334}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-371}

-   is_a: [attribute](attribute.md)

## Children {#children-341}

## Used by {#used-by-334}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-294}

+---------------+--------------------------------------------------+
| **Aliases:**  | > extreme_unusual_properties/heavy metals method |
+---------------+--------------------------------------------------+
| **Mappings:** | > mixs:heavy_metals_meth                         |
+---------------+--------------------------------------------------+

# Slot: height_carper_fiber

The average carpet fiber height in the indoor environment

URI:
[nmdc:height_carper_fiber](https://microbiomedata/meta/height_carper_fiber)

## Domain and Range {#domain-and-range-335}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-372}

-   is_a: [attribute](attribute.md)

## Children {#children-342}

## Used by {#used-by-335}

## Other properties {#other-properties-295}

+---------------+----------------------------+
| **Aliases:**  | > height carpet fiber mat  |
+---------------+----------------------------+
| **Mappings:** | > mixs:height_carper_fiber |
+---------------+----------------------------+

# Slot: herbicide_regm

"Information about treatment involving use of herbicides; information
about treatment involving use of growth hormones; should include the
name of herbicide, amount administered, treatment regimen including how
many times the treatment was repeated, how long each treatment lasted,
and the start and end time of the entire treatment; can include multiple
regimens"

URI: [nmdc:herbicide_regm](https://microbiomedata/meta/herbicide_regm)

## Domain and Range {#domain-and-range-336}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-373}

-   is_a: [attribute](attribute.md)

## Children {#children-343}

## Used by {#used-by-336}

## Other properties {#other-properties-296}

+---------------+-----------------------+
| **Aliases:**  | > herbicide regimen   |
+---------------+-----------------------+
| **Mappings:** | > mixs:herbicide_regm |
+---------------+-----------------------+

# Slot: highest similarity score

TODO: Yuri to fill in

URI:
<https://microbiomedata/schema/workflow_execution_activity/highest_similarity_score>

## Domain and Range {#domain-and-range-337}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-374}

## Children {#children-344}

-   [metabolite quantification➞highest similarity
    score](metabolite_quantification_highest_similarity_score.md)

## Used by {#used-by-337}

# Slot: horizon

Specific layer in the land area which measures parallel to the soil
surface and possesses physical characteristics which differ from the
layers above and beneath

URI: [nmdc:horizon](https://microbiomedata/meta/horizon)

## Domain and Range {#domain-and-range-338}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-375}

-   is_a: [attribute](attribute.md)

## Children {#children-345}

## Used by {#used-by-338}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-297}

+---------------+----------------+
| **Aliases:**  | > horizon      |
+---------------+----------------+
| **Mappings:** | > mixs:horizon |
+---------------+----------------+

# Slot: horizon_meth

Reference or method used in determining the horizon

URI: [nmdc:horizon_meth](https://microbiomedata/meta/horizon_meth)

## Domain and Range {#domain-and-range-339}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-376}

-   is_a: [attribute](attribute.md)

## Children {#children-346}

## Used by {#used-by-339}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-298}

+---------------+---------------------+
| **Aliases:**  | > horizon method    |
+---------------+---------------------+
| **Mappings:** | > mixs:horizon_meth |
+---------------+---------------------+

# Slot: host_age

"Age of host at the time of sampling; relevant scale depends on species
and study, e.g. Could be seconds for amoebae or centuries for trees"

URI: [nmdc:host_age](https://microbiomedata/meta/host_age)

## Domain and Range {#domain-and-range-340}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-377}

-   is_a: [attribute](attribute.md)

## Children {#children-347}

## Used by {#used-by-340}

## Other properties {#other-properties-299}

+---------------+-----------------+
| **Aliases:**  | > host age      |
+---------------+-----------------+
| **Mappings:** | > mixs:host_age |
+---------------+-----------------+

# Slot: host_blood_press_diast

"Resting diastolic blood pressure, measured as mm mercury"

URI:
[nmdc:host_blood_press_diast](https://microbiomedata/meta/host_blood_press_diast)

## Domain and Range {#domain-and-range-341}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-378}

-   is_a: [attribute](attribute.md)

## Children {#children-348}

## Used by {#used-by-341}

## Other properties {#other-properties-300}

+---------------+---------------------------------+
| **Aliases:**  | > host blood pressure diastolic |
+---------------+---------------------------------+
| **Mappings:** | > mixs:host_blood_press_diast   |
+---------------+---------------------------------+

# Slot: host_blood_press_syst

"Resting systolic blood pressure, measured as mm mercury"

URI:
[nmdc:host_blood_press_syst](https://microbiomedata/meta/host_blood_press_syst)

## Domain and Range {#domain-and-range-342}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-379}

-   is_a: [attribute](attribute.md)

## Children {#children-349}

## Used by {#used-by-342}

## Other properties {#other-properties-301}

+---------------+--------------------------------+
| **Aliases:**  | > host blood pressure systolic |
+---------------+--------------------------------+
| **Mappings:** | > mixs:host_blood_press_syst   |
+---------------+--------------------------------+

# Slot: host_body_habitat

Original body habitat where the sample was obtained from

URI:
[nmdc:host_body_habitat](https://microbiomedata/meta/host_body_habitat)

## Domain and Range {#domain-and-range-343}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-380}

-   is_a: [attribute](attribute.md)

## Children {#children-350}

## Used by {#used-by-343}

## Other properties {#other-properties-302}

+---------------+--------------------------+
| **Aliases:**  | > host body habitat      |
+---------------+--------------------------+
| **Mappings:** | > mixs:host_body_habitat |
+---------------+--------------------------+

# Slot: host_body_mass_index

"Body mass index, calculated as weight/(height)squared"

URI:
[nmdc:host_body_mass_index](https://microbiomedata/meta/host_body_mass_index)

## Domain and Range {#domain-and-range-344}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-381}

-   is_a: [attribute](attribute.md)

## Children {#children-351}

## Used by {#used-by-344}

## Other properties {#other-properties-303}

+---------------+-----------------------------+
| **Aliases:**  | > host body-mass index      |
+---------------+-----------------------------+
| **Mappings:** | > mixs:host_body_mass_index |
+---------------+-----------------------------+

# Slot: host_body_product

"Substance produced by the body, e.g. Stool, mucus, where the sample was
obtained from. For foundational model of anatomy ontology (fma) or
Uber-anatomy ontology (UBERON) terms, please see
<https://www.ebi.ac.uk/ols/ontologies/fma> or
<https://www.ebi.ac.uk/ols/ontologies/uberon>"

URI:
[nmdc:host_body_product](https://microbiomedata/meta/host_body_product)

## Domain and Range {#domain-and-range-345}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-382}

-   is_a: [attribute](attribute.md)

## Children {#children-352}

## Used by {#used-by-345}

## Other properties {#other-properties-304}

+---------------+--------------------------+
| **Aliases:**  | > host body product      |
+---------------+--------------------------+
| **Mappings:** | > mixs:host_body_product |
+---------------+--------------------------+

# Slot: host_body_site

"Name of body site where the sample was obtained from, such as a
specific organ or tissue (tongue, lung etc...). For foundational model
of anatomy ontology (fma) (v 4.11.0) or Uber-anatomy ontology (UBERON)
(v releases/2014-06-15) terms, please see
<http://purl.bioontology.org/ontology/FMA> or
<http://purl.bioontology.org/ontology/UBERON>"

URI: [nmdc:host_body_site](https://microbiomedata/meta/host_body_site)

## Domain and Range {#domain-and-range-346}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-383}

-   is_a: [attribute](attribute.md)

## Children {#children-353}

## Used by {#used-by-346}

## Other properties {#other-properties-305}

+---------------+-----------------------+
| **Aliases:**  | > host body site      |
+---------------+-----------------------+
| **Mappings:** | > mixs:host_body_site |
+---------------+-----------------------+

# Slot: host_body_temp

Core body temperature of the host when sample was collected

URI: [nmdc:host_body_temp](https://microbiomedata/meta/host_body_temp)

## Domain and Range {#domain-and-range-347}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-384}

-   is_a: [attribute](attribute.md)

## Children {#children-354}

## Used by {#used-by-347}

## Other properties {#other-properties-306}

+---------------+-------------------------+
| **Aliases:**  | > host body temperature |
+---------------+-------------------------+
| **Mappings:** | > mixs:host_body_temp   |
+---------------+-------------------------+

# Slot: host_color

The color of host

URI: [nmdc:host_color](https://microbiomedata/meta/host_color)

## Domain and Range {#domain-and-range-348}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-385}

-   is_a: [attribute](attribute.md)

## Children {#children-355}

## Used by {#used-by-348}

## Other properties {#other-properties-307}

+---------------+-------------------+
| **Aliases:**  | > host color      |
+---------------+-------------------+
| **Mappings:** | > mixs:host_color |
+---------------+-------------------+

# Slot: host_common_name

"Common name of the host, e.g. Human"

URI:
[nmdc:host_common_name](https://microbiomedata/meta/host_common_name)

## Domain and Range {#domain-and-range-349}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-386}

-   is_a: [attribute](attribute.md)

## Children {#children-356}

## Used by {#used-by-349}

## Other properties {#other-properties-308}

+---------------+-------------------------+
| **Aliases:**  | > host common name      |
+---------------+-------------------------+
| **Mappings:** | > mixs:host_common_name |
+---------------+-------------------------+

# Slot: host_diet

"Type of diet depending on the host, for animals omnivore, herbivore
etc., for humans high-fat, meditteranean etc.; can include multiple diet
types"

URI: [nmdc:host_diet](https://microbiomedata/meta/host_diet)

## Domain and Range {#domain-and-range-350}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-387}

-   is_a: [attribute](attribute.md)

## Children {#children-357}

## Used by {#used-by-350}

## Other properties {#other-properties-309}

+---------------+------------------+
| **Aliases:**  | > host diet      |
+---------------+------------------+
| **Mappings:** | > mixs:host_diet |
+---------------+------------------+

# Slot: host_disease_stat

"List of diseases with which the host has been diagnosed; can include
multiple diagnoses. The value of the field depends on host; for humans
the terms should be chosen from do (disease ontology) at
<http://www.disease-ontology.org>, other hosts are free text"

URI:
[nmdc:host_disease_stat](https://microbiomedata/meta/host_disease_stat)

## Domain and Range {#domain-and-range-351}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-388}

-   is_a: [attribute](attribute.md)

## Children {#children-358}

## Used by {#used-by-351}

## Other properties {#other-properties-310}

+---------------+--------------------------+
| **Aliases:**  | > host disease status    |
+---------------+--------------------------+
| **Mappings:** | > mixs:host_disease_stat |
+---------------+--------------------------+

# Slot: host_dry_mass

Measurement of dry mass

URI: [nmdc:host_dry_mass](https://microbiomedata/meta/host_dry_mass)

## Domain and Range {#domain-and-range-352}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-389}

-   is_a: [attribute](attribute.md)

## Children {#children-359}

## Used by {#used-by-352}

## Other properties {#other-properties-311}

+---------------+----------------------+
| **Aliases:**  | > host dry mass      |
+---------------+----------------------+
| **Mappings:** | > mixs:host_dry_mass |
+---------------+----------------------+

# Slot: host_family_relationship

Relationships to other hosts in the same study; can include multiple
relationships

URI:
[nmdc:host_family_relationship](https://microbiomedata/meta/host_family_relationship)

## Domain and Range {#domain-and-range-353}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-390}

-   is_a: [attribute](attribute.md)

## Children {#children-360}

## Used by {#used-by-353}

## Other properties {#other-properties-312}

+---------------+---------------------------------+
| **Aliases:**  | > host family relationship      |
+---------------+---------------------------------+
| **Mappings:** | > mixs:host_family_relationship |
+---------------+---------------------------------+

# Slot: host_genotype

Observed genotype

URI: [nmdc:host_genotype](https://microbiomedata/meta/host_genotype)

## Domain and Range {#domain-and-range-354}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-391}

-   is_a: [attribute](attribute.md)

## Children {#children-361}

## Used by {#used-by-354}

## Other properties {#other-properties-313}

+---------------+----------------------+
| **Aliases:**  | > host genotype      |
+---------------+----------------------+
| **Mappings:** | > mixs:host_genotype |
+---------------+----------------------+

# Slot: host_growth_cond

Literature reference giving growth conditions of the host

URI:
[nmdc:host_growth_cond](https://microbiomedata/meta/host_growth_cond)

## Domain and Range {#domain-and-range-355}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-392}

-   is_a: [attribute](attribute.md)

## Children {#children-362}

## Used by {#used-by-355}

## Other properties {#other-properties-314}

+---------------+--------------------------+
| **Aliases:**  | > host growth conditions |
+---------------+--------------------------+
| **Mappings:** | > mixs:host_growth_cond  |
+---------------+--------------------------+

# Slot: host_height

The height of subject

URI: [nmdc:host_height](https://microbiomedata/meta/host_height)

## Domain and Range {#domain-and-range-356}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-393}

-   is_a: [attribute](attribute.md)

## Children {#children-363}

## Used by {#used-by-356}

## Other properties {#other-properties-315}

+---------------+--------------------+
| **Aliases:**  | > host height      |
+---------------+--------------------+
| **Mappings:** | > mixs:host_height |
+---------------+--------------------+

# Slot: host_hiv_stat

"HIV status of subject, if yes HAART initiation status should also be
indicated as \[YES or NO\]"

URI: [nmdc:host_hiv_stat](https://microbiomedata/meta/host_hiv_stat)

## Domain and Range {#domain-and-range-357}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-394}

-   is_a: [attribute](attribute.md)

## Children {#children-364}

## Used by {#used-by-357}

## Other properties {#other-properties-316}

+---------------+----------------------+
| **Aliases:**  | > host HIV status    |
+---------------+----------------------+
| **Mappings:** | > mixs:host_hiv_stat |
+---------------+----------------------+

# Slot: host_infra_specific_name

Taxonomic information about the host below subspecies level

URI:
[nmdc:host_infra_specific_name](https://microbiomedata/meta/host_infra_specific_name)

## Domain and Range {#domain-and-range-358}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-395}

-   is_a: [attribute](attribute.md)

## Children {#children-365}

## Used by {#used-by-358}

## Other properties {#other-properties-317}

+---------------+---------------------------------+
| **Aliases:**  | > host infra-specific name      |
+---------------+---------------------------------+
| **Mappings:** | > mixs:host_infra_specific_name |
+---------------+---------------------------------+

# Slot: host_infra_specific_rank

"Taxonomic rank information about the host below subspecies level, such
as variety, form, rank etc."

URI:
[nmdc:host_infra_specific_rank](https://microbiomedata/meta/host_infra_specific_rank)

## Domain and Range {#domain-and-range-359}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-396}

-   is_a: [attribute](attribute.md)

## Children {#children-366}

## Used by {#used-by-359}

## Other properties {#other-properties-318}

+---------------+---------------------------------+
| **Aliases:**  | > host infra-specific rank      |
+---------------+---------------------------------+
| **Mappings:** | > mixs:host_infra_specific_rank |
+---------------+---------------------------------+

# Slot: host_last_meal

Content of last meal and time since feeding; can include multiple values

URI: [nmdc:host_last_meal](https://microbiomedata/meta/host_last_meal)

## Domain and Range {#domain-and-range-360}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-397}

-   is_a: [attribute](attribute.md)

## Children {#children-367}

## Used by {#used-by-360}

## Other properties {#other-properties-319}

+---------------+-----------------------+
| **Aliases:**  | > host last meal      |
+---------------+-----------------------+
| **Mappings:** | > mixs:host_last_meal |
+---------------+-----------------------+

# Slot: host_length

The length of subject

URI: [nmdc:host_length](https://microbiomedata/meta/host_length)

## Domain and Range {#domain-and-range-361}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-398}

-   is_a: [attribute](attribute.md)

## Children {#children-368}

## Used by {#used-by-361}

## Other properties {#other-properties-320}

+---------------+--------------------+
| **Aliases:**  | > host length      |
+---------------+--------------------+
| **Mappings:** | > mixs:host_length |
+---------------+--------------------+

# Slot: host_life_stage

Description of life stage of host

URI: [nmdc:host_life_stage](https://microbiomedata/meta/host_life_stage)

## Domain and Range {#domain-and-range-362}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-399}

-   is_a: [attribute](attribute.md)

## Children {#children-369}

## Used by {#used-by-362}

## Other properties {#other-properties-321}

+---------------+------------------------+
| **Aliases:**  | > host life stage      |
+---------------+------------------------+
| **Mappings:** | > mixs:host_life_stage |
+---------------+------------------------+

# Slot: host_name

URI: [nmdc:host_name](https://microbiomedata/meta/host_name)

## Domain and Range {#domain-and-range-363}

None → 0..1 [String](types/String.md)

## Parents {#parents-400}

## Children {#children-370}

## Used by {#used-by-363}

-   [Biosample](Biosample.md)

# Slot: host_occupation

Most frequent job performed by subject

URI: [nmdc:host_occupation](https://microbiomedata/meta/host_occupation)

## Domain and Range {#domain-and-range-364}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-401}

-   is_a: [attribute](attribute.md)

## Children {#children-371}

## Used by {#used-by-364}

## Other properties {#other-properties-322}

+---------------+------------------------+
| **Aliases:**  | > host occupation      |
+---------------+------------------------+
| **Mappings:** | > mixs:host_occupation |
+---------------+------------------------+

# Slot: host_phenotype

"Phenotype of human or other host. For phenotypic quality ontology
(pato) (v 2018-03-27) terms, please see
<http://purl.bioontology.org/ontology/pato>. For Human Phenotype
Ontology (HP) (v 2018-06-13) please see
<http://purl.bioontology.org/ontology/HP>"

URI: [nmdc:host_phenotype](https://microbiomedata/meta/host_phenotype)

## Domain and Range {#domain-and-range-365}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-402}

-   is_a: [attribute](attribute.md)

## Children {#children-372}

## Used by {#used-by-365}

## Other properties {#other-properties-323}

+---------------+-----------------------+
| **Aliases:**  | > host phenotype      |
+---------------+-----------------------+
| **Mappings:** | > mixs:host_phenotype |
+---------------+-----------------------+

# Slot: host_pred_appr

Tool or approach used for host prediction

URI: [nmdc:host_pred_appr](https://microbiomedata/meta/host_pred_appr)

## Domain and Range {#domain-and-range-366}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-403}

-   is_a: [attribute](attribute.md)

## Children {#children-373}

## Used by {#used-by-366}

## Other properties {#other-properties-324}

+-----------------+----------------------------+
| **Aliases:**    | > host prediction approach |
+-----------------+----------------------------+
| **Mappings:**   | > mixs:host_pred_appr      |
+-----------------+----------------------------+
| **In Subsets:** | > sequencing               |
+-----------------+----------------------------+

# Slot: host_pred_est_acc

"For each tool or approach used for host prediction, estimated false
discovery rates should be included, either computed de novo or from the
literature"

URI:
[nmdc:host_pred_est_acc](https://microbiomedata/meta/host_pred_est_acc)

## Domain and Range {#domain-and-range-367}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-404}

-   is_a: [attribute](attribute.md)

## Children {#children-374}

## Used by {#used-by-367}

## Other properties {#other-properties-325}

+-----------------+--------------------------------------+
| **Aliases:**    | > host prediction estimated accuracy |
+-----------------+--------------------------------------+
| **Mappings:**   | > mixs:host_pred_est_acc             |
+-----------------+--------------------------------------+
| **In Subsets:** | > sequencing                         |
+-----------------+--------------------------------------+

# Slot: host_pulse

"Resting pulse, measured as beats per minute"

URI: [nmdc:host_pulse](https://microbiomedata/meta/host_pulse)

## Domain and Range {#domain-and-range-368}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-405}

-   is_a: [attribute](attribute.md)

## Children {#children-375}

## Used by {#used-by-368}

## Other properties {#other-properties-326}

+---------------+-------------------+
| **Aliases:**  | > host pulse      |
+---------------+-------------------+
| **Mappings:** | > mixs:host_pulse |
+---------------+-------------------+

# Slot: host_sex

Physical sex of the host

URI: [nmdc:host_sex](https://microbiomedata/meta/host_sex)

## Domain and Range {#domain-and-range-369}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-406}

-   is_a: [attribute](attribute.md)

## Children {#children-376}

## Used by {#used-by-369}

## Other properties {#other-properties-327}

+---------------+-----------------+
| **Aliases:**  | > host sex      |
+---------------+-----------------+
| **Mappings:** | > mixs:host_sex |
+---------------+-----------------+

# Slot: host_shape

Morphological shape of host

URI: [nmdc:host_shape](https://microbiomedata/meta/host_shape)

## Domain and Range {#domain-and-range-370}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-407}

-   is_a: [attribute](attribute.md)

## Children {#children-377}

## Used by {#used-by-370}

## Other properties {#other-properties-328}

+---------------+-------------------+
| **Aliases:**  | > host shape      |
+---------------+-------------------+
| **Mappings:** | > mixs:host_shape |
+---------------+-------------------+

# Slot: host_spec_range

The NCBI taxonomy identifier of the specific host if it is known

URI: [nmdc:host_spec_range](https://microbiomedata/meta/host_spec_range)

## Domain and Range {#domain-and-range-371}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-408}

-   is_a: [attribute](attribute.md)

## Children {#children-378}

## Used by {#used-by-371}

## Other properties {#other-properties-329}

+-----------------+--------------------------------+
| **Aliases:**    | > host specificity or range    |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:host_spec_range         |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: host_subject_id

"A unique identifier by which each subject can be referred to,
de-identified, e.g. #131"

URI: [nmdc:host_subject_id](https://microbiomedata/meta/host_subject_id)

## Domain and Range {#domain-and-range-372}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-409}

-   is_a: [attribute](attribute.md)

## Children {#children-379}

## Used by {#used-by-372}

## Other properties {#other-properties-330}

+---------------+------------------------+
| **Aliases:**  | > host subject id      |
+---------------+------------------------+
| **Mappings:** | > mixs:host_subject_id |
+---------------+------------------------+

# Slot: host_substrate

The growth substrate of the host

URI: [nmdc:host_substrate](https://microbiomedata/meta/host_substrate)

## Domain and Range {#domain-and-range-373}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-410}

-   is_a: [attribute](attribute.md)

## Children {#children-380}

## Used by {#used-by-373}

## Other properties {#other-properties-331}

+---------------+-----------------------+
| **Aliases:**  | > host substrate      |
+---------------+-----------------------+
| **Mappings:** | > mixs:host_substrate |
+---------------+-----------------------+

# Slot: host_taxid

"NCBI taxon id of the host, e.g. 9606"

URI: [nmdc:host_taxid](https://microbiomedata/meta/host_taxid)

## Domain and Range {#domain-and-range-374}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-411}

-   is_a: [attribute](attribute.md)

## Children {#children-381}

## Used by {#used-by-374}

## Other properties {#other-properties-332}

+---------------+-------------------+
| **Aliases:**  | > host taxid      |
+---------------+-------------------+
| **Mappings:** | > mixs:host_taxid |
+---------------+-------------------+

# Slot: host_tot_mass

"Total mass of the host at collection, the unit depends on host"

URI: [nmdc:host_tot_mass](https://microbiomedata/meta/host_tot_mass)

## Domain and Range {#domain-and-range-375}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-412}

-   is_a: [attribute](attribute.md)

## Children {#children-382}

## Used by {#used-by-375}

## Other properties {#other-properties-333}

+---------------+----------------------+
| **Aliases:**  | > host total mass    |
+---------------+----------------------+
| **Mappings:** | > mixs:host_tot_mass |
+---------------+----------------------+

# Slot: host_wet_mass

Measurement of wet mass

URI: [nmdc:host_wet_mass](https://microbiomedata/meta/host_wet_mass)

## Domain and Range {#domain-and-range-376}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-413}

-   is_a: [attribute](attribute.md)

## Children {#children-383}

## Used by {#used-by-376}

## Other properties {#other-properties-334}

+---------------+----------------------+
| **Aliases:**  | > host wet mass      |
+---------------+----------------------+
| **Mappings:** | > mixs:host_wet_mass |
+---------------+----------------------+

# Slot: hrt

"Whether subject had hormone replacement theraphy, and if yes start
date"

URI: [nmdc:hrt](https://microbiomedata/meta/hrt)

## Domain and Range {#domain-and-range-377}

None → 0..1 [TimestampValue](TimestampValue.md)

## Parents {#parents-414}

-   is_a: [attribute](attribute.md)

## Children {#children-384}

## Used by {#used-by-377}

## Other properties {#other-properties-335}

+---------------+------------+
| **Aliases:**  | > HRT      |
+---------------+------------+
| **Mappings:** | > mixs:hrt |
+---------------+------------+

# Slot: humidity

"Amount of water vapour in the air, at the time of sampling"

URI: [nmdc:humidity](https://microbiomedata/meta/humidity)

## Domain and Range {#domain-and-range-378}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-415}

-   is_a: [attribute](attribute.md)

## Children {#children-385}

## Used by {#used-by-378}

## Other properties {#other-properties-336}

+---------------+-----------------+
| **Aliases:**  | > humidity      |
+---------------+-----------------+
| **Mappings:** | > mixs:humidity |
+---------------+-----------------+

# Slot: humidity_regm

"Information about treatment involving an exposure to varying degree of
humidity; information about treatment involving use of growth hormones;
should include amount of humidity administered, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include multiple regimens"

URI: [nmdc:humidity_regm](https://microbiomedata/meta/humidity_regm)

## Domain and Range {#domain-and-range-379}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-416}

-   is_a: [attribute](attribute.md)

## Children {#children-386}

## Used by {#used-by-379}

## Other properties {#other-properties-337}

+---------------+----------------------+
| **Aliases:**  | > humidity regimen   |
+---------------+----------------------+
| **Mappings:** | > mixs:humidity_regm |
+---------------+----------------------+

# Slot: hysterectomy

Specification of whether hysterectomy was performed

URI: [nmdc:hysterectomy](https://microbiomedata/meta/hysterectomy)

## Domain and Range {#domain-and-range-380}

None → 0..1 [BooleanValue](BooleanValue.md)

## Parents {#parents-417}

-   is_a: [attribute](attribute.md)

## Children {#children-387}

## Used by {#used-by-380}

## Other properties {#other-properties-338}

+---------------+---------------------+
| **Aliases:**  | > hysterectomy      |
+---------------+---------------------+
| **Mappings:** | > mixs:hysterectomy |
+---------------+---------------------+

# Slot: id

A unique identifier for a thing. Must be either a CURIE shorthand for a
URI or a complete URI

URI: <https://microbiomedata/schema/workflow_execution_activity/id>

## Domain and Range {#domain-and-range-381}

None → 1..1 [String](types/String.md)

## Parents {#parents-418}

## Children {#children-388}

-   [person➞id](person_id.md)

## Used by {#used-by-381}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [Activity](Activity.md)
-   [ChemicalEntity](ChemicalEntity.md)
-   [EnvironmentalMaterialTerm](EnvironmentalMaterialTerm.md)
-   [GeneProduct](GeneProduct.md)
-   [Instrument](Instrument.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md)
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [NamedThing](NamedThing.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md)
-   [OntologyClass](OntologyClass.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)

# Slot: identifier

URI: [nmdc:identifier](https://microbiomedata/meta/identifier)

## Domain and Range {#domain-and-range-382}

None → 0..1 [String](types/String.md)

## Parents {#parents-419}

## Children {#children-389}

## Used by {#used-by-382}

-   [Biosample](Biosample.md)

# Slot: ihmc_ethnicity

Ethnicity of the subject

URI: [nmdc:ihmc_ethnicity](https://microbiomedata/meta/ihmc_ethnicity)

## Domain and Range {#domain-and-range-383}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-420}

-   is_a: [attribute](attribute.md)

## Children {#children-390}

## Used by {#used-by-383}

## Other properties {#other-properties-339}

+---------------+-----------------------+
| **Aliases:**  | > IHMC ethnicity      |
+---------------+-----------------------+
| **Mappings:** | > mixs:ihmc_ethnicity |
+---------------+-----------------------+

# Slot: ihmc_medication_code

Can include multiple medication codes

URI:
[nmdc:ihmc_medication_code](https://microbiomedata/meta/ihmc_medication_code)

## Domain and Range {#domain-and-range-384}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-421}

-   is_a: [attribute](attribute.md)

## Children {#children-391}

## Used by {#used-by-384}

## Other properties {#other-properties-340}

+---------------+-----------------------------+
| **Aliases:**  | > IHMC medication code      |
+---------------+-----------------------------+
| **Mappings:** | > mixs:ihmc_medication_code |
+---------------+-----------------------------+

# Slot: inchi

URI: <https://microbiomedata/schema/workflow_execution_activity/inchi>

## Domain and Range {#domain-and-range-385}

None → 0..1 [String](types/String.md)

## Parents {#parents-422}

## Children {#children-392}

-   [chemical entity➞inchi](chemical_entity_inchi.md)

## Used by {#used-by-385}

# Slot: inchi key

URI:
<https://microbiomedata/schema/workflow_execution_activity/inchi_key>

## Domain and Range {#domain-and-range-386}

None → 0..1 [String](types/String.md)

## Parents {#parents-423}

## Children {#children-393}

-   [chemical entity➞inchi key](chemical_entity_inchi_key.md)

## Used by {#used-by-386}

# NMDC-Workflow-Exectution schema

## Classes

-   [MAGBin](MAGBin.md)
-   [Activity](Activity.md) - a provence-generating activity
    -   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)
        -Represents an instance of an execution of a particular workflow
        -   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
        -   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
        -   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
        -   [MetagenomeAssembly](MetagenomeAssembly.md)
        -   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
        -   [MetatranscriptomeActivity](MetatranscriptomeActivity.md) -A
            metatranscriptome activity that e.g. pools assembly and
            annotation activity.
        -   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
        -   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
        -   [NomAnalysisActivity](NomAnalysisActivity.md)
        -   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
        -   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [Agent](Agent.md) - a provence-generating agent
-   [AttributeValue](AttributeValue.md) - The value for any value of a
    attribute for a sample. This object can hold both the un-normalized
    atomic value and the structured value
    -   [BooleanValue](BooleanValue.md) - A value that is a boolean
    -   [ControlledTermValue](ControlledTermValue.md) - A controlled
        term or class from an ontology
    -   [GeolocationValue](GeolocationValue.md) - A normalized value for
        a location on the earth's surface
    -   [ImageValue](ImageValue.md) - An attribute value representing an
        image.
    -   [IntegerValue](IntegerValue.md) - A value that is an integer
    -   [PersonValue](PersonValue.md) - An attribute value representing
        a person
    -   [QuantityValue](QuantityValue.md) - A simple quantity, e.g. 2cm
    -   [TextValue](TextValue.md) - A basic string value
    -   [TimestampValue](TimestampValue.md) - A value that is a
        timestamp. The range should be ISO-8601
    -   [UrlValue](UrlValue.md) - A value that is a string that conforms
        to URL syntax
-   [MetaboliteQuantification](MetaboliteQuantification.md) - This is
    used to link a metabolomics analysis workflow to a specific
    metabolite
-   [NamedThing](NamedThing.md) - a databased entity or concept/class
    -   [GeneProduct](GeneProduct.md) - A molecule encoded by a gene
        that has an evolved function
    -   [Instrument](Instrument.md) - A material entity that is designed
        to perform a function in a scientific investigation, but is not
        a reagent\[OBI\].
    -   [OntologyClass](OntologyClass.md)
        -   [ChemicalEntity](ChemicalEntity.md) - An atom or molecule
            that can be represented with a chemical formula. Include
            lipids, glycans, natural products, drugs. There may be
            different terms for distinct acid-base forms, protonation
            states
        -   [EnvironmentalMaterialTerm](EnvironmentalMaterialTerm.md)
    -   [Person](Person.md) - represents a person, such as a researcher
-   [PeptideQuantification](PeptideQuantification.md) - This is used to
    link a metaproteomics analysis workflow to a specific peptide
    sequence and related information
-   [ProteinQuantification](ProteinQuantification.md) - This is used to
    link a metaproteomics analysis workflow to a specific protein

## Mixins

## Slots

-   [GOLD identifiers](GOLD_identifiers.md)
-   [INSDC identifiers](INSDC_identifiers.md) - Any identifier covered
    by the International Nucleotide Sequence Database Collaboration
-   [MGnify identifiers](MGnify_identifiers.md)
-   [abstract](abstract.md) - The abstract of manuscript/grant
    associated with the entity; i.e., a summary of the resource.
-   [acted on behalf of](acted_on_behalf_of.md)
-   [all proteins](all_proteins.md) - the list of protein identifiers
    that are associated with the peptide sequence
    -   [peptide quantification➞all
        proteins](peptide_quantification_all_proteins.md)
    -   [protein quantification➞all
        proteins](protein_quantification_all_proteins.md) - the grouped
        list of protein identifiers associated with the peptide
        sequences that were grouped to a best protein
-   [alternate emails](alternate_emails.md) - One or more other email
    addresses for an entity.
-   [alternative descriptions](alternative_descriptions.md) - A list of
    alternative descriptions for the entity. The distinction between
    desciption and alternative descriptions is application-specific.
-   [alternative identifiers](alternative_identifiers.md) - A list of
    alternative identifiers for the entity.
    -   [external database
        identifiers](external_database_identifiers.md) - Link to
        corresponding identifier in external database
        -   [GOLD sequencing project
            identifiers](GOLD_sequencing_project_identifiers.md)
            -\['identifiers for corresponding sequencing project in
            GOLD'\]
        -   [INSDC experiment
            identifiers](INSDC_experiment_identifiers.md)
        -   [analysis identifiers](analysis_identifiers.md)
            -   [GOLD analysis project
                identifiers](GOLD_analysis_project_identifiers.md)
                -\['identifiers for corresponding analysis project in
                GOLD'\]
            -   [INSDC analysis
                identifiers](INSDC_analysis_identifiers.md)
            -   [MGnify analysis
                identifiers](MGnify_analysis_identifiers.md)
        -   [omics processing
            identifiers](omics_processing_identifiers.md)
        -   [sample identifiers](sample_identifiers.md)
            -   [GOLD sample identifiers](GOLD_sample_identifiers.md)
                -\['identifiers for corresponding sample in GOLD'\]
            -   [INSDC biosample
                identifiers](INSDC_biosample_identifiers.md)
                -\['identifiers for corresponding sample in INSDC'\]
            -   [INSDC secondary sample
                identifiers](INSDC_secondary_sample_identifiers.md)
                -\['secondary identifiers for corresponding sample in
                INSDC'\]
        -   [study identifiers](study_identifiers.md)
            -   [GOLD study identifiers](GOLD_study_identifiers.md)
                -\['identifiers for corresponding project in GOLD'\]
            -   [INSDC SRA ENA study
                identifiers](INSDC_SRA_ENA_study_identifiers.md)
                -\['identifiers for corresponding project in INSDC SRA /
                ENA'\]
            -   [INSDC bioproject
                identifiers](INSDC_bioproject_identifiers.md)
                -\['identifiers for corresponding project in INSDC
                Bioproject'\]
            -   [MGnify project
                identifiers](MGnify_project_identifiers.md)
                -\['identifiers for corresponding project in MGnify'\]
-   [alternative names](alternative_names.md) - A list of alternative
    names used to refer to the entity. The distinction between name and
    alternative names is application-specific.
-   [alternative titles](alternative_titles.md) - A list of alternative
    titles for the entity. The distinction between title and alternative
    titles is application-specific.
-   [assembly identifiers](assembly_identifiers.md)
    -   [INSDC assembly identifiers](INSDC_assembly_identifiers.md)
-   [attribute](attribute.md) - A attribute of a biosample. Examples:
    depth, habitat, material. For NMDC, attributes SHOULD be mapped to
    terms within a MIxS template
    -   [display order](display_order.md) - When rendering information,
        this attribute to specify the order in which the information
        should be rendered.
    -   [execution resource](execution_resource.md) - Example:
        NERSC-Cori
        -   [workflow execution activity➞execution
            resource](workflow_execution_activity_execution_resource.md)
    -   [file size bytes](file_size_bytes.md) - Size of the file in
        bytes
    -   [git url](git_url.md) - Example:
        <https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>
        -   [workflow execution activity➞git
            url](workflow_execution_activity_git_url.md)
    -   [md5 checksum](md5_checksum.md) - MD5 checksum of file
        (pre-compressed)
    -   [url](url.md)
-   [best protein](best_protein.md) - the specific protein identifier
    most correctly associated with the peptide sequence
    -   [peptide quantification➞best
        protein](peptide_quantification_best_protein.md)
    -   [protein quantification➞best
        protein](protein_quantification_best_protein.md) - the specific
        protein identifier most correctly grouped to its associated
        peptide sequences
-   [binned contig num](binned_contig_num.md)
-   [chemical formula](chemical_formula.md) - A generic grouping for
    miolecular formulae and empirican formulae
    -   [chemical entity➞chemical
        formula](chemical_entity_chemical_formula.md)
-   [description](description.md) - a human-readable description of a
    thing
-   [email](email.md) - An email address for an entity such as a person.
    This should be the primarly email address used.
-   [ended at time](ended_at_time.md)
    -   [workflow execution activity➞ended at
        time](workflow_execution_activity_ended_at_time.md)
-   [has boolean value](has_boolean_value.md) - Links a quantity value
    to a boolean
-   [has calibration](has_calibration.md) - TODO: Yuri to fill in
    -   [metabolomics analysis activity➞has
        calibration](metabolomics_analysis_activity_has_calibration.md)
    -   [nom analysis activity➞has
        calibration](nom_analysis_activity_has_calibration.md) - A
        reference to a file that holds calibration information.
-   [has input](has_input.md) - An input to a process.
    -   [workflow execution activity➞has
        input](workflow_execution_activity_has_input.md)
        -   [read QC analysis activity➞has
            input](read_QC_analysis_activity_has_input.md)
-   [has metabolite quantifications](has_metabolite_quantifications.md)
    -   [metabolomics analysis activity➞has metabolite
        quantifications](metabolomics_analysis_activity_has_metabolite_quantifications.md)
-   [has numeric value](has_numeric_value.md) - Links a quantity value
    to a number
    -   [has maximum numeric value](has_maximum_numeric_value.md) - The
        maximum value part, expressed as number, of the quantity value
        when the value covers a range.
    -   [has minimum numeric value](has_minimum_numeric_value.md) - The
        minimum value part, expressed as number, of the quantity value
        when the value covers a range.
    -   [quantity value➞has numeric
        value](quantity_value_has_numeric_value.md) - The number part of
        the quantity
-   [has output](has_output.md) - An output biosample to a processing
    step
    -   [workflow execution activity➞has
        output](workflow_execution_activity_has_output.md)
        -   [read QC analysis activity➞has
            output](read_QC_analysis_activity_has_output.md)
-   [has peptide quantifications](has_peptide_quantifications.md)
    -   [metaproteomics analysis activity➞has peptide
        quantifications](metaproteomics_analysis_activity_has_peptide_quantifications.md)
-   [has raw value](has_raw_value.md) - The value that was specified for
    an annotation in raw form, i.e. a string. E.g. "2 cm" or "2-4 cm"
    -   [geolocation value➞has raw
        value](geolocation_value_has_raw_value.md) - The raw value for a
        geolocation should follow {lat} {long}
    -   [person value➞has raw value](person_value_has_raw_value.md) -The
        full name of the Investgator in format FIRST LAST.
    -   [quantity value➞has raw value](quantity_value_has_raw_value.md)
        \- Unnormalized atomic string representation, should in syntax
        {number} {unit}
-   [has unit](has_unit.md) - Links a quantity value to a unit
    -   [quantity value➞has unit](quantity_value_has_unit.md) - The unit
        of the quantity
-   [highest similarity score](highest_similarity_score.md) - TODO: Yuri
    to fill in
    -   [metabolite quantification➞highest similarity
        score](metabolite_quantification_highest_similarity_score.md)
-   [id](id.md) - A unique identifier for a thing. Must be either a
    CURIE shorthand for a URI or a complete URI
    -   [person➞id](person_id.md) - Should be an ORCID. Specify in CURIE
        format. E.g ORCID:0000-1111-...
-   [inchi](inchi.md)
    -   [chemical entity➞inchi](chemical_entity_inchi.md)
-   [inchi key](inchi_key.md)
    -   [chemical entity➞inchi key](chemical_entity_inchi_key.md)
-   [input contig num](input_contig_num.md)
-   [input_read_bases](input_read_bases.md) - TODO
    -   [read QC analysis
        activity➞input_read_bases](read_QC_analysis_activity_input_read_bases.md)
-   [keywords](keywords.md) - A list of keywords that used to associate
    the entity. Keywords SHOULD come from controlled vocabularies,
    including MESH, ENVO.
-   [language](language.md) - Should use ISO 639-1 code e.g. "en", "fr"
-   [latitude](latitude.md) - latitude
-   [longitude](longitude.md) - longitude
-   [lowDepth contig num](lowDepth_contig_num.md)
-   [➞bin name](mAGBin__bin_name.md)
-   [➞bin quality](mAGBin__bin_quality.md)
-   [➞completeness](mAGBin__completeness.md)
-   [➞contamination](mAGBin__contamination.md)
-   [➞gene count](mAGBin__gene_count.md)
-   [➞gtdbtk class](mAGBin__gtdbtk_class.md)
-   [➞gtdbtk domain](mAGBin__gtdbtk_domain.md)
-   [➞gtdbtk family](mAGBin__gtdbtk_family.md)
-   [➞gtdbtk genus](mAGBin__gtdbtk_genus.md)
-   [➞gtdbtk order](mAGBin__gtdbtk_order.md)
-   [➞gtdbtk phylum](mAGBin__gtdbtk_phylum.md)
-   [➞gtdbtk species](mAGBin__gtdbtk_species.md)
-   [➞num 16s](mAGBin__num_16s.md)
-   [➞num 23s](mAGBin__num_23s.md)
-   [➞num 5s](mAGBin__num_5s.md)
-   [➞num tRNA](mAGBin__num_tRNA.md)
-   [➞number of contig](mAGBin__number_of_contig.md)
-   [mags list](mags_list.md)
-   [metabolite quantified](metabolite_quantified.md) - the specific
    metabolite identifier
    -   [metabolite quantification➞metabolite
        quantified](metabolite_quantification_metabolite_quantified.md)
-   [metagenome assembly parameter](metagenome_assembly_parameter.md)
    -   [asm_score](asm_score.md) - A score for comparing metagenomic
        assembly quality from same sample.
    -   [contig_bp](contig_bp.md) - Total size in bp of all contigs.
    -   [contigs](contigs.md) - The sum of the (length\*log(length)) of
        all contigs, times some constant. Increase the contiguity, the
        score will increase
    -   [ctg_L50](ctg_L50.md) - Given a set of contigs, the L50 is
        defined as the sequence length of the shortest contig at 50% of
        the total genome length.
    -   [ctg_L90](ctg_L90.md) - The L90 statistic is less than or equal
        to the L50 statistic; it is the length for which the collection
        of all contigs of that length or longer contains at least 90% of
        the sum of the lengths of all contigs.
    -   [ctg_N50](ctg_N50.md) - Given a set of contigs, each with its
        own length, the N50 count is defined as the smallest number of
        contigs whose length sum makes up half of genome size.
    -   [ctg_N90](ctg_N90.md) - Given a set of contigs, each with its
        own length, the N90 count is defined as the smallest number of
        contigs whose length sum makes up 90% of genome size.
    -   [ctg_logsum](ctg_logsum.md) - Maximum contig length.
    -   [ctg_max](ctg_max.md) - Maximum contig length.
    -   [ctg_powsum](ctg_powsum.md) - Powersum of all contigs is the
        same as logsum except that it uses the sum of
        (length\*(length\^P)) for some power P (default P=0.25).
    -   [gap_pct](gap_pct.md) - The gap size percentage of all
        scaffolds.
    -   [gc_avg](gc_avg.md) - Average of GC content of all contigs.
    -   [gc_std](gc_std.md) - Standard deviation of GC content of all
        contigs.
    -   [num_aligned_reads](num_aligned_reads.md) - The sequence count
        number of input reads aligned to assembled contigs.
    -   [num_input_reads](num_input_reads.md) - The sequence count
        number of input reads for assembly.
    -   [scaf_L50](scaf_L50.md) - Given a set of scaffolds, the L50 is
        defined as the sequence length of the shortest scaffold at 50%
        of the total genome length.
    -   [scaf_L90](scaf_L90.md) - The L90 statistic is less than or
        equal to the L50 statistic; it is the length for which the
        collection of all scaffolds of that length or longer contains at
        least 90% of the sum of the lengths of all scaffolds.
    -   [scaf_N50](scaf_N50.md) - Given a set of scaffolds, each with
        its own length, the N50 count is defined as the smallest number
        of scaffolds whose length sum makes up half of genome size.
    -   [scaf_N90](scaf_N90.md) - Given a set of scaffolds, each with
        its own length, the N90 count is defined as the smallest number
        of scaffolds whose length sum makes up 90% of genome size.
    -   [scaf_bp](scaf_bp.md) - Total size in bp of all scaffolds.
    -   [scaf_l\_gt50K](scaf_l_gt50K.md) - Total size in bp of all
        scaffolds greater than 50 KB.
    -   [scaf_logsum](scaf_logsum.md) - The sum of the
        (length\*log(length)) of all scaffolds, times some constant.
        Increase the contiguity, the score will increase
    -   [scaf_max](scaf_max.md) - Maximum scaffold length.
    -   [scaf_n\_gt50K](scaf_n_gt50K.md) - Total sequence count of
        scaffolds greater than 50 KB.
    -   [scaf_pct_gt50K](scaf_pct_gt50K.md) - Total sequence size
        percentage of scaffolds greater than 50 KB.
    -   [scaf_powsum](scaf_powsum.md) - Powersum of all scaffolds is the
        same as logsum except that it uses the sum of
        (length\*(length\^P)) for some power P (default P=0.25).
    -   [scaffolds](scaffolds.md) - Total sequence count of all
        scaffolds.
-   [min_q\_value](min_q_value.md) - smallest Q-Value associated with
    the peptide sequence as provided by MSGFPlus tool
    -   [peptide
        quantification➞min_q\_value](peptide_quantification_min_q_value.md)
-   [name](name.md) - A human readable label for an entity
    -   [person value➞name](person_value_name.md) - The full name of the
        Investgator. It should follow the format FIRST \[MIDDLE NAME\|
        MIDDLE INITIAL\] LAST, where MIDDLE NAME\| MIDDLE INITIAL is
        optional.
-   [objective](objective.md) - The scientific objectives associated
    with the entity. It SHOULD correspond to scientific norms for
    objectives field in a structured abstract.
-   [orcid](orcid.md) - The ORICD of a person.
-   [output_read_bases](output_read_bases.md) - TODO
    -   [read QC analysis
        activity➞output_read_bases](read_QC_analysis_activity_output_read_bases.md)
-   [part of](part_of.md) - Links a resource to another resource that
    either logically or physically includes it.
-   [peptide sequence](peptide_sequence.md)
    -   [peptide quantification➞peptide
        sequence](peptide_quantification_peptide_sequence.md)
-   [peptide_sequence_count](peptide_sequence_count.md) - count of
    peptide sequences grouped to the best_protein
    -   [protein
        quantification➞peptide_sequence_count](protein_quantification_peptide_sequence_count.md)
-   [peptide_spectral_count](peptide_spectral_count.md) - sum of filter
    passing MS2 spectra associated with the peptide sequence within a
    given LC-MS/MS data file
    -   [peptide
        quantification➞peptide_spectral_count](peptide_quantification_peptide_spectral_count.md)
-   [peptide_sum_masic_abundance](peptide_sum_masic_abundance.md)
    -combined MS1 extracted ion chromatograms derived from MS2 spectra
    associated with the peptide sequence from a given LC-MS/MS data file
    using the MASIC tool
    -   [peptide
        quantification➞peptide_sum_masic_abundance](peptide_quantification_peptide_sum_masic_abundance.md)
-   [profile image url](profile_image_url.md) - A url that points to an
    image of a person.
-   [protein_spectral_count](protein_spectral_count.md) - sum of filter
    passing MS2 spectra associated with the best protein within a given
    LC-MS/MS data file
    -   [protein
        quantification➞protein_spectral_count](protein_quantification_protein_spectral_count.md)
-   [protein_sum_masic_abundance](protein_sum_masic_abundance.md)
    -combined MS1 extracted ion chromatograms derived from MS2 spectra
    associated with the best protein from a given LC-MS/MS data file
    using the MASIC tool
    -   [protein
        quantification➞protein_sum_masic_abundance](protein_quantification_protein_sum_masic_abundance.md)
-   [publications](publications.md) - A list of publications that are
    assocatiated with the entity. The publicatons SHOULD be given using
    an identifier, such as a DOI or Pubmed ID, if possible.
-   [read QC analysis statistic](read_QC_analysis_statistic.md)
    -   [input base count](input_base_count.md) - The nucleotide base
        count number of input reads for QC analysis.
    -   [input read count](input_read_count.md) - The sequence count
        number of input reads for QC analysis.
    -   [output base count](output_base_count.md) - After QC analysis
        nucleotide base count number.
    -   [output read count](output_read_count.md) - After QC analysis
        sequence count number.
-   [smiles](smiles.md) - A string encoding of a molecular graph, no
    chiral or isotopic information. There are usually a large number of
    valid SMILES which represent a given structure. For example, CCO,
    OCC and C(O)C all specify the structure of ethanol.
    -   [chemical entity➞smiles](chemical_entity_smiles.md)
-   [started at time](started_at_time.md)
    -   [workflow execution activity➞started at
        time](workflow_execution_activity_started_at_time.md)
-   [term](term.md) - pointer to an ontology class
-   [title](title.md) - A name given to the entity that differs from the
    name/label programatically assigned to it. For example, when
    extracting study information for GOLD, the GOLD system has assigned
    a name/label. However, for display purposes, we may also wish the
    capture the title of the proposal that was used to fund the study.
-   [too short contig num](too_short_contig_num.md)
-   [type](type.md) - An optional string that specifies the type object.
    This is used to allow for searches for different kinds of objects.
    -   [attribute value➞type](attribute_value_type.md) - An optional
        string that specified the type of object.
    -   [workflow execution
        activity➞type](workflow_execution_activity_type.md)
-   [unbinned contig num](unbinned_contig_num.md)
-   [used](used.md)
    -   [metabolomics analysis
        activity➞used](metabolomics_analysis_activity_used.md) - The
        instrument used to collect the data used in the analysis
    -   [metaproteomics analysis
        activity➞used](metaproteomics_analysis_activity_used.md) - The
        instrument used to collect the data used in the analysis
    -   [nom analysis activity➞used](nom_analysis_activity_used.md) -The
        instrument used to collect the data used in the analysis
-   [was associated with](was_associated_with.md)
    -   [workflow execution activity➞was associated
        with](workflow_execution_activity_was_associated_with.md) - the
        agent/entity associated with the generation of the file
-   [was generated by](was_generated_by.md)
-   [was informed by](was_informed_by.md)
    -   [workflow execution activity➞was informed
        by](workflow_execution_activity_was_informed_by.md)
-   [websites](websites.md) - A list of websites that are assocatiated
    with the entity.

## Enums

## Subsets

-   [Environment](Environment.md)
-   [Investigation](Investigation.md)
-   [MixsExtension](MixsExtension.md)
-   [NucleicAcidSequenceSource](NucleicAcidSequenceSource.md)
-   [Sequencing](Sequencing.md)
-   [Soil](Soil.md)
-   [WorkflowSubset](WorkflowSubset.md) - Subset consisting of just the
    workflow exectution activities

## Types

### Built in

-   **Bool**
-   **Decimal**
-   **ElementIdentifier**
-   **NCName**
-   **NodeIdentifier**
-   **URI**
-   **URIorCURIE**
-   **XSDDate**
-   **XSDDateTime**
-   **XSDTime**
-   **float**
-   **int**
-   **str**

### Defined

-   [Boolean](types/Boolean.md) (**Bool**) - A binary (true or false)
    value
-   [Bytes](types/Bytes.md) (**int**) - An integer value that
    corresponds to a size in bytes
-   [Date](types/Date.md) (**XSDDate**) - a date (year, month and day)
    in an idealized calendar
-   [Datetime](types/Datetime.md) (**XSDDateTime**) - The combination of
    a date and time
-   [Decimal](types/Decimal.md) (**Decimal**) - A real number with
    arbitrary precision that conforms to the xsd:decimal specification
-   [DecimalDegree](types/DecimalDegree.md) (**float**) - A decimal
    degree expresses latitude or longitude as decimal fractions.
-   [Double](types/Double.md) (**float**) - A real number that conforms
    to the xsd:double specification
-   [ExternalIdentifier](types/ExternalIdentifier.md)
    ([Uriorcurie](types/Uriorcurie.md)) - A CURIE representing an
    external identifier
-   [Float](types/Float.md) (**float**) - A real number that conforms to
    the xsd:float specification
-   [Integer](types/Integer.md) (**int**) - An integer
-   [LanguageCode](types/LanguageCode.md) (**str**) - A language code
    conforming to ISO_639-1
-   [Ncname](types/Ncname.md) (**NCName**) - Prefix part of CURIE
-   [Nodeidentifier](types/Nodeidentifier.md) (**NodeIdentifier**) - A
    URI, CURIE or BNODE that represents a node in a model.
-   [Objectidentifier](types/Objectidentifier.md)
    (**ElementIdentifier**) - A URI or CURIE that represents an object
    in the model.
-   [String](types/String.md) (**str**) - A character string
-   [Time](types/Time.md) (**XSDTime**) - A time object represents a
    (local) time of day, independent of any particular day
-   [Unit](types/Unit.md) (**str**)
-   [Uri](types/Uri.md) (**URI**) - a complete URI
-   [Uriorcurie](types/Uriorcurie.md) (**URIorCURIE**) - a URI or a
    CURIE

# Slot: indoor_space

"A distinguishable space within a structure, the purpose for which
discrete areas of a building is used"

URI: [nmdc:indoor_space](https://microbiomedata/meta/indoor_space)

## Domain and Range {#domain-and-range-387}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-424}

-   is_a: [attribute](attribute.md)

## Children {#children-394}

## Used by {#used-by-387}

## Other properties {#other-properties-341}

+---------------+---------------------+
| **Aliases:**  | > indoor space      |
+---------------+---------------------+
| **Mappings:** | > mixs:indoor_space |
+---------------+---------------------+

# Slot: indoor_surf

Type of indoor surface

URI: [nmdc:indoor_surf](https://microbiomedata/meta/indoor_surf)

## Domain and Range {#domain-and-range-388}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-425}

-   is_a: [attribute](attribute.md)

## Children {#children-395}

## Used by {#used-by-388}

## Other properties {#other-properties-342}

+---------------+--------------------+
| **Aliases:**  | > indoor surface   |
+---------------+--------------------+
| **Mappings:** | > mixs:indoor_surf |
+---------------+--------------------+

# Slot: indust_eff_percent

Percentage of industrial effluents received by wastewater treatment
plant

URI:
[nmdc:indust_eff_percent](https://microbiomedata/meta/indust_eff_percent)

## Domain and Range {#domain-and-range-389}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-426}

-   is_a: [attribute](attribute.md)

## Children {#children-396}

## Used by {#used-by-389}

## Other properties {#other-properties-343}

+---------------+-------------------------------+
| **Aliases:**  | > industrial effluent percent |
+---------------+-------------------------------+
| **Mappings:** | > mixs:indust_eff_percent     |
+---------------+-------------------------------+

# Slot: inorg_particles

"Concentration of particles such as sand, grit, metal particles,
ceramics, etc.; can include multiple particles"

URI: [nmdc:inorg_particles](https://microbiomedata/meta/inorg_particles)

## Domain and Range {#domain-and-range-390}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-427}

-   is_a: [attribute](attribute.md)

## Children {#children-397}

## Used by {#used-by-390}

## Other properties {#other-properties-344}

+---------------+------------------------+
| **Aliases:**  | > inorganic particles  |
+---------------+------------------------+
| **Mappings:** | > mixs:inorg_particles |
+---------------+------------------------+

# Slot: input base count

The nucleotide base count number of input reads for QC analysis.

URI:
<https://microbiomedata/schema/workflow_execution_activity/input_base_count>

## Domain and Range {#domain-and-range-391}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-428}

-   is_a: [read QC analysis statistic](read_QC_analysis_statistic.md)

## Children {#children-398}

## Used by {#used-by-391}

-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)

# Slot: input contig num

URI:
<https://microbiomedata/schema/workflow_execution_activity/input_contig_num>

## Domain and Range {#domain-and-range-392}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-429}

## Children {#children-399}

## Used by {#used-by-392}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)

# Slot: input_read_bases

TODO

URI:
<https://microbiomedata/schema/workflow_execution_activity/input_read_bases>

## Domain and Range {#domain-and-range-393}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-430}

## Children {#children-400}

-   [read QC analysis
    activity➞input_read_bases](read_QC_analysis_activity_input_read_bases.md)

## Used by {#used-by-393}

# Slot: input read count

The sequence count number of input reads for QC analysis.

URI:
<https://microbiomedata/schema/workflow_execution_activity/input_read_count>

## Domain and Range {#domain-and-range-394}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-431}

-   is_a: [read QC analysis statistic](read_QC_analysis_statistic.md)

## Children {#children-401}

## Used by {#used-by-394}

-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)

# Slot: inside_lux

The recorded value at sampling time (power density)

URI: [nmdc:inside_lux](https://microbiomedata/meta/inside_lux)

## Domain and Range {#domain-and-range-395}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-432}

-   is_a: [attribute](attribute.md)

## Children {#children-402}

## Used by {#used-by-395}

## Other properties {#other-properties-345}

+---------------+--------------------+
| **Aliases:**  | > inside lux light |
+---------------+--------------------+
| **Mappings:** | > mixs:inside_lux  |
+---------------+--------------------+

# Slot: instrument_name

The name of the instrument that was used for processing the sample.

URI: [nmdc:instrument_name](https://microbiomedata/meta/instrument_name)

## Domain and Range {#domain-and-range-396}

None → 0..1 [String](types/String.md)

## Parents {#parents-433}

## Children {#children-403}

## Used by {#used-by-396}

-   [OmicsProcessing](OmicsProcessing.md)

# Slot: int_wall_cond

The physical condition of the wall at the time of sampling; photos or
video preferred; use drawings to indicate location of damaged areas

URI: [nmdc:int_wall_cond](https://microbiomedata/meta/int_wall_cond)

## Domain and Range {#domain-and-range-397}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-434}

-   is_a: [attribute](attribute.md)

## Children {#children-404}

## Used by {#used-by-397}

## Other properties {#other-properties-346}

+---------------+---------------------------+
| **Aliases:**  | > interior wall condition |
+---------------+---------------------------+
| **Mappings:** | > mixs:int_wall_cond      |
+---------------+---------------------------+

# Slot: investigation_type

"Nucleic Acid Sequence Report is the root element of all MIGS/MIMS
compliant reports as standardized by Genomic Standards Consortium. This
field is either eukaryote,bacteria,virus,plasmid,organelle,
metagenome,mimarks-survey, mimarks-specimen, metatranscriptome, single
amplified genome, metagenome-assembled genome, or uncultivated viral
genome"

URI:
[nmdc:investigation_type](https://microbiomedata/meta/investigation_type)

## Domain and Range {#domain-and-range-398}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-435}

-   is_a: [attribute](attribute.md)

## Children {#children-405}

## Used by {#used-by-398}

## Other properties {#other-properties-347}

+-----------------+---------------------------+
| **Aliases:**    | > investigation type      |
+-----------------+---------------------------+
| **Mappings:**   | > mixs:investigation_type |
+-----------------+---------------------------+
| **In Subsets:** | > investigation           |
+-----------------+---------------------------+

# Slot: is balanced

URI: [nmdc:is_balanced](https://microbiomedata/meta/is_balanced)

## Domain and Range {#domain-and-range-399}

None → 0..1 [Boolean](types/Boolean.md)

## Parents {#parents-436}

## Children {#children-406}

-   [reaction➞is balanced](reaction_is_balanced.md)

## Used by {#used-by-399}

# Slot: is diastereoselective

URI:
[nmdc:is_diastereoselective](https://microbiomedata/meta/is_diastereoselective)

## Domain and Range {#domain-and-range-400}

None → 0..1 [Boolean](types/Boolean.md)

## Parents {#parents-437}

## Children {#children-407}

-   [reaction➞is diastereoselective](reaction_is_diastereoselective.md)

## Used by {#used-by-400}

# Slot: is fully characterized

False if includes R-groups

URI:
[nmdc:is_fully_characterized](https://microbiomedata/meta/is_fully_characterized)

## Domain and Range {#domain-and-range-401}

None → 0..1 [Boolean](types/Boolean.md)

## Parents {#parents-438}

## Children {#children-408}

-   [reaction➞is fully
    characterized](reaction_is_fully_characterized.md)

## Used by {#used-by-401}

# Slot: is stereo

URI: [nmdc:is_stereo](https://microbiomedata/meta/is_stereo)

## Domain and Range {#domain-and-range-402}

None → 0..1 [Boolean](types/Boolean.md)

## Parents {#parents-439}

## Children {#children-409}

-   [reaction➞is stereo](reaction_is_stereo.md)

## Used by {#used-by-402}

# Slot: is transport

URI: [nmdc:is_transport](https://microbiomedata/meta/is_transport)

## Domain and Range {#domain-and-range-403}

None → 0..1 [Boolean](types/Boolean.md)

## Parents {#parents-440}

## Children {#children-410}

-   [reaction➞is transport](reaction_is_transport.md)

## Used by {#used-by-403}

# Slot: isol_growth_condt

"Publication reference in the form of pubmed ID (pmid), digital object
identifier (doi) or url for isolation and growth condition
specifications of the organism/material"

URI:
[nmdc:isol_growth_condt](https://microbiomedata/meta/isol_growth_condt)

## Domain and Range {#domain-and-range-404}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-441}

-   is_a: [attribute](attribute.md)

## Children {#children-411}

## Used by {#used-by-404}

## Other properties {#other-properties-348}

+-----------------+----------------------------------+
| **Aliases:**    | > isolation and growth condition |
+-----------------+----------------------------------+
| **Mappings:**   | > mixs:isol_growth_condt         |
+-----------------+----------------------------------+
| **In Subsets:** | > nucleic acid sequence source   |
+-----------------+----------------------------------+

# Slot: iw_bt_date_well

Injection water breakthrough date per well following a secondary and/or
tertiary recovery

URI: [nmdc:iw_bt_date_well](https://microbiomedata/meta/iw_bt_date_well)

## Domain and Range {#domain-and-range-405}

None → 0..1 [TimestampValue](TimestampValue.md)

## Parents {#parents-442}

-   is_a: [attribute](attribute.md)

## Children {#children-412}

## Used by {#used-by-405}

## Other properties {#other-properties-349}

+---------------+------------------------------------------------------+
| **Aliases:**  | > injection water breakthrough date of specific well |
+---------------+------------------------------------------------------+
| **Mappings:** | > mixs:iw_bt_date_well                               |
+---------------+------------------------------------------------------+

# Slot: iwf

Proportion of the produced fluids derived from injected water at the
time of sampling. (e.g. 87%)

URI: [nmdc:iwf](https://microbiomedata/meta/iwf)

## Domain and Range {#domain-and-range-406}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-443}

-   is_a: [attribute](attribute.md)

## Children {#children-413}

## Used by {#used-by-406}

## Other properties {#other-properties-350}

+---------------+----------------------------+
| **Aliases:**  | > injection water fraction |
+---------------+----------------------------+
| **Mappings:** | > mixs:iwf                 |
+---------------+----------------------------+

# Slot: keywords

A list of keywords that used to associate the entity. Keywords SHOULD
come from controlled vocabularies, including MESH, ENVO.

URI:
<https://microbiomedata/schema/workflow_execution_activity/keywords>

## Domain and Range {#domain-and-range-407}

None → 0..\* [String](types/String.md)

## Parents {#parents-444}

## Children {#children-414}

## Used by {#used-by-407}

## Other properties {#other-properties-351}

  **Mappings:**    dcterms:subject
  ---------------- -----------------
                   
  Slot: kidney_d   isord

History of kidney disorders; can include multiple disorders

URI: [nmdc:kidney_disord](https://microbiomedata/meta/kidney_disord)

## Domain and Range {#domain-and-range-408}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-445}

-   is_a: [attribute](attribute.md)

## Children {#children-415}

## Used by {#used-by-408}

## Other properties {#other-properties-352}

+---------------+-------------------------+
| **Aliases:**  | > urine/kidney disorder |
+---------------+-------------------------+
| **Mappings:** | > mixs:kidney_disord    |
+---------------+-------------------------+

# Slot: language

Should use ISO 639-1 code e.g. "en", "fr"

URI:
<https://microbiomedata/schema/workflow_execution_activity/language>

## Domain and Range {#domain-and-range-409}

None → 0..1 [LanguageCode](types/LanguageCode.md)

## Parents {#parents-446}

## Children {#children-416}

## Used by {#used-by-409}

-   [TextValue](TextValue.md)

# Slot: last_clean

"The last time the floor was cleaned (swept, mopped, vacuumed)"

URI: [nmdc:last_clean](https://microbiomedata/meta/last_clean)

## Domain and Range {#domain-and-range-410}

None → 0..1 [TimestampValue](TimestampValue.md)

## Parents {#parents-447}

-   is_a: [attribute](attribute.md)

## Children {#children-417}

## Used by {#used-by-410}

## Other properties {#other-properties-353}

+---------------+-----------------------------------+
| **Aliases:**  | > last time swept/mopped/vacuumed |
+---------------+-----------------------------------+
| **Mappings:** | > mixs:last_clean                 |
+---------------+-----------------------------------+

# Slot: lat_lon

The geographical origin of the sample as defined by latitude and
longitude. The values should be reported in decimal degrees and in WGS84
system

URI: [nmdc:lat_lon](https://microbiomedata/meta/lat_lon)

## Domain and Range {#domain-and-range-411}

None → 0..1 [GeolocationValue](GeolocationValue.md)

## Parents {#parents-448}

-   is_a: [attribute](attribute.md)

## Children {#children-418}

-   [biosample➞lat_lon](biosample_lat_lon.md)

## Used by {#used-by-411}

## Other properties {#other-properties-354}

+-----------------+------------------------------------------------+
| **Aliases:**    | > geographic location (latitude and longitude) |
+-----------------+------------------------------------------------+
| **Mappings:**   | > mixs:lat_lon                                 |
+-----------------+------------------------------------------------+
| **In Subsets:** | > environment                                  |
+-----------------+------------------------------------------------+

# Slot: latitude

latitude

URI:
<https://microbiomedata/schema/workflow_execution_activity/latitude>

## Domain and Range {#domain-and-range-412}

[GeolocationValue](GeolocationValue.md) → 0..1
[DecimalDegree](types/DecimalDegree.md)

## Parents {#parents-449}

## Children {#children-419}

## Used by {#used-by-412}

-   [GeolocationValue](GeolocationValue.md)

## Other properties {#other-properties-355}

+---------------+-------------------+
| **Mappings:** | > wgs:lat         |
+---------------+-------------------+
|               | > schema:latitude |
+---------------+-------------------+

# Slot: left participants

URI:
[nmdc:left_participants](https://microbiomedata/meta/left_participants)

## Domain and Range {#domain-and-range-413}

None → 0..\* [ReactionParticipant](ReactionParticipant.md)

## Parents {#parents-450}

-   is_a: [has participants](has_participants.md)

## Children {#children-420}

## Used by {#used-by-413}

# Slot: lib_layout

"Specify whether to expect single, paired, or other configuration of
reads"

URI: [nmdc:lib_layout](https://microbiomedata/meta/lib_layout)

## Domain and Range {#domain-and-range-414}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-451}

-   is_a: [attribute](attribute.md)

## Children {#children-421}

## Used by {#used-by-414}

## Other properties {#other-properties-356}

+-----------------+-------------------+
| **Aliases:**    | > library layout  |
+-----------------+-------------------+
| **Mappings:**   | > mixs:lib_layout |
+-----------------+-------------------+
| **In Subsets:** | > sequencing      |
+-----------------+-------------------+

# Slot: lib_reads_seqd

Total number of clones sequenced from the library

URI: [nmdc:lib_reads_seqd](https://microbiomedata/meta/lib_reads_seqd)

## Domain and Range {#domain-and-range-415}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-452}

-   is_a: [attribute](attribute.md)

## Children {#children-422}

## Used by {#used-by-415}

## Other properties {#other-properties-357}

+-----------------+---------------------------+
| **Aliases:**    | > library reads sequenced |
+-----------------+---------------------------+
| **Mappings:**   | > mixs:lib_reads_seqd     |
+-----------------+---------------------------+
| **In Subsets:** | > sequencing              |
+-----------------+---------------------------+

# Slot: lib_screen

Specific enrichment or screening methods applied before and/or after
creating libraries

URI: [nmdc:lib_screen](https://microbiomedata/meta/lib_screen)

## Domain and Range {#domain-and-range-416}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-453}

-   is_a: [attribute](attribute.md)

## Children {#children-423}

## Used by {#used-by-416}

## Other properties {#other-properties-358}

+-----------------+------------------------------+
| **Aliases:**    | > library screening strategy |
+-----------------+------------------------------+
| **Mappings:**   | > mixs:lib_screen            |
+-----------------+------------------------------+
| **In Subsets:** | > sequencing                 |
+-----------------+------------------------------+

# Slot: lib_size

Total number of clones in the library prepared for the project

URI: [nmdc:lib_size](https://microbiomedata/meta/lib_size)

## Domain and Range {#domain-and-range-417}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-454}

-   is_a: [attribute](attribute.md)

## Children {#children-424}

## Used by {#used-by-417}

## Other properties {#other-properties-359}

+-----------------+-----------------+
| **Aliases:**    | > library size  |
+-----------------+-----------------+
| **Mappings:**   | > mixs:lib_size |
+-----------------+-----------------+
| **In Subsets:** | > sequencing    |
+-----------------+-----------------+

# Slot: lib_vector

Cloning vector type(s) used in construction of libraries

URI: [nmdc:lib_vector](https://microbiomedata/meta/lib_vector)

## Domain and Range {#domain-and-range-418}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-455}

-   is_a: [attribute](attribute.md)

## Children {#children-425}

## Used by {#used-by-418}

## Other properties {#other-properties-360}

+-----------------+-------------------+
| **Aliases:**    | > library vector  |
+-----------------+-------------------+
| **Mappings:**   | > mixs:lib_vector |
+-----------------+-------------------+
| **In Subsets:** | > sequencing      |
+-----------------+-------------------+

# Slot: light_intensity

Measurement of light intensity

URI: [nmdc:light_intensity](https://microbiomedata/meta/light_intensity)

## Domain and Range {#domain-and-range-419}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-456}

-   is_a: [attribute](attribute.md)

## Children {#children-426}

## Used by {#used-by-419}

## Other properties {#other-properties-361}

+---------------+------------------------+
| **Aliases:**  | > light intensity      |
+---------------+------------------------+
| **Mappings:** | > mixs:light_intensity |
+---------------+------------------------+

# Slot: light_regm

"Information about treatment(s) involving exposure to light, including
both light intensity and quality."

URI: [nmdc:light_regm](https://microbiomedata/meta/light_regm)

## Domain and Range {#domain-and-range-420}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-457}

-   is_a: [attribute](attribute.md)

## Children {#children-427}

## Used by {#used-by-420}

## Other properties {#other-properties-362}

+---------------+-------------------+
| **Aliases:**  | > light regimen   |
+---------------+-------------------+
| **Mappings:** | > mixs:light_regm |
+---------------+-------------------+

# Slot: light_type

"Application of light to achieve some practical or aesthetic effect.
Lighting includes the use of both artificial light sources such as lamps
and light fixtures, as well as natural illumination by capturing
daylight. Can also include absence of light"

URI: [nmdc:light_type](https://microbiomedata/meta/light_type)

## Domain and Range {#domain-and-range-421}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-458}

-   is_a: [attribute](attribute.md)

## Children {#children-428}

## Used by {#used-by-421}

## Other properties {#other-properties-363}

+---------------+-------------------+
| **Aliases:**  | > light type      |
+---------------+-------------------+
| **Mappings:** | > mixs:light_type |
+---------------+-------------------+

# Slot: link_addit_analys

Link to additional analysis results performed on the sample

URI:
[nmdc:link_addit_analys](https://microbiomedata/meta/link_addit_analys)

## Domain and Range {#domain-and-range-422}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-459}

-   is_a: [attribute](attribute.md)

## Children {#children-429}

## Used by {#used-by-422}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-364}

+---------------+--------------------------------+
| **Aliases:**  | > links to additional analysis |
+---------------+--------------------------------+
| **Mappings:** | > mixs:link_addit_analys       |
+---------------+--------------------------------+

# Slot: link_class_info

Link to digitized soil maps or other soil classification information

URI: [nmdc:link_class_info](https://microbiomedata/meta/link_class_info)

## Domain and Range {#domain-and-range-423}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-460}

-   is_a: [attribute](attribute.md)

## Children {#children-430}

## Used by {#used-by-423}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-365}

+---------------+--------------------------------------+
| **Aliases:**  | > link to classification information |
+---------------+--------------------------------------+
| **Mappings:** | > mixs:link_class_info               |
+---------------+--------------------------------------+

# Slot: link_climate_info

Link to climate resource

URI:
[nmdc:link_climate_info](https://microbiomedata/meta/link_climate_info)

## Domain and Range {#domain-and-range-424}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-461}

-   is_a: [attribute](attribute.md)

## Children {#children-431}

## Used by {#used-by-424}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-366}

+---------------+-------------------------------+
| **Aliases:**  | > link to climate information |
+---------------+-------------------------------+
| **Mappings:** | > mixs:link_climate_info      |
+---------------+-------------------------------+

# Slot: lithology

"Hydrocarbon resource main lithology (Additional information:
<http://petrowiki.org/Lithology_and_rock_type_determination>).
If""other"" is specified, please propose entry in ""additional info""
field"

URI: [nmdc:lithology](https://microbiomedata/meta/lithology)

## Domain and Range {#domain-and-range-425}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-462}

-   is_a: [attribute](attribute.md)

## Children {#children-432}

## Used by {#used-by-425}

## Other properties {#other-properties-367}

+---------------+------------------+
| **Aliases:**  | > lithology      |
+---------------+------------------+
| **Mappings:** | > mixs:lithology |
+---------------+------------------+

# Slot: liver_disord

History of liver disorders; can include multiple disorders

URI: [nmdc:liver_disord](https://microbiomedata/meta/liver_disord)

## Domain and Range {#domain-and-range-426}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-463}

-   is_a: [attribute](attribute.md)

## Children {#children-433}

## Used by {#used-by-426}

## Other properties {#other-properties-368}

+---------------+---------------------+
| **Aliases:**  | > liver disorder    |
+---------------+---------------------+
| **Mappings:** | > mixs:liver_disord |
+---------------+---------------------+

# Slot: local_class

Soil classification based on local soil classification system

URI: [nmdc:local_class](https://microbiomedata/meta/local_class)

## Domain and Range {#domain-and-range-427}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-464}

-   is_a: [attribute](attribute.md)

## Children {#children-434}

## Used by {#used-by-427}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-369}

+---------------+---------------------------------------+
| **Aliases:**  | > soil_taxonomic/local classification |
+---------------+---------------------------------------+
| **Mappings:** | > mixs:local_class                    |
+---------------+---------------------------------------+

# Slot: local_class_meth

Reference or method used in determining the local soil classification

URI:
[nmdc:local_class_meth](https://microbiomedata/meta/local_class_meth)

## Domain and Range {#domain-and-range-428}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-465}

-   is_a: [attribute](attribute.md)

## Children {#children-435}

## Used by {#used-by-428}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-370}

+---------------+----------------------------------------------+
| **Aliases:**  | > soil_taxonomic/local classification method |
+---------------+----------------------------------------------+
| **Mappings:** | > mixs:local_class_meth                      |
+---------------+----------------------------------------------+

# Slot: location

URI: [nmdc:location](https://microbiomedata/meta/location)

## Domain and Range {#domain-and-range-429}

None → 0..1 [String](types/String.md)

## Parents {#parents-466}

## Children {#children-436}

## Used by {#used-by-429}

-   [Biosample](Biosample.md)

# Slot: longitude

longitude

URI:
<https://microbiomedata/schema/workflow_execution_activity/longitude>

## Domain and Range {#domain-and-range-430}

[GeolocationValue](GeolocationValue.md) → 0..1
[DecimalDegree](types/DecimalDegree.md)

## Parents {#parents-467}

## Children {#children-437}

## Used by {#used-by-430}

-   [GeolocationValue](GeolocationValue.md)

## Other properties {#other-properties-371}

+---------------+--------------------+
| **Mappings:** | > wgs:long         |
+---------------+--------------------+
|               | > schema:longitude |
+---------------+--------------------+

# Slot: lowDepth contig num

URI:
<https://microbiomedata/schema/workflow_execution_activity/lowDepth_contig_num>

## Domain and Range {#domain-and-range-431}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-468}

## Children {#children-438}

## Used by {#used-by-431}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)

# Slot: bin name

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__bin_name>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__bin_name>)

## Domain and Range {#domain-and-range-432}

None → 0..1 [String](types/String.md)

## Parents {#parents-469}

## Children {#children-439}

## Used by {#used-by-432}

-   [MAGBin](MAGBin.md)

# Slot: bin quality

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__bin_quality>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__bin_quality>)

## Domain and Range {#domain-and-range-433}

None → 0..1 [String](types/String.md)

## Parents {#parents-470}

## Children {#children-440}

## Used by {#used-by-433}

-   [MAGBin](MAGBin.md)

# Slot: completeness

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__completeness>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__completeness>)

## Domain and Range {#domain-and-range-434}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-471}

## Children {#children-441}

## Used by {#used-by-434}

-   [MAGBin](MAGBin.md)

# Slot: contamination

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__contamination>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__contamination>)

## Domain and Range {#domain-and-range-435}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-472}

## Children {#children-442}

## Used by {#used-by-435}

-   [MAGBin](MAGBin.md)

# Slot: gene count

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gene_count>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gene_count>)

## Domain and Range {#domain-and-range-436}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-473}

## Children {#children-443}

## Used by {#used-by-436}

-   [MAGBin](MAGBin.md)

# Slot: gtdbtk class

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_class>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_class>)

## Domain and Range {#domain-and-range-437}

None → 0..1 [String](types/String.md)

## Parents {#parents-474}

## Children {#children-444}

## Used by {#used-by-437}

-   [MAGBin](MAGBin.md)

# Slot: gtdbtk domain

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_domain>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_domain>)

## Domain and Range {#domain-and-range-438}

None → 0..1 [String](types/String.md)

## Parents {#parents-475}

## Children {#children-445}

## Used by {#used-by-438}

-   [MAGBin](MAGBin.md)

# Slot: gtdbtk family

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_family>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_family>)

## Domain and Range {#domain-and-range-439}

None → 0..1 [String](types/String.md)

## Parents {#parents-476}

## Children {#children-446}

## Used by {#used-by-439}

-   [MAGBin](MAGBin.md)

# Slot: gtdbtk genus

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_genus>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_genus>)

## Domain and Range {#domain-and-range-440}

None → 0..1 [String](types/String.md)

## Parents {#parents-477}

## Children {#children-447}

## Used by {#used-by-440}

-   [MAGBin](MAGBin.md)

# Slot: gtdbtk order

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_order>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_order>)

## Domain and Range {#domain-and-range-441}

None → 0..1 [String](types/String.md)

## Parents {#parents-478}

## Children {#children-448}

## Used by {#used-by-441}

-   [MAGBin](MAGBin.md)

# Slot: gtdbtk phylum

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_phylum>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_phylum>)

## Domain and Range {#domain-and-range-442}

None → 0..1 [String](types/String.md)

## Parents {#parents-479}

## Children {#children-449}

## Used by {#used-by-442}

-   [MAGBin](MAGBin.md)

# Slot: gtdbtk species

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_species>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__gtdbtk_species>)

## Domain and Range {#domain-and-range-443}

None → 0..1 [String](types/String.md)

## Parents {#parents-480}

## Children {#children-450}

## Used by {#used-by-443}

-   [MAGBin](MAGBin.md)

# Slot: num 16s

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_16s>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_16s>)

## Domain and Range {#domain-and-range-444}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-481}

## Children {#children-451}

## Used by {#used-by-444}

-   [MAGBin](MAGBin.md)

# Slot: num 23s

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_23s>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_23s>)

## Domain and Range {#domain-and-range-445}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-482}

## Children {#children-452}

## Used by {#used-by-445}

-   [MAGBin](MAGBin.md)

# Slot: num 5s

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_5s>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_5s>)

## Domain and Range {#domain-and-range-446}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-483}

## Children {#children-453}

## Used by {#used-by-446}

-   [MAGBin](MAGBin.md)

# Slot: num tRNA

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_tRNA>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__num_tRNA>)

## Domain and Range {#domain-and-range-447}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-484}

## Children {#children-454}

## Used by {#used-by-447}

-   [MAGBin](MAGBin.md)

# Slot: number of contig

URI:
\[<https://microbiomedata/schema/workflow_execution_activity/mAGBin__number_of_contig>\](<https://microbiomedata/schema/workflow_execution_activity/mAGBin__number_of_contig>)

## Domain and Range {#domain-and-range-448}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-485}

## Children {#children-455}

## Used by {#used-by-448}

-   [MAGBin](MAGBin.md)

# Slot: mag_cov_software

Tool(s) used to determine the genome coverage if coverage is used as a
binning parameter in the extraction of genomes from metagenomic datasets

URI:
[nmdc:mag_cov_software](https://microbiomedata/meta/mag_cov_software)

## Domain and Range {#domain-and-range-449}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-486}

-   is_a: [attribute](attribute.md)

## Children {#children-456}

## Used by {#used-by-449}

## Other properties {#other-properties-372}

+-----------------+-------------------------+
| **Aliases:**    | > MAG coverage software |
+-----------------+-------------------------+
| **Mappings:**   | > mixs:mag_cov_software |
+-----------------+-------------------------+
| **In Subsets:** | > sequencing            |
+-----------------+-------------------------+

# Slot: magnesium

Concentration of magnesium in the sample

URI: [nmdc:magnesium](https://microbiomedata/meta/magnesium)

## Domain and Range {#domain-and-range-450}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-487}

-   is_a: [attribute](attribute.md)

## Children {#children-457}

## Used by {#used-by-450}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-373}

+---------------+------------------+
| **Aliases:**  | > magnesium      |
+---------------+------------------+
| **Mappings:** | > mixs:magnesium |
+---------------+------------------+

# Slot: mags activity set

This property links a database object to the set of MAGs analysis
activities.

URI:
[nmdc:mags_activity_set](https://microbiomedata/meta/mags_activity_set)

## Domain and Range {#domain-and-range-451}

[Database](Database.md) → 0..\*
[MAGsAnalysisActivity](MAGsAnalysisActivity.md)

## Parents {#parents-488}

## Children {#children-458}

## Used by {#used-by-451}

-   [Database](Database.md)

# Slot: mags list

URI:
<https://microbiomedata/schema/workflow_execution_activity/mags_list>

## Domain and Range {#domain-and-range-452}

None → 0..\* [MAGBin](MAGBin.md)

## Parents {#parents-489}

## Children {#children-459}

## Used by {#used-by-452}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)

# Slot: maternal_health_stat

Specification of the maternal health status

URI:
[nmdc:maternal_health_stat](https://microbiomedata/meta/maternal_health_stat)

## Domain and Range {#domain-and-range-453}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-490}

-   is_a: [attribute](attribute.md)

## Children {#children-460}

## Used by {#used-by-453}

## Other properties {#other-properties-374}

+---------------+-----------------------------------------+
| **Aliases:**  | > amniotic fluid/maternal health status |
+---------------+-----------------------------------------+
| **Mappings:** | > mixs:maternal_health_stat             |
+---------------+-----------------------------------------+

# Slot: max_occup

The maximum amount of people allowed in the indoor environment

URI: [nmdc:max_occup](https://microbiomedata/meta/max_occup)

## Domain and Range {#domain-and-range-454}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-491}

-   is_a: [attribute](attribute.md)

## Children {#children-461}

## Used by {#used-by-454}

## Other properties {#other-properties-375}

+---------------+---------------------+
| **Aliases:**  | > maximum occupancy |
+---------------+---------------------+
| **Mappings:** | > mixs:max_occup    |
+---------------+---------------------+

# Slot: md5 checksum

MD5 checksum of file (pre-compressed)

URI:
<https://microbiomedata/schema/workflow_execution_activity/md5_checksum>

## Domain and Range {#domain-and-range-455}

None → 0..1 [String](types/String.md)

## Parents {#parents-492}

-   is_a: [attribute](attribute.md)

## Children {#children-462}

## Used by {#used-by-455}

# Slot: mean_frict_vel

Measurement of mean friction velocity

URI: [nmdc:mean_frict_vel](https://microbiomedata/meta/mean_frict_vel)

## Domain and Range {#domain-and-range-456}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-493}

-   is_a: [attribute](attribute.md)

## Children {#children-463}

## Used by {#used-by-456}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-376}

+---------------+--------------------------+
| **Aliases:**  | > mean friction velocity |
+---------------+--------------------------+
| **Mappings:** | > mixs:mean_frict_vel    |
+---------------+--------------------------+

# Slot: mean_peak_frict_vel

Measurement of mean peak friction velocity

URI:
[nmdc:mean_peak_frict_vel](https://microbiomedata/meta/mean_peak_frict_vel)

## Domain and Range {#domain-and-range-457}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-494}

-   is_a: [attribute](attribute.md)

## Children {#children-464}

## Used by {#used-by-457}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-377}

+---------------+-------------------------------+
| **Aliases:**  | > mean peak friction velocity |
+---------------+-------------------------------+
| **Mappings:** | > mixs:mean_peak_frict_vel    |
+---------------+-------------------------------+

# Slot: mech_struc

mechanical structure: a moving structure

URI: [nmdc:mech_struc](https://microbiomedata/meta/mech_struc)

## Domain and Range {#domain-and-range-458}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-495}

-   is_a: [attribute](attribute.md)

## Children {#children-465}

## Used by {#used-by-458}

## Other properties {#other-properties-378}

+---------------+------------------------+
| **Aliases:**  | > mechanical structure |
+---------------+------------------------+
| **Mappings:** | > mixs:mech_struc      |
+---------------+------------------------+

# Slot: mechanical_damage

Information about any mechanical damage exerted on the plant; can
include multiple damages and sites

URI:
[nmdc:mechanical_damage](https://microbiomedata/meta/mechanical_damage)

## Domain and Range {#domain-and-range-459}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-496}

-   is_a: [attribute](attribute.md)

## Children {#children-466}

## Used by {#used-by-459}

## Other properties {#other-properties-379}

+---------------+--------------------------+
| **Aliases:**  | > mechanical damage      |
+---------------+--------------------------+
| **Mappings:** | > mixs:mechanical_damage |
+---------------+--------------------------+

# Slot: medic_hist_perform

Whether full medical history was collected

URI:
[nmdc:medic_hist_perform](https://microbiomedata/meta/medic_hist_perform)

## Domain and Range {#domain-and-range-460}

None → 0..1 [BooleanValue](BooleanValue.md)

## Parents {#parents-497}

-   is_a: [attribute](attribute.md)

## Children {#children-467}

## Used by {#used-by-460}

## Other properties {#other-properties-380}

+---------------+-----------------------------+
| **Aliases:**  | > medical history performed |
+---------------+-----------------------------+
| **Mappings:** | > mixs:medic_hist_perform   |
+---------------+-----------------------------+

# Slot: menarche

Date of most recent menstruation

URI: [nmdc:menarche](https://microbiomedata/meta/menarche)

## Domain and Range {#domain-and-range-461}

None → 0..1 [TimestampValue](TimestampValue.md)

## Parents {#parents-498}

-   is_a: [attribute](attribute.md)

## Children {#children-468}

## Used by {#used-by-461}

## Other properties {#other-properties-381}

+---------------+-----------------+
| **Aliases:**  | > menarche      |
+---------------+-----------------+
| **Mappings:** | > mixs:menarche |
+---------------+-----------------+

# Slot: menopause

Date of onset of menopause

URI: [nmdc:menopause](https://microbiomedata/meta/menopause)

## Domain and Range {#domain-and-range-462}

None → 0..1 [TimestampValue](TimestampValue.md)

## Parents {#parents-499}

-   is_a: [attribute](attribute.md)

## Children {#children-469}

## Used by {#used-by-462}

## Other properties {#other-properties-382}

+---------------+------------------+
| **Aliases:**  | > menopause      |
+---------------+------------------+
| **Mappings:** | > mixs:menopause |
+---------------+------------------+

# Slot: highest similarity score

TODO: Yuri to fill in

URI:
<https://microbiomedata/schema/workflow_execution_activity/metabolite_quantification_highest_similarity_score>

## Domain and Range {#domain-and-range-463}

[MetaboliteQuantification](MetaboliteQuantification.md) → 0..1
[Float](types/Float.md)

## Parents {#parents-500}

-   is_a: [highest similarity score](highest_similarity_score.md)

## Children {#children-470}

## Used by {#used-by-463}

-   [MetaboliteQuantification](MetaboliteQuantification.md)

# Slot: metabolite quantified

the specific metabolite identifier

URI:
<https://microbiomedata/schema/workflow_execution_activity/metabolite_quantification_metabolite_quantified>

## Domain and Range {#domain-and-range-464}

[MetaboliteQuantification](MetaboliteQuantification.md) → 0..1
[ChemicalEntity](ChemicalEntity.md)

## Parents {#parents-501}

-   is_a: [metabolite quantified](metabolite_quantified.md)

## Children {#children-471}

## Used by {#used-by-464}

-   [MetaboliteQuantification](MetaboliteQuantification.md)

# Slot: metabolite quantified

the specific metabolite identifier

URI:
<https://microbiomedata/schema/workflow_execution_activity/metabolite_quantified>

## Domain and Range {#domain-and-range-465}

None → 0..1 [ChemicalEntity](ChemicalEntity.md)

## Parents {#parents-502}

## Children {#children-472}

-   [metabolite quantification➞metabolite
    quantified](metabolite_quantification_metabolite_quantified.md)

## Used by {#used-by-465}

# Slot: has calibration

TODO: Yuri to fill in

URI:
<https://microbiomedata/schema/workflow_execution_activity/metabolomics_analysis_activity_has_calibration>

## Domain and Range {#domain-and-range-466}

[MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md) → 0..1
[String](types/String.md)

## Parents {#parents-503}

-   is_a: [has calibration](has_calibration.md)

## Children {#children-473}

## Used by {#used-by-466}

-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)

# Slot: has metabolite quantifications

URI:
<https://microbiomedata/schema/workflow_execution_activity/metabolomics_analysis_activity_has_metabolite_quantifications>

## Domain and Range {#domain-and-range-467}

[MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md) → 0..\*
[MetaboliteQuantification](MetaboliteQuantification.md)

## Parents {#parents-504}

-   is_a: [has metabolite
    quantifications](has_metabolite_quantifications.md)

## Children {#children-474}

## Used by {#used-by-467}

-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)

# Slot: metabolomics analysis activity set

This property links a database object to the set of metabolomics
analysis activities.

URI:
[nmdc:metabolomics_analysis_activity_set](https://microbiomedata/meta/metabolomics_analysis_activity_set)

## Domain and Range {#domain-and-range-468}

[Database](Database.md) → 0..\*
[MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)

## Parents {#parents-505}

## Children {#children-475}

## Used by {#used-by-468}

-   [Database](Database.md)

# Slot: used

The instrument used to collect the data used in the analysis

URI:
<https://microbiomedata/schema/workflow_execution_activity/metabolomics_analysis_activity_used>

## Domain and Range {#domain-and-range-469}

[MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md) → 0..1
[Instrument](Instrument.md)

## Parents {#parents-506}

-   is_a: [used](used.md)

## Children {#children-476}

## Used by {#used-by-469}

-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)

## Other properties {#other-properties-383}

  **Mappings:**    prov:used
  ---------------- ----------------------------
                   
  Slot: metageno   me annotation activity set

This property links a database object to the set of metagenome
annotation activities.

URI:
[nmdc:metagenome_annotation_activity_set](https://microbiomedata/meta/metagenome_annotation_activity_set)

## Domain and Range {#domain-and-range-470}

[Database](Database.md) → 0..\*
[MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)

## Parents {#parents-507}

## Children {#children-477}

## Used by {#used-by-470}

-   [Database](Database.md)

# Slot: metagenome assembly parameter

URI:
<https://microbiomedata/schema/workflow_execution_activity/metagenome_assembly_parameter>

## Domain and Range {#domain-and-range-471}

None → 0..1 [String](types/String.md)

## Parents {#parents-508}

## Children {#children-478}

-   [asm_score](asm_score.md)
-   [contig_bp](contig_bp.md)
-   [contigs](contigs.md)
-   [ctg_L50](ctg_L50.md)
-   [ctg_L90](ctg_L90.md)
-   [ctg_N50](ctg_N50.md)
-   [ctg_N90](ctg_N90.md)
-   [ctg_logsum](ctg_logsum.md)
-   [ctg_max](ctg_max.md)
-   [ctg_powsum](ctg_powsum.md)
-   [gap_pct](gap_pct.md)
-   [gc_avg](gc_avg.md)
-   [gc_std](gc_std.md)
-   [num_aligned_reads](num_aligned_reads.md)
-   [num_input_reads](num_input_reads.md)
-   [scaf_L50](scaf_L50.md)
-   [scaf_L90](scaf_L90.md)
-   [scaf_N50](scaf_N50.md)
-   [scaf_N90](scaf_N90.md)
-   [scaf_bp](scaf_bp.md)
-   [scaf_l\_gt50K](scaf_l_gt50K.md)
-   [scaf_logsum](scaf_logsum.md)
-   [scaf_max](scaf_max.md)
-   [scaf_n\_gt50K](scaf_n_gt50K.md)
-   [scaf_pct_gt50K](scaf_pct_gt50K.md)
-   [scaf_powsum](scaf_powsum.md)
-   [scaffolds](scaffolds.md)

## Used by {#used-by-471}

# Slot: metagenome assembly set

This property links a database object to the set of metagenome assembly
activities.

URI:
[nmdc:metagenome_assembly_set](https://microbiomedata/meta/metagenome_assembly_set)

## Domain and Range {#domain-and-range-472}

[Database](Database.md) → 0..\*
[MetagenomeAssembly](MetagenomeAssembly.md)

## Parents {#parents-509}

## Children {#children-479}

## Used by {#used-by-472}

-   [Database](Database.md)

# Slot: has peptide quantifications {#slot-has-peptide-quantifications-1}

URI:
<https://microbiomedata/schema/workflow_execution_activity/metaproteomics_analysis_activity_has_peptide_quantifications>

## Domain and Range {#domain-and-range-473}

[MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md) →
0..\* [PeptideQuantification](PeptideQuantification.md)

## Parents {#parents-510}

-   is_a: [has peptide quantifications](has_peptide_quantifications.md)

## Children {#children-480}

## Used by {#used-by-473}

-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)

# Slot: metaproteomics analysis activity set

This property links a database object to the set of metaproteomics
analysis activities.

URI:
[nmdc:metaproteomics_analysis_activity_set](https://microbiomedata/meta/metaproteomics_analysis_activity_set)

## Domain and Range {#domain-and-range-474}

[Database](Database.md) → 0..\*
[MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)

## Parents {#parents-511}

## Children {#children-481}

## Used by {#used-by-474}

-   [Database](Database.md)

# Slot: used

The instrument used to collect the data used in the analysis

URI:
<https://microbiomedata/schema/workflow_execution_activity/metaproteomics_analysis_activity_used>

## Domain and Range {#domain-and-range-475}

[MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md) →
0..1 [Instrument](Instrument.md)

## Parents {#parents-512}

-   is_a: [used](used.md)

## Children {#children-482}

## Used by {#used-by-475}

-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)

## Other properties {#other-properties-384}

  **Mappings:**    prov:used
  ---------------- ------------------------
                   
  Slot: metatran   scriptome activity set

This property links a database object to the set of metatranscriptome
analysis activities.

URI:
[nmdc:metatranscriptome_activity_set](https://microbiomedata/meta/metatranscriptome_activity_set)

## Domain and Range {#domain-and-range-476}

[Database](Database.md) → 0..\*
[MetatranscriptomeActivity](MetatranscriptomeActivity.md)

## Parents {#parents-513}

## Children {#children-483}

## Used by {#used-by-476}

-   [Database](Database.md)

# Slot: methane

Methane (gas) amount or concentration at the time of sampling

URI: [nmdc:methane](https://microbiomedata/meta/methane)

## Domain and Range {#domain-and-range-477}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-514}

-   is_a: [attribute](attribute.md)

## Children {#children-484}

## Used by {#used-by-477}

## Other properties {#other-properties-385}

+---------------+----------------+
| **Aliases:**  | > methane      |
+---------------+----------------+
| **Mappings:** | > mixs:methane |
+---------------+----------------+

# Slot: microbial_biomass

"The part of the organic matter in the soil that constitutes living
microorganisms smaller than 5-10 micrometer. If you keep this, you would
need to have correction factors used for conversion to the final units"

URI:
[nmdc:microbial_biomass](https://microbiomedata/meta/microbial_biomass)

## Domain and Range {#domain-and-range-478}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-515}

-   is_a: [attribute](attribute.md)

## Children {#children-485}

## Used by {#used-by-478}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-386}

+---------------+--------------------------+
| **Aliases:**  | > microbial biomass      |
+---------------+--------------------------+
| **Mappings:** | > mixs:microbial_biomass |
+---------------+--------------------------+

# Slot: microbial_biomass_meth

Reference or method used in determining microbial biomass

URI:
[nmdc:microbial_biomass_meth](https://microbiomedata/meta/microbial_biomass_meth)

## Domain and Range {#domain-and-range-479}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-516}

-   is_a: [attribute](attribute.md)

## Children {#children-486}

## Used by {#used-by-479}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-387}

+---------------+-------------------------------+
| **Aliases:**  | > microbial biomass method    |
+---------------+-------------------------------+
| **Mappings:** | > mixs:microbial_biomass_meth |
+---------------+-------------------------------+

# Slot: mid

"Molecular barcodes, called Multiplex Identifiers (MIDs), that are used
to specifically tag unique samples in a sequencing run. Sequence should
be reported in uppercase letters"

URI: [nmdc:mid](https://microbiomedata/meta/mid)

## Domain and Range {#domain-and-range-480}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-517}

-   is_a: [attribute](attribute.md)

## Children {#children-487}

## Used by {#used-by-480}

## Other properties {#other-properties-388}

+-----------------+-------------------------+
| **Aliases:**    | > multiplex identifiers |
+-----------------+-------------------------+
| **Mappings:**   | > mixs:mid              |
+-----------------+-------------------------+
| **In Subsets:** | > sequencing            |
+-----------------+-------------------------+

# Slot: min_q\_value

smallest Q-Value associated with the peptide sequence as provided by
MSGFPlus tool

URI:
<https://microbiomedata/schema/workflow_execution_activity/min_q_value>

## Domain and Range {#domain-and-range-481}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-518}

## Children {#children-488}

-   [peptide
    quantification➞min_q\_value](peptide_quantification_min_q_value.md)

## Used by {#used-by-481}

## Other properties {#other-properties-389}

  **See also:**       [OBI:0001442](OBI:0001442)
  ------------------- ----------------------------
                      
  Slot: [mineral]()   nutr_regm

"Information about treatment involving the use of mineral supplements;
should include the name of mineral nutrient, amount administered,
treatment regimen including how many times the treatment was repeated,
how long each treatment lasted, and the start and end time of the entire
treatment; can include multiple mineral nutrient regimens"

URI:
[nmdc:mineral_nutr_regm](https://microbiomedata/meta/mineral_nutr_regm)

## Domain and Range {#domain-and-range-482}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-519}

-   is_a: [attribute](attribute.md)

## Children {#children-489}

## Used by {#used-by-482}

## Other properties {#other-properties-390}

+---------------+----------------------------+
| **Aliases:**  | > mineral nutrient regimen |
+---------------+----------------------------+
| **Mappings:** | > mixs:mineral_nutr_regm   |
+---------------+----------------------------+

# Slot: misc_param

"Any other measurement performed or parameter collected, that is not
listed here"

URI: [nmdc:misc_param](https://microbiomedata/meta/misc_param)

## Domain and Range {#domain-and-range-483}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-520}

-   is_a: [attribute](attribute.md)

## Children {#children-490}

## Used by {#used-by-483}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-391}

+---------------+---------------------------+
| **Aliases:**  | > miscellaneous parameter |
+---------------+---------------------------+
| **Mappings:** | > mixs:misc_param         |
+---------------+---------------------------+

# Slot: mixs_url

URI: [nmdc:mixs_url](https://microbiomedata/meta/mixs_url)

## Domain and Range {#domain-and-range-484}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-521}

-   is_a: [attribute](attribute.md)

## Children {#children-491}

## Used by {#used-by-484}

## Other properties {#other-properties-392}

+-----------------+---------------------------------+
| **Aliases:**    | > relevant electronic resources |
+-----------------+---------------------------------+
| **Mappings:**   | > mixs:mixs_url                 |
+-----------------+---------------------------------+
| **In Subsets:** | > sequencing                    |
+-----------------+---------------------------------+

# Slot: mod_date

The last date on which the database information was modified.

URI: [nmdc:mod_date](https://microbiomedata/meta/mod_date)

## Domain and Range {#domain-and-range-485}

None → 0..1 [String](types/String.md)

## Parents {#parents-522}

## Children {#children-492}

## Used by {#used-by-485}

-   [Biosample](Biosample.md)
-   [OmicsProcessing](OmicsProcessing.md)

# Slot: n_alkanes

Concentration of n-alkanes; can include multiple n-alkanes

URI: [nmdc:n_alkanes](https://microbiomedata/meta/n_alkanes)

## Domain and Range {#domain-and-range-486}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-523}

-   is_a: [attribute](attribute.md)

## Children {#children-493}

## Used by {#used-by-486}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-393}

+---------------+------------------+
| **Aliases:**  | > n-alkanes      |
+---------------+------------------+
| **Mappings:** | > mixs:n_alkanes |
+---------------+------------------+

# Slot: name {#slot-name-1}

A human readable label for an entity

URI: <https://microbiomedata/schema/workflow_execution_activity/name>

## Domain and Range {#domain-and-range-487}

None → 0..1 [String](types/String.md)

## Parents {#parents-524}

## Children {#children-494}

-   [person value➞name](person_value_name.md)

## Used by {#used-by-487}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [Activity](Activity.md)
-   [ChemicalEntity](ChemicalEntity.md)
-   [EnvironmentalMaterialTerm](EnvironmentalMaterialTerm.md)
-   [GeneProduct](GeneProduct.md)
-   [Instrument](Instrument.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md)
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [NamedThing](NamedThing.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md)
-   [OntologyClass](OntologyClass.md)
-   [Person](Person.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)

# Slot: ncbi_project_name

URI:
[nmdc:ncbi_project_name](https://microbiomedata/meta/ncbi_project_name)

## Domain and Range {#domain-and-range-488}

None → 0..1 [String](types/String.md)

## Parents {#parents-525}

## Children {#children-495}

## Used by {#used-by-488}

-   [OmicsProcessing](OmicsProcessing.md)

# Slot: ncbi_taxonomy_name

URI:
[nmdc:ncbi_taxonomy_name](https://microbiomedata/meta/ncbi_taxonomy_name)

## Domain and Range {#domain-and-range-489}

None → 0..1 [String](types/String.md)

## Parents {#parents-526}

## Children {#children-496}

## Used by {#used-by-489}

-   [Biosample](Biosample.md)

# Slot: nitrate

Concentration of nitrate in the sample

URI: [nmdc:nitrate](https://microbiomedata/meta/nitrate)

## Domain and Range {#domain-and-range-490}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-527}

-   is_a: [attribute](attribute.md)

## Children {#children-497}

## Used by {#used-by-490}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-394}

+---------------+----------------+
| **Aliases:**  | > nitrate      |
+---------------+----------------+
| **Mappings:** | > mixs:nitrate |
+---------------+----------------+

# Slot: nitrite

Concentration of nitrite in the sample

URI: [nmdc:nitrite](https://microbiomedata/meta/nitrite)

## Domain and Range {#domain-and-range-491}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-528}

-   is_a: [attribute](attribute.md)

## Children {#children-498}

## Used by {#used-by-491}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-395}

+---------------+----------------+
| **Aliases:**  | > nitrite      |
+---------------+----------------+
| **Mappings:** | > mixs:nitrite |
+---------------+----------------+

# Slot: nitro

Concentration of nitrogen (total)

URI: [nmdc:nitro](https://microbiomedata/meta/nitro)

## Domain and Range {#domain-and-range-492}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-529}

-   is_a: [attribute](attribute.md)

## Children {#children-499}

## Used by {#used-by-492}

## Other properties {#other-properties-396}

+---------------+--------------+
| **Aliases:**  | > nitrogen   |
+---------------+--------------+
| **Mappings:** | > mixs:nitro |
+---------------+--------------+

# Slot: nmdc schema version

TODO

URI:
[nmdc:nmdc_schema_version](https://microbiomedata/meta/nmdc_schema_version)

## Domain and Range {#domain-and-range-493}

None → 0..1 [String](types/String.md)

## Parents {#parents-530}

## Children {#children-500}

-   [database➞nmdc schema version](database_nmdc_schema_version.md)

## Used by {#used-by-493}

# Slot: has calibration

A reference to a file that holds calibration information.

URI:
<https://microbiomedata/schema/workflow_execution_activity/nom_analysis_activity_has_calibration>

## Domain and Range {#domain-and-range-494}

[NomAnalysisActivity](NomAnalysisActivity.md) → 0..1
[String](types/String.md)

## Parents {#parents-531}

-   is_a: [has calibration](has_calibration.md)

## Children {#children-501}

## Used by {#used-by-494}

-   [NomAnalysisActivity](NomAnalysisActivity.md)

# Slot: nom analysis activity set

This property links a database object to the set of natural organic
matter (NOM) analysis activities.

URI:
[nmdc:nom_analysis_activity_set](https://microbiomedata/meta/nom_analysis_activity_set)

## Domain and Range {#domain-and-range-495}

[Database](Database.md) → 0..\*
[NomAnalysisActivity](NomAnalysisActivity.md)

## Parents {#parents-532}

## Children {#children-502}

## Used by {#used-by-495}

-   [Database](Database.md)

# Slot: used

The instrument used to collect the data used in the analysis

URI:
<https://microbiomedata/schema/workflow_execution_activity/nom_analysis_activity_used>

## Domain and Range {#domain-and-range-496}

[NomAnalysisActivity](NomAnalysisActivity.md) → 0..1
[Instrument](Instrument.md)

## Parents {#parents-533}

-   is_a: [used](used.md)

## Children {#children-503}

## Used by {#used-by-496}

-   [NomAnalysisActivity](NomAnalysisActivity.md)

## Other properties {#other-properties-397}

  **Mappings:**    prov:used
  ---------------- ---------------
                   
  Slot: non_mine   ral_nutr_regm

"Information about treatment involving the exposure of plant to
non-mineral nutrient such as oxygen, hydrogen or carbon; should include
the name of non-mineral nutrient, amount administered, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include multiple non-mineral nutrient regimens"

URI:
[nmdc:non_mineral_nutr_regm](https://microbiomedata/meta/non_mineral_nutr_regm)

## Domain and Range {#domain-and-range-497}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-534}

-   is_a: [attribute](attribute.md)

## Children {#children-504}

## Used by {#used-by-497}

## Other properties {#other-properties-398}

+---------------+--------------------------------+
| **Aliases:**  | > non-mineral nutrient regimen |
+---------------+--------------------------------+
| **Mappings:** | > mixs:non_mineral_nutr_regm   |
+---------------+--------------------------------+

# Slot: nose_mouth_teeth_throat_disord

History of nose/mouth/teeth/throat disorders; can include multiple
disorders

URI:
[nmdc:nose_mouth_teeth_throat_disord](https://microbiomedata/meta/nose_mouth_teeth_throat_disord)

## Domain and Range {#domain-and-range-498}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-535}

-   is_a: [attribute](attribute.md)

## Children {#children-505}

## Used by {#used-by-498}

## Other properties {#other-properties-399}

+---------------+---------------------------------------+
| **Aliases:**  | > nose/mouth/teeth/throat disorder    |
+---------------+---------------------------------------+
| **Mappings:** | > mixs:nose_mouth_teeth_throat_disord |
+---------------+---------------------------------------+

# Slot: nose_throat_disord

History of nose-throat disorders; can include multiple disorders

URI:
[nmdc:nose_throat_disord](https://microbiomedata/meta/nose_throat_disord)

## Domain and Range {#domain-and-range-499}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-536}

-   is_a: [attribute](attribute.md)

## Children {#children-506}

## Used by {#used-by-499}

## Other properties {#other-properties-400}

+---------------+-----------------------------+
| **Aliases:**  | > lung/nose-throat disorder |
+---------------+-----------------------------+
| **Mappings:** | > mixs:nose_throat_disord   |
+---------------+-----------------------------+

# Slot: nucl_acid_amp

"A link to a literature reference, electronic resource or a standard
operating procedure (SOP), that describes the enzymatic amplification
(PCR, TMA, NASBA) of specific nucleic acids"

URI: [nmdc:nucl_acid_amp](https://microbiomedata/meta/nucl_acid_amp)

## Domain and Range {#domain-and-range-500}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-537}

-   is_a: [attribute](attribute.md)

## Children {#children-507}

## Used by {#used-by-500}

-   [OmicsProcessing](OmicsProcessing.md)

## Other properties {#other-properties-401}

+-----------------+------------------------------+
| **Aliases:**    | > nucleic acid amplification |
+-----------------+------------------------------+
| **Mappings:**   | > mixs:nucl_acid_amp         |
+-----------------+------------------------------+
| **In Subsets:** | > sequencing                 |
+-----------------+------------------------------+

# Slot: nucl_acid_ext

"A link to a literature reference, electronic resource or a standard
operating procedure (SOP), that describes the material separation to
recover the nucleic acid fraction from a sample"

URI: [nmdc:nucl_acid_ext](https://microbiomedata/meta/nucl_acid_ext)

## Domain and Range {#domain-and-range-501}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-538}

-   is_a: [attribute](attribute.md)

## Children {#children-508}

## Used by {#used-by-501}

-   [OmicsProcessing](OmicsProcessing.md)

## Other properties {#other-properties-402}

+-----------------+---------------------------+
| **Aliases:**    | > nucleic acid extraction |
+-----------------+---------------------------+
| **Mappings:**   | > mixs:nucl_acid_ext      |
+-----------------+---------------------------+
| **In Subsets:** | > sequencing              |
+-----------------+---------------------------+

# Slot: num_aligned_reads

The sequence count number of input reads aligned to assembled contigs.

URI:
<https://microbiomedata/schema/workflow_execution_activity/num_aligned_reads>

## Domain and Range {#domain-and-range-502}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-539}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-509}

## Used by {#used-by-502}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: num_input_reads

The sequence count number of input reads for assembly.

URI:
<https://microbiomedata/schema/workflow_execution_activity/num_input_reads>

## Domain and Range {#domain-and-range-503}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-540}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-510}

## Used by {#used-by-503}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: num_replicons

"Reports the number of replicons in a nuclear genome of eukaryotes, in
the genome of a bacterium or archaea or the number of segments in a
segmented virus. Always applied to the haploid chromosome count of a
eukaryote"

URI: [nmdc:num_replicons](https://microbiomedata/meta/num_replicons)

## Domain and Range {#domain-and-range-504}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-541}

-   is_a: [attribute](attribute.md)

## Children {#children-511}

## Used by {#used-by-504}

## Other properties {#other-properties-403}

+-----------------+--------------------------------+
| **Aliases:**    | > number of replicons          |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:num_replicons           |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: number_contig

"Total number of contigs in the cleaned/submitted assembly that makes up
a given genome, SAG, MAG, or UViG"

URI: [nmdc:number_contig](https://microbiomedata/meta/number_contig)

## Domain and Range {#domain-and-range-505}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-542}

-   is_a: [attribute](attribute.md)

## Children {#children-512}

## Used by {#used-by-505}

## Other properties {#other-properties-404}

+-----------------+----------------------+
| **Aliases:**    | > number of contigs  |
+-----------------+----------------------+
| **Mappings:**   | > mixs:number_contig |
+-----------------+----------------------+
| **In Subsets:** | > sequencing         |
+-----------------+----------------------+

# Slot: number_pets

The number of pets residing in the sampled space

URI: [nmdc:number_pets](https://microbiomedata/meta/number_pets)

## Domain and Range {#domain-and-range-506}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-543}

-   is_a: [attribute](attribute.md)

## Children {#children-513}

## Used by {#used-by-506}

## Other properties {#other-properties-405}

+---------------+--------------------+
| **Aliases:**  | > number of pets   |
+---------------+--------------------+
| **Mappings:** | > mixs:number_pets |
+---------------+--------------------+

# Slot: number_plants

The number of plant(s) in the sampling space

URI: [nmdc:number_plants](https://microbiomedata/meta/number_plants)

## Domain and Range {#domain-and-range-507}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-544}

-   is_a: [attribute](attribute.md)

## Children {#children-514}

## Used by {#used-by-507}

## Other properties {#other-properties-406}

+---------------+-------------------------+
| **Aliases:**  | > number of houseplants |
+---------------+-------------------------+
| **Mappings:** | > mixs:number_plants    |
+---------------+-------------------------+

# Slot: number_resident

The number of individuals currently occupying in the sampling location

URI: [nmdc:number_resident](https://microbiomedata/meta/number_resident)

## Domain and Range {#domain-and-range-508}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-545}

-   is_a: [attribute](attribute.md)

## Children {#children-515}

## Used by {#used-by-508}

## Other properties {#other-properties-407}

+---------------+------------------------+
| **Aliases:**  | > number of residents  |
+---------------+------------------------+
| **Mappings:** | > mixs:number_resident |
+---------------+------------------------+

# Slot: object set

Applies to a property that links a database object to a set of objects.
This is necessary in a json document to provide context for a list, and
to allow for a single json object that combines multiple object types

URI: [nmdc:object_set](https://microbiomedata/meta/object_set)

## Domain and Range {#domain-and-range-509}

[Database](Database.md) → 0..\* [String](types/String.md)

## Parents {#parents-546}

## Children {#children-516}

## Used by {#used-by-509}

# Slot: objective

The scientific objectives associated with the entity. It SHOULD
correspond to scientific norms for objectives field in a structured
abstract.

URI:
<https://microbiomedata/schema/workflow_execution_activity/objective>

## Domain and Range {#domain-and-range-510}

None → 0..1 [String](types/String.md)

## Parents {#parents-547}

## Children {#children-517}

## Used by {#used-by-510}

## Other properties {#other-properties-408}

  **Mappings:**    sio:000337
  ---------------- ------------
                   
  Slot: occup_de   nsity_samp

Average number of occupants at time of sampling per square footage

URI:
[nmdc:occup_density_samp](https://microbiomedata/meta/occup_density_samp)

## Domain and Range {#domain-and-range-511}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-548}

-   is_a: [attribute](attribute.md)

## Children {#children-518}

## Used by {#used-by-511}

## Other properties {#other-properties-409}

+---------------+--------------------------------+
| **Aliases:**  | > occupant density at sampling |
+---------------+--------------------------------+
| **Mappings:** | > mixs:occup_density_samp      |
+---------------+--------------------------------+

# Slot: occup_document

The type of documentation of occupancy

URI: [nmdc:occup_document](https://microbiomedata/meta/occup_document)

## Domain and Range {#domain-and-range-512}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-549}

-   is_a: [attribute](attribute.md)

## Children {#children-519}

## Used by {#used-by-512}

## Other properties {#other-properties-410}

+---------------+---------------------------+
| **Aliases:**  | > occupancy documentation |
+---------------+---------------------------+
| **Mappings:** | > mixs:occup_document     |
+---------------+---------------------------+

# Slot: occup_samp

Number of occupants present at time of sample within the given space

URI: [nmdc:occup_samp](https://microbiomedata/meta/occup_samp)

## Domain and Range {#domain-and-range-513}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-550}

-   is_a: [attribute](attribute.md)

## Children {#children-520}

## Used by {#used-by-513}

## Other properties {#other-properties-411}

+---------------+-------------------------+
| **Aliases:**  | > occupancy at sampling |
+---------------+-------------------------+
| **Mappings:** | > mixs:occup_samp       |
+---------------+-------------------------+

# Slot: has input

An input to a process.

URI:
[nmdc:omics_processing_has_input](https://microbiomedata/meta/omics_processing_has_input)

## Domain and Range {#domain-and-range-514}

[OmicsProcessing](OmicsProcessing.md) → 1..\* [Biosample](Biosample.md)

## Parents {#parents-551}

-   is_a: [biosample processing➞has
    input](biosample_processing_has_input.md)

## Children {#children-521}

## Used by {#used-by-514}

-   [OmicsProcessing](OmicsProcessing.md)

## Other properties {#other-properties-412}

  **Aliases:**    input
  --------------- -----------------------
                  
  Slot: omics p   rocessing identifiers

URI:
<https://microbiomedata/schema/workflow_execution_activity/omics_processing_identifiers>

## Domain and Range {#domain-and-range-515}

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents {#parents-552}

-   is_a: [external database
    identifiers](external_database_identifiers.md)

## Children {#children-522}

## Used by {#used-by-515}

# Slot: omics processing set

This property links a database object to the set of omics processings
within it.

URI:
[nmdc:omics_processing_set](https://microbiomedata/meta/omics_processing_set)

## Domain and Range {#domain-and-range-516}

[Database](Database.md) → 0..\* [OmicsProcessing](OmicsProcessing.md)

## Parents {#parents-553}

## Children {#children-523}

## Used by {#used-by-516}

-   [Database](Database.md)

# Slot: omics_type

URI: [nmdc:omics_type](https://microbiomedata/meta/omics_type)

## Domain and Range {#domain-and-range-517}

None → 0..1 [String](types/String.md)

## Parents {#parents-554}

## Children {#children-524}

## Used by {#used-by-517}

# Slot: orcid

The ORICD of a person.

URI: <https://microbiomedata/schema/workflow_execution_activity/orcid>

## Domain and Range {#domain-and-range-518}

[PersonValue](PersonValue.md) → 0..1 [String](types/String.md)

## Parents {#parents-555}

## Children {#children-525}

## Used by {#used-by-518}

-   [PersonValue](PersonValue.md)

# Slot: org_carb

Concentration of organic carbon

URI: [nmdc:org_carb](https://microbiomedata/meta/org_carb)

## Domain and Range {#domain-and-range-519}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-556}

-   is_a: [attribute](attribute.md)

## Children {#children-526}

## Used by {#used-by-519}

## Other properties {#other-properties-413}

+---------------+------------------+
| **Aliases:**  | > organic carbon |
+---------------+------------------+
| **Mappings:** | > mixs:org_carb  |
+---------------+------------------+

# Slot: org_matter

Concentration of organic matter

URI: [nmdc:org_matter](https://microbiomedata/meta/org_matter)

## Domain and Range {#domain-and-range-520}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-557}

-   is_a: [attribute](attribute.md)

## Children {#children-527}

## Used by {#used-by-520}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-414}

+---------------+-------------------+
| **Aliases:**  | > organic matter  |
+---------------+-------------------+
| **Mappings:** | > mixs:org_matter |
+---------------+-------------------+

# Slot: org_nitro

Concentration of organic nitrogen

URI: [nmdc:org_nitro](https://microbiomedata/meta/org_nitro)

## Domain and Range {#domain-and-range-521}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-558}

-   is_a: [attribute](attribute.md)

## Children {#children-528}

## Used by {#used-by-521}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-415}

+---------------+--------------------+
| **Aliases:**  | > organic nitrogen |
+---------------+--------------------+
| **Mappings:** | > mixs:org_nitro   |
+---------------+--------------------+

# Slot: org_particles

"Concentration of particles such as faeces, hairs, food, vomit, paper
fibers, plant material, humus, etc."

URI: [nmdc:org_particles](https://microbiomedata/meta/org_particles)

## Domain and Range {#domain-and-range-522}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-559}

-   is_a: [attribute](attribute.md)

## Children {#children-529}

## Used by {#used-by-522}

## Other properties {#other-properties-416}

+---------------+----------------------+
| **Aliases:**  | > organic particles  |
+---------------+----------------------+
| **Mappings:** | > mixs:org_particles |
+---------------+----------------------+

# Slot: organism_count

"Total cell count of any organism (or group of organisms) per gram,
volume or area of sample, should include name of organism followed by
count. The method that was used for the enumeration (e.g. qPCR, atp,
mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7
cells per ml; qpcr)"

URI: [nmdc:organism_count](https://microbiomedata/meta/organism_count)

## Domain and Range {#domain-and-range-523}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-560}

-   is_a: [attribute](attribute.md)

## Children {#children-530}

## Used by {#used-by-523}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-417}

+---------------+-----------------------+
| **Aliases:**  | > organism count      |
+---------------+-----------------------+
| **Mappings:** | > mixs:organism_count |
+---------------+-----------------------+

# Slot: organism_count_qpcr_info

"If qpcr was used for the cell count, the target gene name, the primer
sequence and the cycling conditions should also be provided. (Example:
16S rrna; FWD:ACGTAGCTATGACGT REV:GTGCTAGTCGAGTAC; initial
denaturation:90C_5min; denaturation:90C_2min; annealing:52C_30 sec;
elongation:72C_30 sec; 90 C for 1 min; final elongation:72C_5min; 30
cycles)"

URI:
[nmdc:organism_count_qpcr_info](https://microbiomedata/meta/organism_count_qpcr_info)

## Domain and Range {#domain-and-range-524}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-561}

-   is_a: [attribute](attribute.md)

## Children {#children-531}

## Used by {#used-by-524}

## Other properties {#other-properties-418}

+---------------+-----------------------------------+
| **Aliases:**  | > organism count qPCR information |
+---------------+-----------------------------------+
| **Mappings:** | > mixs:organism_count_qpcr_info   |
+---------------+-----------------------------------+

# Slot: output base count

After QC analysis nucleotide base count number.

URI:
<https://microbiomedata/schema/workflow_execution_activity/output_base_count>

## Domain and Range {#domain-and-range-525}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-562}

-   is_a: [read QC analysis statistic](read_QC_analysis_statistic.md)

## Children {#children-532}

## Used by {#used-by-525}

-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)

# Slot: output_read_bases

TODO

URI:
<https://microbiomedata/schema/workflow_execution_activity/output_read_bases>

## Domain and Range {#domain-and-range-526}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-563}

## Children {#children-533}

-   [read QC analysis
    activity➞output_read_bases](read_QC_analysis_activity_output_read_bases.md)

## Used by {#used-by-526}

# Slot: output read count

After QC analysis sequence count number.

URI:
<https://microbiomedata/schema/workflow_execution_activity/output_read_count>

## Domain and Range {#domain-and-range-527}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-564}

-   is_a: [read QC analysis statistic](read_QC_analysis_statistic.md)

## Children {#children-534}

## Used by {#used-by-527}

-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)

# Slot: owc_tvdss

Depth of the original oil water contact (OWC) zone (average) (m TVDSS)

URI: [nmdc:owc_tvdss](https://microbiomedata/meta/owc_tvdss)

## Domain and Range {#domain-and-range-528}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-565}

-   is_a: [attribute](attribute.md)

## Children {#children-535}

## Used by {#used-by-528}

## Other properties {#other-properties-419}

+---------------+---------------------------+
| **Aliases:**  | > oil water contact depth |
+---------------+---------------------------+
| **Mappings:** | > mixs:owc_tvdss          |
+---------------+---------------------------+

# Slot: oxy_stat_samp

Oxygenation status of sample

URI: [nmdc:oxy_stat_samp](https://microbiomedata/meta/oxy_stat_samp)

## Domain and Range {#domain-and-range-529}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-566}

-   is_a: [attribute](attribute.md)

## Children {#children-536}

## Used by {#used-by-529}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-420}

+---------------+--------------------------------+
| **Aliases:**  | > oxygenation status of sample |
+---------------+--------------------------------+
| **Mappings:** | > mixs:oxy_stat_samp           |
+---------------+--------------------------------+

# Slot: oxygen

Oxygen (gas) amount or concentration at the time of sampling

URI: [nmdc:oxygen](https://microbiomedata/meta/oxygen)

## Domain and Range {#domain-and-range-530}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-567}

-   is_a: [attribute](attribute.md)

## Children {#children-537}

## Used by {#used-by-530}

## Other properties {#other-properties-421}

+---------------+---------------+
| **Aliases:**  | > oxygen      |
+---------------+---------------+
| **Mappings:** | > mixs:oxygen |
+---------------+---------------+

# Slot: part of

Links a resource to another resource that either logically or physically
includes it.

URI: <https://microbiomedata/schema/workflow_execution_activity/part_of>

## Domain and Range {#domain-and-range-531}

[NamedThing](NamedThing.md) → 0..\* [NamedThing](NamedThing.md)

## Parents {#parents-568}

## Children {#children-538}

## Used by {#used-by-531}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md)
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)

## Other properties {#other-properties-422}

+---------------+--------------------+
| **Aliases:**  | > is part of       |
+---------------+--------------------+
| **Mappings:** | > dcterms:isPartOf |
+---------------+--------------------+

# Slot: part_org_carb

Concentration of particulate organic carbon

URI: [nmdc:part_org_carb](https://microbiomedata/meta/part_org_carb)

## Domain and Range {#domain-and-range-532}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-569}

-   is_a: [attribute](attribute.md)

## Children {#children-539}

## Used by {#used-by-532}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-423}

+---------------+------------------------------+
| **Aliases:**  | > particulate organic carbon |
+---------------+------------------------------+
| **Mappings:** | > mixs:part_org_carb         |
+---------------+------------------------------+

# Slot: part_org_nitro

Concentration of particulate organic nitrogen

URI: [nmdc:part_org_nitro](https://microbiomedata/meta/part_org_nitro)

## Domain and Range {#domain-and-range-533}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-570}

-   is_a: [attribute](attribute.md)

## Children {#children-540}

## Used by {#used-by-533}

## Other properties {#other-properties-424}

+---------------+--------------------------------+
| **Aliases:**  | > particulate organic nitrogen |
+---------------+--------------------------------+
| **Mappings:** | > mixs:part_org_nitro          |
+---------------+--------------------------------+

# Slot: particle_class

"Particles are classified, based on their size, into six general
categories:clay, silt, sand, gravel, cobbles, and boulders; should
include amount of particle preceded by the name of the particle type;
can include multiple values"

URI: [nmdc:particle_class](https://microbiomedata/meta/particle_class)

## Domain and Range {#domain-and-range-534}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-571}

-   is_a: [attribute](attribute.md)

## Children {#children-541}

## Used by {#used-by-534}

## Other properties {#other-properties-425}

+---------------+---------------------------+
| **Aliases:**  | > particle classification |
+---------------+---------------------------+
| **Mappings:** | > mixs:particle_class     |
+---------------+---------------------------+

# Slot: pathogenicity

To what is the entity pathogenic

URI: [nmdc:pathogenicity](https://microbiomedata/meta/pathogenicity)

## Domain and Range {#domain-and-range-535}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-572}

-   is_a: [attribute](attribute.md)

## Children {#children-542}

## Used by {#used-by-535}

## Other properties {#other-properties-426}

+-----------------+--------------------------------+
| **Aliases:**    | > known pathogenicity          |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:pathogenicity           |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: has_part {#slot-has_part-1}

A pathway can be broken down to a series of reaction step

URI:
[nmdc:pathway_has_part](https://microbiomedata/meta/pathway_has_part)

## Domain and Range {#domain-and-range-536}

[Pathway](Pathway.md) → 0..\* [Reaction](Reaction.md)

## Parents {#parents-573}

-   is_a: [has_part](has_part.md)

## Children {#children-543}

## Used by {#used-by-536}

-   [Pathway](Pathway.md)

# Slot: pcr_cond

Description of reaction conditions and components of PCR in the form of
'initial denaturation:94degC_1.5min; annealing=...'

URI: [nmdc:pcr_cond](https://microbiomedata/meta/pcr_cond)

## Domain and Range {#domain-and-range-537}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-574}

-   is_a: [attribute](attribute.md)

## Children {#children-544}

## Used by {#used-by-537}

-   [OmicsProcessing](OmicsProcessing.md)

## Other properties {#other-properties-427}

+-----------------+------------------+
| **Aliases:**    | > pcr conditions |
+-----------------+------------------+
| **Mappings:**   | > mixs:pcr_cond  |
+-----------------+------------------+
| **In Subsets:** | > sequencing     |
+-----------------+------------------+

# Slot: pcr_primers

"PCR primers that were used to amplify the sequence of the targeted
gene, locus or subfragment. This field should contain all the primers
used for a single PCR reaction if multiple forward or reverse primers
are present in a single PCR reaction. The primer sequence should be
reported in uppercase letters"

URI: [nmdc:pcr_primers](https://microbiomedata/meta/pcr_primers)

## Domain and Range {#domain-and-range-538}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-575}

-   is_a: [attribute](attribute.md)

## Children {#children-545}

## Used by {#used-by-538}

-   [OmicsProcessing](OmicsProcessing.md)

## Other properties {#other-properties-428}

+-----------------+--------------------+
| **Aliases:**    | > pcr primers      |
+-----------------+--------------------+
| **Mappings:**   | > mixs:pcr_primers |
+-----------------+--------------------+
| **In Subsets:** | > sequencing       |
+-----------------+--------------------+

# Slot: all proteins

the list of protein identifiers that are associated with the peptide
sequence

URI:
<https://microbiomedata/schema/workflow_execution_activity/peptide_quantification_all_proteins>

## Domain and Range {#domain-and-range-539}

[PeptideQuantification](PeptideQuantification.md) → 0..\*
[GeneProduct](GeneProduct.md)

## Parents {#parents-576}

-   is_a: [all proteins](all_proteins.md)

## Children {#children-546}

## Used by {#used-by-539}

-   [PeptideQuantification](PeptideQuantification.md)

# Slot: best protein

the specific protein identifier most correctly associated with the
peptide sequence

URI:
<https://microbiomedata/schema/workflow_execution_activity/peptide_quantification_best_protein>

## Domain and Range {#domain-and-range-540}

[PeptideQuantification](PeptideQuantification.md) → 0..1
[GeneProduct](GeneProduct.md)

## Parents {#parents-577}

-   is_a: [best protein](best_protein.md)

## Children {#children-547}

## Used by {#used-by-540}

-   [PeptideQuantification](PeptideQuantification.md)

# Slot: min_q\_value

smallest Q-Value associated with the peptide sequence as provided by
MSGFPlus tool

URI:
<https://microbiomedata/schema/workflow_execution_activity/peptide_quantification_min_q_value>

## Domain and Range {#domain-and-range-541}

[PeptideQuantification](PeptideQuantification.md) → 0..1
[Float](types/Float.md)

## Parents {#parents-578}

-   is_a: [min_q\_value](min_q_value.md)

## Children {#children-548}

## Used by {#used-by-541}

-   [PeptideQuantification](PeptideQuantification.md)

## Other properties {#other-properties-429}

  **See also:**   [OBI:0001442](OBI:0001442)
  --------------- ----------------------------
                  
  Slot: peptide   sequence

URI:
<https://microbiomedata/schema/workflow_execution_activity/peptide_quantification_peptide_sequence>

## Domain and Range {#domain-and-range-542}

[PeptideQuantification](PeptideQuantification.md) → 0..1
[String](types/String.md)

## Parents {#parents-579}

-   is_a: [peptide sequence](peptide_sequence.md)

## Children {#children-549}

## Used by {#used-by-542}

-   [PeptideQuantification](PeptideQuantification.md)

# Slot: peptide_spectral_count

sum of filter passing MS2 spectra associated with the peptide sequence
within a given LC-MS/MS data file

URI:
<https://microbiomedata/schema/workflow_execution_activity/peptide_quantification_peptide_spectral_count>

## Domain and Range {#domain-and-range-543}

[PeptideQuantification](PeptideQuantification.md) → 0..1
[Integer](types/Integer.md)

## Parents {#parents-580}

-   is_a: [peptide_spectral_count](peptide_spectral_count.md)

## Children {#children-550}

## Used by {#used-by-543}

-   [PeptideQuantification](PeptideQuantification.md)

# Slot: peptide_sum_masic_abundance

combined MS1 extracted ion chromatograms derived from MS2 spectra
associated with the peptide sequence from a given LC-MS/MS data file
using the MASIC tool

URI:
<https://microbiomedata/schema/workflow_execution_activity/peptide_quantification_peptide_sum_masic_abundance>

## Domain and Range {#domain-and-range-544}

[PeptideQuantification](PeptideQuantification.md) → 0..1
[Integer](types/Integer.md)

## Parents {#parents-581}

-   is_a: [peptide_sum_masic_abundance](peptide_sum_masic_abundance.md)

## Children {#children-551}

## Used by {#used-by-544}

-   [PeptideQuantification](PeptideQuantification.md)

# Slot: peptide sequence {#slot-peptide-sequence-1}

URI:
<https://microbiomedata/schema/workflow_execution_activity/peptide_sequence>

## Domain and Range {#domain-and-range-545}

None → 0..1 [String](types/String.md)

## Parents {#parents-582}

## Children {#children-552}

-   [peptide quantification➞peptide
    sequence](peptide_quantification_peptide_sequence.md)

## Used by {#used-by-545}

# Slot: peptide_sequence_count

count of peptide sequences grouped to the best_protein

URI:
<https://microbiomedata/schema/workflow_execution_activity/peptide_sequence_count>

## Domain and Range {#domain-and-range-546}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-583}

## Children {#children-553}

-   [protein
    quantification➞peptide_sequence_count](protein_quantification_peptide_sequence_count.md)

## Used by {#used-by-546}

# Slot: peptide_spectral_count

sum of filter passing MS2 spectra associated with the peptide sequence
within a given LC-MS/MS data file

URI:
<https://microbiomedata/schema/workflow_execution_activity/peptide_spectral_count>

## Domain and Range {#domain-and-range-547}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-584}

## Children {#children-554}

-   [peptide
    quantification➞peptide_spectral_count](peptide_quantification_peptide_spectral_count.md)

## Used by {#used-by-547}

# Slot: peptide_sum_masic_abundance

combined MS1 extracted ion chromatograms derived from MS2 spectra
associated with the peptide sequence from a given LC-MS/MS data file
using the MASIC tool

URI:
<https://microbiomedata/schema/workflow_execution_activity/peptide_sum_masic_abundance>

## Domain and Range {#domain-and-range-548}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-585}

## Children {#children-555}

-   [peptide
    quantification➞peptide_sum_masic_abundance](peptide_quantification_peptide_sum_masic_abundance.md)

## Used by {#used-by-548}

# Slot: permeability

Measure of the ability of a hydrocarbon resource to allow fluids to pass
through it. (Additional information:
<https://en.wikipedia.org/wiki/Permeability_(earth_sciences)>)

URI: [nmdc:permeability](https://microbiomedata/meta/permeability)

## Domain and Range {#domain-and-range-549}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-586}

-   is_a: [attribute](attribute.md)

## Children {#children-556}

## Used by {#used-by-549}

## Other properties {#other-properties-430}

+---------------+---------------------+
| **Aliases:**  | > permeability      |
+---------------+---------------------+
| **Mappings:** | > mixs:permeability |
+---------------+---------------------+

# Slot: id

Should be an ORCID. Specify in CURIE format. E.g ORCID:0000-1111-...

URI:
<https://microbiomedata/schema/workflow_execution_activity/person_id>

## Domain and Range {#domain-and-range-550}

[Person](Person.md) → 1..1 [String](types/String.md)

## Parents {#parents-587}

-   is_a: [id](id.md)

## Children {#children-557}

## Used by {#used-by-550}

-   [Person](Person.md)

# Slot: has raw value

The full name of the Investgator in format FIRST LAST.

URI:
<https://microbiomedata/schema/workflow_execution_activity/person_value_has_raw_value>

## Domain and Range {#domain-and-range-551}

[PersonValue](PersonValue.md) → 0..1 [String](types/String.md)

## Parents {#parents-588}

-   is_a: [has raw value](has_raw_value.md)

## Children {#children-558}

## Used by {#used-by-551}

-   [PersonValue](PersonValue.md)

# Slot: name {#slot-name-2}

The full name of the Investgator. It should follow the format FIRST
\[MIDDLE NAME\| MIDDLE INITIAL\] LAST, where MIDDLE NAME\| MIDDLE
INITIAL is optional.

URI:
<https://microbiomedata/schema/workflow_execution_activity/person_value_name>

## Domain and Range {#domain-and-range-552}

[PersonValue](PersonValue.md) → 0..1 [String](types/String.md)

## Parents {#parents-589}

-   is_a: [name](name.md)

## Children {#children-559}

## Used by {#used-by-552}

-   [PersonValue](PersonValue.md)

# Slot: perturbation

"Type of perturbation, e.g. chemical administration, physical
disturbance, etc., coupled with perturbation regimen including how many
times the perturbation was repeated, how long each perturbation lasted,
and the start and end time of the entire perturbation period; can
include multiple perturbation types"

URI: [nmdc:perturbation](https://microbiomedata/meta/perturbation)

## Domain and Range {#domain-and-range-553}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-590}

-   is_a: [attribute](attribute.md)

## Children {#children-560}

## Used by {#used-by-553}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-431}

+---------------+---------------------+
| **Aliases:**  | > perturbation      |
+---------------+---------------------+
| **Mappings:** | > mixs:perturbation |
+---------------+---------------------+

# Slot: pesticide_regm

"Information about treatment involving use of insecticides; should
include the name of pesticide, amount administered, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include multiple pesticide regimens"

URI: [nmdc:pesticide_regm](https://microbiomedata/meta/pesticide_regm)

## Domain and Range {#domain-and-range-554}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-591}

-   is_a: [attribute](attribute.md)

## Children {#children-561}

## Used by {#used-by-554}

## Other properties {#other-properties-432}

+---------------+-----------------------+
| **Aliases:**  | > pesticide regimen   |
+---------------+-----------------------+
| **Mappings:** | > mixs:pesticide_regm |
+---------------+-----------------------+

# Slot: pet_farm_animal

"Specification of presence of pets or farm animals in the environment of
subject, if yes the animals should be specified; can include multiple
animals present"

URI: [nmdc:pet_farm_animal](https://microbiomedata/meta/pet_farm_animal)

## Domain and Range {#domain-and-range-555}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-592}

-   is_a: [attribute](attribute.md)

## Children {#children-562}

## Used by {#used-by-555}

## Other properties {#other-properties-433}

+---------------+------------------------------------+
| **Aliases:**  | > presence of pets or farm animals |
+---------------+------------------------------------+
| **Mappings:** | > mixs:pet_farm_animal             |
+---------------+------------------------------------+

# Slot: petroleum_hydrocarb

Concentration of petroleum hydrocarbon

URI:
[nmdc:petroleum_hydrocarb](https://microbiomedata/meta/petroleum_hydrocarb)

## Domain and Range {#domain-and-range-556}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-593}

-   is_a: [attribute](attribute.md)

## Children {#children-563}

## Used by {#used-by-556}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-434}

+---------------+----------------------------+
| **Aliases:**  | > petroleum hydrocarbon    |
+---------------+----------------------------+
| **Mappings:** | > mixs:petroleum_hydrocarb |
+---------------+----------------------------+

# Slot: ph

"Ph measurement of the sample, or liquid portion of sample, or aqueous
phase of the fluid"

URI: [nmdc:ph](https://microbiomedata/meta/ph)

## Domain and Range {#domain-and-range-557}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-594}

-   is_a: [attribute](attribute.md)

## Children {#children-564}

## Used by {#used-by-557}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-435}

+---------------+-----------+
| **Aliases:**  | > pH      |
+---------------+-----------+
| **Mappings:** | > mixs:ph |
+---------------+-----------+

# Slot: ph_meth

Reference or method used in determining ph

URI: [nmdc:ph_meth](https://microbiomedata/meta/ph_meth)

## Domain and Range {#domain-and-range-558}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-595}

-   is_a: [attribute](attribute.md)

## Children {#children-565}

## Used by {#used-by-558}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-436}

+---------------+----------------+
| **Aliases:**  | > pH method    |
+---------------+----------------+
| **Mappings:** | > mixs:ph_meth |
+---------------+----------------+

# Slot: ph_regm

"Information about treatment involving exposure of plants to varying
levels of ph of the growth media, treatment regimen including how many
times the treatment was repeated, how long each treatment lasted, and
the start and end time of the entire treatment; can include multiple
regimen"

URI: [nmdc:ph_regm](https://microbiomedata/meta/ph_regm)

## Domain and Range {#domain-and-range-559}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-596}

-   is_a: [attribute](attribute.md)

## Children {#children-566}

## Used by {#used-by-559}

## Other properties {#other-properties-437}

+---------------+----------------+
| **Aliases:**  | > pH regimen   |
+---------------+----------------+
| **Mappings:** | > mixs:ph_regm |
+---------------+----------------+

# Slot: phaeopigments

Concentration of phaeopigments; can include multiple phaeopigments

URI: [nmdc:phaeopigments](https://microbiomedata/meta/phaeopigments)

## Domain and Range {#domain-and-range-560}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-597}

-   is_a: [attribute](attribute.md)

## Children {#children-567}

## Used by {#used-by-560}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-438}

+---------------+----------------------+
| **Aliases:**  | > phaeopigments      |
+---------------+----------------------+
| **Mappings:** | > mixs:phaeopigments |
+---------------+----------------------+

# Slot: phase

The phase for a coding sequence entity. For example, phase of a CDS as
represented in a GFF3 with a value of 0, 1 or 2.

URI: [nmdc:phase](https://microbiomedata/meta/phase)

## Domain and Range {#domain-and-range-561}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-598}

## Children {#children-568}

-   [genome feature➞phase](genome_feature_phase.md)

## Used by {#used-by-561}

## Other properties {#other-properties-439}

+----------------------------------+----------------------------------+
| **Exact Mappings:**              | biolink:phase                    |
+==================================+==================================+
| Slot: phosphate ===============  |                                  |
+----------------------------------+----------------------------------+
| Concentration of pho             | sphate                           |
+----------------------------------+----------------------------------+
| URI: \`nmdc:phosphate            | > \<<https://microb              |
|                                  | iomedata/meta/phosphate>\>\`\_\_ |
+----------------------------------+----------------------------------+
| ## Domain a                      | -562:                            |
| nd Range {#domain-and-range-560} |                                  |
+----------------------------------+----------------------------------+
| None → 0..1 \`Quantit            | yValue                           |
|                                  | \<QuantityValue.md\>\`\_\_       |
| ## Parents {#parents-599}        |                                  |
+----------------------------------+----------------------------------+
| -   is_a: \`attribute            | \<attribute.md\>\`\_\_           |
|                                  |                                  |
| ## Children {#children-569}      |                                  |
|                                  |                                  |
| ## Used by {#used-by-562}        |                                  |
+----------------------------------+----------------------------------+
| ## Other pr                      | -440:                            |
| operties {#other-properties-438} |                                  |
+----------------------------------+----------------------------------+
| ============= ======             | ========                         |
+----------------------------------+----------------------------------+
| **Aliases:** phosp               | hate                             |
+----------------------------------+----------------------------------+
| **Mappings:** mixs:              | phosphate                        |
+----------------------------------+----------------------------------+

# Slot: phosplipid_fatt_acid

Concentration of phospholipid fatty acids; can include multiple values

URI:
[nmdc:phosplipid_fatt_acid](https://microbiomedata/meta/phosplipid_fatt_acid)

## Domain and Range {#domain-and-range-563}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-600}

-   is_a: [attribute](attribute.md)

## Children {#children-570}

## Used by {#used-by-563}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-441}

+---------------+-----------------------------+
| **Aliases:**  | > phospholipid fatty acid   |
+---------------+-----------------------------+
| **Mappings:** | > mixs:phosplipid_fatt_acid |
+---------------+-----------------------------+

# Slot: photon_flux

Measurement of photon flux

URI: [nmdc:photon_flux](https://microbiomedata/meta/photon_flux)

## Domain and Range {#domain-and-range-564}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-601}

-   is_a: [attribute](attribute.md)

## Children {#children-571}

## Used by {#used-by-564}

## Other properties {#other-properties-442}

+---------------+--------------------+
| **Aliases:**  | > photon flux      |
+---------------+--------------------+
| **Mappings:** | > mixs:photon_flux |
+---------------+--------------------+

# Slot: plant_growth_med

"Specification of the media for growing the plants or tissue cultured
samples, e.g. soil, aeroponic, hydroponic, in vitro solid culture
medium, in vitro liquid culture medium. Recommended value is a specific
value from EO:plant growth medium (follow this link for terms
<http://purl.obolibrary.org/obo/EO_0007147>) or other controlled
vocabulary"

URI:
[nmdc:plant_growth_med](https://microbiomedata/meta/plant_growth_med)

## Domain and Range {#domain-and-range-565}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-602}

-   is_a: [attribute](attribute.md)

## Children {#children-572}

## Used by {#used-by-565}

## Other properties {#other-properties-443}

+---------------+-------------------------+
| **Aliases:**  | > plant growth medium   |
+---------------+-------------------------+
| **Mappings:** | > mixs:plant_growth_med |
+---------------+-------------------------+

# Slot: plant_product

"Substance produced by the plant, where the sample was obtained from"

URI: [nmdc:plant_product](https://microbiomedata/meta/plant_product)

## Domain and Range {#domain-and-range-566}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-603}

-   is_a: [attribute](attribute.md)

## Children {#children-573}

## Used by {#used-by-566}

## Other properties {#other-properties-444}

+---------------+----------------------+
| **Aliases:**  | > plant product      |
+---------------+----------------------+
| **Mappings:** | > mixs:plant_product |
+---------------+----------------------+

# Slot: plant_sex

"Sex of the reproductive parts on the whole plant, e.g. pistillate,
staminate, monoecieous, hermaphrodite."

URI: [nmdc:plant_sex](https://microbiomedata/meta/plant_sex)

## Domain and Range {#domain-and-range-567}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-604}

-   is_a: [attribute](attribute.md)

## Children {#children-574}

## Used by {#used-by-567}

## Other properties {#other-properties-445}

+---------------+------------------+
| **Aliases:**  | > plant sex      |
+---------------+------------------+
| **Mappings:** | > mixs:plant_sex |
+---------------+------------------+

# Slot: plant_struc

"Name of plant structure the sample was obtained from; for Plant
Ontology (PO) (v releases/2017-12-14) terms, see
<http://purl.bioontology.org/ontology/PO>, e.g. petiole epidermis
(PO_0000051). If an individual flower is sampled, the sex of it can be
recorded here."

URI: [nmdc:plant_struc](https://microbiomedata/meta/plant_struc)

## Domain and Range {#domain-and-range-568}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-605}

-   is_a: [attribute](attribute.md)

## Children {#children-575}

## Used by {#used-by-568}

## Other properties {#other-properties-446}

+---------------+--------------------+
| **Aliases:**  | > plant structure  |
+---------------+--------------------+
| **Mappings:** | > mixs:plant_struc |
+---------------+--------------------+

# Slot: ploidy

"The ploidy level of the genome (e.g. allopolyploid, haploid, diploid,
triploid, tetraploid). It has implications for the downstream study of
duplicated gene and regions of the genomes (and perhaps for difficulties
in assembly). For terms, please select terms listed under class ploidy
(PATO:001374) of Phenotypic Quality Ontology (PATO), and for a browser
of PATO (v 2018-03-27) please refer to
<http://purl.bioontology.org/ontology/PATO>"

URI: [nmdc:ploidy](https://microbiomedata/meta/ploidy)

## Domain and Range {#domain-and-range-569}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-606}

-   is_a: [attribute](attribute.md)

## Children {#children-576}

## Used by {#used-by-569}

## Other properties {#other-properties-447}

+-----------------+--------------------------------+
| **Aliases:**    | > ploidy                       |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:ploidy                  |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: pollutants

"Pollutant types and, amount or concentrations measured at the time of
sampling; can report multiple pollutants by entering numeric values
preceded by name of pollutant"

URI: [nmdc:pollutants](https://microbiomedata/meta/pollutants)

## Domain and Range {#domain-and-range-570}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-607}

-   is_a: [attribute](attribute.md)

## Children {#children-577}

## Used by {#used-by-570}

## Other properties {#other-properties-448}

+---------------+-------------------+
| **Aliases:**  | > pollutants      |
+---------------+-------------------+
| **Mappings:** | > mixs:pollutants |
+---------------+-------------------+

# Slot: pool_dna_extracts

"Indicate whether multiple DNA extractions were mixed. If the answer
yes, the number of extracts that were pooled should be given"

URI:
[nmdc:pool_dna_extracts](https://microbiomedata/meta/pool_dna_extracts)

## Domain and Range {#domain-and-range-571}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-608}

-   is_a: [attribute](attribute.md)

## Children {#children-578}

## Used by {#used-by-571}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-449}

+---------------+-------------------------------------+
| **Aliases:**  | > pooling of DNA extracts (if done) |
+---------------+-------------------------------------+
| **Mappings:** | > mixs:pool_dna_extracts            |
+---------------+-------------------------------------+

# Slot: porosity

Porosity of deposited sediment is volume of voids divided by the total
volume of sample

URI: [nmdc:porosity](https://microbiomedata/meta/porosity)

## Domain and Range {#domain-and-range-572}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-609}

-   is_a: [attribute](attribute.md)

## Children {#children-579}

## Used by {#used-by-572}

## Other properties {#other-properties-450}

+---------------+-----------------+
| **Aliases:**  | > porosity      |
+---------------+-----------------+
| **Mappings:** | > mixs:porosity |
+---------------+-----------------+

# Slot: potassium

Concentration of potassium in the sample

URI: [nmdc:potassium](https://microbiomedata/meta/potassium)

## Domain and Range {#domain-and-range-573}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-610}

-   is_a: [attribute](attribute.md)

## Children {#children-580}

## Used by {#used-by-573}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-451}

+---------------+------------------+
| **Aliases:**  | > potassium      |
+---------------+------------------+
| **Mappings:** | > mixs:potassium |
+---------------+------------------+

# Slot: pour_point

"Temperature at which a liquid becomes semi solid and loses its flow
characteristics. In crude oil a highB pour pointB is generally
associated with a high paraffin content, typically found in crude
deriving from a larger proportion of plant material. (soure:
<https://en.wikipedia.org/wiki/pour_point>)"

URI: [nmdc:pour_point](https://microbiomedata/meta/pour_point)

## Domain and Range {#domain-and-range-574}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-611}

-   is_a: [attribute](attribute.md)

## Children {#children-581}

## Used by {#used-by-574}

## Other properties {#other-properties-452}

+---------------+-------------------+
| **Aliases:**  | > pour point      |
+---------------+-------------------+
| **Mappings:** | > mixs:pour_point |
+---------------+-------------------+

# Slot: pre_treatment

The process of pre-treatment removes materials that can be easily
collected from the raw wastewater

URI: [nmdc:pre_treatment](https://microbiomedata/meta/pre_treatment)

## Domain and Range {#domain-and-range-575}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-612}

-   is_a: [attribute](attribute.md)

## Children {#children-582}

## Used by {#used-by-575}

## Other properties {#other-properties-453}

+---------------+----------------------+
| **Aliases:**  | > pre-treatment      |
+---------------+----------------------+
| **Mappings:** | > mixs:pre_treatment |
+---------------+----------------------+

# Slot: pred_genome_struc

Expected structure of the viral genome

URI:
[nmdc:pred_genome_struc](https://microbiomedata/meta/pred_genome_struc)

## Domain and Range {#domain-and-range-576}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-613}

-   is_a: [attribute](attribute.md)

## Children {#children-583}

## Used by {#used-by-576}

## Other properties {#other-properties-454}

+-----------------+------------------------------+
| **Aliases:**    | > predicted genome structure |
+-----------------+------------------------------+
| **Mappings:**   | > mixs:pred_genome_struc     |
+-----------------+------------------------------+
| **In Subsets:** | > sequencing                 |
+-----------------+------------------------------+

# Slot: pred_genome_type

Type of genome predicted for the UViG

URI:
[nmdc:pred_genome_type](https://microbiomedata/meta/pred_genome_type)

## Domain and Range {#domain-and-range-577}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-614}

-   is_a: [attribute](attribute.md)

## Children {#children-584}

## Used by {#used-by-577}

## Other properties {#other-properties-455}

+-----------------+-------------------------+
| **Aliases:**    | > predicted genome type |
+-----------------+-------------------------+
| **Mappings:**   | > mixs:pred_genome_type |
+-----------------+-------------------------+
| **In Subsets:** | > sequencing            |
+-----------------+-------------------------+

# Slot: pregnancy

Date due of pregnancy

URI: [nmdc:pregnancy](https://microbiomedata/meta/pregnancy)

## Domain and Range {#domain-and-range-578}

None → 0..1 [TimestampValue](TimestampValue.md)

## Parents {#parents-615}

-   is_a: [attribute](attribute.md)

## Children {#children-585}

## Used by {#used-by-578}

## Other properties {#other-properties-456}

+---------------+------------------+
| **Aliases:**  | > pregnancy      |
+---------------+------------------+
| **Mappings:** | > mixs:pregnancy |
+---------------+------------------+

# Slot: pres_animal

The number and type of animals present in the sampling space

URI: [nmdc:pres_animal](https://microbiomedata/meta/pres_animal)

## Domain and Range {#domain-and-range-579}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-616}

-   is_a: [attribute](attribute.md)

## Children {#children-586}

## Used by {#used-by-579}

## Other properties {#other-properties-457}

+---------------+-----------------------------------------+
| **Aliases:**  | > presence of pets, animals, or insects |
+---------------+-----------------------------------------+
| **Mappings:** | > mixs:pres_animal                      |
+---------------+-----------------------------------------+

# Slot: pressure

"Pressure to which the sample is subject to, in atmospheres"

URI: [nmdc:pressure](https://microbiomedata/meta/pressure)

## Domain and Range {#domain-and-range-580}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-617}

-   is_a: [attribute](attribute.md)

## Children {#children-587}

## Used by {#used-by-580}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-458}

+---------------+-----------------+
| **Aliases:**  | > pressure      |
+---------------+-----------------+
| **Mappings:** | > mixs:pressure |
+---------------+-----------------+

# Slot: previous_land_use

Previous land use and dates

URI:
[nmdc:previous_land_use](https://microbiomedata/meta/previous_land_use)

## Domain and Range {#domain-and-range-581}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-618}

-   is_a: [attribute](attribute.md)

## Children {#children-588}

## Used by {#used-by-581}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-459}

+---------------+-----------------------------+
| **Aliases:**  | > history/previous land use |
+---------------+-----------------------------+
| **Mappings:** | > mixs:previous_land_use    |
+---------------+-----------------------------+

# Slot: previous_land_use_meth

Reference or method used in determining previous land use and dates

URI:
[nmdc:previous_land_use_meth](https://microbiomedata/meta/previous_land_use_meth)

## Domain and Range {#domain-and-range-582}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-619}

-   is_a: [attribute](attribute.md)

## Children {#children-589}

## Used by {#used-by-582}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-460}

+---------------+------------------------------------+
| **Aliases:**  | > history/previous land use method |
+---------------+------------------------------------+
| **Mappings:** | > mixs:previous_land_use_meth      |
+---------------+------------------------------------+

# Slot: primary_prod

"Measurement of primary production, generally measured as isotope
uptake"

URI: [nmdc:primary_prod](https://microbiomedata/meta/primary_prod)

## Domain and Range {#domain-and-range-583}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-620}

-   is_a: [attribute](attribute.md)

## Children {#children-590}

## Used by {#used-by-583}

## Other properties {#other-properties-461}

+---------------+----------------------+
| **Aliases:**  | > primary production |
+---------------+----------------------+
| **Mappings:** | > mixs:primary_prod  |
+---------------+----------------------+

# Slot: primary_treatment

The process to produce both a generally homogeneous liquid capable of
being treated biologically and a sludge that can be separately treated
or processed

URI:
[nmdc:primary_treatment](https://microbiomedata/meta/primary_treatment)

## Domain and Range {#domain-and-range-584}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-621}

-   is_a: [attribute](attribute.md)

## Children {#children-591}

## Used by {#used-by-584}

## Other properties {#other-properties-462}

+---------------+--------------------------+
| **Aliases:**  | > primary treatment      |
+---------------+--------------------------+
| **Mappings:** | > mixs:primary_treatment |
+---------------+--------------------------+

# Slot: principal investigator

represents the PI

URI:
[nmdc:principal_investigator](https://microbiomedata/meta/principal_investigator)

## Domain and Range {#domain-and-range-585}

None → 0..1 [PersonValue](PersonValue.md)

## Parents {#parents-622}

-   is_a: [attribute](attribute.md)

## Children {#children-592}

## Used by {#used-by-585}

-   [OmicsProcessing](OmicsProcessing.md)
-   [Study](Study.md)

## Other properties {#other-properties-463}

  **Aliases:**    PI
  --------------- -----------------
                  
  Slot: process   ing_institution

The organization that processed the sample.

URI:
[nmdc:processing_institution](https://microbiomedata/meta/processing_institution)

## Domain and Range {#domain-and-range-586}

None → 0..1 [String](types/String.md)

## Parents {#parents-623}

## Children {#children-593}

## Used by {#used-by-586}

-   [OmicsProcessing](OmicsProcessing.md)

# Slot: prod_rate

Oil and/or gas production rates per well (e.g. 524 m3 / day)

URI: [nmdc:prod_rate](https://microbiomedata/meta/prod_rate)

## Domain and Range {#domain-and-range-587}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-624}

-   is_a: [attribute](attribute.md)

## Children {#children-594}

## Used by {#used-by-587}

## Other properties {#other-properties-464}

+---------------+-------------------+
| **Aliases:**  | > production rate |
+---------------+-------------------+
| **Mappings:** | > mixs:prod_rate  |
+---------------+-------------------+

# Slot: prod_start_date

Date of field's first production

URI: [nmdc:prod_start_date](https://microbiomedata/meta/prod_start_date)

## Domain and Range {#domain-and-range-588}

None → 0..1 [TimestampValue](TimestampValue.md)

## Parents {#parents-625}

-   is_a: [attribute](attribute.md)

## Children {#children-595}

## Used by {#used-by-588}

## Other properties {#other-properties-465}

+---------------+-------------------------+
| **Aliases:**  | > production start date |
+---------------+-------------------------+
| **Mappings:** | > mixs:prod_start_date  |
+---------------+-------------------------+

# Slot: profile image url

A url that points to an image of a person.

URI:
<https://microbiomedata/schema/workflow_execution_activity/profile_image_url>

## Domain and Range {#domain-and-range-589}

[PersonValue](PersonValue.md) → 0..1 [String](types/String.md)

## Parents {#parents-626}

## Children {#children-596}

## Used by {#used-by-589}

-   [PersonValue](PersonValue.md)

# Slot: profile_position

Cross-sectional position in the hillslope where sample was
collected.sample area position in relation to surrounding areas

URI:
[nmdc:profile_position](https://microbiomedata/meta/profile_position)

## Domain and Range {#domain-and-range-590}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-627}

-   is_a: [attribute](attribute.md)

## Children {#children-597}

## Used by {#used-by-590}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-466}

+---------------+-------------------------+
| **Aliases:**  | > profile position      |
+---------------+-------------------------+
| **Mappings:** | > mixs:profile_position |
+---------------+-------------------------+

# Slot: project_name

Name of the project within which the sequencing was organized

URI: [nmdc:project_name](https://microbiomedata/meta/project_name)

## Domain and Range {#domain-and-range-591}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-628}

-   is_a: [attribute](attribute.md)

## Children {#children-598}

## Used by {#used-by-591}

## Other properties {#other-properties-467}

+-----------------+---------------------+
| **Aliases:**    | > project name      |
+-----------------+---------------------+
| **Mappings:**   | > mixs:project_name |
+-----------------+---------------------+
| **In Subsets:** | > investigation     |
+-----------------+---------------------+

# Slot: propagation

"This field is specific to different taxa. For phages: lytic/lysogenic,
for plasmids: incompatibility group, for eukaryotes: sexual/asexual
(Note: there is the strong opinion to name phage propagation obligately
lytic or temperate, therefore we also give this choice"

URI: [nmdc:propagation](https://microbiomedata/meta/propagation)

## Domain and Range {#domain-and-range-592}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-629}

-   is_a: [attribute](attribute.md)

## Children {#children-599}

## Used by {#used-by-592}

## Other properties {#other-properties-468}

+-----------------+--------------------------------+
| **Aliases:**    | > propagation                  |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:propagation             |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: proport_woa_temperature

URI:
[nmdc:proport_woa_temperature](https://microbiomedata/meta/proport_woa_temperature)

## Domain and Range {#domain-and-range-593}

None → 0..1 [String](types/String.md)

## Parents {#parents-630}

## Children {#children-600}

## Used by {#used-by-593}

-   [Biosample](Biosample.md)

# Slot: all proteins

the grouped list of protein identifiers associated with the peptide
sequences that were grouped to a best protein

URI:
<https://microbiomedata/schema/workflow_execution_activity/protein_quantification_all_proteins>

## Domain and Range {#domain-and-range-594}

[ProteinQuantification](ProteinQuantification.md) → 0..\*
[GeneProduct](GeneProduct.md)

## Parents {#parents-631}

-   is_a: [all proteins](all_proteins.md)

## Children {#children-601}

## Used by {#used-by-594}

-   [ProteinQuantification](ProteinQuantification.md)

# Slot: best protein

the specific protein identifier most correctly grouped to its associated
peptide sequences

URI:
<https://microbiomedata/schema/workflow_execution_activity/protein_quantification_best_protein>

## Domain and Range {#domain-and-range-595}

[ProteinQuantification](ProteinQuantification.md) → 0..1
[GeneProduct](GeneProduct.md)

## Parents {#parents-632}

-   is_a: [best protein](best_protein.md)

## Children {#children-602}

## Used by {#used-by-595}

-   [ProteinQuantification](ProteinQuantification.md)

# Slot: peptide_sequence_count

count of peptide sequences grouped to the best_protein

URI:
<https://microbiomedata/schema/workflow_execution_activity/protein_quantification_peptide_sequence_count>

## Domain and Range {#domain-and-range-596}

[ProteinQuantification](ProteinQuantification.md) → 0..1
[Integer](types/Integer.md)

## Parents {#parents-633}

-   is_a: [peptide_sequence_count](peptide_sequence_count.md)

## Children {#children-603}

## Used by {#used-by-596}

-   [ProteinQuantification](ProteinQuantification.md)

# Slot: protein_spectral_count

sum of filter passing MS2 spectra associated with the best protein
within a given LC-MS/MS data file

URI:
<https://microbiomedata/schema/workflow_execution_activity/protein_quantification_protein_spectral_count>

## Domain and Range {#domain-and-range-597}

[ProteinQuantification](ProteinQuantification.md) → 0..1
[Integer](types/Integer.md)

## Parents {#parents-634}

-   is_a: [protein_spectral_count](protein_spectral_count.md)

## Children {#children-604}

## Used by {#used-by-597}

-   [ProteinQuantification](ProteinQuantification.md)

# Slot: protein_sum_masic_abundance

combined MS1 extracted ion chromatograms derived from MS2 spectra
associated with the best protein from a given LC-MS/MS data file using
the MASIC tool

URI:
<https://microbiomedata/schema/workflow_execution_activity/protein_quantification_protein_sum_masic_abundance>

## Domain and Range {#domain-and-range-598}

[ProteinQuantification](ProteinQuantification.md) → 0..1
[Integer](types/Integer.md)

## Parents {#parents-635}

-   is_a: [protein_sum_masic_abundance](protein_sum_masic_abundance.md)

## Children {#children-605}

## Used by {#used-by-598}

-   [ProteinQuantification](ProteinQuantification.md)

# Slot: protein_spectral_count

sum of filter passing MS2 spectra associated with the best protein
within a given LC-MS/MS data file

URI:
<https://microbiomedata/schema/workflow_execution_activity/protein_spectral_count>

## Domain and Range {#domain-and-range-599}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-636}

## Children {#children-606}

-   [protein
    quantification➞protein_spectral_count](protein_quantification_protein_spectral_count.md)

## Used by {#used-by-599}

# Slot: protein_sum_masic_abundance

combined MS1 extracted ion chromatograms derived from MS2 spectra
associated with the best protein from a given LC-MS/MS data file using
the MASIC tool

URI:
<https://microbiomedata/schema/workflow_execution_activity/protein_sum_masic_abundance>

## Domain and Range {#domain-and-range-600}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-637}

## Children {#children-607}

-   [protein
    quantification➞protein_sum_masic_abundance](protein_quantification_protein_sum_masic_abundance.md)

## Used by {#used-by-600}

# Slot: publications

A list of publications that are assocatiated with the entity. The
publicatons SHOULD be given using an identifier, such as a DOI or Pubmed
ID, if possible.

URI:
<https://microbiomedata/schema/workflow_execution_activity/publications>

## Domain and Range {#domain-and-range-601}

None → 0..\* [String](types/String.md)

## Parents {#parents-638}

## Children {#children-608}

## Used by {#used-by-601}

# Slot: pulmonary_disord

History of pulmonary disorders; can include multiple disorders

URI:
[nmdc:pulmonary_disord](https://microbiomedata/meta/pulmonary_disord)

## Domain and Range {#domain-and-range-602}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-639}

-   is_a: [attribute](attribute.md)

## Children {#children-609}

## Used by {#used-by-602}

## Other properties {#other-properties-469}

+---------------+---------------------------+
| **Aliases:**  | > lung/pulmonary disorder |
+---------------+---------------------------+
| **Mappings:** | > mixs:pulmonary_disord   |
+---------------+---------------------------+

# Slot: quad_pos

The quadrant position of the sampling room within the building

URI: [nmdc:quad_pos](https://microbiomedata/meta/quad_pos)

## Domain and Range {#domain-and-range-603}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-640}

-   is_a: [attribute](attribute.md)

## Children {#children-610}

## Used by {#used-by-603}

## Other properties {#other-properties-470}

+---------------+---------------------+
| **Aliases:**  | > quadrant position |
+---------------+---------------------+
| **Mappings:** | > mixs:quad_pos     |
+---------------+---------------------+

# Slot: has numeric value

The number part of the quantity

URI:
<https://microbiomedata/schema/workflow_execution_activity/quantity_value_has_numeric_value>

## Domain and Range {#domain-and-range-604}

[QuantityValue](QuantityValue.md) → 0..1 [Double](types/Double.md)

## Parents {#parents-641}

-   is_a: [has numeric value](has_numeric_value.md)

## Children {#children-611}

## Used by {#used-by-604}

-   [QuantityValue](QuantityValue.md)

## Other properties {#other-properties-471}

+---------------+---------------------+
| **Mappings:** | > qud:quantityValue |
+---------------+---------------------+
|               | > schema:value      |
+---------------+---------------------+

# Slot: has raw value

Unnormalized atomic string representation, should in syntax {number}
{unit}

URI:
<https://microbiomedata/schema/workflow_execution_activity/quantity_value_has_raw_value>

## Domain and Range {#domain-and-range-605}

[QuantityValue](QuantityValue.md) → 0..1 [String](types/String.md)

## Parents {#parents-642}

-   is_a: [has raw value](has_raw_value.md)

## Children {#children-612}

## Used by {#used-by-605}

-   [QuantityValue](QuantityValue.md)

# Slot: has unit

The unit of the quantity

URI:
<https://microbiomedata/schema/workflow_execution_activity/quantity_value_has_unit>

## Domain and Range {#domain-and-range-606}

[QuantityValue](QuantityValue.md) → 0..1 [Unit](types/Unit.md)

## Parents {#parents-643}

-   is_a: [has unit](has_unit.md)

## Children {#children-613}

## Used by {#used-by-606}

-   [QuantityValue](QuantityValue.md)

## Other properties {#other-properties-472}

+---------------+-------------------+
| **Aliases:**  | > scale           |
+---------------+-------------------+
| **Mappings:** | > qud:unit        |
+---------------+-------------------+
|               | > schema:unitCode |
+---------------+-------------------+

# Slot: radiation_regm

"Information about treatment involving exposure of plant or a plant part
to a particular radiation regimen; should include the radiation type,
amount or intensity administered, treatment regimen including how many
times the treatment was repeated, how long each treatment lasted, and
the start and end time of the entire treatment; can include multiple
radiation regimens"

URI: [nmdc:radiation_regm](https://microbiomedata/meta/radiation_regm)

## Domain and Range {#domain-and-range-607}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-644}

-   is_a: [attribute](attribute.md)

## Children {#children-614}

## Used by {#used-by-607}

## Other properties {#other-properties-473}

+---------------+-----------------------+
| **Aliases:**  | > radiation regimen   |
+---------------+-----------------------+
| **Mappings:** | > mixs:radiation_regm |
+---------------+-----------------------+

# Slot: rainfall_regm

"Information about treatment involving an exposure to a given amount of
rainfall, treatment regimen including how many times the treatment was
repeated, how long each treatment lasted, and the start and end time of
the entire treatment; can include multiple regimens"

URI: [nmdc:rainfall_regm](https://microbiomedata/meta/rainfall_regm)

## Domain and Range {#domain-and-range-608}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-645}

-   is_a: [attribute](attribute.md)

## Children {#children-615}

## Used by {#used-by-608}

## Other properties {#other-properties-474}

+---------------+----------------------+
| **Aliases:**  | > rainfall regimen   |
+---------------+----------------------+
| **Mappings:** | > mixs:rainfall_regm |
+---------------+----------------------+

# Slot: direction

One of l-\>r, r-\>l, bidirectional, neutral

URI:
[nmdc:reaction_direction](https://microbiomedata/meta/reaction_direction)

## Domain and Range {#domain-and-range-609}

[Reaction](Reaction.md) → 0..1 [String](types/String.md)

## Parents {#parents-646}

-   is_a: [direction](direction.md)

## Children {#children-616}

## Used by {#used-by-609}

-   [Reaction](Reaction.md)

# Slot: is balanced

URI:
[nmdc:reaction_is_balanced](https://microbiomedata/meta/reaction_is_balanced)

## Domain and Range {#domain-and-range-610}

[Reaction](Reaction.md) → 0..1 [Boolean](types/Boolean.md)

## Parents {#parents-647}

-   is_a: [is balanced](is_balanced.md)

## Children {#children-617}

## Used by {#used-by-610}

-   [Reaction](Reaction.md)

# Slot: is diastereoselective

URI:
[nmdc:reaction_is_diastereoselective](https://microbiomedata/meta/reaction_is_diastereoselective)

## Domain and Range {#domain-and-range-611}

[Reaction](Reaction.md) → 0..1 [Boolean](types/Boolean.md)

## Parents {#parents-648}

-   is_a: [is diastereoselective](is_diastereoselective.md)

## Children {#children-618}

## Used by {#used-by-611}

-   [Reaction](Reaction.md)

# Slot: is fully characterized

False if includes R-groups

URI:
[nmdc:reaction_is_fully_characterized](https://microbiomedata/meta/reaction_is_fully_characterized)

## Domain and Range {#domain-and-range-612}

[Reaction](Reaction.md) → 0..1 [Boolean](types/Boolean.md)

## Parents {#parents-649}

-   is_a: [is fully characterized](is_fully_characterized.md)

## Children {#children-619}

## Used by {#used-by-612}

-   [Reaction](Reaction.md)

# Slot: is stereo

URI:
[nmdc:reaction_is_stereo](https://microbiomedata/meta/reaction_is_stereo)

## Domain and Range {#domain-and-range-613}

[Reaction](Reaction.md) → 0..1 [Boolean](types/Boolean.md)

## Parents {#parents-650}

-   is_a: [is stereo](is_stereo.md)

## Children {#children-620}

## Used by {#used-by-613}

-   [Reaction](Reaction.md)

# Slot: is transport

URI:
[nmdc:reaction_is_transport](https://microbiomedata/meta/reaction_is_transport)

## Domain and Range {#domain-and-range-614}

[Reaction](Reaction.md) → 0..1 [Boolean](types/Boolean.md)

## Parents {#parents-651}

-   is_a: [is transport](is_transport.md)

## Children {#children-621}

## Used by {#used-by-614}

-   [Reaction](Reaction.md)

# Slot: left participants

URI:
[nmdc:reaction_left_participants](https://microbiomedata/meta/reaction_left_participants)

## Domain and Range {#domain-and-range-615}

[Reaction](Reaction.md) → 0..\*
[ReactionParticipant](ReactionParticipant.md)

## Parents {#parents-652}

-   is_a: [has participants](has_participants.md)

## Children {#children-622}

## Used by {#used-by-615}

-   [Reaction](Reaction.md)

# Slot: chemical

URI:
[nmdc:reaction_participant_chemical](https://microbiomedata/meta/reaction_participant_chemical)

## Domain and Range {#domain-and-range-616}

[ReactionParticipant](ReactionParticipant.md) → 0..1
[ChemicalEntity](ChemicalEntity.md)

## Parents {#parents-653}

-   is_a: [chemical](chemical.md)

## Children {#children-623}

## Used by {#used-by-616}

-   [ReactionParticipant](ReactionParticipant.md)

# Slot: stoichiometry

URI:
[nmdc:reaction_participant_stoichiometry](https://microbiomedata/meta/reaction_participant_stoichiometry)

## Domain and Range {#domain-and-range-617}

[ReactionParticipant](ReactionParticipant.md) → 0..1
[Integer](types/Integer.md)

## Parents {#parents-654}

-   is_a: [stoichiometry](stoichiometry.md)

## Children {#children-624}

## Used by {#used-by-617}

-   [ReactionParticipant](ReactionParticipant.md)

# Slot: right participants

URI:
[nmdc:reaction_right_participants](https://microbiomedata/meta/reaction_right_participants)

## Domain and Range {#domain-and-range-618}

[Reaction](Reaction.md) → 0..\*
[ReactionParticipant](ReactionParticipant.md)

## Parents {#parents-655}

-   is_a: [has participants](has_participants.md)

## Children {#children-625}

## Used by {#used-by-618}

-   [Reaction](Reaction.md)

# Slot: smarts string

URI:
[nmdc:reaction_smarts_string](https://microbiomedata/meta/reaction_smarts_string)

## Domain and Range {#domain-and-range-619}

[Reaction](Reaction.md) → 0..1 [String](types/String.md)

## Parents {#parents-656}

-   is_a: [smarts string](smarts_string.md)

## Children {#children-626}

## Used by {#used-by-619}

-   [Reaction](Reaction.md)

# Slot: reactor_type

"Anaerobic digesters can be designed and engineered to operate using a
number of different process configurations, as batch or continuous,
mesophilic, high solid or low solid, and single stage or multistage"

URI: [nmdc:reactor_type](https://microbiomedata/meta/reactor_type)

## Domain and Range {#domain-and-range-620}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-657}

-   is_a: [attribute](attribute.md)

## Children {#children-627}

## Used by {#used-by-620}

## Other properties {#other-properties-475}

+---------------+---------------------+
| **Aliases:**  | > reactor type      |
+---------------+---------------------+
| **Mappings:** | > mixs:reactor_type |
+---------------+---------------------+

# Slot: has input

An input to a process.

URI:
<https://microbiomedata/schema/workflow_execution_activity/read_QC_analysis_activity_has_input>

## Domain and Range {#domain-and-range-621}

[ReadQCAnalysisActivity](ReadQCAnalysisActivity.md) → 1..\*
[NamedThing](NamedThing.md)

## Parents {#parents-658}

-   is_a: [workflow execution activity➞has
    input](workflow_execution_activity_has_input.md)

## Children {#children-628}

## Used by {#used-by-621}

-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)

## Other properties {#other-properties-476}

  **Aliases:**    input
  --------------- -----------
                  
                  output-1:
  Slot: has out   put

An output biosample to a processing step

URI:
<https://microbiomedata/schema/workflow_execution_activity/read_QC_analysis_activity_has_output>

## Domain and Range {#domain-and-range-622}

[ReadQCAnalysisActivity](ReadQCAnalysisActivity.md) → 1..\*
[NamedThing](NamedThing.md)

## Parents {#parents-659}

-   is_a: [workflow execution activity➞has
    output](workflow_execution_activity_has_output.md)

## Children {#children-629}

## Used by {#used-by-622}

-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)

## Other properties {#other-properties-477}

  **Aliases:**    output
  --------------- -----------------
                  
                  t_read_bases-1:
  Slot: input_r   ead_bases

TODO

URI:
<https://microbiomedata/schema/workflow_execution_activity/read_QC_analysis_activity_input_read_bases>

## Domain and Range {#domain-and-range-623}

[ReadQCAnalysisActivity](ReadQCAnalysisActivity.md) → 0..1
[Float](types/Float.md)

## Parents {#parents-660}

-   is_a: [input_read_bases](input_read_bases.md)

## Children {#children-630}

## Used by {#used-by-623}

-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)

# Slot: output_read_bases

TODO

URI:
<https://microbiomedata/schema/workflow_execution_activity/read_QC_analysis_activity_output_read_bases>

## Domain and Range {#domain-and-range-624}

[ReadQCAnalysisActivity](ReadQCAnalysisActivity.md) → 0..1
[Float](types/Float.md)

## Parents {#parents-661}

-   is_a: [output_read_bases](output_read_bases.md)

## Children {#children-631}

## Used by {#used-by-624}

-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)

# Slot: read QC analysis activity set

This property links a database object to the set of read QC analysis
activities.

URI:
[nmdc:read_QC_analysis_activity_set](https://microbiomedata/meta/read_QC_analysis_activity_set)

## Domain and Range {#domain-and-range-625}

[Database](Database.md) → 0..\*
[ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)

## Parents {#parents-662}

## Children {#children-632}

## Used by {#used-by-625}

-   [Database](Database.md)

# Slot: read QC analysis statistic

URI:
<https://microbiomedata/schema/workflow_execution_activity/read_QC_analysis_statistic>

## Domain and Range {#domain-and-range-626}

None → 0..1 [String](types/String.md)

## Parents {#parents-663}

## Children {#children-633}

-   [input base count](input_base_count.md)
-   [input read count](input_read_count.md)
-   [output base count](output_base_count.md)
-   [output read count](output_read_count.md)

## Used by {#used-by-626}

# Slot: read based analysis activity set

This property links a database object to the set of read based analysis
activities.

URI:
[nmdc:read_based_analysis_activity_set](https://microbiomedata/meta/read_based_analysis_activity_set)

## Domain and Range {#domain-and-range-627}

[Database](Database.md) → 0..\*
[ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)

## Parents {#parents-664}

## Children {#children-634}

## Used by {#used-by-627}

-   [Database](Database.md)

# Slot: reassembly_bin

Has an assembly been performed on a genome bin extracted from a
metagenomic assembly?

URI: [nmdc:reassembly_bin](https://microbiomedata/meta/reassembly_bin)

## Domain and Range {#domain-and-range-628}

None → 0..1 [BooleanValue](BooleanValue.md)

## Parents {#parents-665}

-   is_a: [attribute](attribute.md)

## Children {#children-635}

## Used by {#used-by-628}

## Other properties {#other-properties-478}

+-----------------+---------------------------+
| **Aliases:**    | > reassembly post binning |
+-----------------+---------------------------+
| **Mappings:**   | > mixs:reassembly_bin     |
+-----------------+---------------------------+
| **In Subsets:** | > sequencing              |
+-----------------+---------------------------+

# Slot: redox_potential

"Redox potential, measured relative to a hydrogen cell, indicating
oxidation or reduction potential"

URI: [nmdc:redox_potential](https://microbiomedata/meta/redox_potential)

## Domain and Range {#domain-and-range-629}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-666}

-   is_a: [attribute](attribute.md)

## Children {#children-636}

## Used by {#used-by-629}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-479}

+---------------+------------------------+
| **Aliases:**  | > redox potential      |
+---------------+------------------------+
| **Mappings:** | > mixs:redox_potential |
+---------------+------------------------+

# Slot: ref_biomaterial

"Primary publication if isolated before genome publication; otherwise,
primary genome report"

URI: [nmdc:ref_biomaterial](https://microbiomedata/meta/ref_biomaterial)

## Domain and Range {#domain-and-range-630}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-667}

-   is_a: [attribute](attribute.md)

## Children {#children-637}

## Used by {#used-by-630}

## Other properties {#other-properties-480}

+-----------------+--------------------------------+
| **Aliases:**    | > reference for biomaterial    |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:ref_biomaterial         |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: ref_db

"List of database(s) used for ORF annotation, along with version number
and reference to website or publication"

URI: [nmdc:ref_db](https://microbiomedata/meta/ref_db)

## Domain and Range {#domain-and-range-631}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-668}

-   is_a: [attribute](attribute.md)

## Children {#children-638}

## Used by {#used-by-631}

## Other properties {#other-properties-481}

+-----------------+-------------------------+
| **Aliases:**    | > reference database(s) |
+-----------------+-------------------------+
| **Mappings:**   | > mixs:ref_db           |
+-----------------+-------------------------+
| **In Subsets:** | > sequencing            |
+-----------------+-------------------------+

# Slot: rel_air_humidity

"Partial vapor and air pressure, density of the vapor and air, or by the
actual mass of the vapor and air"

URI:
[nmdc:rel_air_humidity](https://microbiomedata/meta/rel_air_humidity)

## Domain and Range {#domain-and-range-632}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-669}

-   is_a: [attribute](attribute.md)

## Children {#children-639}

## Used by {#used-by-632}

## Other properties {#other-properties-482}

+---------------+-------------------------+
| **Aliases:**  | > relative air humidity |
+---------------+-------------------------+
| **Mappings:** | > mixs:rel_air_humidity |
+---------------+-------------------------+

# Slot: rel_humidity_out

The recorded outside relative humidity value at the time of sampling

URI:
[nmdc:rel_humidity_out](https://microbiomedata/meta/rel_humidity_out)

## Domain and Range {#domain-and-range-633}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-670}

-   is_a: [attribute](attribute.md)

## Children {#children-640}

## Used by {#used-by-633}

## Other properties {#other-properties-483}

+---------------+-----------------------------+
| **Aliases:**  | > outside relative humidity |
+---------------+-----------------------------+
| **Mappings:** | > mixs:rel_humidity_out     |
+---------------+-----------------------------+

# Slot: rel_samp_loc

The sampling location within the train car

URI: [nmdc:rel_samp_loc](https://microbiomedata/meta/rel_samp_loc)

## Domain and Range {#domain-and-range-634}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-671}

-   is_a: [attribute](attribute.md)

## Children {#children-641}

## Used by {#used-by-634}

## Other properties {#other-properties-484}

+---------------+------------------------------+
| **Aliases:**  | > relative sampling location |
+---------------+------------------------------+
| **Mappings:** | > mixs:rel_samp_loc          |
+---------------+------------------------------+

# Slot: rel_to_oxygen

"Is this organism an aerobe, anaerobe? Please note that aerobic and
anaerobic are valid descriptors for microbial environments"

URI: [nmdc:rel_to_oxygen](https://microbiomedata/meta/rel_to_oxygen)

## Domain and Range {#domain-and-range-635}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-672}

-   is_a: [attribute](attribute.md)

## Children {#children-642}

## Used by {#used-by-635}

## Other properties {#other-properties-485}

+-----------------+--------------------------------+
| **Aliases:**    | > relationship to oxygen       |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:rel_to_oxygen           |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: relevant protocols

URI:
[nmdc:relevant_protocols](https://microbiomedata/meta/relevant_protocols)

## Domain and Range {#domain-and-range-636}

None → 0..\* [String](types/String.md)

## Parents {#parents-673}

## Children {#children-643}

## Used by {#used-by-636}

-   [Study](Study.md)

# Slot: reservoir

Name of the reservoir (e.g. Carapebus)

URI: [nmdc:reservoir](https://microbiomedata/meta/reservoir)

## Domain and Range {#domain-and-range-637}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-674}

-   is_a: [attribute](attribute.md)

## Children {#children-644}

## Used by {#used-by-637}

## Other properties {#other-properties-486}

+---------------+------------------+
| **Aliases:**  | > reservoir name |
+---------------+------------------+
| **Mappings:** | > mixs:reservoir |
+---------------+------------------+

# Slot: resins_pc

"Saturate, Aromatic, Resin and AsphalteneB (SARA) is an analysis method
that dividesB crude oilB components according to their polarizability
and polarity. There are three main methods to obtain SARA results. The
most popular one is known as the Iatroscan TLC-FID and is referred to as
IP-143 (source:
<https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene>)"

URI: [nmdc:resins_pc](https://microbiomedata/meta/resins_pc)

## Domain and Range {#domain-and-range-638}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-675}

-   is_a: [attribute](attribute.md)

## Children {#children-645}

## Used by {#used-by-638}

## Other properties {#other-properties-487}

+---------------+------------------+
| **Aliases:**  | > resins wt%     |
+---------------+------------------+
| **Mappings:** | > mixs:resins_pc |
+---------------+------------------+

# Slot: resp_part_matter

"Concentration of substances that remain suspended in the air, and
comprise mixtures of organic and inorganic substances (PM10 and PM2.5);
can report multiple PM's by entering numeric values preceded by name of
PM"

URI:
[nmdc:resp_part_matter](https://microbiomedata/meta/resp_part_matter)

## Domain and Range {#domain-and-range-639}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-676}

-   is_a: [attribute](attribute.md)

## Children {#children-646}

## Used by {#used-by-639}

## Other properties {#other-properties-488}

+---------------+---------------------------------+
| **Aliases:**  | > respirable particulate matter |
+---------------+---------------------------------+
| **Mappings:** | > mixs:resp_part_matter         |
+---------------+---------------------------------+

# Slot: right participants

URI:
[nmdc:right_participants](https://microbiomedata/meta/right_participants)

## Domain and Range {#domain-and-range-640}

None → 0..\* [ReactionParticipant](ReactionParticipant.md)

## Parents {#parents-677}

-   is_a: [has participants](has_participants.md)

## Children {#children-647}

## Used by {#used-by-640}

# Slot: room_air_exch_rate

The rate at which outside air replaces indoor air in a given space

URI:
[nmdc:room_air_exch_rate](https://microbiomedata/meta/room_air_exch_rate)

## Domain and Range {#domain-and-range-641}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-678}

-   is_a: [attribute](attribute.md)

## Children {#children-648}

## Used by {#used-by-641}

## Other properties {#other-properties-489}

+---------------+---------------------------+
| **Aliases:**  | > room air exchange rate  |
+---------------+---------------------------+
| **Mappings:** | > mixs:room_air_exch_rate |
+---------------+---------------------------+

# Slot: room_architec_element

"The unique details and component parts that, together, form the
architecture of a distinguisahable space within a built structure"

URI:
[nmdc:room_architec_element](https://microbiomedata/meta/room_architec_element)

## Domain and Range {#domain-and-range-642}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-679}

-   is_a: [attribute](attribute.md)

## Children {#children-649}

## Used by {#used-by-642}

## Other properties {#other-properties-490}

+---------------+-------------------------------+
| **Aliases:**  | > room architectural elements |
+---------------+-------------------------------+
| **Mappings:** | > mixs:room_architec_element  |
+---------------+-------------------------------+

# Slot: room_condt

The condition of the room at the time of sampling

URI: [nmdc:room_condt](https://microbiomedata/meta/room_condt)

## Domain and Range {#domain-and-range-643}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-680}

-   is_a: [attribute](attribute.md)

## Children {#children-650}

## Used by {#used-by-643}

## Other properties {#other-properties-491}

+---------------+-------------------+
| **Aliases:**  | > room condition  |
+---------------+-------------------+
| **Mappings:** | > mixs:room_condt |
+---------------+-------------------+

# Slot: room_connected

List of rooms connected to the sampling room by a doorway

URI: [nmdc:room_connected](https://microbiomedata/meta/room_connected)

## Domain and Range {#domain-and-range-644}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-681}

-   is_a: [attribute](attribute.md)

## Children {#children-651}

## Used by {#used-by-644}

## Other properties {#other-properties-492}

+---------------+--------------------------------+
| **Aliases:**  | > rooms connected by a doorway |
+---------------+--------------------------------+
| **Mappings:** | > mixs:room_connected          |
+---------------+--------------------------------+

# Slot: room_count

The total count of rooms in the built structure including all room types

URI: [nmdc:room_count](https://microbiomedata/meta/room_count)

## Domain and Range {#domain-and-range-645}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-682}

-   is_a: [attribute](attribute.md)

## Children {#children-652}

## Used by {#used-by-645}

## Other properties {#other-properties-493}

+---------------+-------------------+
| **Aliases:**  | > room count      |
+---------------+-------------------+
| **Mappings:** | > mixs:room_count |
+---------------+-------------------+

# Slot: room_dim

"The length, width and height of sampling room"

URI: [nmdc:room_dim](https://microbiomedata/meta/room_dim)

## Domain and Range {#domain-and-range-646}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-683}

-   is_a: [attribute](attribute.md)

## Children {#children-653}

## Used by {#used-by-646}

## Other properties {#other-properties-494}

+---------------+-------------------+
| **Aliases:**  | > room dimensions |
+---------------+-------------------+
| **Mappings:** | > mixs:room_dim   |
+---------------+-------------------+

# Slot: room_door_dist

Distance between doors (meters) in the hallway between the sampling room
and adjacent rooms

URI: [nmdc:room_door_dist](https://microbiomedata/meta/room_door_dist)

## Domain and Range {#domain-and-range-647}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-684}

-   is_a: [attribute](attribute.md)

## Children {#children-654}

## Used by {#used-by-647}

## Other properties {#other-properties-495}

+---------------+-----------------------+
| **Aliases:**  | > room door distance  |
+---------------+-----------------------+
| **Mappings:** | > mixs:room_door_dist |
+---------------+-----------------------+

# Slot: room_door_share

"List of room(s) (room number, room name) sharing a door with the
sampling room"

URI: [nmdc:room_door_share](https://microbiomedata/meta/room_door_share)

## Domain and Range {#domain-and-range-648}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-685}

-   is_a: [attribute](attribute.md)

## Children {#children-655}

## Used by {#used-by-648}

## Other properties {#other-properties-496}

+---------------+----------------------------------------------+
| **Aliases:**  | > rooms that share a door with sampling room |
+---------------+----------------------------------------------+
| **Mappings:** | > mixs:room_door_share                       |
+---------------+----------------------------------------------+

# Slot: room_hallway

"List of room(s) (room number, room name) located in the same hallway as
sampling room"

URI: [nmdc:room_hallway](https://microbiomedata/meta/room_hallway)

## Domain and Range {#domain-and-range-649}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-686}

-   is_a: [attribute](attribute.md)

## Children {#children-656}

## Used by {#used-by-649}

## Other properties {#other-properties-497}

+---------------+--------------------------------------+
| **Aliases:**  | > rooms that are on the same hallway |
+---------------+--------------------------------------+
| **Mappings:** | > mixs:room_hallway                  |
+---------------+--------------------------------------+

# Slot: room_loc

The position of the room within the building

URI: [nmdc:room_loc](https://microbiomedata/meta/room_loc)

## Domain and Range {#domain-and-range-650}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-687}

-   is_a: [attribute](attribute.md)

## Children {#children-657}

## Used by {#used-by-650}

## Other properties {#other-properties-498}

+---------------+-----------------------------+
| **Aliases:**  | > room location in building |
+---------------+-----------------------------+
| **Mappings:** | > mixs:room_loc             |
+---------------+-----------------------------+

# Slot: room_moist_damage_hist

The history of moisture damage or mold in the past 12 months. Number of
events of moisture damage or mold observed

URI:
[nmdc:room_moist_damage_hist](https://microbiomedata/meta/room_moist_damage_hist)

## Domain and Range {#domain-and-range-651}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-688}

-   is_a: [attribute](attribute.md)

## Children {#children-658}

## Used by {#used-by-651}

## Other properties {#other-properties-499}

+---------------+----------------------------------------+
| **Aliases:**  | > room moisture damage or mold history |
+---------------+----------------------------------------+
| **Mappings:** | > mixs:room_moist_damage_hist          |
+---------------+----------------------------------------+

# Slot: room_net_area

The net floor area of sampling room. Net area excludes wall thicknesses

URI: [nmdc:room_net_area](https://microbiomedata/meta/room_net_area)

## Domain and Range {#domain-and-range-652}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-689}

-   is_a: [attribute](attribute.md)

## Children {#children-659}

## Used by {#used-by-652}

## Other properties {#other-properties-500}

+---------------+----------------------+
| **Aliases:**  | > room net area      |
+---------------+----------------------+
| **Mappings:** | > mixs:room_net_area |
+---------------+----------------------+

# Slot: room_occup

Count of room occupancy at time of sampling

URI: [nmdc:room_occup](https://microbiomedata/meta/room_occup)

## Domain and Range {#domain-and-range-653}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-690}

-   is_a: [attribute](attribute.md)

## Children {#children-660}

## Used by {#used-by-653}

## Other properties {#other-properties-501}

+---------------+-------------------+
| **Aliases:**  | > room occupancy  |
+---------------+-------------------+
| **Mappings:** | > mixs:room_occup |
+---------------+-------------------+

# Slot: room_samp_pos

The horizontal sampling position in the room relative to architectural
elements

URI: [nmdc:room_samp_pos](https://microbiomedata/meta/room_samp_pos)

## Domain and Range {#domain-and-range-654}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-691}

-   is_a: [attribute](attribute.md)

## Children {#children-661}

## Used by {#used-by-654}

## Other properties {#other-properties-502}

+---------------+--------------------------+
| **Aliases:**  | > room sampling position |
+---------------+--------------------------+
| **Mappings:** | > mixs:room_samp_pos     |
+---------------+--------------------------+

# Slot: room_type

The main purpose or activity of the sampling room. A room is any
distinguishable space within a structure

URI: [nmdc:room_type](https://microbiomedata/meta/room_type)

## Domain and Range {#domain-and-range-655}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-692}

-   is_a: [attribute](attribute.md)

## Children {#children-662}

## Used by {#used-by-655}

## Other properties {#other-properties-503}

+---------------+------------------+
| **Aliases:**  | > room type      |
+---------------+------------------+
| **Mappings:** | > mixs:room_type |
+---------------+------------------+

# Slot: room_vol

Volume of sampling room

URI: [nmdc:room_vol](https://microbiomedata/meta/room_vol)

## Domain and Range {#domain-and-range-656}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-693}

-   is_a: [attribute](attribute.md)

## Children {#children-663}

## Used by {#used-by-656}

## Other properties {#other-properties-504}

+---------------+-----------------+
| **Aliases:**  | > room volume   |
+---------------+-----------------+
| **Mappings:** | > mixs:room_vol |
+---------------+-----------------+

# Slot: room_wall_share

"List of room(s) (room number, room name) sharing a wall with the
sampling room"

URI: [nmdc:room_wall_share](https://microbiomedata/meta/room_wall_share)

## Domain and Range {#domain-and-range-657}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-694}

-   is_a: [attribute](attribute.md)

## Children {#children-664}

## Used by {#used-by-657}

## Other properties {#other-properties-505}

+---------------+----------------------------------------------+
| **Aliases:**  | > rooms that share a wall with sampling room |
+---------------+----------------------------------------------+
| **Mappings:** | > mixs:room_wall_share                       |
+---------------+----------------------------------------------+

# Slot: room_window_count

Number of windows in the room

URI:
[nmdc:room_window_count](https://microbiomedata/meta/room_window_count)

## Domain and Range {#domain-and-range-658}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-695}

-   is_a: [attribute](attribute.md)

## Children {#children-665}

## Used by {#used-by-658}

## Other properties {#other-properties-506}

+---------------+--------------------------+
| **Aliases:**  | > room window count      |
+---------------+--------------------------+
| **Mappings:** | > mixs:room_window_count |
+---------------+--------------------------+

# Slot: root_cond

"Relevant rooting conditions such as field plot size, sowing density,
container dimensions, number of plants per container."

URI: [nmdc:root_cond](https://microbiomedata/meta/root_cond)

## Domain and Range {#domain-and-range-659}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-696}

-   is_a: [attribute](attribute.md)

## Children {#children-666}

## Used by {#used-by-659}

## Other properties {#other-properties-507}

+---------------+----------------------+
| **Aliases:**  | > rooting conditions |
+---------------+----------------------+
| **Mappings:** | > mixs:root_cond     |
+---------------+----------------------+

# Slot: root_med_carbon

Source of organic carbon in the culture rooting medium; e.g. sucrose.

URI: [nmdc:root_med_carbon](https://microbiomedata/meta/root_med_carbon)

## Domain and Range {#domain-and-range-660}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-697}

-   is_a: [attribute](attribute.md)

## Children {#children-667}

## Used by {#used-by-660}

## Other properties {#other-properties-508}

+---------------+-------------------------+
| **Aliases:**  | > rooting medium carbon |
+---------------+-------------------------+
| **Mappings:** | > mixs:root_med_carbon  |
+---------------+-------------------------+

# Slot: root_med_macronutr

"Measurement of the culture rooting medium macronutrients (N,P, K, Ca,
Mg, S); e.g. KH2PO4 (170B mg/L)."

URI:
[nmdc:root_med_macronutr](https://microbiomedata/meta/root_med_macronutr)

## Domain and Range {#domain-and-range-661}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-698}

-   is_a: [attribute](attribute.md)

## Children {#children-668}

## Used by {#used-by-661}

## Other properties {#other-properties-509}

+---------------+---------------------------------+
| **Aliases:**  | > rooting medium macronutrients |
+---------------+---------------------------------+
| **Mappings:** | > mixs:root_med_macronutr       |
+---------------+---------------------------------+

# Slot: root_med_micronutr

"Measurement of the culture rooting medium micronutrients (Fe, Mn, Zn,
B, Cu, Mo); e.g. H3BO3 (6.2B mg/L)."

URI:
[nmdc:root_med_micronutr](https://microbiomedata/meta/root_med_micronutr)

## Domain and Range {#domain-and-range-662}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-699}

-   is_a: [attribute](attribute.md)

## Children {#children-669}

## Used by {#used-by-662}

## Other properties {#other-properties-510}

+---------------+---------------------------------+
| **Aliases:**  | > rooting medium micronutrients |
+---------------+---------------------------------+
| **Mappings:** | > mixs:root_med_micronutr       |
+---------------+---------------------------------+

# Slot: root_med_ph

pH measurement of the culture rooting medium; e.g. 5.5.

URI: [nmdc:root_med_ph](https://microbiomedata/meta/root_med_ph)

## Domain and Range {#domain-and-range-663}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-700}

-   is_a: [attribute](attribute.md)

## Children {#children-670}

## Used by {#used-by-663}

## Other properties {#other-properties-511}

+---------------+---------------------+
| **Aliases:**  | > rooting medium pH |
+---------------+---------------------+
| **Mappings:** | > mixs:root_med_ph  |
+---------------+---------------------+

# Slot: root_med_regl

"Growth regulators in the culture rooting medium such as cytokinins,
auxins, gybberellins, abscisic acid; e.g. 0.5B mg/L NAA."

URI: [nmdc:root_med_regl](https://microbiomedata/meta/root_med_regl)

## Domain and Range {#domain-and-range-664}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-701}

-   is_a: [attribute](attribute.md)

## Children {#children-671}

## Used by {#used-by-664}

## Other properties {#other-properties-512}

+---------------+-----------------------------+
| **Aliases:**  | > rooting medium regulators |
+---------------+-----------------------------+
| **Mappings:** | > mixs:root_med_regl        |
+---------------+-----------------------------+

# Slot: root_med_solid

Specification of the solidifying agent in the culture rooting medium;
e.g. agar.

URI: [nmdc:root_med_solid](https://microbiomedata/meta/root_med_solid)

## Domain and Range {#domain-and-range-665}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-702}

-   is_a: [attribute](attribute.md)

## Children {#children-672}

## Used by {#used-by-665}

## Other properties {#other-properties-513}

+---------------+-----------------------------+
| **Aliases:**  | > rooting medium solidifier |
+---------------+-----------------------------+
| **Mappings:** | > mixs:root_med_solid       |
+---------------+-----------------------------+

# Slot: root_med_suppl

"Organic supplements of the culture rooting medium, such as vitamins,
amino acids, organic acids, antibiotics activated charcoal;
e.g. nicotinic acid (0.5B mg/L)."

URI: [nmdc:root_med_suppl](https://microbiomedata/meta/root_med_suppl)

## Domain and Range {#domain-and-range-666}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-703}

-   is_a: [attribute](attribute.md)

## Children {#children-673}

## Used by {#used-by-666}

## Other properties {#other-properties-514}

+---------------+--------------------------------------+
| **Aliases:**  | > rooting medium organic supplements |
+---------------+--------------------------------------+
| **Mappings:** | > mixs:root_med_suppl                |
+---------------+--------------------------------------+

# Slot: salinity

"Salinity is the total concentration of all dissolved salts in a water
sample. While salinity can be measured by a complete chemical analysis,
this method is difficult and time consuming. More often, it is instead
derived from the conductivity measurement. This is known as practical
salinity. These derivations compare the specific conductance of the
sample to a salinity standard such as seawater"

URI: [nmdc:salinity](https://microbiomedata/meta/salinity)

## Domain and Range {#domain-and-range-667}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-704}

-   is_a: [attribute](attribute.md)

## Children {#children-674}

## Used by {#used-by-667}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-515}

+---------------+-----------------+
| **Aliases:**  | > salinity      |
+---------------+-----------------+
| **Mappings:** | > mixs:salinity |
+---------------+-----------------+

# Slot: salinity_category

Categorcial description of the sample's salinity. Examples: halophile,
halotolerant, hypersaline, huryhaline

URI:
[nmdc:salinity_category](https://microbiomedata/meta/salinity_category)

## Domain and Range {#domain-and-range-668}

None → 0..1 [String](types/String.md)

## Parents {#parents-705}

## Children {#children-675}

## Used by {#used-by-668}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-516}

  **See also:**    <https://github.com/microbiomedata/nmdc-metadata/pull/297>
  ---------------- ------------------------------------------------------------
                   
  Slot: salinity   \_meth

Reference or method used in determining salinity

URI: [nmdc:salinity_meth](https://microbiomedata/meta/salinity_meth)

## Domain and Range {#domain-and-range-669}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-706}

-   is_a: [attribute](attribute.md)

## Children {#children-676}

## Used by {#used-by-669}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-517}

+---------------+----------------------------------------------+
| **Aliases:**  | > extreme_unusual_properties/salinity method |
+---------------+----------------------------------------------+
| **Mappings:** | > mixs:salinity_meth                         |
+---------------+----------------------------------------------+

# Slot: salt_regm

"Information about treatment involving use of salts as supplement to
liquid and soil growth media; should include the name of salt, amount
administered, treatment regimen including how many times the treatment
was repeated, how long each treatment lasted, and the start and end time
of the entire treatment; can include multiple salt regimens"

URI: [nmdc:salt_regm](https://microbiomedata/meta/salt_regm)

## Domain and Range {#domain-and-range-670}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-707}

-   is_a: [attribute](attribute.md)

## Children {#children-677}

## Used by {#used-by-670}

## Other properties {#other-properties-518}

+---------------+------------------+
| **Aliases:**  | > salt regimen   |
+---------------+------------------+
| **Mappings:** | > mixs:salt_regm |
+---------------+------------------+

# Slot: samp_capt_status

Reason for the sample

URI:
[nmdc:samp_capt_status](https://microbiomedata/meta/samp_capt_status)

## Domain and Range {#domain-and-range-671}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-708}

-   is_a: [attribute](attribute.md)

## Children {#children-678}

## Used by {#used-by-671}

## Other properties {#other-properties-519}

+---------------+-------------------------+
| **Aliases:**  | > sample capture status |
+---------------+-------------------------+
| **Mappings:** | > mixs:samp_capt_status |
+---------------+-------------------------+

# Slot: samp_collect_device

The method or device employed for collecting the sample

URI:
[nmdc:samp_collect_device](https://microbiomedata/meta/samp_collect_device)

## Domain and Range {#domain-and-range-672}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-709}

-   is_a: [attribute](attribute.md)

## Children {#children-679}

## Used by {#used-by-672}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-520}

+-----------------+--------------------------------------+
| **Aliases:**    | > sample collection device or method |
+-----------------+--------------------------------------+
| **Mappings:**   | > mixs:samp_collect_device           |
+-----------------+--------------------------------------+
| **In Subsets:** | > nucleic acid sequence source       |
+-----------------+--------------------------------------+

# Slot: samp_collection_point

"Sampling point on the asset were sample was collected (e.g. Wellhead,
storage tank, separator, etc). If""other"" is specified, please propose
entry in ""additional info"" field"

URI:
[nmdc:samp_collection_point](https://microbiomedata/meta/samp_collection_point)

## Domain and Range {#domain-and-range-673}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-710}

-   is_a: [attribute](attribute.md)

## Children {#children-680}

## Used by {#used-by-673}

## Other properties {#other-properties-521}

+---------------+------------------------------+
| **Aliases:**  | > sample collection point    |
+---------------+------------------------------+
| **Mappings:** | > mixs:samp_collection_point |
+---------------+------------------------------+

# Slot: samp_dis_stage

"Stage of the disease at the time of sample collection,
e.g. inoculation, penetration, infection, growth and reproduction,
dissemination of pathogen."

URI: [nmdc:samp_dis_stage](https://microbiomedata/meta/samp_dis_stage)

## Domain and Range {#domain-and-range-674}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-711}

-   is_a: [attribute](attribute.md)

## Children {#children-681}

## Used by {#used-by-674}

## Other properties {#other-properties-522}

+---------------+------------------------+
| **Aliases:**  | > sample disease stage |
+---------------+------------------------+
| **Mappings:** | > mixs:samp_dis_stage  |
+---------------+------------------------+

# Slot: samp_floor

"The floor of the building, where the sampling room is located"

URI: [nmdc:samp_floor](https://microbiomedata/meta/samp_floor)

## Domain and Range {#domain-and-range-675}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-712}

-   is_a: [attribute](attribute.md)

## Children {#children-682}

## Used by {#used-by-675}

## Other properties {#other-properties-523}

+---------------+-------------------+
| **Aliases:**  | > sampling floor  |
+---------------+-------------------+
| **Mappings:** | > mixs:samp_floor |
+---------------+-------------------+

# Slot: samp_loc_corr_rate

"Metal corrosion rate is the speed of metal deterioration due to
environmental conditions. As environmental conditions change corrosion
rates change accordingly. Therefore, long term corrosion rates are
generally more informative than short term rates and for that reason
they are preferred during reporting. In the case of suspected MIC,
corrosion rate measurements at the time of sampling might provide
insights into the involvement of certain microbial community members in
MIC as well as potential microbial interplays"

URI:
[nmdc:samp_loc_corr_rate](https://microbiomedata/meta/samp_loc_corr_rate)

## Domain and Range {#domain-and-range-676}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-713}

-   is_a: [attribute](attribute.md)

## Children {#children-683}

## Used by {#used-by-676}

## Other properties {#other-properties-524}

+---------------+-------------------------------------+
| **Aliases:**  | > corrosion rate at sample location |
+---------------+-------------------------------------+
| **Mappings:** | > mixs:samp_loc_corr_rate           |
+---------------+-------------------------------------+

# Slot: samp_mat_process

"Any processing applied to the sample during or after retrieving the
sample from environment. This field accepts OBI, for a browser of OBI (v
2018-02-12) terms please see <http://purl.bioontology.org/ontology/OBI>"

URI:
[nmdc:samp_mat_process](https://microbiomedata/meta/samp_mat_process)

## Domain and Range {#domain-and-range-677}

None → 0..1 [ControlledTermValue](ControlledTermValue.md)

## Parents {#parents-714}

-   is_a: [attribute](attribute.md)

## Children {#children-684}

## Used by {#used-by-677}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-525}

+-----------------+--------------------------------+
| **Aliases:**    | > sample material processing   |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:samp_mat_process        |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: samp_md

"In non deviated well, measured depth is equal to the true vertical
depth, TVD (TVD=TVDSS plus the reference or datum it refers to). In
deviated wells, the MD is the length of trajectory of the borehole
measured from the same reference or datum. Common datums used are ground
level (GL), drilling rig floor (DF), rotary table (RT), kelly bushing
(KB) and mean sea level (MSL). If""other"" is specified, please propose
entry in ""additional info"" field "

URI: [nmdc:samp_md](https://microbiomedata/meta/samp_md)

## Domain and Range {#domain-and-range-678}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-715}

-   is_a: [attribute](attribute.md)

## Children {#children-685}

## Used by {#used-by-678}

## Other properties {#other-properties-526}

+---------------+-------------------------+
| **Aliases:**  | > sample measured depth |
+---------------+-------------------------+
| **Mappings:** | > mixs:samp_md          |
+---------------+-------------------------+

# Slot: samp_preserv

"Preservative added to the sample (e.g. Rnalater, alcohol, formaldehyde,
etc.). Where appropriate include volume added (e.g. Rnalater; 2 ml)"

URI: [nmdc:samp_preserv](https://microbiomedata/meta/samp_preserv)

## Domain and Range {#domain-and-range-679}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-716}

-   is_a: [attribute](attribute.md)

## Children {#children-686}

## Used by {#used-by-679}

## Other properties {#other-properties-527}

+---------------+--------------------------------+
| **Aliases:**  | > preservative added to sample |
+---------------+--------------------------------+
| **Mappings:** | > mixs:samp_preserv            |
+---------------+--------------------------------+

# Slot: samp_room_id

Sampling room number. This ID should be consistent with the designations
on the building floor plans

URI: [nmdc:samp_room_id](https://microbiomedata/meta/samp_room_id)

## Domain and Range {#domain-and-range-680}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-717}

-   is_a: [attribute](attribute.md)

## Children {#children-687}

## Used by {#used-by-680}

## Other properties {#other-properties-528}

+---------------+----------------------------+
| **Aliases:**  | > sampling room ID or name |
+---------------+----------------------------+
| **Mappings:** | > mixs:samp_room_id        |
+---------------+----------------------------+

# Slot: samp_salinity

"Salinity is the total concentration of all dissolved salts in a liquid
or solid (in the form of an extract obtained by centrifugation) sample.
While salinity can be measured by a complete chemical analysis, this
method is difficult and time consuming. More often, it is instead
derived from the conductivity measurement. This is known as practical
salinity. These derivations compare the specific conductance of the
sample to a salinity standard such as seawater"

URI: [nmdc:samp_salinity](https://microbiomedata/meta/samp_salinity)

## Domain and Range {#domain-and-range-681}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-718}

-   is_a: [attribute](attribute.md)

## Children {#children-688}

## Used by {#used-by-681}

## Other properties {#other-properties-529}

+---------------+----------------------+
| **Aliases:**  | > sample salinity    |
+---------------+----------------------+
| **Mappings:** | > mixs:samp_salinity |
+---------------+----------------------+

# Slot: samp_size

"Amount or size of sample (volume, mass or area) that was collected"

URI: [nmdc:samp_size](https://microbiomedata/meta/samp_size)

## Domain and Range {#domain-and-range-682}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-719}

-   is_a: [attribute](attribute.md)

## Children {#children-689}

## Used by {#used-by-682}

## Other properties {#other-properties-530}

+-----------------+--------------------------------------+
| **Aliases:**    | > amount or size of sample collected |
+-----------------+--------------------------------------+
| **Mappings:**   | > mixs:samp_size                     |
+-----------------+--------------------------------------+
| **In Subsets:** | > nucleic acid sequence source       |
+-----------------+--------------------------------------+

# Slot: samp_sort_meth

"Method by which samples are sorted; open face filter collecting total
suspended particles, prefilter to remove particles larger than X
micrometers in diameter, where common values of X would be 10 and 2.5
full size sorting in a cascade impactor."

URI: [nmdc:samp_sort_meth](https://microbiomedata/meta/samp_sort_meth)

## Domain and Range {#domain-and-range-683}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-720}

-   is_a: [attribute](attribute.md)

## Children {#children-690}

## Used by {#used-by-683}

## Other properties {#other-properties-531}

+---------------+------------------------------+
| **Aliases:**  | > sample size sorting method |
+---------------+------------------------------+
| **Mappings:** | > mixs:samp_sort_meth        |
+---------------+------------------------------+

# Slot: samp_store_dur

Duration for which the sample was stored

URI: [nmdc:samp_store_dur](https://microbiomedata/meta/samp_store_dur)

## Domain and Range {#domain-and-range-684}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-721}

-   is_a: [attribute](attribute.md)

## Children {#children-691}

## Used by {#used-by-684}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-532}

+---------------+---------------------------+
| **Aliases:**  | > sample storage duration |
+---------------+---------------------------+
| **Mappings:** | > mixs:samp_store_dur     |
+---------------+---------------------------+

# Slot: samp_store_loc

"Location at which sample was stored, usually name of a specific
freezer/room"

URI: [nmdc:samp_store_loc](https://microbiomedata/meta/samp_store_loc)

## Domain and Range {#domain-and-range-685}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-722}

-   is_a: [attribute](attribute.md)

## Children {#children-692}

## Used by {#used-by-685}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-533}

+---------------+---------------------------+
| **Aliases:**  | > sample storage location |
+---------------+---------------------------+
| **Mappings:** | > mixs:samp_store_loc     |
+---------------+---------------------------+

# Slot: samp_store_temp

"Temperature at which sample was stored, e.g. -80 degree Celsius"

URI: [nmdc:samp_store_temp](https://microbiomedata/meta/samp_store_temp)

## Domain and Range {#domain-and-range-686}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-723}

-   is_a: [attribute](attribute.md)

## Children {#children-693}

## Used by {#used-by-686}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-534}

+---------------+------------------------------+
| **Aliases:**  | > sample storage temperature |
+---------------+------------------------------+
| **Mappings:** | > mixs:samp_store_temp       |
+---------------+------------------------------+

# Slot: samp_subtype

"Name of sample sub-type. For example if""sample type"" is ""Produced
Water"" then subtype could be ""Oil Phase"" or ""Water Phase"".
If""other"" is specified, please propose entry in ""additional info""
field"

URI: [nmdc:samp_subtype](https://microbiomedata/meta/samp_subtype)

## Domain and Range {#domain-and-range-687}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-724}

-   is_a: [attribute](attribute.md)

## Children {#children-694}

## Used by {#used-by-687}

## Other properties {#other-properties-535}

+---------------+---------------------+
| **Aliases:**  | > sample subtype    |
+---------------+---------------------+
| **Mappings:** | > mixs:samp_subtype |
+---------------+---------------------+

# Slot: samp_time_out

The recent and long term history of outside sampling

URI: [nmdc:samp_time_out](https://microbiomedata/meta/samp_time_out)

## Domain and Range {#domain-and-range-688}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-725}

-   is_a: [attribute](attribute.md)

## Children {#children-695}

## Used by {#used-by-688}

## Other properties {#other-properties-536}

+---------------+-------------------------+
| **Aliases:**  | > sampling time outside |
+---------------+-------------------------+
| **Mappings:** | > mixs:samp_time_out    |
+---------------+-------------------------+

# Slot: samp_transport_cond

Sample transport duration (in days or hrs) and temperature the sample
was exposed to (e.g. 5.5 days; 20 B0C)

URI:
[nmdc:samp_transport_cond](https://microbiomedata/meta/samp_transport_cond)

## Domain and Range {#domain-and-range-689}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-726}

-   is_a: [attribute](attribute.md)

## Children {#children-696}

## Used by {#used-by-689}

## Other properties {#other-properties-537}

+---------------+-------------------------------+
| **Aliases:**  | > sample transport conditions |
+---------------+-------------------------------+
| **Mappings:** | > mixs:samp_transport_cond    |
+---------------+-------------------------------+

# Slot: samp_tvdss

Depth of the sample i.e. The vertical distance between the sea level and
the sampled position in the subsurface. Depth can be reported as an
interval for subsurface samples e.g. 1325.75-1362.25 m

URI: [nmdc:samp_tvdss](https://microbiomedata/meta/samp_tvdss)

## Domain and Range {#domain-and-range-690}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-727}

-   is_a: [attribute](attribute.md)

## Children {#children-697}

## Used by {#used-by-690}

## Other properties {#other-properties-538}

+---------------+-------------------------------------+
| **Aliases:**  | > sample true vertical depth subsea |
+---------------+-------------------------------------+
| **Mappings:** | > mixs:samp_tvdss                   |
+---------------+-------------------------------------+

# Slot: samp_type

"Type of material (i.e. sample) collected. Includes types like core,
rock trimmings, drill cuttings, piping section, coupon, pigging debris,
solid deposit, produced fluid, produced water, injected water, swabs,
etc. If""other"" is specified, please propose entry in ""additional
info"" field"

URI: [nmdc:samp_type](https://microbiomedata/meta/samp_type)

## Domain and Range {#domain-and-range-691}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-728}

-   is_a: [attribute](attribute.md)

## Children {#children-698}

## Used by {#used-by-691}

## Other properties {#other-properties-539}

+---------------+------------------+
| **Aliases:**  | > sample type    |
+---------------+------------------+
| **Mappings:** | > mixs:samp_type |
+---------------+------------------+

# Slot: samp_vol_we_dna_ext

"Volume (ml), weight (g) of processed sample, or surface area swabbed
from sample for DNA extraction"

URI:
[nmdc:samp_vol_we_dna_ext](https://microbiomedata/meta/samp_vol_we_dna_ext)

## Domain and Range {#domain-and-range-692}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-729}

-   is_a: [attribute](attribute.md)

## Children {#children-699}

## Used by {#used-by-692}

-   [Biosample](Biosample.md)
-   [OmicsProcessing](OmicsProcessing.md)

## Other properties {#other-properties-540}

+---------------+----------------------------------------------+
| **Aliases:**  | > sample volume or weight for DNA extraction |
+---------------+----------------------------------------------+
| **Mappings:** | > mixs:samp_vol_we_dna_ext                   |
+---------------+----------------------------------------------+

# Slot: samp_weather

The weather on the sampling day

URI: [nmdc:samp_weather](https://microbiomedata/meta/samp_weather)

## Domain and Range {#domain-and-range-693}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-730}

-   is_a: [attribute](attribute.md)

## Children {#children-700}

## Used by {#used-by-693}

## Other properties {#other-properties-541}

+---------------+------------------------+
| **Aliases:**  | > sampling day weather |
+---------------+------------------------+
| **Mappings:** | > mixs:samp_weather    |
+---------------+------------------------+

# Slot: samp_well_name

Name of the well (e.g. BXA1123) where sample was taken

URI: [nmdc:samp_well_name](https://microbiomedata/meta/samp_well_name)

## Domain and Range {#domain-and-range-694}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-731}

-   is_a: [attribute](attribute.md)

## Children {#children-701}

## Used by {#used-by-694}

## Other properties {#other-properties-542}

+---------------+-----------------------+
| **Aliases:**  | > sample well name    |
+---------------+-----------------------+
| **Mappings:** | > mixs:samp_well_name |
+---------------+-----------------------+

# Slot: sample_collection_day

URI:
[nmdc:sample_collection_day](https://microbiomedata/meta/sample_collection_day)

## Domain and Range {#domain-and-range-695}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-732}

## Children {#children-702}

## Used by {#used-by-695}

# Slot: sample_collection_hour

URI:
[nmdc:sample_collection_hour](https://microbiomedata/meta/sample_collection_hour)

## Domain and Range {#domain-and-range-696}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-733}

## Children {#children-703}

## Used by {#used-by-696}

# Slot: sample_collection_minute

URI:
[nmdc:sample_collection_minute](https://microbiomedata/meta/sample_collection_minute)

## Domain and Range {#domain-and-range-697}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-734}

## Children {#children-704}

## Used by {#used-by-697}

# Slot: sample_collection_month

URI:
[nmdc:sample_collection_month](https://microbiomedata/meta/sample_collection_month)

## Domain and Range {#domain-and-range-698}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-735}

## Children {#children-705}

## Used by {#used-by-698}

# Slot: sample_collection_site

URI:
[nmdc:sample_collection_site](https://microbiomedata/meta/sample_collection_site)

## Domain and Range {#domain-and-range-699}

None → 0..1 [String](types/String.md)

## Parents {#parents-736}

## Children {#children-706}

## Used by {#used-by-699}

-   [Biosample](Biosample.md)

# Slot: sample_collection_year

URI:
[nmdc:sample_collection_year](https://microbiomedata/meta/sample_collection_year)

## Domain and Range {#domain-and-range-700}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-737}

## Children {#children-707}

## Used by {#used-by-700}

# Slot: sample identifiers

URI:
<https://microbiomedata/schema/workflow_execution_activity/sample_identifiers>

## Domain and Range {#domain-and-range-701}

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents {#parents-738}

-   is_a: [external database
    identifiers](external_database_identifiers.md)

## Children {#children-708}

-   [GOLD sample identifiers](GOLD_sample_identifiers.md)
-   [INSDC biosample identifiers](INSDC_biosample_identifiers.md)
-   [INSDC secondary sample
    identifiers](INSDC_secondary_sample_identifiers.md)

## Used by {#used-by-701}

# Slot: saturates_pc

"Saturate, Aromatic, Resin and AsphalteneB (SARA) is an analysis method
that dividesB crude oilB components according to their polarizability
and polarity. There are three main methods to obtain SARA results. The
most popular one is known as the Iatroscan TLC-FID and is referred to as
IP-143 (source:
<https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene>)"

URI: [nmdc:saturates_pc](https://microbiomedata/meta/saturates_pc)

## Domain and Range {#domain-and-range-702}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-739}

-   is_a: [attribute](attribute.md)

## Children {#children-709}

## Used by {#used-by-702}

## Other properties {#other-properties-543}

+---------------+---------------------+
| **Aliases:**  | > saturates wt%     |
+---------------+---------------------+
| **Mappings:** | > mixs:saturates_pc |
+---------------+---------------------+

# Slot: scaf_L50

Given a set of scaffolds, the L50 is defined as the sequence length of
the shortest scaffold at 50% of the total genome length.

URI:
<https://microbiomedata/schema/workflow_execution_activity/scaf_L50>

## Domain and Range {#domain-and-range-703}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-740}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-710}

## Used by {#used-by-703}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: scaf_L90

The L90 statistic is less than or equal to the L50 statistic; it is the
length for which the collection of all scaffolds of that length or
longer contains at least 90% of the sum of the lengths of all scaffolds.

URI:
<https://microbiomedata/schema/workflow_execution_activity/scaf_L90>

## Domain and Range {#domain-and-range-704}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-741}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-711}

## Used by {#used-by-704}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: scaf_N50

Given a set of scaffolds, each with its own length, the N50 count is
defined as the smallest number of scaffolds whose length sum makes up
half of genome size.

URI:
<https://microbiomedata/schema/workflow_execution_activity/scaf_N50>

## Domain and Range {#domain-and-range-705}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-742}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-712}

## Used by {#used-by-705}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: scaf_N90

Given a set of scaffolds, each with its own length, the N90 count is
defined as the smallest number of scaffolds whose length sum makes up
90% of genome size.

URI:
<https://microbiomedata/schema/workflow_execution_activity/scaf_N90>

## Domain and Range {#domain-and-range-706}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-743}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-713}

## Used by {#used-by-706}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: scaf_bp

Total size in bp of all scaffolds.

URI: <https://microbiomedata/schema/workflow_execution_activity/scaf_bp>

## Domain and Range {#domain-and-range-707}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-744}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-714}

## Used by {#used-by-707}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: scaf_l\_gt50K

Total size in bp of all scaffolds greater than 50 KB.

URI:
<https://microbiomedata/schema/workflow_execution_activity/scaf_l_gt50K>

## Domain and Range {#domain-and-range-708}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-745}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-715}

## Used by {#used-by-708}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: scaf_logsum

The sum of the (length\*log(length)) of all scaffolds, times some
constant. Increase the contiguity, the score will increase

URI:
<https://microbiomedata/schema/workflow_execution_activity/scaf_logsum>

## Domain and Range {#domain-and-range-709}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-746}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-716}

## Used by {#used-by-709}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: scaf_max

Maximum scaffold length.

URI:
<https://microbiomedata/schema/workflow_execution_activity/scaf_max>

## Domain and Range {#domain-and-range-710}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-747}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-717}

## Used by {#used-by-710}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: scaf_n\_gt50K

Total sequence count of scaffolds greater than 50 KB.

URI:
<https://microbiomedata/schema/workflow_execution_activity/scaf_n_gt50K>

## Domain and Range {#domain-and-range-711}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-748}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-718}

## Used by {#used-by-711}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: scaf_pct_gt50K

Total sequence size percentage of scaffolds greater than 50 KB.

URI:
<https://microbiomedata/schema/workflow_execution_activity/scaf_pct_gt50K>

## Domain and Range {#domain-and-range-712}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-749}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-719}

## Used by {#used-by-712}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: scaf_powsum

Powersum of all scaffolds is the same as logsum except that it uses the
sum of (length\*(length\^P)) for some power P (default P=0.25).

URI:
<https://microbiomedata/schema/workflow_execution_activity/scaf_powsum>

## Domain and Range {#domain-and-range-713}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-750}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-720}

## Used by {#used-by-713}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: scaffolds

Total sequence count of all scaffolds.

URI:
<https://microbiomedata/schema/workflow_execution_activity/scaffolds>

## Domain and Range {#domain-and-range-714}

None → 0..1 [Float](types/Float.md)

## Parents {#parents-751}

-   is_a: [metagenome assembly
    parameter](metagenome_assembly_parameter.md)

## Children {#children-721}

## Used by {#used-by-714}

-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)

# Slot: season

The season when sampling occurred

URI: [nmdc:season](https://microbiomedata/meta/season)

## Domain and Range {#domain-and-range-715}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-752}

-   is_a: [attribute](attribute.md)

## Children {#children-722}

## Used by {#used-by-715}

## Other properties {#other-properties-544}

+---------------+---------------+
| **Aliases:**  | > season      |
+---------------+---------------+
| **Mappings:** | > mixs:season |
+---------------+---------------+

# Slot: season_environment

"Treatment involving an exposure to a particular season (e.g. Winter,
summer, rabi, rainy etc.), treatment regimen including how many times
the treatment was repeated, how long each treatment lasted, and the
start and end time of the entire treatment"

URI:
[nmdc:season_environment](https://microbiomedata/meta/season_environment)

## Domain and Range {#domain-and-range-716}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-753}

-   is_a: [attribute](attribute.md)

## Children {#children-723}

## Used by {#used-by-716}

## Other properties {#other-properties-545}

+---------------+---------------------------+
| **Aliases:**  | > seasonal environment    |
+---------------+---------------------------+
| **Mappings:** | > mixs:season_environment |
+---------------+---------------------------+

# Slot: season_precpt

"The average of all seasonal precipitation values known, or an estimated
equivalent value derived by such methods as regional indexes or
Isohyetal maps."

URI: [nmdc:season_precpt](https://microbiomedata/meta/season_precpt)

## Domain and Range {#domain-and-range-717}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-754}

-   is_a: [attribute](attribute.md)

## Children {#children-724}

## Used by {#used-by-717}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-546}

+---------------+-------------------------------+
| **Aliases:**  | > mean seasonal precipitation |
+---------------+-------------------------------+
| **Mappings:** | > mixs:season_precpt          |
+---------------+-------------------------------+

# Slot: season_temp

Mean seasonal temperature

URI: [nmdc:season_temp](https://microbiomedata/meta/season_temp)

## Domain and Range {#domain-and-range-718}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-755}

-   is_a: [attribute](attribute.md)

## Children {#children-725}

## Used by {#used-by-718}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-547}

+---------------+-----------------------------+
| **Aliases:**  | > mean seasonal temperature |
+---------------+-----------------------------+
| **Mappings:** | > mixs:season_temp          |
+---------------+-----------------------------+

# Slot: season_use

The seasons the space is occupied

URI: [nmdc:season_use](https://microbiomedata/meta/season_use)

## Domain and Range {#domain-and-range-719}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-756}

-   is_a: [attribute](attribute.md)

## Children {#children-726}

## Used by {#used-by-719}

## Other properties {#other-properties-548}

+---------------+-------------------+
| **Aliases:**  | > seasonal use    |
+---------------+-------------------+
| **Mappings:** | > mixs:season_use |
+---------------+-------------------+

# Slot: secondary_treatment

The process for substantially degrading the biological content of the
sewage

URI:
[nmdc:secondary_treatment](https://microbiomedata/meta/secondary_treatment)

## Domain and Range {#domain-and-range-720}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-757}

-   is_a: [attribute](attribute.md)

## Children {#children-727}

## Used by {#used-by-720}

## Other properties {#other-properties-549}

+---------------+----------------------------+
| **Aliases:**  | > secondary treatment      |
+---------------+----------------------------+
| **Mappings:** | > mixs:secondary_treatment |
+---------------+----------------------------+

# Slot: sediment_type

Information about the sediment type based on major constituents

URI: [nmdc:sediment_type](https://microbiomedata/meta/sediment_type)

## Domain and Range {#domain-and-range-721}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-758}

-   is_a: [attribute](attribute.md)

## Children {#children-728}

## Used by {#used-by-721}

## Other properties {#other-properties-550}

+---------------+----------------------+
| **Aliases:**  | > sediment type      |
+---------------+----------------------+
| **Mappings:** | > mixs:sediment_type |
+---------------+----------------------+

# Slot: seq_meth

"Sequencing method used; e.g. Sanger, pyrosequencing, ABI-solid"

URI: [nmdc:seq_meth](https://microbiomedata/meta/seq_meth)

## Domain and Range {#domain-and-range-722}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-759}

-   is_a: [attribute](attribute.md)

## Children {#children-729}

## Used by {#used-by-722}

-   [OmicsProcessing](OmicsProcessing.md)

## Other properties {#other-properties-551}

+-----------------+---------------------+
| **Aliases:**    | > sequencing method |
+-----------------+---------------------+
| **Mappings:**   | > mixs:seq_meth     |
+-----------------+---------------------+
| **In Subsets:** | > sequencing        |
+-----------------+---------------------+

# Slot: seq_quality_check

"Indicate if the sequence has been called by automatic systems (none) or
undergone a manual editing procedure (e.g. by inspecting the raw data or
chromatograms). Applied only for sequences that are not submitted to
SRA,ENA or DRA"

URI:
[nmdc:seq_quality_check](https://microbiomedata/meta/seq_quality_check)

## Domain and Range {#domain-and-range-723}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-760}

-   is_a: [attribute](attribute.md)

## Children {#children-730}

## Used by {#used-by-723}

-   [OmicsProcessing](OmicsProcessing.md)

## Other properties {#other-properties-552}

+-----------------+--------------------------+
| **Aliases:**    | > sequence quality check |
+-----------------+--------------------------+
| **Mappings:**   | > mixs:seq_quality_check |
+-----------------+--------------------------+
| **In Subsets:** | > sequencing             |
+-----------------+--------------------------+

# Slot: seqid {#slot-seqid-1}

The ID of the landmark used to establish the coordinate system for the
current feature.

URI: [nmdc:seqid](https://microbiomedata/meta/seqid)

## Domain and Range {#domain-and-range-724}

None → 1..1 [String](types/String.md)

## Parents {#parents-761}

## Children {#children-731}

-   [genome feature➞seqid](genome_feature_seqid.md)

## Used by {#used-by-724}

# Slot: sewage_type

Type of wastewater treatment plant as municipial or industrial

URI: [nmdc:sewage_type](https://microbiomedata/meta/sewage_type)

## Domain and Range {#domain-and-range-725}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-762}

-   is_a: [attribute](attribute.md)

## Children {#children-732}

## Used by {#used-by-725}

## Other properties {#other-properties-553}

+---------------+--------------------+
| **Aliases:**  | > sewage type      |
+---------------+--------------------+
| **Mappings:** | > mixs:sewage_type |
+---------------+--------------------+

# Slot: sexual_act

Current sexual partner and frequency of sex

URI: [nmdc:sexual_act](https://microbiomedata/meta/sexual_act)

## Domain and Range {#domain-and-range-726}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-763}

-   is_a: [attribute](attribute.md)

## Children {#children-733}

## Used by {#used-by-726}

## Other properties {#other-properties-554}

+---------------+-------------------+
| **Aliases:**  | > sexual activity |
+---------------+-------------------+
| **Mappings:** | > mixs:sexual_act |
+---------------+-------------------+

# Slot: shading_device_cond

The physical condition of the shading device at the time of sampling

URI:
[nmdc:shading_device_cond](https://microbiomedata/meta/shading_device_cond)

## Domain and Range {#domain-and-range-727}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-764}

-   is_a: [attribute](attribute.md)

## Children {#children-734}

## Used by {#used-by-727}

## Other properties {#other-properties-555}

+---------------+----------------------------+
| **Aliases:**  | > shading device condition |
+---------------+----------------------------+
| **Mappings:** | > mixs:shading_device_cond |
+---------------+----------------------------+

# Slot: shading_device_loc

The location of the shading device in relation to the built structure

URI:
[nmdc:shading_device_loc](https://microbiomedata/meta/shading_device_loc)

## Domain and Range {#domain-and-range-728}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-765}

-   is_a: [attribute](attribute.md)

## Children {#children-735}

## Used by {#used-by-728}

## Other properties {#other-properties-556}

+---------------+---------------------------+
| **Aliases:**  | > shading device location |
+---------------+---------------------------+
| **Mappings:** | > mixs:shading_device_loc |
+---------------+---------------------------+

# Slot: shading_device_mat

The material the shading device is composed of

URI:
[nmdc:shading_device_mat](https://microbiomedata/meta/shading_device_mat)

## Domain and Range {#domain-and-range-729}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-766}

-   is_a: [attribute](attribute.md)

## Children {#children-736}

## Used by {#used-by-729}

## Other properties {#other-properties-557}

+---------------+---------------------------+
| **Aliases:**  | > shading device material |
+---------------+---------------------------+
| **Mappings:** | > mixs:shading_device_mat |
+---------------+---------------------------+

# Slot: shading_device_type

The type of shading device

URI:
[nmdc:shading_device_type](https://microbiomedata/meta/shading_device_type)

## Domain and Range {#domain-and-range-730}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-767}

-   is_a: [attribute](attribute.md)

## Children {#children-737}

## Used by {#used-by-730}

## Other properties {#other-properties-558}

+---------------+----------------------------+
| **Aliases:**  | > shading device type      |
+---------------+----------------------------+
| **Mappings:** | > mixs:shading_device_type |
+---------------+----------------------------+

# Slot: shading_device_water_mold

Signs of the presence of mold or mildew on the shading device

URI:
[nmdc:shading_device_water_mold](https://microbiomedata/meta/shading_device_water_mold)

## Domain and Range {#domain-and-range-731}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-768}

-   is_a: [attribute](attribute.md)

## Children {#children-738}

## Used by {#used-by-731}

## Other properties {#other-properties-559}

+---------------+--------------------------------------+
| **Aliases:**  | > shading device signs of water/mold |
+---------------+--------------------------------------+
| **Mappings:** | > mixs:shading_device_water_mold     |
+---------------+--------------------------------------+

# Slot: sieving

Collection design of pooled samples and/or sieve size and amount of
sample sieved

URI: [nmdc:sieving](https://microbiomedata/meta/sieving)

## Domain and Range {#domain-and-range-732}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-769}

-   is_a: [attribute](attribute.md)

## Children {#children-739}

## Used by {#used-by-732}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-560}

+---------------+-------------------------------------+
| **Aliases:**  | > composite design/sieving (if any) |
+---------------+-------------------------------------+
| **Mappings:** | > mixs:sieving                      |
+---------------+-------------------------------------+

# Slot: silicate

Concentration of silicate

URI: [nmdc:silicate](https://microbiomedata/meta/silicate)

## Domain and Range {#domain-and-range-733}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-770}

-   is_a: [attribute](attribute.md)

## Children {#children-740}

## Used by {#used-by-733}

## Other properties {#other-properties-561}

+---------------+-----------------+
| **Aliases:**  | > silicate      |
+---------------+-----------------+
| **Mappings:** | > mixs:silicate |
+---------------+-----------------+

# Slot: sim_search_meth

"Tool used to compare ORFs with database, along with version and cutoffs
used"

URI: [nmdc:sim_search_meth](https://microbiomedata/meta/sim_search_meth)

## Domain and Range {#domain-and-range-734}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-771}

-   is_a: [attribute](attribute.md)

## Children {#children-741}

## Used by {#used-by-734}

## Other properties {#other-properties-562}

+-----------------+----------------------------+
| **Aliases:**    | > similarity search method |
+-----------------+----------------------------+
| **Mappings:**   | > mixs:sim_search_meth     |
+-----------------+----------------------------+
| **In Subsets:** | > sequencing               |
+-----------------+----------------------------+

# Slot: single_cell_lysis_appr

Method used to free DNA from interior of the cell(s) or particle(s)

URI:
[nmdc:single_cell_lysis_appr](https://microbiomedata/meta/single_cell_lysis_appr)

## Domain and Range {#domain-and-range-735}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-772}

-   is_a: [attribute](attribute.md)

## Children {#children-742}

## Used by {#used-by-735}

## Other properties {#other-properties-563}

+-----------------+------------------------------------------------+
| **Aliases:**    | > single cell or viral particle lysis approach |
+-----------------+------------------------------------------------+
| **Mappings:**   | > mixs:single_cell_lysis_appr                  |
+-----------------+------------------------------------------------+
| **In Subsets:** | > sequencing                                   |
+-----------------+------------------------------------------------+

# Slot: single_cell_lysis_prot

Name of the kit or standard protocol used for cell(s) or particle(s)
lysis

URI:
[nmdc:single_cell_lysis_prot](https://microbiomedata/meta/single_cell_lysis_prot)

## Domain and Range {#domain-and-range-736}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-773}

-   is_a: [attribute](attribute.md)

## Children {#children-743}

## Used by {#used-by-736}

## Other properties {#other-properties-564}

+-----------------+----------------------------------------------------+
| **Aliases:**    | > single cell or viral particle lysis kit protocol |
+-----------------+----------------------------------------------------+
| **Mappings:**   | > mixs:single_cell_lysis_prot                      |
+-----------------+----------------------------------------------------+
| **In Subsets:** | > sequencing                                       |
+-----------------+----------------------------------------------------+

# Slot: size_frac

Filtering pore size used in sample preparation

URI: [nmdc:size_frac](https://microbiomedata/meta/size_frac)

## Domain and Range {#domain-and-range-737}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-774}

-   is_a: [attribute](attribute.md)

## Children {#children-744}

## Used by {#used-by-737}

## Other properties {#other-properties-565}

+-----------------+--------------------------------+
| **Aliases:**    | > size fraction selected       |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:size_frac               |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: size_frac_low

Refers to the mesh/pore size used to pre-filter/pre-sort the sample.
Materials larger than the size threshold are excluded from the sample

URI: [nmdc:size_frac_low](https://microbiomedata/meta/size_frac_low)

## Domain and Range {#domain-and-range-738}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-775}

-   is_a: [attribute](attribute.md)

## Children {#children-745}

## Used by {#used-by-738}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-566}

+---------------+---------------------------------+
| **Aliases:**  | > size-fraction lower threshold |
+---------------+---------------------------------+
| **Mappings:** | > mixs:size_frac_low            |
+---------------+---------------------------------+

# Slot: size_frac_up

Refers to the mesh/pore size used to retain the sample. Materials
smaller than the size threshold are excluded from the sample

URI: [nmdc:size_frac_up](https://microbiomedata/meta/size_frac_up)

## Domain and Range {#domain-and-range-739}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-776}

-   is_a: [attribute](attribute.md)

## Children {#children-746}

## Used by {#used-by-739}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-567}

+---------------+---------------------------------+
| **Aliases:**  | > size-fraction upper threshold |
+---------------+---------------------------------+
| **Mappings:** | > mixs:size_frac_up             |
+---------------+---------------------------------+

# Slot: slope_aspect

"The direction a slope faces. While looking down a slope use a compass
to record the direction you are facing (direction or degrees); e.g., nw
or 315 degrees. This measure provides an indication of sun and wind
exposure that will influence soil temperature and evapotranspiration."

URI: [nmdc:slope_aspect](https://microbiomedata/meta/slope_aspect)

## Domain and Range {#domain-and-range-740}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-777}

-   is_a: [attribute](attribute.md)

## Children {#children-747}

## Used by {#used-by-740}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-568}

+---------------+---------------------+
| **Aliases:**  | > slope aspect      |
+---------------+---------------------+
| **Mappings:** | > mixs:slope_aspect |
+---------------+---------------------+

# Slot: slope_gradient

Commonly called 'slope'. The angle between ground surface and a
horizontal line (in percent). This is the direction that overland water
would flow. This measure is usually taken with a hand level meter or
clinometer

URI: [nmdc:slope_gradient](https://microbiomedata/meta/slope_gradient)

## Domain and Range {#domain-and-range-741}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-778}

-   is_a: [attribute](attribute.md)

## Children {#children-748}

## Used by {#used-by-741}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-569}

+---------------+-----------------------+
| **Aliases:**  | > slope gradient      |
+---------------+-----------------------+
| **Mappings:** | > mixs:slope_gradient |
+---------------+-----------------------+

# Slot: sludge_retent_time

The time activated sludge remains in reactor

URI:
[nmdc:sludge_retent_time](https://microbiomedata/meta/sludge_retent_time)

## Domain and Range {#domain-and-range-742}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-779}

-   is_a: [attribute](attribute.md)

## Children {#children-749}

## Used by {#used-by-742}

## Other properties {#other-properties-570}

+---------------+---------------------------+
| **Aliases:**  | > sludge retention time   |
+---------------+---------------------------+
| **Mappings:** | > mixs:sludge_retent_time |
+---------------+---------------------------+

# Slot: smarts string

URI: [nmdc:smarts_string](https://microbiomedata/meta/smarts_string)

## Domain and Range {#domain-and-range-743}

None → 0..1 [String](types/String.md)

## Parents {#parents-780}

## Children {#children-750}

-   [reaction➞smarts string](reaction_smarts_string.md)

## Used by {#used-by-743}

# Slot: smiles

A string encoding of a molecular graph, no chiral or isotopic
information. There are usually a large number of valid SMILES which
represent a given structure. For example, CCO, OCC and C(O)C all specify
the structure of ethanol.

URI: <https://microbiomedata/schema/workflow_execution_activity/smiles>

## Domain and Range {#domain-and-range-744}

None → 0..\* [String](types/String.md)

## Parents {#parents-781}

## Children {#children-751}

-   [chemical entity➞smiles](chemical_entity_smiles.md)

## Used by {#used-by-744}

# Slot: smoker

Specification of smoking status

URI: [nmdc:smoker](https://microbiomedata/meta/smoker)

## Domain and Range {#domain-and-range-745}

None → 0..1 [BooleanValue](BooleanValue.md)

## Parents {#parents-782}

-   is_a: [attribute](attribute.md)

## Children {#children-752}

## Used by {#used-by-745}

## Other properties {#other-properties-571}

+---------------+---------------+
| **Aliases:**  | > smoker      |
+---------------+---------------+
| **Mappings:** | > mixs:smoker |
+---------------+---------------+

# Slot: sodium

Sodium concentration in the sample

URI: [nmdc:sodium](https://microbiomedata/meta/sodium)

## Domain and Range {#domain-and-range-746}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-783}

-   is_a: [attribute](attribute.md)

## Children {#children-753}

## Used by {#used-by-746}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-572}

+---------------+---------------+
| **Aliases:**  | > sodium      |
+---------------+---------------+
| **Mappings:** | > mixs:sodium |
+---------------+---------------+

# Slot: soil_annual_season_temp

URI:
[nmdc:soil_annual_season_temp](https://microbiomedata/meta/soil_annual_season_temp)

## Domain and Range {#domain-and-range-747}

None → 0..1 [String](types/String.md)

## Parents {#parents-784}

## Children {#children-754}

## Used by {#used-by-747}

# Slot: soil_type

Soil series name or other lower-level classification

URI: [nmdc:soil_type](https://microbiomedata/meta/soil_type)

## Domain and Range {#domain-and-range-748}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-785}

-   is_a: [attribute](attribute.md)

## Children {#children-755}

## Used by {#used-by-748}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-573}

+---------------+------------------+
| **Aliases:**  | > soil type      |
+---------------+------------------+
| **Mappings:** | > mixs:soil_type |
+---------------+------------------+

# Slot: soil_type_meth

Reference or method used in determining soil series name or other
lower-level classification

URI: [nmdc:soil_type_meth](https://microbiomedata/meta/soil_type_meth)

## Domain and Range {#domain-and-range-749}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-786}

-   is_a: [attribute](attribute.md)

## Children {#children-756}

## Used by {#used-by-749}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-574}

+---------------+-----------------------+
| **Aliases:**  | > soil type method    |
+---------------+-----------------------+
| **Mappings:** | > mixs:soil_type_meth |
+---------------+-----------------------+

# Slot: solar_irradiance

The amount of solar energy that arrives at a specific area of a surface
during a specific time interval

URI:
[nmdc:solar_irradiance](https://microbiomedata/meta/solar_irradiance)

## Domain and Range {#domain-and-range-750}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-787}

-   is_a: [attribute](attribute.md)

## Children {#children-757}

## Used by {#used-by-750}

## Other properties {#other-properties-575}

+---------------+-------------------------+
| **Aliases:**  | > solar irradiance      |
+---------------+-------------------------+
| **Mappings:** | > mixs:solar_irradiance |
+---------------+-------------------------+

# Slot: soluble_inorg_mat

"Concentration of substances such as ammonia, road-salt, sea-salt,
cyanide, hydrogen sulfide, thiocyanates, thiosulfates, etc."

URI:
[nmdc:soluble_inorg_mat](https://microbiomedata/meta/soluble_inorg_mat)

## Domain and Range {#domain-and-range-751}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-788}

-   is_a: [attribute](attribute.md)

## Children {#children-758}

## Used by {#used-by-751}

## Other properties {#other-properties-576}

+---------------+------------------------------+
| **Aliases:**  | > soluble inorganic material |
+---------------+------------------------------+
| **Mappings:** | > mixs:soluble_inorg_mat     |
+---------------+------------------------------+

# Slot: soluble_iron_micromol

URI:
[nmdc:soluble_iron_micromol](https://microbiomedata/meta/soluble_iron_micromol)

## Domain and Range {#domain-and-range-752}

None → 0..1 [String](types/String.md)

## Parents {#parents-789}

## Children {#children-759}

## Used by {#used-by-752}

-   [Biosample](Biosample.md)

# Slot: soluble_org_mat

"Concentration of substances such as urea, fruit sugars, soluble
proteins, drugs, pharmaceuticals, etc."

URI: [nmdc:soluble_org_mat](https://microbiomedata/meta/soluble_org_mat)

## Domain and Range {#domain-and-range-753}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-790}

-   is_a: [attribute](attribute.md)

## Children {#children-760}

## Used by {#used-by-753}

## Other properties {#other-properties-577}

+---------------+----------------------------+
| **Aliases:**  | > soluble organic material |
+---------------+----------------------------+
| **Mappings:** | > mixs:soluble_org_mat     |
+---------------+----------------------------+

# Slot: soluble_react_phosp

Concentration of soluble reactive phosphorus

URI:
[nmdc:soluble_react_phosp](https://microbiomedata/meta/soluble_react_phosp)

## Domain and Range {#domain-and-range-754}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-791}

-   is_a: [attribute](attribute.md)

## Children {#children-761}

## Used by {#used-by-754}

## Other properties {#other-properties-578}

+---------------+-------------------------------+
| **Aliases:**  | > soluble reactive phosphorus |
+---------------+-------------------------------+
| **Mappings:** | > mixs:soluble_react_phosp    |
+---------------+-------------------------------+

# Slot: sop

"Standard operating procedures used in assembly and/or annotation of
genomes, metagenomes or environmental sequences"

URI: [nmdc:sop](https://microbiomedata/meta/sop)

## Domain and Range {#domain-and-range-755}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-792}

-   is_a: [attribute](attribute.md)

## Children {#children-762}

## Used by {#used-by-755}

## Other properties {#other-properties-579}

+-----------------+------------------------------------------+
| **Aliases:**    | > relevant standard operating procedures |
+-----------------+------------------------------------------+
| **Mappings:**   | > mixs:sop                               |
+-----------------+------------------------------------------+
| **In Subsets:** | > sequencing                             |
+-----------------+------------------------------------------+

# Slot: sort_tech

Method used to sort/isolate cells or particles of interest

URI: [nmdc:sort_tech](https://microbiomedata/meta/sort_tech)

## Domain and Range {#domain-and-range-756}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-793}

-   is_a: [attribute](attribute.md)

## Children {#children-763}

## Used by {#used-by-756}

## Other properties {#other-properties-580}

+-----------------+----------------------+
| **Aliases:**    | > sorting technology |
+-----------------+----------------------+
| **Mappings:**   | > mixs:sort_tech     |
+-----------------+----------------------+
| **In Subsets:** | > sequencing         |
+-----------------+----------------------+

# Slot: source_mat_id

"A unique identifier assigned to a material sample (as defined by
<http://rs.tdwg.org/dwc/terms/materialSampleID>, and as opposed to a
particular digital record of a material sample) used for extracting
nucleic acids, and subsequent sequencing. The identifier can refer
either to the original material collected or to any derived sub-samples.
The INSDC qualifiers /specimen_voucher, /bio_material, or
/culture_collection may or may not share the same value as the
source_mat_id field. For instance, the /specimen_voucher qualifier and
source_mat_id may both contain 'UAM:Herps:14' , referring to both the
specimen voucher and sampled tissue with the same identifier. However,
the /culture_collection qualifier may refer to a value from an initial
culture (e.g. ATCC:11775) while source_mat_id would refer to an
identifier from some derived culture from which the nucleic acids were
extracted (e.g. xatc123 or ark:/2154/R2)."

URI: [nmdc:source_mat_id](https://microbiomedata/meta/source_mat_id)

## Domain and Range {#domain-and-range-757}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-794}

-   is_a: [attribute](attribute.md)

## Children {#children-764}

## Used by {#used-by-757}

## Other properties {#other-properties-581}

+-----------------+--------------------------------+
| **Aliases:**    | > source material identifiers  |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:source_mat_id           |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: source_uvig

Type of dataset from which the UViG was obtained

URI: [nmdc:source_uvig](https://microbiomedata/meta/source_uvig)

## Domain and Range {#domain-and-range-758}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-795}

-   is_a: [attribute](attribute.md)

## Children {#children-765}

## Used by {#used-by-758}

## Other properties {#other-properties-582}

+-----------------+--------------------------------+
| **Aliases:**    | > source of UViGs              |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:source_uvig             |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: space_typ_state

Customary or normal state of the space

URI: [nmdc:space_typ_state](https://microbiomedata/meta/space_typ_state)

## Domain and Range {#domain-and-range-759}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-796}

-   is_a: [attribute](attribute.md)

## Children {#children-766}

## Used by {#used-by-759}

## Other properties {#other-properties-583}

+---------------+------------------------+
| **Aliases:**  | > space typical state  |
+---------------+------------------------+
| **Mappings:** | > mixs:space_typ_state |
+---------------+------------------------+

# Slot: special_diet

Specification of special diet; can include multiple special diets

URI: [nmdc:special_diet](https://microbiomedata/meta/special_diet)

## Domain and Range {#domain-and-range-760}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-797}

-   is_a: [attribute](attribute.md)

## Children {#children-767}

## Used by {#used-by-760}

## Other properties {#other-properties-584}

+---------------+---------------------+
| **Aliases:**  | > special diet      |
+---------------+---------------------+
| **Mappings:** | > mixs:special_diet |
+---------------+---------------------+

# Slot: specific

"The building specifications. If design is chosen, indicate phase:
conceptual, schematic, design development, construction documents"

URI: [nmdc:specific](https://microbiomedata/meta/specific)

## Domain and Range {#domain-and-range-761}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-798}

-   is_a: [attribute](attribute.md)

## Children {#children-768}

## Used by {#used-by-761}

## Other properties {#other-properties-585}

+---------------+------------------+
| **Aliases:**  | > specifications |
+---------------+------------------+
| **Mappings:** | > mixs:specific  |
+---------------+------------------+

# Slot: specific_ecosystem

TODO

URI:
[nmdc:specific_ecosystem](https://microbiomedata/meta/specific_ecosystem)

## Domain and Range {#domain-and-range-762}

None → 0..1 [String](types/String.md)

## Parents {#parents-799}

-   is_a: [gold_path_field](gold_path_field.md)

## Children {#children-769}

## Used by {#used-by-762}

-   [Biosample](Biosample.md)
-   [Study](Study.md)

# Slot: specific_host

"If there is a host involved, please provide its taxid (or environmental
if not actually isolated from the dead or alive host - i.e. a pathogen
could be isolated from a swipe of a bench etc) and report whether it is
a laboratory or natural host)"

URI: [nmdc:specific_host](https://microbiomedata/meta/specific_host)

## Domain and Range {#domain-and-range-763}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-800}

-   is_a: [attribute](attribute.md)

## Children {#children-770}

## Used by {#used-by-763}

## Other properties {#other-properties-586}

+-----------------+--------------------------------+
| **Aliases:**    | > specific host                |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:specific_host           |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: specific_humidity

"The mass of water vapour in a unit mass of moist air, usually expressed
as grams of vapour per kilogram of air, or, in air conditioning, as
grains per pound."

URI:
[nmdc:specific_humidity](https://microbiomedata/meta/specific_humidity)

## Domain and Range {#domain-and-range-764}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-801}

-   is_a: [attribute](attribute.md)

## Children {#children-771}

## Used by {#used-by-764}

## Other properties {#other-properties-587}

+---------------+--------------------------+
| **Aliases:**  | > specific humidity      |
+---------------+--------------------------+
| **Mappings:** | > mixs:specific_humidity |
+---------------+--------------------------+

# Slot: sr_dep_env

"Source rock depositional environment
(<https://en.wikipedia.org/wiki/Source_rock>). If""other"" is specified,
please propose entry in ""additional info"" field"

URI: [nmdc:sr_dep_env](https://microbiomedata/meta/sr_dep_env)

## Domain and Range {#domain-and-range-765}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-802}

-   is_a: [attribute](attribute.md)

## Children {#children-772}

## Used by {#used-by-765}

## Other properties {#other-properties-588}

+---------------+----------------------------------------+
| **Aliases:**  | > source rock depositional environment |
+---------------+----------------------------------------+
| **Mappings:** | > mixs:sr_dep_env                      |
+---------------+----------------------------------------+

# Slot: sr_geol_age

"Geological age of source rock (Additional info:
<https://en.wikipedia.org/wiki/Period_(geology)>). If""other"" is
specified, please propose entry in ""additional info"" field"

URI: [nmdc:sr_geol_age](https://microbiomedata/meta/sr_geol_age)

## Domain and Range {#domain-and-range-766}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-803}

-   is_a: [attribute](attribute.md)

## Children {#children-773}

## Used by {#used-by-766}

## Other properties {#other-properties-589}

+---------------+------------------------------+
| **Aliases:**  | > source rock geological age |
+---------------+------------------------------+
| **Mappings:** | > mixs:sr_geol_age           |
+---------------+------------------------------+

# Slot: sr_kerog_type

"Origin of kerogen. Type I: Algal (aquatic), Type II: planktonic and
soft plant material (aquatic or terrestrial), Type III: terrestrial
woody/ fibrous plant material (terrestrial), Type IV: oxidized recycled
woody debris (terrestrial) (additional information:
<https://en.wikipedia.org/wiki/Kerogen>). If""other"" is specified,
please propose entry in ""additional info"" field"

URI: [nmdc:sr_kerog_type](https://microbiomedata/meta/sr_kerog_type)

## Domain and Range {#domain-and-range-767}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-804}

-   is_a: [attribute](attribute.md)

## Children {#children-774}

## Used by {#used-by-767}

## Other properties {#other-properties-590}

+---------------+----------------------------+
| **Aliases:**  | > source rock kerogen type |
+---------------+----------------------------+
| **Mappings:** | > mixs:sr_kerog_type       |
+---------------+----------------------------+

# Slot: sr_lithology

"Lithology of source rock (<https://en.wikipedia.org/wiki/Source_rock>).
If""other"" is specified, please propose entry in ""additional info""
field"

URI: [nmdc:sr_lithology](https://microbiomedata/meta/sr_lithology)

## Domain and Range {#domain-and-range-768}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-805}

-   is_a: [attribute](attribute.md)

## Children {#children-775}

## Used by {#used-by-768}

## Other properties {#other-properties-591}

+---------------+-------------------------+
| **Aliases:**  | > source rock lithology |
+---------------+-------------------------+
| **Mappings:** | > mixs:sr_lithology     |
+---------------+-------------------------+

# Slot: standing_water_regm

"Treatment involving an exposure to standing water during a plant's life
span, types can be flood water or standing water, treatment regimen
including how many times the treatment was repeated, how long each
treatment lasted, and the start and end time of the entire treatment;
can include multiple regimens"

URI:
[nmdc:standing_water_regm](https://microbiomedata/meta/standing_water_regm)

## Domain and Range {#domain-and-range-769}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-806}

-   is_a: [attribute](attribute.md)

## Children {#children-776}

## Used by {#used-by-769}

## Other properties {#other-properties-592}

+---------------+----------------------------+
| **Aliases:**  | > standing water regimen   |
+---------------+----------------------------+
| **Mappings:** | > mixs:standing_water_regm |
+---------------+----------------------------+

# Slot: start

The start of the feature in positive 1-based integer coordinates

URI: [nmdc:start](https://microbiomedata/meta/start)

## Domain and Range {#domain-and-range-770}

None → 1..1 [Integer](types/Integer.md)

## Parents {#parents-807}

-   is_a: [gff coordinate](gff_coordinate.md)

## Children {#children-777}

## Used by {#used-by-770}

## Other properties {#other-properties-593}

  **Close Mappings:**    biolink:start_interbase_coordinate
  ---------------------- ------------------------------------
                         
  Slot: started at tim   e

URI:
<https://microbiomedata/schema/workflow_execution_activity/started_at_time>

## Domain and Range {#domain-and-range-771}

None → 0..1 [String](types/String.md)

## Parents {#parents-808}

## Children {#children-778}

-   [workflow execution activity➞started at
    time](workflow_execution_activity_started_at_time.md)

## Used by {#used-by-771}

-   [Activity](Activity.md)

## Other properties {#other-properties-594}

  **Mappings:**    prov:startedAtTime
  ---------------- --------------------
                   
                   hiometry-1:
  Slot: stoichio   metry

URI: [nmdc:stoichiometry](https://microbiomedata/meta/stoichiometry)

## Domain and Range {#domain-and-range-772}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-809}

## Children {#children-779}

-   [reaction
    participant➞stoichiometry](reaction_participant_stoichiometry.md)

## Used by {#used-by-772}

# Slot: store_cond

Explain how and for how long the soil sample was stored before DNA
extraction

URI: [nmdc:store_cond](https://microbiomedata/meta/store_cond)

## Domain and Range {#domain-and-range-773}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-810}

-   is_a: [attribute](attribute.md)

## Children {#children-780}

## Used by {#used-by-773}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-595}

+---------------+-------------------------------------------+
| **Aliases:**  | > storage conditions (fresh/frozen/other) |
+---------------+-------------------------------------------+
| **Mappings:** | > mixs:store_cond                         |
+---------------+-------------------------------------------+

# Slot: strand {#slot-strand-1}

The strand on which a feature is located. Has a value of '+' (sense
strand or forward strand) or '-' (anti-sense strand or reverse strand).

URI: [nmdc:strand](https://microbiomedata/meta/strand)

## Domain and Range {#domain-and-range-774}

None → 0..1 [String](types/String.md)

## Parents {#parents-811}

## Children {#children-781}

-   [genome feature➞strand](genome_feature_strand.md)

## Used by {#used-by-774}

## Other properties {#other-properties-596}

  **Exact Mappings:**    biolink:strand
  ---------------------- ----------------
                         
  Slot: study_complt_s   tat

"Specification of study completion status, if no the reason should be
specified"

URI:
[nmdc:study_complt_stat](https://microbiomedata/meta/study_complt_stat)

## Domain and Range {#domain-and-range-775}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-812}

-   is_a: [attribute](attribute.md)

## Children {#children-782}

## Used by {#used-by-775}

## Other properties {#other-properties-597}

+---------------+---------------------------+
| **Aliases:**  | > study completion status |
+---------------+---------------------------+
| **Mappings:** | > mixs:study_complt_stat  |
+---------------+---------------------------+

# Slot: doi

The dataset citation for this study

URI: [nmdc:study_doi](https://microbiomedata/meta/study_doi)

## Domain and Range {#domain-and-range-776}

[Study](Study.md) → 0..1 [AttributeValue](AttributeValue.md)

## Parents {#parents-813}

-   is_a: [doi](doi.md)

## Children {#children-783}

## Used by {#used-by-776}

-   [Study](Study.md)

# Slot: study identifiers

URI:
<https://microbiomedata/schema/workflow_execution_activity/study_identifiers>

## Domain and Range {#domain-and-range-777}

None → 0..\* [ExternalIdentifier](types/ExternalIdentifier.md)

## Parents {#parents-814}

-   is_a: [external database
    identifiers](external_database_identifiers.md)

## Children {#children-784}

-   [GOLD study identifiers](GOLD_study_identifiers.md)
-   [INSDC SRA ENA study
    identifiers](INSDC_SRA_ENA_study_identifiers.md)
-   [INSDC bioproject identifiers](INSDC_bioproject_identifiers.md)
-   [MGnify project identifiers](MGnify_project_identifiers.md)

## Used by {#used-by-777}

# Slot: study image

Links a study to one or more images.

URI: [nmdc:study_image](https://microbiomedata/meta/study_image)

## Domain and Range {#domain-and-range-778}

[Study](Study.md) → 0..\* [ImageValue](ImageValue.md)

## Parents {#parents-815}

## Children {#children-785}

## Used by {#used-by-778}

-   [Study](Study.md)

# Slot: study set

This property links a database object to the set of studies within it.

URI: [nmdc:study_set](https://microbiomedata/meta/study_set)

## Domain and Range {#domain-and-range-779}

[Database](Database.md) → 0..\* [Study](Study.md)

## Parents {#parents-816}

## Children {#children-786}

## Used by {#used-by-779}

-   [Database](Database.md)

# Slot: subject

URI: [nmdc:subject](https://microbiomedata/meta/subject)

## Domain and Range {#domain-and-range-780}

None → 0..1 [GeneProduct](GeneProduct.md)

## Parents {#parents-817}

## Children {#children-787}

## Used by {#used-by-780}

-   [FunctionalAnnotation](FunctionalAnnotation.md)

# Slot: submitted_to_insdc

"Depending on the study (large-scale e.g. done with next generation
sequencing technology, or small-scale) sequences have to be submitted to
SRA (Sequence Read Archive), DRA (DDBJ Read Archive) or via the
classical Webin/Sequin systems to Genbank, ENA and DDBJ. Although this
field is mandatory, it is meant as a self-test field, therefore it is
not necessary to include this field in contextual data submitted to
databases"

URI:
[nmdc:submitted_to_insdc](https://microbiomedata/meta/submitted_to_insdc)

## Domain and Range {#domain-and-range-781}

None → 0..1 [BooleanValue](BooleanValue.md)

## Parents {#parents-818}

-   is_a: [attribute](attribute.md)

## Children {#children-788}

## Used by {#used-by-781}

## Other properties {#other-properties-598}

+-----------------+---------------------------+
| **Aliases:**    | > submitted to insdc      |
+-----------------+---------------------------+
| **Mappings:**   | > mixs:submitted_to_insdc |
+-----------------+---------------------------+
| **In Subsets:** | > investigation           |
+-----------------+---------------------------+

# Slot: subspecf_gen_lin

"This should provide further information about the genetic distinctness
of the sequenced organism by recording additional information
e.g. serovar, serotype, biotype, ecotype, or any relevant genetic typing
schemes like Group I plasmid. It can also contain alternative taxonomic
information. It should contain both the lineage name, and the lineage
rank, i.e. biovar:abc123"

URI:
[nmdc:subspecf_gen_lin](https://microbiomedata/meta/subspecf_gen_lin)

## Domain and Range {#domain-and-range-782}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-819}

-   is_a: [attribute](attribute.md)

## Children {#children-789}

## Used by {#used-by-782}

## Other properties {#other-properties-599}

+-----------------+--------------------------------+
| **Aliases:**    | > subspecific genetic lineage  |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:subspecf_gen_lin        |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: substructure_type

The substructure or under building is that largely hidden section of the
building which is built off the foundations to the ground floor level

URI:
[nmdc:substructure_type](https://microbiomedata/meta/substructure_type)

## Domain and Range {#domain-and-range-783}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-820}

-   is_a: [attribute](attribute.md)

## Children {#children-790}

## Used by {#used-by-783}

## Other properties {#other-properties-600}

+---------------+--------------------------+
| **Aliases:**  | > substructure type      |
+---------------+--------------------------+
| **Mappings:** | > mixs:substructure_type |
+---------------+--------------------------+

# Slot: subsurface_depth

URI:
[nmdc:subsurface_depth](https://microbiomedata/meta/subsurface_depth)

## Domain and Range {#domain-and-range-784}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-821}

## Children {#children-791}

## Used by {#used-by-784}

-   [Biosample](Biosample.md)

# Slot: subsurface_depth2

URI:
[nmdc:subsurface_depth2](https://microbiomedata/meta/subsurface_depth2)

## Domain and Range {#domain-and-range-785}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-822}

## Children {#children-792}

## Used by {#used-by-785}

-   [Biosample](Biosample.md)

# Slot: sulfate

Concentration of sulfate in the sample

URI: [nmdc:sulfate](https://microbiomedata/meta/sulfate)

## Domain and Range {#domain-and-range-786}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-823}

-   is_a: [attribute](attribute.md)

## Children {#children-793}

## Used by {#used-by-786}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-601}

+---------------+----------------+
| **Aliases:**  | > sulfate      |
+---------------+----------------+
| **Mappings:** | > mixs:sulfate |
+---------------+----------------+

# Slot: sulfate_fw

Original sulfate concentration in the hydrocarbon resource

URI: [nmdc:sulfate_fw](https://microbiomedata/meta/sulfate_fw)

## Domain and Range {#domain-and-range-787}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-824}

-   is_a: [attribute](attribute.md)

## Children {#children-794}

## Used by {#used-by-787}

## Other properties {#other-properties-602}

+---------------+------------------------------+
| **Aliases:**  | > sulfate in formation water |
+---------------+------------------------------+
| **Mappings:** | > mixs:sulfate_fw            |
+---------------+------------------------------+

# Slot: sulfide

Concentration of sulfide in the sample

URI: [nmdc:sulfide](https://microbiomedata/meta/sulfide)

## Domain and Range {#domain-and-range-788}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-825}

-   is_a: [attribute](attribute.md)

## Children {#children-795}

## Used by {#used-by-788}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-603}

+---------------+----------------+
| **Aliases:**  | > sulfide      |
+---------------+----------------+
| **Mappings:** | > mixs:sulfide |
+---------------+----------------+

# Slot: surf_air_cont

Contaminant identified on surface

URI: [nmdc:surf_air_cont](https://microbiomedata/meta/surf_air_cont)

## Domain and Range {#domain-and-range-789}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-826}

-   is_a: [attribute](attribute.md)

## Children {#children-796}

## Used by {#used-by-789}

## Other properties {#other-properties-604}

+---------------+---------------------------+
| **Aliases:**  | > surface-air contaminant |
+---------------+---------------------------+
| **Mappings:** | > mixs:surf_air_cont      |
+---------------+---------------------------+

# Slot: surf_humidity

Surfaces: water activity as a function of air and material moisture

URI: [nmdc:surf_humidity](https://microbiomedata/meta/surf_humidity)

## Domain and Range {#domain-and-range-790}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-827}

-   is_a: [attribute](attribute.md)

## Children {#children-797}

## Used by {#used-by-790}

## Other properties {#other-properties-605}

+---------------+----------------------+
| **Aliases:**  | > surface humidity   |
+---------------+----------------------+
| **Mappings:** | > mixs:surf_humidity |
+---------------+----------------------+

# Slot: surf_material

Surface materials at the point of sampling

URI: [nmdc:surf_material](https://microbiomedata/meta/surf_material)

## Domain and Range {#domain-and-range-791}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-828}

-   is_a: [attribute](attribute.md)

## Children {#children-798}

## Used by {#used-by-791}

## Other properties {#other-properties-606}

+---------------+----------------------+
| **Aliases:**  | > surface material   |
+---------------+----------------------+
| **Mappings:** | > mixs:surf_material |
+---------------+----------------------+

# Slot: surf_moisture

Water held on a surface

URI: [nmdc:surf_moisture](https://microbiomedata/meta/surf_moisture)

## Domain and Range {#domain-and-range-792}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-829}

-   is_a: [attribute](attribute.md)

## Children {#children-799}

## Used by {#used-by-792}

## Other properties {#other-properties-607}

+---------------+----------------------+
| **Aliases:**  | > surface moisture   |
+---------------+----------------------+
| **Mappings:** | > mixs:surf_moisture |
+---------------+----------------------+

# Slot: surf_moisture_ph

ph measurement of surface

URI:
[nmdc:surf_moisture_ph](https://microbiomedata/meta/surf_moisture_ph)

## Domain and Range {#domain-and-range-793}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-830}

-   is_a: [attribute](attribute.md)

## Children {#children-800}

## Used by {#used-by-793}

## Other properties {#other-properties-608}

+---------------+-------------------------+
| **Aliases:**  | > surface moisture pH   |
+---------------+-------------------------+
| **Mappings:** | > mixs:surf_moisture_ph |
+---------------+-------------------------+

# Slot: surf_temp

Temperature of the surface at the time of sampling

URI: [nmdc:surf_temp](https://microbiomedata/meta/surf_temp)

## Domain and Range {#domain-and-range-794}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-831}

-   is_a: [attribute](attribute.md)

## Children {#children-801}

## Used by {#used-by-794}

## Other properties {#other-properties-609}

+---------------+-----------------------+
| **Aliases:**  | > surface temperature |
+---------------+-----------------------+
| **Mappings:** | > mixs:surf_temp      |
+---------------+-----------------------+

# Slot: suspend_part_matter

Concentration of suspended particulate matter

URI:
[nmdc:suspend_part_matter](https://microbiomedata/meta/suspend_part_matter)

## Domain and Range {#domain-and-range-795}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-832}

-   is_a: [attribute](attribute.md)

## Children {#children-802}

## Used by {#used-by-795}

## Other properties {#other-properties-610}

+---------------+--------------------------------+
| **Aliases:**  | > suspended particulate matter |
+---------------+--------------------------------+
| **Mappings:** | > mixs:suspend_part_matter     |
+---------------+--------------------------------+

# Slot: suspend_solids

"Concentration of substances including a wide variety of material, such
as silt, decaying plant and animal matter; can include multiple
substances"

URI: [nmdc:suspend_solids](https://microbiomedata/meta/suspend_solids)

## Domain and Range {#domain-and-range-796}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-833}

-   is_a: [attribute](attribute.md)

## Children {#children-803}

## Used by {#used-by-796}

## Other properties {#other-properties-611}

+---------------+-----------------------+
| **Aliases:**  | > suspended solids    |
+---------------+-----------------------+
| **Mappings:** | > mixs:suspend_solids |
+---------------+-----------------------+

# Slot: tan

Total Acid NumberB (TAN) is a measurement of acidity that is determined
by the amount ofB potassium hydroxideB in milligrams that is needed to
neutralize the acids in one gram of oil.B It is an important quality
measurement ofB crude oil. (source:
<https://en.wikipedia.org/wiki/Total_acid_number>)

URI: [nmdc:tan](https://microbiomedata/meta/tan)

## Domain and Range {#domain-and-range-797}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-834}

-   is_a: [attribute](attribute.md)

## Children {#children-804}

## Used by {#used-by-797}

## Other properties {#other-properties-612}

+---------------+---------------------+
| **Aliases:**  | > total acid number |
+---------------+---------------------+
| **Mappings:** | > mixs:tan          |
+---------------+---------------------+

# Slot: target_gene

Targeted gene or locus name for marker gene studies

URI: [nmdc:target_gene](https://microbiomedata/meta/target_gene)

## Domain and Range {#domain-and-range-798}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-835}

-   is_a: [attribute](attribute.md)

## Children {#children-805}

## Used by {#used-by-798}

-   [OmicsProcessing](OmicsProcessing.md)

## Other properties {#other-properties-613}

+-----------------+--------------------+
| **Aliases:**    | > target gene      |
+-----------------+--------------------+
| **Mappings:**   | > mixs:target_gene |
+-----------------+--------------------+
| **In Subsets:** | > sequencing       |
+-----------------+--------------------+

# Slot: target_subfragment

Name of subfragment of a gene or locus. Important to e.g. identify
special regions on marker genes like V6 on 16S rRNA

URI:
[nmdc:target_subfragment](https://microbiomedata/meta/target_subfragment)

## Domain and Range {#domain-and-range-799}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-836}

-   is_a: [attribute](attribute.md)

## Children {#children-806}

## Used by {#used-by-799}

-   [OmicsProcessing](OmicsProcessing.md)

## Other properties {#other-properties-614}

+-----------------+---------------------------+
| **Aliases:**    | > target subfragment      |
+-----------------+---------------------------+
| **Mappings:**   | > mixs:target_subfragment |
+-----------------+---------------------------+
| **In Subsets:** | > sequencing              |
+-----------------+---------------------------+

# Slot: tax_class

"Method used for taxonomic classification, along with reference database
used, classification rank, and thresholds used to classify new genomes"

URI: [nmdc:tax_class](https://microbiomedata/meta/tax_class)

## Domain and Range {#domain-and-range-800}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-837}

-   is_a: [attribute](attribute.md)

## Children {#children-807}

## Used by {#used-by-800}

## Other properties {#other-properties-615}

+-----------------+----------------------------+
| **Aliases:**    | > taxonomic classification |
+-----------------+----------------------------+
| **Mappings:**   | > mixs:tax_class           |
+-----------------+----------------------------+
| **In Subsets:** | > sequencing               |
+-----------------+----------------------------+

# Slot: tax_ident

The phylogenetic marker(s) used to assign an organism name to the SAG or
MAG

URI: [nmdc:tax_ident](https://microbiomedata/meta/tax_ident)

## Domain and Range {#domain-and-range-801}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-838}

-   is_a: [attribute](attribute.md)

## Children {#children-808}

## Used by {#used-by-801}

## Other properties {#other-properties-616}

+-----------------+-----------------------------+
| **Aliases:**    | > taxonomic identity marker |
+-----------------+-----------------------------+
| **Mappings:**   | > mixs:tax_ident            |
+-----------------+-----------------------------+
| **In Subsets:** | > sequencing                |
+-----------------+-----------------------------+

# Slot: temp

Temperature of the sample at the time of sampling

URI: [nmdc:temp](https://microbiomedata/meta/temp)

## Domain and Range {#domain-and-range-802}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-839}

-   is_a: [attribute](attribute.md)

## Children {#children-809}

## Used by {#used-by-802}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-617}

+---------------+---------------+
| **Aliases:**  | > temperature |
+---------------+---------------+
| **Mappings:** | > mixs:temp   |
+---------------+---------------+

# Slot: temp_out

The recorded temperature value at sampling time outside

URI: [nmdc:temp_out](https://microbiomedata/meta/temp_out)

## Domain and Range {#domain-and-range-803}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-840}

-   is_a: [attribute](attribute.md)

## Children {#children-810}

## Used by {#used-by-803}

## Other properties {#other-properties-618}

+---------------+-----------------------------+
| **Aliases:**  | > temperature outside house |
+---------------+-----------------------------+
| **Mappings:** | > mixs:temp_out             |
+---------------+-----------------------------+

# Slot: term

pointer to an ontology class

URI: <https://microbiomedata/schema/workflow_execution_activity/term>

## Domain and Range {#domain-and-range-804}

[ControlledTermValue](ControlledTermValue.md) → 0..1
[OntologyClass](OntologyClass.md)

## Parents {#parents-841}

## Children {#children-811}

## Used by {#used-by-804}

-   [ControlledTermValue](ControlledTermValue.md)

## Other properties {#other-properties-619}

  **Mappings:**    rdf:type
  ---------------- -------------
                   
  Slot: tertiary   \_treatment

The process providing a final treatment stage to raise the effluent
quality before it is discharged to the receiving environment

URI:
[nmdc:tertiary_treatment](https://microbiomedata/meta/tertiary_treatment)

## Domain and Range {#domain-and-range-805}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-842}

-   is_a: [attribute](attribute.md)

## Children {#children-812}

## Used by {#used-by-805}

## Other properties {#other-properties-620}

+---------------+---------------------------+
| **Aliases:**  | > tertiary treatment      |
+---------------+---------------------------+
| **Mappings:** | > mixs:tertiary_treatment |
+---------------+---------------------------+

# Slot: texture

"The relative proportion of different grain sizes of mineral particles
in a soil, as described using a standard system; express as % sand (50
um to 2 mm), silt (2 um to 50 um), and clay (\<2 um) with textural name
(e.g., silty clay loam) optional."

URI: [nmdc:texture](https://microbiomedata/meta/texture)

## Domain and Range {#domain-and-range-806}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-843}

-   is_a: [attribute](attribute.md)

## Children {#children-813}

## Used by {#used-by-806}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-621}

+---------------+----------------+
| **Aliases:**  | > texture      |
+---------------+----------------+
| **Mappings:** | > mixs:texture |
+---------------+----------------+

# Slot: texture_meth

Reference or method used in determining soil texture

URI: [nmdc:texture_meth](https://microbiomedata/meta/texture_meth)

## Domain and Range {#domain-and-range-807}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-844}

-   is_a: [attribute](attribute.md)

## Children {#children-814}

## Used by {#used-by-807}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-622}

+---------------+---------------------+
| **Aliases:**  | > texture method    |
+---------------+---------------------+
| **Mappings:** | > mixs:texture_meth |
+---------------+---------------------+

# Slot: tidal_stage

Stage of tide

URI: [nmdc:tidal_stage](https://microbiomedata/meta/tidal_stage)

## Domain and Range {#domain-and-range-808}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-845}

-   is_a: [attribute](attribute.md)

## Children {#children-815}

## Used by {#used-by-808}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-623}

+---------------+--------------------+
| **Aliases:**  | > tidal stage      |
+---------------+--------------------+
| **Mappings:** | > mixs:tidal_stage |
+---------------+--------------------+

# Slot: tillage

Note method(s) used for tilling

URI: [nmdc:tillage](https://microbiomedata/meta/tillage)

## Domain and Range {#domain-and-range-809}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-846}

-   is_a: [attribute](attribute.md)

## Children {#children-816}

## Used by {#used-by-809}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-624}

+---------------+-------------------+
| **Aliases:**  | > history/tillage |
+---------------+-------------------+
| **Mappings:** | > mixs:tillage    |
+---------------+-------------------+

# Slot: time_last_toothbrush

Specification of the time since last toothbrushing

URI:
[nmdc:time_last_toothbrush](https://microbiomedata/meta/time_last_toothbrush)

## Domain and Range {#domain-and-range-810}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-847}

-   is_a: [attribute](attribute.md)

## Children {#children-817}

## Used by {#used-by-810}

## Other properties {#other-properties-625}

+---------------+---------------------------------+
| **Aliases:**  | > time since last toothbrushing |
+---------------+---------------------------------+
| **Mappings:** | > mixs:time_last_toothbrush     |
+---------------+---------------------------------+

# Slot: time_since_last_wash

Specification of the time since last wash

URI:
[nmdc:time_since_last_wash](https://microbiomedata/meta/time_since_last_wash)

## Domain and Range {#domain-and-range-811}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-848}

-   is_a: [attribute](attribute.md)

## Children {#children-818}

## Used by {#used-by-811}

## Other properties {#other-properties-626}

+---------------+-----------------------------+
| **Aliases:**  | > time since last wash      |
+---------------+-----------------------------+
| **Mappings:** | > mixs:time_since_last_wash |
+---------------+-----------------------------+

# Slot: tiss_cult_growth_med

Description of plant tissue culture growth media used

URI:
[nmdc:tiss_cult_growth_med](https://microbiomedata/meta/tiss_cult_growth_med)

## Domain and Range {#domain-and-range-812}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-849}

-   is_a: [attribute](attribute.md)

## Children {#children-819}

## Used by {#used-by-812}

## Other properties {#other-properties-627}

+---------------+-------------------------------+
| **Aliases:**  | > tissue culture growth media |
+---------------+-------------------------------+
| **Mappings:** | > mixs:tiss_cult_growth_med   |
+---------------+-------------------------------+

# Slot: title

A name given to the entity that differs from the name/label
programatically assigned to it. For example, when extracting study
information for GOLD, the GOLD system has assigned a name/label.
However, for display purposes, we may also wish the capture the title of
the proposal that was used to fund the study.

URI: <https://microbiomedata/schema/workflow_execution_activity/title>

## Domain and Range {#domain-and-range-813}

None → 0..1 [String](types/String.md)

## Parents {#parents-850}

## Children {#children-820}

## Used by {#used-by-813}

## Other properties {#other-properties-628}

+----------------------------------+----------------------------------+
| **Exact Mappings:**              | dcterms:title                    |
+==================================+==================================+
| Slot: toluene =============      |                                  |
+----------------------------------+----------------------------------+
| Concentration of tol             | uene in the sample               |
+----------------------------------+----------------------------------+
| URI: \`nmdc:toluene \<           | <https://micr                    |
|                                  | obiomedata/meta/toluene>\>\`\_\_ |
+----------------------------------+----------------------------------+
| ## Domain a                      | -814:                            |
| nd Range {#domain-and-range-812} |                                  |
+----------------------------------+----------------------------------+
| None → 0..1 \`Quantit            | yValue                           |
|                                  | \<QuantityValue.md\>\`\_\_       |
| ## Parents {#parents-851}        |                                  |
+----------------------------------+----------------------------------+
| -   is_a: \`attribute            | \<attribute.md\>\`\_\_           |
|                                  |                                  |
| ## Children {#children-821}      |                                  |
|                                  |                                  |
| ## Used by {#used-by-814}        |                                  |
+----------------------------------+----------------------------------+
| ## Other pr                      | -629:                            |
| operties {#other-properties-627} |                                  |
+----------------------------------+----------------------------------+
| ============= ======             | ======                           |
+----------------------------------+----------------------------------+
| **Aliases:** tolue               | ne                               |
+----------------------------------+----------------------------------+
| **Mappings:** mixs:              | toluene                          |
+----------------------------------+----------------------------------+

# Slot: too short contig num

URI:
<https://microbiomedata/schema/workflow_execution_activity/too_short_contig_num>

## Domain and Range {#domain-and-range-815}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-852}

## Children {#children-822}

## Used by {#used-by-815}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)

# Slot: tot_carb

Total carbon content

URI: [nmdc:tot_carb](https://microbiomedata/meta/tot_carb)

## Domain and Range {#domain-and-range-816}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-853}

-   is_a: [attribute](attribute.md)

## Children {#children-823}

## Used by {#used-by-816}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-630}

+---------------+-----------------+
| **Aliases:**  | > total carbon  |
+---------------+-----------------+
| **Mappings:** | > mixs:tot_carb |
+---------------+-----------------+

# Slot: tot_depth_water_col

Measurement of total depth of water column

URI:
[nmdc:tot_depth_water_col](https://microbiomedata/meta/tot_depth_water_col)

## Domain and Range {#domain-and-range-817}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-854}

-   is_a: [attribute](attribute.md)

## Children {#children-824}

## Used by {#used-by-817}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-631}

+---------------+-------------------------------+
| **Aliases:**  | > total depth of water column |
+---------------+-------------------------------+
| **Mappings:** | > mixs:tot_depth_water_col    |
+---------------+-------------------------------+

# Slot: tot_diss_nitro

"Total dissolved nitrogen concentration, reported as nitrogen, measured
by: total dissolved nitrogen = NH4 + NO3NO2 + dissolved organic
nitrogen"

URI: [nmdc:tot_diss_nitro](https://microbiomedata/meta/tot_diss_nitro)

## Domain and Range {#domain-and-range-818}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-855}

-   is_a: [attribute](attribute.md)

## Children {#children-825}

## Used by {#used-by-818}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-632}

+---------------+----------------------------+
| **Aliases:**  | > total dissolved nitrogen |
+---------------+----------------------------+
| **Mappings:** | > mixs:tot_diss_nitro      |
+---------------+----------------------------+

# Slot: tot_inorg_nitro

Total inorganic nitrogen content

URI: [nmdc:tot_inorg_nitro](https://microbiomedata/meta/tot_inorg_nitro)

## Domain and Range {#domain-and-range-819}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-856}

-   is_a: [attribute](attribute.md)

## Children {#children-826}

## Used by {#used-by-819}

## Other properties {#other-properties-633}

+---------------+----------------------------+
| **Aliases:**  | > total inorganic nitrogen |
+---------------+----------------------------+
| **Mappings:** | > mixs:tot_inorg_nitro     |
+---------------+----------------------------+

# Slot: tot_iron

Concentration of total iron in the sample

URI: [nmdc:tot_iron](https://microbiomedata/meta/tot_iron)

## Domain and Range {#domain-and-range-820}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-857}

-   is_a: [attribute](attribute.md)

## Children {#children-827}

## Used by {#used-by-820}

## Other properties {#other-properties-634}

+---------------+-----------------+
| **Aliases:**  | > total iron    |
+---------------+-----------------+
| **Mappings:** | > mixs:tot_iron |
+---------------+-----------------+

# Slot: tot_nitro

"Total nitrogen concentration of water samples, calculated by: total
nitrogen = total dissolved nitrogen + particulate nitrogen. Can also be
measured without filtering, reported as nitrogen"

URI: [nmdc:tot_nitro](https://microbiomedata/meta/tot_nitro)

## Domain and Range {#domain-and-range-821}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-858}

-   is_a: [attribute](attribute.md)

## Children {#children-828}

## Used by {#used-by-821}

## Other properties {#other-properties-635}

+---------------+--------------------------------+
| **Aliases:**  | > total nitrogen concentration |
+---------------+--------------------------------+
| **Mappings:** | > mixs:tot_nitro               |
+---------------+--------------------------------+

# Slot: tot_nitro_content

Total nitrogen content of the sample

URI:
[nmdc:tot_nitro_content](https://microbiomedata/meta/tot_nitro_content)

## Domain and Range {#domain-and-range-822}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-859}

-   is_a: [attribute](attribute.md)

## Children {#children-829}

## Used by {#used-by-822}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-636}

+---------------+--------------------------+
| **Aliases:**  | > total nitrogen content |
+---------------+--------------------------+
| **Mappings:** | > mixs:tot_nitro_content |
+---------------+--------------------------+

# Slot: tot_nitro_content_meth

Reference or method used in determining the total nitrogen

URI:
[nmdc:tot_nitro_content_meth](https://microbiomedata/meta/tot_nitro_content_meth)

## Domain and Range {#domain-and-range-823}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-860}

-   is_a: [attribute](attribute.md)

## Children {#children-830}

## Used by {#used-by-823}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-637}

+---------------+---------------------------------+
| **Aliases:**  | > total nitrogen content method |
+---------------+---------------------------------+
| **Mappings:** | > mixs:tot_nitro_content_meth   |
+---------------+---------------------------------+

# Slot: tot_org_c\_meth

Reference or method used in determining total organic carbon

URI: [nmdc:tot_org_c\_meth](https://microbiomedata/meta/tot_org_c_meth)

## Domain and Range {#domain-and-range-824}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-861}

-   is_a: [attribute](attribute.md)

## Children {#children-831}

## Used by {#used-by-824}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-638}

+---------------+-------------------------------+
| **Aliases:**  | > total organic carbon method |
+---------------+-------------------------------+
| **Mappings:** | > mixs:tot_org_c\_meth        |
+---------------+-------------------------------+

# Slot: tot_org_carb

"Definition for soil: total organic carbon content of the soil,
definition otherwise: total organic carbon content"

URI: [nmdc:tot_org_carb](https://microbiomedata/meta/tot_org_carb)

## Domain and Range {#domain-and-range-825}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-862}

-   is_a: [attribute](attribute.md)

## Children {#children-832}

## Used by {#used-by-825}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-639}

+---------------+------------------------+
| **Aliases:**  | > total organic carbon |
+---------------+------------------------+
| **Mappings:** | > mixs:tot_org_carb    |
+---------------+------------------------+

# Slot: tot_part_carb

Total particulate carbon content

URI: [nmdc:tot_part_carb](https://microbiomedata/meta/tot_part_carb)

## Domain and Range {#domain-and-range-826}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-863}

-   is_a: [attribute](attribute.md)

## Children {#children-833}

## Used by {#used-by-826}

## Other properties {#other-properties-640}

+---------------+----------------------------+
| **Aliases:**  | > total particulate carbon |
+---------------+----------------------------+
| **Mappings:** | > mixs:tot_part_carb       |
+---------------+----------------------------+

# Slot: tot_phosp

"Total phosphorus concentration in the sample, calculated by: total
phosphorus = total dissolved phosphorus + particulate phosphorus"

URI: [nmdc:tot_phosp](https://microbiomedata/meta/tot_phosp)

## Domain and Range {#domain-and-range-827}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-864}

-   is_a: [attribute](attribute.md)

## Children {#children-834}

## Used by {#used-by-827}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-641}

+---------------+--------------------+
| **Aliases:**  | > total phosphorus |
+---------------+--------------------+
| **Mappings:** | > mixs:tot_phosp   |
+---------------+--------------------+

# Slot: tot_phosphate

Total amount or concentration of phosphate

URI: [nmdc:tot_phosphate](https://microbiomedata/meta/tot_phosphate)

## Domain and Range {#domain-and-range-828}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-865}

-   is_a: [attribute](attribute.md)

## Children {#children-835}

## Used by {#used-by-828}

## Other properties {#other-properties-642}

+---------------+----------------------+
| **Aliases:**  | > total phosphate    |
+---------------+----------------------+
| **Mappings:** | > mixs:tot_phosphate |
+---------------+----------------------+

# Slot: tot_sulfur

Concentration of total sulfur in the sample

URI: [nmdc:tot_sulfur](https://microbiomedata/meta/tot_sulfur)

## Domain and Range {#domain-and-range-829}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-866}

-   is_a: [attribute](attribute.md)

## Children {#children-836}

## Used by {#used-by-829}

## Other properties {#other-properties-643}

+---------------+-------------------+
| **Aliases:**  | > total sulfur    |
+---------------+-------------------+
| **Mappings:** | > mixs:tot_sulfur |
+---------------+-------------------+

# Slot: train_line

The subway line name

URI: [nmdc:train_line](https://microbiomedata/meta/train_line)

## Domain and Range {#domain-and-range-830}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-867}

-   is_a: [attribute](attribute.md)

## Children {#children-837}

## Used by {#used-by-830}

## Other properties {#other-properties-644}

+---------------+-------------------+
| **Aliases:**  | > train line      |
+---------------+-------------------+
| **Mappings:** | > mixs:train_line |
+---------------+-------------------+

# Slot: train_stat_loc

The train station collection location

URI: [nmdc:train_stat_loc](https://microbiomedata/meta/train_stat_loc)

## Domain and Range {#domain-and-range-831}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-868}

-   is_a: [attribute](attribute.md)

## Children {#children-838}

## Used by {#used-by-831}

## Other properties {#other-properties-645}

+---------------+-------------------------------------+
| **Aliases:**  | > train station collection location |
+---------------+-------------------------------------+
| **Mappings:** | > mixs:train_stat_loc               |
+---------------+-------------------------------------+

# Slot: train_stop_loc

The train stop collection location

URI: [nmdc:train_stop_loc](https://microbiomedata/meta/train_stop_loc)

## Domain and Range {#domain-and-range-832}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-869}

-   is_a: [attribute](attribute.md)

## Children {#children-839}

## Used by {#used-by-832}

## Other properties {#other-properties-646}

+---------------+----------------------------------+
| **Aliases:**  | > train stop collection location |
+---------------+----------------------------------+
| **Mappings:** | > mixs:train_stop_loc            |
+---------------+----------------------------------+

# Slot: travel_out_six_month

Specification of the countries travelled in the last six months; can
include multiple travels

URI:
[nmdc:travel_out_six_month](https://microbiomedata/meta/travel_out_six_month)

## Domain and Range {#domain-and-range-833}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-870}

-   is_a: [attribute](attribute.md)

## Children {#children-840}

## Used by {#used-by-833}

## Other properties {#other-properties-647}

+---------------+-------------------------------------------------+
| **Aliases:**  | > travel outside the country in last six months |
+---------------+-------------------------------------------------+
| **Mappings:** | > mixs:travel_out_six_month                     |
+---------------+-------------------------------------------------+

# Slot: trna_ext_software

Tools used for tRNA identification

URI:
[nmdc:trna_ext_software](https://microbiomedata/meta/trna_ext_software)

## Domain and Range {#domain-and-range-834}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-871}

-   is_a: [attribute](attribute.md)

## Children {#children-841}

## Used by {#used-by-834}

## Other properties {#other-properties-648}

+-----------------+----------------------------+
| **Aliases:**    | > tRNA extraction software |
+-----------------+----------------------------+
| **Mappings:**   | > mixs:trna_ext_software   |
+-----------------+----------------------------+
| **In Subsets:** | > sequencing               |
+-----------------+----------------------------+

# Slot: trnas

The total number of tRNAs identified from the SAG or MAG

URI: [nmdc:trnas](https://microbiomedata/meta/trnas)

## Domain and Range {#domain-and-range-835}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-872}

-   is_a: [attribute](attribute.md)

## Children {#children-842}

## Used by {#used-by-835}

## Other properties {#other-properties-649}

+-----------------+--------------------------------------+
| **Aliases:**    | > number of standard tRNAs extracted |
+-----------------+--------------------------------------+
| **Mappings:**   | > mixs:trnas                         |
+-----------------+--------------------------------------+
| **In Subsets:** | > sequencing                         |
+-----------------+--------------------------------------+

# Slot: trophic_level

Trophic levels are the feeding position in a food chain. Microbes can be
a range of producers (e.g. chemolithotroph)

URI: [nmdc:trophic_level](https://microbiomedata/meta/trophic_level)

## Domain and Range {#domain-and-range-836}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-873}

-   is_a: [attribute](attribute.md)

## Children {#children-843}

## Used by {#used-by-836}

## Other properties {#other-properties-650}

+-----------------+--------------------------------+
| **Aliases:**    | > trophic level                |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:trophic_level           |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: turbidity

Measure of the amount of cloudiness or haziness in water caused by
individual particles

URI: [nmdc:turbidity](https://microbiomedata/meta/turbidity)

## Domain and Range {#domain-and-range-837}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-874}

-   is_a: [attribute](attribute.md)

## Children {#children-844}

## Used by {#used-by-837}

## Other properties {#other-properties-651}

+---------------+------------------+
| **Aliases:**  | > turbidity      |
+---------------+------------------+
| **Mappings:** | > mixs:turbidity |
+---------------+------------------+

# Slot: tvdss_of_hcr_pressure

True vertical depth subsea (TVDSS) of the hydrocarbon resource where the
original pressure was measured (e.g. 1578 m )

URI:
[nmdc:tvdss_of_hcr_pressure](https://microbiomedata/meta/tvdss_of_hcr_pressure)

## Domain and Range {#domain-and-range-838}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-875}

-   is_a: [attribute](attribute.md)

## Children {#children-845}

## Used by {#used-by-838}

## Other properties {#other-properties-652}

+---------------+--------------------------------------------------+
| **Aliases:**  | > depth (TVDSS) of hydrocarbon resource pressure |
+---------------+--------------------------------------------------+
| **Mappings:** | > mixs:tvdss_of_hcr_pressure                     |
+---------------+--------------------------------------------------+

# Slot: tvdss_of_hcr_temp

True vertical depth subsea (TVDSS) of the hydrocarbon resource where the
original temperature was measured (e.g. 1345 m)

URI:
[nmdc:tvdss_of_hcr_temp](https://microbiomedata/meta/tvdss_of_hcr_temp)

## Domain and Range {#domain-and-range-839}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-876}

-   is_a: [attribute](attribute.md)

## Children {#children-846}

## Used by {#used-by-839}

## Other properties {#other-properties-653}

+---------------+-----------------------------------------------------+
| **Aliases:**  | > depth (TVDSS) of hydrocarbon resource temperature |
+---------------+-----------------------------------------------------+
| **Mappings:** | > mixs:tvdss_of_hcr_temp                            |
+---------------+-----------------------------------------------------+

# Slot: twin_sibling

Specification of twin sibling presence

URI: [nmdc:twin_sibling](https://microbiomedata/meta/twin_sibling)

## Domain and Range {#domain-and-range-840}

None → 0..1 [BooleanValue](BooleanValue.md)

## Parents {#parents-877}

-   is_a: [attribute](attribute.md)

## Children {#children-847}

## Used by {#used-by-840}

## Other properties {#other-properties-654}

+---------------+-------------------------+
| **Aliases:**  | > twin sibling presence |
+---------------+-------------------------+
| **Mappings:** | > mixs:twin_sibling     |
+---------------+-------------------------+

# Slot: typ_occup_density

Customary or normal density of occupants

URI:
[nmdc:typ_occup_density](https://microbiomedata/meta/typ_occup_density)

## Domain and Range {#domain-and-range-841}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-878}

-   is_a: [attribute](attribute.md)

## Children {#children-848}

## Used by {#used-by-841}

## Other properties {#other-properties-655}

+---------------+----------------------------+
| **Aliases:**  | > typical occupant density |
+---------------+----------------------------+
| **Mappings:** | > mixs:typ_occup_density   |
+---------------+----------------------------+

# Slot: type

An optional string that specifies the type object. This is used to allow
for searches for different kinds of objects.

URI: <https://microbiomedata/schema/workflow_execution_activity/type>

## Domain and Range {#domain-and-range-842}

None → 0..1 [String](types/String.md)

## Parents {#parents-879}

## Children {#children-849}

-   [attribute value➞type](attribute_value_type.md)
-   [workflow execution
    activity➞type](workflow_execution_activity_type.md)

## Used by {#used-by-842}

-   [MAGBin](MAGBin.md)

## Other properties {#other-properties-656}

+---------------+-----------------------------------------------------+
| **Examples:** | > Example(value='nmdc:Biosample', description=None) |
+---------------+-----------------------------------------------------+
|               | > Example(value='nmdc:Study', description=None)     |
+---------------+-----------------------------------------------------+

# Slot: unbinned contig num

URI:
<https://microbiomedata/schema/workflow_execution_activity/unbinned_contig_num>

## Domain and Range {#domain-and-range-843}

None → 0..1 [Integer](types/Integer.md)

## Parents {#parents-880}

## Children {#children-850}

## Used by {#used-by-843}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)

# Slot: urine_collect_meth

Specification of urine collection method

URI:
[nmdc:urine_collect_meth](https://microbiomedata/meta/urine_collect_meth)

## Domain and Range {#domain-and-range-844}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-881}

-   is_a: [attribute](attribute.md)

## Children {#children-851}

## Used by {#used-by-844}

## Other properties {#other-properties-657}

+---------------+---------------------------+
| **Aliases:**  | > urine/collection method |
+---------------+---------------------------+
| **Mappings:** | > mixs:urine_collect_meth |
+---------------+---------------------------+

# Slot: url

URI: <https://microbiomedata/schema/workflow_execution_activity/url>

## Domain and Range {#domain-and-range-845}

None → 0..1 [String](types/String.md)

## Parents {#parents-882}

-   is_a: [attribute](attribute.md)

## Children {#children-852}

## Used by {#used-by-845}

-   [ImageValue](ImageValue.md)

# Slot: urogenit_disord

"History of urogenital disorders, can include multiple disorders"

URI: [nmdc:urogenit_disord](https://microbiomedata/meta/urogenit_disord)

## Domain and Range {#domain-and-range-846}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-883}

-   is_a: [attribute](attribute.md)

## Children {#children-853}

## Used by {#used-by-846}

## Other properties {#other-properties-658}

+---------------+------------------------+
| **Aliases:**  | > urogenital disorder  |
+---------------+------------------------+
| **Mappings:** | > mixs:urogenit_disord |
+---------------+------------------------+

# Slot: urogenit_tract_disor

History of urogenitaltract disorders; can include multiple disorders

URI:
[nmdc:urogenit_tract_disor](https://microbiomedata/meta/urogenit_tract_disor)

## Domain and Range {#domain-and-range-847}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-884}

-   is_a: [attribute](attribute.md)

## Children {#children-854}

## Used by {#used-by-847}

## Other properties {#other-properties-659}

+---------------+-----------------------------------+
| **Aliases:**  | > urine/urogenital tract disorder |
+---------------+-----------------------------------+
| **Mappings:** | > mixs:urogenit_tract_disor       |
+---------------+-----------------------------------+

# Slot: used

URI: <https://microbiomedata/schema/workflow_execution_activity/used>

## Domain and Range {#domain-and-range-848}

[Activity](Activity.md) → 0..1 [String](types/String.md)

## Parents {#parents-885}

## Children {#children-855}

-   [metabolomics analysis
    activity➞used](metabolomics_analysis_activity_used.md)
-   [metaproteomics analysis
    activity➞used](metaproteomics_analysis_activity_used.md)
-   [nom analysis activity➞used](nom_analysis_activity_used.md)

## Used by {#used-by-848}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [Activity](Activity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md)
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)

## Other properties {#other-properties-660}

  **Mappings:**    prov:used
  ---------------- -----------
                   
  Slot: ventilat   ion_rate

Ventilation rate of the system in the sampled premises

URI:
[nmdc:ventilation_rate](https://microbiomedata/meta/ventilation_rate)

## Domain and Range {#domain-and-range-849}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-886}

-   is_a: [attribute](attribute.md)

## Children {#children-856}

## Used by {#used-by-849}

## Other properties {#other-properties-661}

+---------------+-------------------------+
| **Aliases:**  | > ventilation rate      |
+---------------+-------------------------+
| **Mappings:** | > mixs:ventilation_rate |
+---------------+-------------------------+

# Slot: ventilation_type

Ventilation system used in the sampled premises

URI:
[nmdc:ventilation_type](https://microbiomedata/meta/ventilation_type)

## Domain and Range {#domain-and-range-850}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-887}

-   is_a: [attribute](attribute.md)

## Children {#children-857}

## Used by {#used-by-850}

## Other properties {#other-properties-662}

+---------------+-------------------------+
| **Aliases:**  | > ventilation type      |
+---------------+-------------------------+
| **Mappings:** | > mixs:ventilation_type |
+---------------+-------------------------+

# Slot: vfa

Concentration of Volatile Fatty Acids in the sample

URI: [nmdc:vfa](https://microbiomedata/meta/vfa)

## Domain and Range {#domain-and-range-851}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-888}

-   is_a: [attribute](attribute.md)

## Children {#children-858}

## Used by {#used-by-851}

## Other properties {#other-properties-663}

+---------------+------------------------+
| **Aliases:**  | > volatile fatty acids |
+---------------+------------------------+
| **Mappings:** | > mixs:vfa             |
+---------------+------------------------+

# Slot: vfa_fw

Original volatile fatty acid concentration in the hydrocarbon resource

URI: [nmdc:vfa_fw](https://microbiomedata/meta/vfa_fw)

## Domain and Range {#domain-and-range-852}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-889}

-   is_a: [attribute](attribute.md)

## Children {#children-859}

## Used by {#used-by-852}

## Other properties {#other-properties-664}

+---------------+--------------------------+
| **Aliases:**  | > vfa in formation water |
+---------------+--------------------------+
| **Mappings:** | > mixs:vfa_fw            |
+---------------+--------------------------+

# Slot: vir_ident_software

"Tool(s) used for the identification of UViG as a viral genome, software
or protocol name including version number, parameters, and cutoffs used"

URI:
[nmdc:vir_ident_software](https://microbiomedata/meta/vir_ident_software)

## Domain and Range {#domain-and-range-853}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-890}

-   is_a: [attribute](attribute.md)

## Children {#children-860}

## Used by {#used-by-853}

## Other properties {#other-properties-665}

+-----------------+---------------------------------+
| **Aliases:**    | > viral identification software |
+-----------------+---------------------------------+
| **Mappings:**   | > mixs:vir_ident_software       |
+-----------------+---------------------------------+
| **In Subsets:** | > sequencing                    |
+-----------------+---------------------------------+

# Slot: virus_enrich_appr

"List of approaches used to enrich the sample for viruses, if any"

URI:
[nmdc:virus_enrich_appr](https://microbiomedata/meta/virus_enrich_appr)

## Domain and Range {#domain-and-range-854}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-891}

-   is_a: [attribute](attribute.md)

## Children {#children-861}

## Used by {#used-by-854}

## Other properties {#other-properties-666}

+-----------------+--------------------------------+
| **Aliases:**    | > virus enrichment approach    |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:virus_enrich_appr       |
+-----------------+--------------------------------+
| **In Subsets:** | > nucleic acid sequence source |
+-----------------+--------------------------------+

# Slot: vis_media

The building visual media

URI: [nmdc:vis_media](https://microbiomedata/meta/vis_media)

## Domain and Range {#domain-and-range-855}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-892}

-   is_a: [attribute](attribute.md)

## Children {#children-862}

## Used by {#used-by-855}

## Other properties {#other-properties-667}

+---------------+------------------+
| **Aliases:**  | > visual media   |
+---------------+------------------+
| **Mappings:** | > mixs:vis_media |
+---------------+------------------+

# Slot: viscosity

A measure of oil's resistanceB to gradual deformation byB shear stressB
orB tensile stress (e.g. 3.5 cp; 100 B0C)

URI: [nmdc:viscosity](https://microbiomedata/meta/viscosity)

## Domain and Range {#domain-and-range-856}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-893}

-   is_a: [attribute](attribute.md)

## Children {#children-863}

## Used by {#used-by-856}

## Other properties {#other-properties-668}

+---------------+------------------+
| **Aliases:**  | > viscosity      |
+---------------+------------------+
| **Mappings:** | > mixs:viscosity |
+---------------+------------------+

# Slot: volatile_org_comp

Concentration of carbon-based chemicals that easily evaporate at room
temperature; can report multiple volatile organic compounds by entering
numeric values preceded by name of compound

URI:
[nmdc:volatile_org_comp](https://microbiomedata/meta/volatile_org_comp)

## Domain and Range {#domain-and-range-857}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-894}

-   is_a: [attribute](attribute.md)

## Children {#children-864}

## Used by {#used-by-857}

## Other properties {#other-properties-669}

+---------------+------------------------------+
| **Aliases:**  | > volatile organic compounds |
+---------------+------------------------------+
| **Mappings:** | > mixs:volatile_org_comp     |
+---------------+------------------------------+

# Slot: votu_class_appr

"Cutoffs and approach used when clustering new UViGs in Rspecies-levelS
vOTUs. Note that results from standard 95% ANI / 85% AF clustering
should be provided alongside vOTUS defined from another set of
thresholds, even if the latter are the ones primarily used during the
analysis"

URI: [nmdc:votu_class_appr](https://microbiomedata/meta/votu_class_appr)

## Domain and Range {#domain-and-range-858}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-895}

-   is_a: [attribute](attribute.md)

## Children {#children-865}

## Used by {#used-by-858}

## Other properties {#other-properties-670}

+-----------------+--------------------------------+
| **Aliases:**    | > vOTU classification approach |
+-----------------+--------------------------------+
| **Mappings:**   | > mixs:votu_class_appr         |
+-----------------+--------------------------------+
| **In Subsets:** | > sequencing                   |
+-----------------+--------------------------------+

# Slot: votu_db

"Reference database (i.e. sequences not generated as part of the current
study) used to cluster new genomes in""species-level"" vOTUs, if any"

URI: [nmdc:votu_db](https://microbiomedata/meta/votu_db)

## Domain and Range {#domain-and-range-859}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-896}

-   is_a: [attribute](attribute.md)

## Children {#children-866}

## Used by {#used-by-859}

## Other properties {#other-properties-671}

+-----------------+-----------------+
| **Aliases:**    | > vOTU database |
+-----------------+-----------------+
| **Mappings:**   | > mixs:votu_db  |
+-----------------+-----------------+
| **In Subsets:** | > sequencing    |
+-----------------+-----------------+

# Slot: votu_seq_comp_appr

"Tool and thresholds used to compare sequences when
computing""species-level"" vOTUs"

URI:
[nmdc:votu_seq_comp_appr](https://microbiomedata/meta/votu_seq_comp_appr)

## Domain and Range {#domain-and-range-860}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-897}

-   is_a: [attribute](attribute.md)

## Children {#children-867}

## Used by {#used-by-860}

## Other properties {#other-properties-672}

+-----------------+-------------------------------------+
| **Aliases:**    | > vOTU sequence comparison approach |
+-----------------+-------------------------------------+
| **Mappings:**   | > mixs:votu_seq_comp_appr           |
+-----------------+-------------------------------------+
| **In Subsets:** | > sequencing                        |
+-----------------+-------------------------------------+

# Slot: wall_area

The total area of the sampled room's walls

URI: [nmdc:wall_area](https://microbiomedata/meta/wall_area)

## Domain and Range {#domain-and-range-861}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-898}

-   is_a: [attribute](attribute.md)

## Children {#children-868}

## Used by {#used-by-861}

## Other properties {#other-properties-673}

+---------------+------------------+
| **Aliases:**  | > wall area      |
+---------------+------------------+
| **Mappings:** | > mixs:wall_area |
+---------------+------------------+

# Slot: wall_const_type

The building class of the wall defined by the composition of the
building elements and fire-resistance rating.

URI: [nmdc:wall_const_type](https://microbiomedata/meta/wall_const_type)

## Domain and Range {#domain-and-range-862}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-899}

-   is_a: [attribute](attribute.md)

## Children {#children-869}

## Used by {#used-by-862}

## Other properties {#other-properties-674}

+---------------+--------------------------+
| **Aliases:**  | > wall construction type |
+---------------+--------------------------+
| **Mappings:** | > mixs:wall_const_type   |
+---------------+--------------------------+

# Slot: wall_finish_mat

The material utilized to finish the outer most layer of the wall

URI: [nmdc:wall_finish_mat](https://microbiomedata/meta/wall_finish_mat)

## Domain and Range {#domain-and-range-863}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-900}

-   is_a: [attribute](attribute.md)

## Children {#children-870}

## Used by {#used-by-863}

## Other properties {#other-properties-675}

+---------------+------------------------+
| **Aliases:**  | > wall finish material |
+---------------+------------------------+
| **Mappings:** | > mixs:wall_finish_mat |
+---------------+------------------------+

# Slot: wall_height

The average height of the walls in the sampled room

URI: [nmdc:wall_height](https://microbiomedata/meta/wall_height)

## Domain and Range {#domain-and-range-864}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-901}

-   is_a: [attribute](attribute.md)

## Children {#children-871}

## Used by {#used-by-864}

## Other properties {#other-properties-676}

+---------------+--------------------+
| **Aliases:**  | > wall height      |
+---------------+--------------------+
| **Mappings:** | > mixs:wall_height |
+---------------+--------------------+

# Slot: wall_loc

The relative location of the wall within the room

URI: [nmdc:wall_loc](https://microbiomedata/meta/wall_loc)

## Domain and Range {#domain-and-range-865}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-902}

-   is_a: [attribute](attribute.md)

## Children {#children-872}

## Used by {#used-by-865}

## Other properties {#other-properties-677}

+---------------+-----------------+
| **Aliases:**  | > wall location |
+---------------+-----------------+
| **Mappings:** | > mixs:wall_loc |
+---------------+-----------------+

# Slot: wall_surf_treatment

The surface treatment of interior wall

URI:
[nmdc:wall_surf_treatment](https://microbiomedata/meta/wall_surf_treatment)

## Domain and Range {#domain-and-range-866}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-903}

-   is_a: [attribute](attribute.md)

## Children {#children-873}

## Used by {#used-by-866}

## Other properties {#other-properties-678}

+---------------+----------------------------+
| **Aliases:**  | > wall surface treatment   |
+---------------+----------------------------+
| **Mappings:** | > mixs:wall_surf_treatment |
+---------------+----------------------------+

# Slot: wall_texture

"The feel, appearance, or consistency of a wall surface"

URI: [nmdc:wall_texture](https://microbiomedata/meta/wall_texture)

## Domain and Range {#domain-and-range-867}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-904}

-   is_a: [attribute](attribute.md)

## Children {#children-874}

## Used by {#used-by-867}

## Other properties {#other-properties-679}

+---------------+---------------------+
| **Aliases:**  | > wall texture      |
+---------------+---------------------+
| **Mappings:** | > mixs:wall_texture |
+---------------+---------------------+

# Slot: wall_thermal_mass

The ability of the wall to provide inertia against temperature
fluctuations. Generally this means concrete or concrete block that is
either exposed or covered only with paint

URI:
[nmdc:wall_thermal_mass](https://microbiomedata/meta/wall_thermal_mass)

## Domain and Range {#domain-and-range-868}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-905}

-   is_a: [attribute](attribute.md)

## Children {#children-875}

## Used by {#used-by-868}

## Other properties {#other-properties-680}

+---------------+--------------------------+
| **Aliases:**  | > wall thermal mass      |
+---------------+--------------------------+
| **Mappings:** | > mixs:wall_thermal_mass |
+---------------+--------------------------+

# Slot: wall_water_mold

Signs of the presence of mold or mildew on a wall

URI: [nmdc:wall_water_mold](https://microbiomedata/meta/wall_water_mold)

## Domain and Range {#domain-and-range-869}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-906}

-   is_a: [attribute](attribute.md)

## Children {#children-876}

## Used by {#used-by-869}

## Other properties {#other-properties-681}

+---------------+----------------------------+
| **Aliases:**  | > wall signs of water/mold |
+---------------+----------------------------+
| **Mappings:** | > mixs:wall_water_mold     |
+---------------+----------------------------+

# Slot: was associated with

URI:
<https://microbiomedata/schema/workflow_execution_activity/was_associated_with>

## Domain and Range {#domain-and-range-870}

None → 0..1 [Agent](Agent.md)

## Parents {#parents-907}

## Children {#children-877}

-   [workflow execution activity➞was associated
    with](workflow_execution_activity_was_associated_with.md)

## Used by {#used-by-870}

-   [Activity](Activity.md)

## Other properties {#other-properties-682}

  **Mappings:**    prov:wasAssociatedWith
  ---------------- ------------------------
                   
                   enerated-by-1:
  Slot: was gene   rated by

URI:
<https://microbiomedata/schema/workflow_execution_activity/was_generated_by>

## Domain and Range {#domain-and-range-871}

None → 0..1 [Activity](Activity.md)

## Parents {#parents-908}

## Children {#children-878}

## Used by {#used-by-871}

-   [AttributeValue](AttributeValue.md)
-   [BooleanValue](BooleanValue.md)
-   [ControlledTermValue](ControlledTermValue.md)
-   [GeolocationValue](GeolocationValue.md)
-   [ImageValue](ImageValue.md)
-   [IntegerValue](IntegerValue.md)
-   [PersonValue](PersonValue.md)
-   [QuantityValue](QuantityValue.md)
-   [TextValue](TextValue.md)
-   [TimestampValue](TimestampValue.md)
-   [UrlValue](UrlValue.md)

## Other properties {#other-properties-683}

  **Mappings:**    prov:wasGeneratedBy
  ---------------- ---------------------
                   
  Slot: was info   rmed by

URI:
<https://microbiomedata/schema/workflow_execution_activity/was_informed_by>

## Domain and Range {#domain-and-range-872}

None → 0..1 [Activity](Activity.md)

## Parents {#parents-909}

## Children {#children-879}

-   [workflow execution activity➞was informed
    by](workflow_execution_activity_was_informed_by.md)

## Used by {#used-by-872}

-   [Activity](Activity.md)
-   [Agent](Agent.md)

## Other properties {#other-properties-684}

  **Mappings:**    prov:wasInformedBy
  ---------------- --------------------
                   
  Slot: wastewat   er_type

"The origin of wastewater such as human waste, rainfall, storm drains,
etc."

URI: [nmdc:wastewater_type](https://microbiomedata/meta/wastewater_type)

## Domain and Range {#domain-and-range-873}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-910}

-   is_a: [attribute](attribute.md)

## Children {#children-880}

## Used by {#used-by-873}

## Other properties {#other-properties-685}

+---------------+------------------------+
| **Aliases:**  | > wastewater type      |
+---------------+------------------------+
| **Mappings:** | > mixs:wastewater_type |
+---------------+------------------------+

# Slot: water_content

Water content measurement

URI: [nmdc:water_content](https://microbiomedata/meta/water_content)

## Domain and Range {#domain-and-range-874}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-911}

-   is_a: [attribute](attribute.md)

## Children {#children-881}

## Used by {#used-by-874}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-686}

+---------------+----------------------+
| **Aliases:**  | > water content      |
+---------------+----------------------+
| **Mappings:** | > mixs:water_content |
+---------------+----------------------+

# Slot: water_content_soil_meth

Reference or method used in determining the water content of soil

URI:
[nmdc:water_content_soil_meth](https://microbiomedata/meta/water_content_soil_meth)

## Domain and Range {#domain-and-range-875}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-912}

-   is_a: [attribute](attribute.md)

## Children {#children-882}

## Used by {#used-by-875}

-   [Biosample](Biosample.md)

## Other properties {#other-properties-687}

+---------------+--------------------------------+
| **Aliases:**  | > water content method         |
+---------------+--------------------------------+
| **Mappings:** | > mixs:water_content_soil_meth |
+---------------+--------------------------------+

# Slot: water_current

Measurement of magnitude and direction of flow within a fluid

URI: [nmdc:water_current](https://microbiomedata/meta/water_current)

## Domain and Range {#domain-and-range-876}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-913}

-   is_a: [attribute](attribute.md)

## Children {#children-883}

## Used by {#used-by-876}

## Other properties {#other-properties-688}

+---------------+----------------------+
| **Aliases:**  | > water current      |
+---------------+----------------------+
| **Mappings:** | > mixs:water_current |
+---------------+----------------------+

# Slot: water_cut

Current amount of water (%) in a produced fluid stream; or the average
of the combined streams

URI: [nmdc:water_cut](https://microbiomedata/meta/water_cut)

## Domain and Range {#domain-and-range-877}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-914}

-   is_a: [attribute](attribute.md)

## Children {#children-884}

## Used by {#used-by-877}

## Other properties {#other-properties-689}

+---------------+------------------+
| **Aliases:**  | > water cut      |
+---------------+------------------+
| **Mappings:** | > mixs:water_cut |
+---------------+------------------+

# Slot: water_feat_size

The size of the water feature

URI: [nmdc:water_feat_size](https://microbiomedata/meta/water_feat_size)

## Domain and Range {#domain-and-range-878}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-915}

-   is_a: [attribute](attribute.md)

## Children {#children-885}

## Used by {#used-by-878}

## Other properties {#other-properties-690}

+---------------+------------------------+
| **Aliases:**  | > water feature size   |
+---------------+------------------------+
| **Mappings:** | > mixs:water_feat_size |
+---------------+------------------------+

# Slot: water_feat_type

The type of water feature present within the building being sampled

URI: [nmdc:water_feat_type](https://microbiomedata/meta/water_feat_type)

## Domain and Range {#domain-and-range-879}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-916}

-   is_a: [attribute](attribute.md)

## Children {#children-886}

## Used by {#used-by-879}

## Other properties {#other-properties-691}

+---------------+------------------------+
| **Aliases:**  | > water feature type   |
+---------------+------------------------+
| **Mappings:** | > mixs:water_feat_type |
+---------------+------------------------+

# Slot: water_production_rate

Water production rates per well (e.g. 987 m3 / day)

URI:
[nmdc:water_production_rate](https://microbiomedata/meta/water_production_rate)

## Domain and Range {#domain-and-range-880}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-917}

-   is_a: [attribute](attribute.md)

## Children {#children-887}

## Used by {#used-by-880}

## Other properties {#other-properties-692}

+---------------+------------------------------+
| **Aliases:**  | > water production rate      |
+---------------+------------------------------+
| **Mappings:** | > mixs:water_production_rate |
+---------------+------------------------------+

# Slot: water_temp_regm

"Information about treatment involving an exposure to water with varying
degree of temperature, treatment regimen including how many times the
treatment was repeated, how long each treatment lasted, and the start
and end time of the entire treatment; can include multiple regimens"

URI: [nmdc:water_temp_regm](https://microbiomedata/meta/water_temp_regm)

## Domain and Range {#domain-and-range-881}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-918}

-   is_a: [attribute](attribute.md)

## Children {#children-888}

## Used by {#used-by-881}

## Other properties {#other-properties-693}

+---------------+-----------------------------+
| **Aliases:**  | > water temperature regimen |
+---------------+-----------------------------+
| **Mappings:** | > mixs:water_temp_regm      |
+---------------+-----------------------------+

# Slot: watering_regm

"Information about treatment involving an exposure to watering
frequencies, treatment regimen including how many times the treatment
was repeated, how long each treatment lasted, and the start and end time
of the entire treatment; can include multiple regimens"

URI: [nmdc:watering_regm](https://microbiomedata/meta/watering_regm)

## Domain and Range {#domain-and-range-882}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-919}

-   is_a: [attribute](attribute.md)

## Children {#children-889}

## Used by {#used-by-882}

## Other properties {#other-properties-694}

+---------------+----------------------+
| **Aliases:**  | > watering regimen   |
+---------------+----------------------+
| **Mappings:** | > mixs:watering_regm |
+---------------+----------------------+

# Slot: websites

A list of websites that are assocatiated with the entity.

URI:
<https://microbiomedata/schema/workflow_execution_activity/websites>

## Domain and Range {#domain-and-range-883}

None → 0..\* [String](types/String.md)

## Parents {#parents-920}

## Children {#children-890}

## Used by {#used-by-883}

-   [PersonValue](PersonValue.md)

# Slot: weekday

The day of the week when sampling occurred

URI: [nmdc:weekday](https://microbiomedata/meta/weekday)

## Domain and Range {#domain-and-range-884}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-921}

-   is_a: [attribute](attribute.md)

## Children {#children-891}

## Used by {#used-by-884}

## Other properties {#other-properties-695}

+---------------+----------------+
| **Aliases:**  | > weekday      |
+---------------+----------------+
| **Mappings:** | > mixs:weekday |
+---------------+----------------+

# Slot: weight_loss_3\_month

"Specification of weight loss in the last three months, if yes should be
further specified to include amount of weight loss"

URI:
[nmdc:weight_loss_3\_month](https://microbiomedata/meta/weight_loss_3_month)

## Domain and Range {#domain-and-range-885}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-922}

-   is_a: [attribute](attribute.md)

## Children {#children-892}

## Used by {#used-by-885}

## Other properties {#other-properties-696}

+---------------+------------------------------------+
| **Aliases:**  | > weight loss in last three months |
+---------------+------------------------------------+
| **Mappings:** | > mixs:weight_loss_3\_month        |
+---------------+------------------------------------+

# Slot: wga_amp_appr

Method used to amplify genomic DNA in preparation for sequencing

URI: [nmdc:wga_amp_appr](https://microbiomedata/meta/wga_amp_appr)

## Domain and Range {#domain-and-range-886}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-923}

-   is_a: [attribute](attribute.md)

## Children {#children-893}

## Used by {#used-by-886}

## Other properties {#other-properties-697}

+-----------------+------------------------------+
| **Aliases:**    | > WGA amplification approach |
+-----------------+------------------------------+
| **Mappings:**   | > mixs:wga_amp_appr          |
+-----------------+------------------------------+
| **In Subsets:** | > sequencing                 |
+-----------------+------------------------------+

# Slot: wga_amp_kit

Kit used to amplify genomic DNA in preparation for sequencing

URI: [nmdc:wga_amp_kit](https://microbiomedata/meta/wga_amp_kit)

## Domain and Range {#domain-and-range-887}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-924}

-   is_a: [attribute](attribute.md)

## Children {#children-894}

## Used by {#used-by-887}

## Other properties {#other-properties-698}

+-----------------+-------------------------+
| **Aliases:**    | > WGA amplification kit |
+-----------------+-------------------------+
| **Mappings:**   | > mixs:wga_amp_kit      |
+-----------------+-------------------------+
| **In Subsets:** | > sequencing            |
+-----------------+-------------------------+

# Slot: win

A unique identifier of a well or wellbore. This is part of the Global
Framework for Well Identification initiative which is compiled by the
Professional Petroleum Data Management Association (PPDM) in an effort
to improve well identification systems. (Supporting information:
<https://ppdm.org/> and <http://dl.ppdm.org/dl/690>)

URI: [nmdc:win](https://microbiomedata/meta/win)

## Domain and Range {#domain-and-range-888}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-925}

-   is_a: [attribute](attribute.md)

## Children {#children-895}

## Used by {#used-by-888}

## Other properties {#other-properties-699}

+---------------+------------------------------+
| **Aliases:**  | > well identification number |
+---------------+------------------------------+
| **Mappings:** | > mixs:win                   |
+---------------+------------------------------+

# Slot: wind_direction

Wind direction is the direction from which a wind originates

URI: [nmdc:wind_direction](https://microbiomedata/meta/wind_direction)

## Domain and Range {#domain-and-range-889}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-926}

-   is_a: [attribute](attribute.md)

## Children {#children-896}

## Used by {#used-by-889}

## Other properties {#other-properties-700}

+---------------+-----------------------+
| **Aliases:**  | > wind direction      |
+---------------+-----------------------+
| **Mappings:** | > mixs:wind_direction |
+---------------+-----------------------+

# Slot: wind_speed

Speed of wind measured at the time of sampling

URI: [nmdc:wind_speed](https://microbiomedata/meta/wind_speed)

## Domain and Range {#domain-and-range-890}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-927}

-   is_a: [attribute](attribute.md)

## Children {#children-897}

## Used by {#used-by-890}

## Other properties {#other-properties-701}

+---------------+-------------------+
| **Aliases:**  | > wind speed      |
+---------------+-------------------+
| **Mappings:** | > mixs:wind_speed |
+---------------+-------------------+

# Slot: window_cond

The physical condition of the window at the time of sampling

URI: [nmdc:window_cond](https://microbiomedata/meta/window_cond)

## Domain and Range {#domain-and-range-891}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-928}

-   is_a: [attribute](attribute.md)

## Children {#children-898}

## Used by {#used-by-891}

## Other properties {#other-properties-702}

+---------------+--------------------+
| **Aliases:**  | > window condition |
+---------------+--------------------+
| **Mappings:** | > mixs:window_cond |
+---------------+--------------------+

# Slot: window_cover

The type of window covering

URI: [nmdc:window_cover](https://microbiomedata/meta/window_cover)

## Domain and Range {#domain-and-range-892}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-929}

-   is_a: [attribute](attribute.md)

## Children {#children-899}

## Used by {#used-by-892}

## Other properties {#other-properties-703}

+---------------+---------------------+
| **Aliases:**  | > window covering   |
+---------------+---------------------+
| **Mappings:** | > mixs:window_cover |
+---------------+---------------------+

# Slot: window_horiz_pos

The horizontal position of the window on the wall

URI:
[nmdc:window_horiz_pos](https://microbiomedata/meta/window_horiz_pos)

## Domain and Range {#domain-and-range-893}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-930}

-   is_a: [attribute](attribute.md)

## Children {#children-900}

## Used by {#used-by-893}

## Other properties {#other-properties-704}

+---------------+------------------------------+
| **Aliases:**  | > window horizontal position |
+---------------+------------------------------+
| **Mappings:** | > mixs:window_horiz_pos      |
+---------------+------------------------------+

# Slot: window_loc

The relative location of the window within the room

URI: [nmdc:window_loc](https://microbiomedata/meta/window_loc)

## Domain and Range {#domain-and-range-894}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-931}

-   is_a: [attribute](attribute.md)

## Children {#children-901}

## Used by {#used-by-894}

## Other properties {#other-properties-705}

+---------------+-------------------+
| **Aliases:**  | > window location |
+---------------+-------------------+
| **Mappings:** | > mixs:window_loc |
+---------------+-------------------+

# Slot: window_mat

The type of material used to finish a window

URI: [nmdc:window_mat](https://microbiomedata/meta/window_mat)

## Domain and Range {#domain-and-range-895}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-932}

-   is_a: [attribute](attribute.md)

## Children {#children-902}

## Used by {#used-by-895}

## Other properties {#other-properties-706}

+---------------+-------------------+
| **Aliases:**  | > window material |
+---------------+-------------------+
| **Mappings:** | > mixs:window_mat |
+---------------+-------------------+

# Slot: window_open_freq

The number of times windows are opened per week

URI:
[nmdc:window_open_freq](https://microbiomedata/meta/window_open_freq)

## Domain and Range {#domain-and-range-896}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-933}

-   is_a: [attribute](attribute.md)

## Children {#children-903}

## Used by {#used-by-896}

## Other properties {#other-properties-707}

+---------------+-------------------------+
| **Aliases:**  | > window open frequency |
+---------------+-------------------------+
| **Mappings:** | > mixs:window_open_freq |
+---------------+-------------------------+

# Slot: window_size

The window's length and width

URI: [nmdc:window_size](https://microbiomedata/meta/window_size)

## Domain and Range {#domain-and-range-897}

None → 0..1 [QuantityValue](QuantityValue.md)

## Parents {#parents-934}

-   is_a: [attribute](attribute.md)

## Children {#children-904}

## Used by {#used-by-897}

## Other properties {#other-properties-708}

+---------------+--------------------+
| **Aliases:**  | > window area/size |
+---------------+--------------------+
| **Mappings:** | > mixs:window_size |
+---------------+--------------------+

# Slot: window_status

Defines whether the windows were open or closed during environmental
testing

URI: [nmdc:window_status](https://microbiomedata/meta/window_status)

## Domain and Range {#domain-and-range-898}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-935}

-   is_a: [attribute](attribute.md)

## Children {#children-905}

## Used by {#used-by-898}

## Other properties {#other-properties-709}

+---------------+----------------------+
| **Aliases:**  | > window status      |
+---------------+----------------------+
| **Mappings:** | > mixs:window_status |
+---------------+----------------------+

# Slot: window_type

The type of windows

URI: [nmdc:window_type](https://microbiomedata/meta/window_type)

## Domain and Range {#domain-and-range-899}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-936}

-   is_a: [attribute](attribute.md)

## Children {#children-906}

## Used by {#used-by-899}

## Other properties {#other-properties-710}

+---------------+--------------------+
| **Aliases:**  | > window type      |
+---------------+--------------------+
| **Mappings:** | > mixs:window_type |
+---------------+--------------------+

# Slot: window_vert_pos

The vertical position of the window on the wall

URI: [nmdc:window_vert_pos](https://microbiomedata/meta/window_vert_pos)

## Domain and Range {#domain-and-range-900}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-937}

-   is_a: [attribute](attribute.md)

## Children {#children-907}

## Used by {#used-by-900}

## Other properties {#other-properties-711}

+---------------+----------------------------+
| **Aliases:**  | > window vertical position |
+---------------+----------------------------+
| **Mappings:** | > mixs:window_vert_pos     |
+---------------+----------------------------+

# Slot: window_water_mold

Signs of the presence of mold or mildew on the window.

URI:
[nmdc:window_water_mold](https://microbiomedata/meta/window_water_mold)

## Domain and Range {#domain-and-range-901}

None → 0..1 [TextValue](TextValue.md)

## Parents {#parents-938}

-   is_a: [attribute](attribute.md)

## Children {#children-908}

## Used by {#used-by-901}

## Other properties {#other-properties-712}

+---------------+------------------------------+
| **Aliases:**  | > window signs of water/mold |
+---------------+------------------------------+
| **Mappings:** | > mixs:window_water_mold     |
+---------------+------------------------------+

# Slot: ended at time

URI:
<https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_ended_at_time>

## Domain and Range {#domain-and-range-902}

[WorkflowExecutionActivity](WorkflowExecutionActivity.md) → 1..1
[String](types/String.md)

## Parents {#parents-939}

-   is_a: [ended at time](ended_at_time.md)

## Children {#children-909}

## Used by {#used-by-902}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md)
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)

## Other properties {#other-properties-713}

  **Mappings:**    prov:endedAtTime
  ---------------- ------------------
                   
                   tion-resource-1:
  Slot: executio   n resource

Example: NERSC-Cori

URI:
<https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_execution_resource>

## Domain and Range {#domain-and-range-903}

[WorkflowExecutionActivity](WorkflowExecutionActivity.md) → 1..1
[String](types/String.md)

## Parents {#parents-940}

-   is_a: [execution resource](execution_resource.md)

## Children {#children-910}

## Used by {#used-by-903}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md)
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)

# Slot: git url

Example:
<https://github.com/microbiomedata/mg_annotation/releases/tag/0.1>

URI:
<https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_git_url>

## Domain and Range {#domain-and-range-904}

[WorkflowExecutionActivity](WorkflowExecutionActivity.md) → 1..1
[String](types/String.md)

## Parents {#parents-941}

-   is_a: [git url](git_url.md)

## Children {#children-911}

## Used by {#used-by-904}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md)
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)

# Slot: has input

An input to a process.

URI:
<https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_has_input>

## Domain and Range {#domain-and-range-905}

[WorkflowExecutionActivity](WorkflowExecutionActivity.md) → 1..\*
[NamedThing](NamedThing.md)

## Parents {#parents-942}

-   is_a: [has input](has_input.md)

## Children {#children-912}

-   [read QC analysis activity➞has
    input](read_QC_analysis_activity_has_input.md)

## Used by {#used-by-905}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md)
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)

## Other properties {#other-properties-714}

  **Aliases:**    input
  --------------- -----------
                  
                  output-2:
  Slot: has out   put

An output biosample to a processing step

URI:
<https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_has_output>

## Domain and Range {#domain-and-range-906}

[WorkflowExecutionActivity](WorkflowExecutionActivity.md) → 1..\*
[NamedThing](NamedThing.md)

## Parents {#parents-943}

-   is_a: [has output](has_output.md)

## Children {#children-913}

-   [read QC analysis activity➞has
    output](read_QC_analysis_activity_has_output.md)

## Used by {#used-by-906}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md)
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)

## Other properties {#other-properties-715}

+---------------+----------------+
| **Aliases:**  | output         |
+===============+================+
|               |                |
+---------------+----------------+
|               | ted-at-time-1: |
+---------------+----------------+
| Slot: started | > at time      |
+---------------+----------------+

URI:
<https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_started_at_time>

## Domain and Range {#domain-and-range-907}

[WorkflowExecutionActivity](WorkflowExecutionActivity.md) → 1..1
[String](types/String.md)

## Parents {#parents-944}

-   is_a: [started at time](started_at_time.md)

## Children {#children-914}

## Used by {#used-by-907}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md)
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)

## Other properties {#other-properties-716}

+----------------------------------+----------------------------------+
| **Mappings:**                    | prov:startedAtTime               |
+==================================+==================================+
|                                  |                                  |
+----------------------------------+----------------------------------+
| # Slot: type {#slot-type-4}      | 4:                               |
+----------------------------------+----------------------------------+
| An optional st                   | ring that specifies the type     |
|                                  | object. This is used to allow    |
+----------------------------------+----------------------------------+
| for searches f                   | or different kinds of objects.   |
|                                  |                                  |
| URI:                             |                                  |
+----------------------------------+----------------------------------+
| <https://microb>                 | iomedata/sch                     |
|                                  | ema/workflow_execution_activity/ |
|                                  | workflow_execution_activity_type |
+----------------------------------+----------------------------------+
|                                  | -range-908:                      |
+----------------------------------+----------------------------------+
| Domain and Ran                   | ge                               |
+----------------------------------+----------------------------------+
| ------------------------------   | \--                              |
+----------------------------------+----------------------------------+
| \`WorkflowExecu                  | tionActivity                     |
|                                  | \<Work                           |
|                                  | flowExecutionActivity.md\>\`\_\_ |
|                                  | → 1..1                           |
+----------------------------------+----------------------------------+
| \`String \<types                 | /String.md\>\`\_\_               |
+----------------------------------+----------------------------------+
| ## Parents {#parents-945}        | 5:                               |
+----------------------------------+----------------------------------+
| -   is_a: \`type                 | > \<type.md\>\`\_\_              |
+----------------------------------+----------------------------------+
| ## Children {#children-915}      | 15:                              |
+----------------------------------+----------------------------------+
| ## Used by {#used-by-908}        | 8:                               |
+----------------------------------+----------------------------------+
| \- \`MAGsAnalys                  | isActivity                       |
|                                  | \                                |
|                                  | <MAGsAnalysisActivity.md\>\`\_\_ |
+----------------------------------+----------------------------------+
| \- \`Metabolomi                  | csAnalysisActivity               |
|                                  | \<Metabol                        |
|                                  | omicsAnalysisActivity.md\>\`\_\_ |
+----------------------------------+----------------------------------+
| \- \`Metagenome                  | AnnotationActivity               |
|                                  | \<Metagen                        |
|                                  | omeAnnotationActivity.md\>\`\_\_ |
+----------------------------------+----------------------------------+
| \- \`Metagenome                  | Assembly                         |
|                                  | \<MetagenomeAssembly.md\>\`\_\_  |
+----------------------------------+----------------------------------+
| \- \`Metaproteo                  | micsAnalysisActivity             |
|                                  | \<Metaprote                      |
|                                  | omicsAnalysisActivity.md\>\`\_\_ |
+----------------------------------+----------------------------------+
| \- \`Metatransc                  | riptomeActivity                  |
|                                  | \<Meta                           |
|                                  | transcriptomeActivity.md\>\`\_\_ |
+----------------------------------+----------------------------------+
| \- \`Metatransc                  | riptomeAnnotationActivity        |
|                                  | \<Metatranscript                 |
|                                  | omeAnnotationActivity.md\>\`\_\_ |
+----------------------------------+----------------------------------+
| \- \`Metatransc                  | riptomeAssembly                  |
|                                  | \<Meta                           |
|                                  | transcriptomeAssembly.md\>\`\_\_ |
+----------------------------------+----------------------------------+
| \- \`NomAnalysi                  | sActivity                        |
|                                  | \<NomAnalysisActivity.md\>\`\_\_ |
+----------------------------------+----------------------------------+
| \- \`ReadQCAnal                  | ysisActivity                     |
|                                  | \<R                              |
|                                  | eadQCAnalysisActivity.md\>\`\_\_ |
+----------------------------------+----------------------------------+
| \- \`ReadBasedA                  | nalysisActivity                  |
|                                  | \<Read                           |
|                                  | BasedAnalysisActivity.md\>\`\_\_ |
+----------------------------------+----------------------------------+
| -   \`WorkflowEx                 | ecutionActivity                  |
|                                  | \<Work                           |
|                                  | flowExecutionActivity.md\>\`\_\_ |
+----------------------------------+----------------------------------+
|                                  | erties-717:                      |
+----------------------------------+----------------------------------+
| Other properti                   | es                               |
+----------------------------------+----------------------------------+
| ------------------------------   | \--                              |
+----------------------------------+----------------------------------+
| =============                    | =================                |
|                                  | ================================ |
+----------------------------------+----------------------------------+
| **Examples:**                    | >                                |
|                                  |  Example(value='nmdc:Biosample', |
|                                  | > description=None)              |
+----------------------------------+----------------------------------+
|                                  | > Example(value='nmdc:Study',    |
|                                  | > description=None)              |
+----------------------------------+----------------------------------+

# Slot: was associated with

the agent/entity associated with the generation of the file

URI:
<https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_was_associated_with>

## Domain and Range {#domain-and-range-909}

[WorkflowExecutionActivity](WorkflowExecutionActivity.md) → 0..1
[WorkflowExecutionActivity](WorkflowExecutionActivity.md)

## Parents {#parents-946}

-   is_a: [was associated with](was_associated_with.md)

## Children {#children-916}

## Used by {#used-by-909}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md)
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)

## Other properties {#other-properties-718}

  **Mappings:**    prov:wasAssociatedWith
  ---------------- ------------------------
                   
                   nformed-by-1:
  Slot: was info   rmed by

URI:
<https://microbiomedata/schema/workflow_execution_activity/workflow_execution_activity_was_informed_by>

## Domain and Range {#domain-and-range-910}

[WorkflowExecutionActivity](WorkflowExecutionActivity.md) → 1..1
[Activity](Activity.md)

## Parents {#parents-947}

-   is_a: [was informed by](was_informed_by.md)

## Children {#children-917}

## Used by {#used-by-910}

-   [MAGsAnalysisActivity](MAGsAnalysisActivity.md)
-   [MetabolomicsAnalysisActivity](MetabolomicsAnalysisActivity.md)
-   [MetagenomeAnnotationActivity](MetagenomeAnnotationActivity.md)
-   [MetagenomeAssembly](MetagenomeAssembly.md)
-   [MetaproteomicsAnalysisActivity](MetaproteomicsAnalysisActivity.md)
-   [MetatranscriptomeActivity](MetatranscriptomeActivity.md)
-   [MetatranscriptomeAnnotationActivity](MetatranscriptomeAnnotationActivity.md)
-   [MetatranscriptomeAssembly](MetatranscriptomeAssembly.md)
-   [NomAnalysisActivity](NomAnalysisActivity.md)
-   [ReadQCAnalysisActivity](ReadQCAnalysisActivity.md)
-   [ReadBasedAnalysisActivity](ReadBasedAnalysisActivity.md)
-   [WorkflowExecutionActivity](WorkflowExecutionActivity.md)

## Other properties {#other-properties-719}

+----------------------------------+----------------------------------+
| **Mappings:**                    | prov:wasInformedBy               |
+==================================+==================================+
| Slot: xylene ============        |                                  |
+----------------------------------+----------------------------------+
| Concentration                    | of xylene in the sample          |
+----------------------------------+----------------------------------+
| URI: \`nmdc:xyl                  | ene                              |
|                                  | \<<https://mic                   |
|                                  | robiomedata/meta/xylene>\>\`\_\_ |
+----------------------------------+----------------------------------+
|                                  | -range-911:                      |
+----------------------------------+----------------------------------+
| Domain and Ran                   | ge                               |
+----------------------------------+----------------------------------+
| ------------------------------   | \--                              |
+----------------------------------+----------------------------------+
| None → 0..1 \`Q                  | uantityValue                     |
|                                  | \<QuantityValue.md\>\`\_\_       |
+----------------------------------+----------------------------------+
| ## Parents {#parents-948}        | 8:                               |
+----------------------------------+----------------------------------+
| -   is_a: \`attr                 | ibute \<attribute.md\>\`\_\_     |
+----------------------------------+----------------------------------+
| ## Children {#children-918}      | 18:                              |
+----------------------------------+----------------------------------+
| ## Used by {#used-by-911}        | 1:                               |
+----------------------------------+----------------------------------+
|                                  | erties-720:                      |
+----------------------------------+----------------------------------+
| Other properti                   | es                               |
+----------------------------------+----------------------------------+
| ------------------------------   | \--                              |
+----------------------------------+----------------------------------+
| =============                    | ===========                      |
+----------------------------------+----------------------------------+
| **Aliases:**                     | > xylene                         |
+----------------------------------+----------------------------------+
| **Mappings:**                    | > mixs:xylene                    |
+----------------------------------+----------------------------------+
